# Bioinfomatics Repositories

Collection of bioinformatician and their repositories.

- [Top50](https://github.com/xhqsm/bioinfomatics-repositories#Top50)
- [Repositories](https://github.com/xhqsm/bioinfomatics-repositories#repositories)

**Update 2020-06-23**

## Top50

| ID | User | Type |Name | Stars |
| :-: | :-: | :-: | :-: | :-: |
| 1 | [lh3](https://github.com/lh3) | One |Heng Li | 5015 |
| 2 | [broadinstitute](https://github.com/broadinstitute) | Team |Broad Institute | 4867 |
| 3 | [brentp](https://github.com/brentp) | One |Brent Pedersen | 3037 |
| 4 | [samtools](https://github.com/samtools) | Team |samtools | 2386 |
| 5 | [crazyhottommy](https://github.com/crazyhottommy) | One |Ming Tang | 2152 |
| 6 | [arq5x](https://github.com/arq5x) | One |Aaron Quinlan | 1770 |
| 7 | [seandavi](https://github.com/seandavi) | One |Sean Davis | 1744 |
| 8 | [biocore](https://github.com/biocore) | Team |biocore | 1666 |
| 9 | [opengene](https://github.com/opengene) | Team |OpenGene - Open Source Genetics Toolbox | 1618 |
| 10 | [illumina](https://github.com/illumina) | Team |Illumina | 1588 |
| 11 | [griffithlab](https://github.com/griffithlab) | Team |The Griffith Lab | 1523 |
| 12 | [ncbi](https://github.com/ncbi) | Team |NCBI - National Center for Biotechnology Information/NLM/NIH | 1517 |
| 13 | [pacificbiosciences](https://github.com/pacificbiosciences) | Team |Pacific Biosciences | 1505 |
| 14 | [stephenturner](https://github.com/stephenturner) | One |Stephen Turner | 1455 |
| 15 | [benlangmead](https://github.com/benlangmead) | One |Ben | 1389 |
| 16 | [tseemann](https://github.com/tseemann) | One |Torsten Seemann | 1258 |
| 17 | [danielecook](https://github.com/danielecook) | One |Daniel E Cook | 1253 |
| 18 | [bioconda](https://github.com/bioconda) | Team |Bioconda | 1251 |
| 19 | [ekg](https://github.com/ekg) | One |Erik Garrison | 1244 |
| 20 | [ewels](https://github.com/ewels) | One |Phil Ewels | 1000 |
| 21 | [deeptools](https://github.com/deeptools) | Team |The deepTools ecosystem | 982 |
| 22 | [vgteam](https://github.com/vgteam) | Team |vgteam | 900 |
| 23 | [ensembl](https://github.com/ensembl) | Team |Ensembl Project | 838 |
| 24 | [tanghaibao](https://github.com/tanghaibao) | One |Haibao Tang | 809 |
| 25 | [nf-core](https://github.com/nf-core) | Team |nf-core | 742 |
| 26 | [quinlan-lab](https://github.com/quinlan-lab) | Team |quinlan-lab | 729 |
| 27 | [wglab](https://github.com/wglab) | Team |Wang Genomics Lab | 683 |
| 28 | [openvax](https://github.com/openvax) | Team |OpenVax | 627 |
| 29 | [statgen](https://github.com/statgen) | Team |Center for Statistical Genetics | 614 |
| 30 | [daler](https://github.com/daler) | One |Ryan Dale | 519 |
| 31 | [daehwankimlab](https://github.com/daehwankimlab) | Team |Kim Lab at UT Southwestern Medical Center | 497 |
| 32 | [hall-lab](https://github.com/hall-lab) | Team |Ira Hall lab | 452 |
| 33 | [pysam-developers](https://github.com/pysam-developers) | Team |pysam-developers | 440 |
| 34 | [biod](https://github.com/biod) | Team |BioD | 417 |
| 35 | [cgatoxford](https://github.com/cgatoxford) | Team |CGAT | 414 |
| 36 | [mdshw5](https://github.com/mdshw5) | One |Matt Shirley | 395 |
| 37 | [bbuchfink](https://github.com/bbuchfink) | One |Benjamin Buchfink | 395 |
| 38 | [felixkrueger](https://github.com/felixkrueger) | One |Felix Krueger | 393 |
| 39 | [bioinform](https://github.com/bioinform) | Team |The Bioinformatics Repository | 387 |
| 40 | [ga4gh](https://github.com/ga4gh) | Team |Global Alliance for Genomics and Health | 356 |
| 41 | [genome](https://github.com/genome) | Team |The McDonnell Genome Institute | 353 |
| 42 | [dzerbino](https://github.com/dzerbino) | One |Daniel Zerbino | 340 |
| 43 | [qiime2](https://github.com/qiime2) | Team |QIIME 2 | 326 |
| 44 | [benjjneb](https://github.com/benjjneb) | One |Benjamin Callahan | 317 |
| 45 | [nanopore-wgs-consortium](https://github.com/nanopore-wgs-consortium) | Team |nanopore-wgs-consortium | 311 |
| 46 | [dpryan79](https://github.com/dpryan79) | One |Devon Ryan | 296 |
| 47 | [ding-lab](https://github.com/ding-lab) | Team |Li Ding's Lab | 282 |
| 48 | [bwa-mem2](https://github.com/bwa-mem2) | Team |bwa-mem2 | 281 |
| 49 | [vcftools](https://github.com/vcftools) | Team |vcftools | 279 |
| 50 | [astrazeneca-ngs](https://github.com/astrazeneca-ngs) | Team |AstraZeneca - NGS Team | 276 |
## Repositories

##### [seandavi](https://github.com/seandavi)/[awesome-single-cell](https://github.com/seandavi/awesome-single-cell)

ID = 1, Stars = 1355, Language = None, Update = 2020-06-02

Community-curated list of software packages and data resources for single-cell, including RNA-seq, ATAC-seq, etc.
##### [bioconda](https://github.com/bioconda)/[bioconda-recipes](https://github.com/bioconda/bioconda-recipes)

ID = 2, Stars = 1172, Language = Shell, Update = 2020-06-23

Conda recipes for the bioconda channel.
##### [stephenturner](https://github.com/stephenturner)/[oneliners](https://github.com/stephenturner/oneliners)

ID = 3, Stars = 1109, Language = None, Update = 2019-06-10

Useful bash one-liners for bioinformatics.
##### [danielecook](https://github.com/danielecook)/[Awesome-Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics)

ID = 4, Stars = 1025, Language = None, Update = 2020-06-11

A curated list of awesome Bioinformatics libraries and software.
##### [samtools](https://github.com/samtools)/[samtools](https://github.com/samtools/samtools)

ID = 5, Stars = 934, Language = C, Update = 2020-06-17

Tools (written in C using htslib) for manipulating next-generation sequencing data
##### [griffithlab](https://github.com/griffithlab)/[rnaseq_tutorial](https://github.com/griffithlab/rnaseq_tutorial)

ID = 6, Stars = 932, Language = R, Update = 2020-06-19

Informatics for RNA-seq: A web resource for analysis on the cloud. Educational tutorials and working pipelines for RNA-seq analysis including an introduction to: cloud computing, critical file formats, reference genomes, gene annotation, expression, differential expression, alternative splicing, data visualization, and interpretation.
##### [lh3](https://github.com/lh3)/[bwa](https://github.com/lh3/bwa)

ID = 7, Stars = 857, Language = C, Update = 2020-06-04

Burrow-Wheeler Aligner for short-read alignment (see minimap2 for long-read alignment)
##### [broadinstitute](https://github.com/broadinstitute)/[gatk](https://github.com/broadinstitute/gatk)

ID = 8, Stars = 830, Language = Java, Update = 2020-06-22

Official code repository for GATK versions 4 and up
##### [opengene](https://github.com/opengene)/[fastp](https://github.com/OpenGene/fastp)

ID = 9, Stars = 779, Language = C++, Update = 2020-06-19

An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...)
##### [lh3](https://github.com/lh3)/[minimap2](https://github.com/lh3/minimap2)

ID = 10, Stars = 745, Language = C, Update = 2020-06-12

A versatile pairwise aligner for genomic and spliced nucleotide sequences
##### [lh3](https://github.com/lh3)/[seqtk](https://github.com/lh3/seqtk)

ID = 11, Stars = 682, Language = C, Update = 2020-06-16

Toolkit for processing sequences in FASTA/Q formats
##### [vgteam](https://github.com/vgteam)/[vg](https://github.com/vgteam/vg)

ID = 12, Stars = 645, Language = C++, Update = 2020-06-22

tools for working with genome variation graphs
##### [ewels](https://github.com/ewels)/[MultiQC](https://github.com/ewels/MultiQC)

ID = 13, Stars = 624, Language = Python, Update = 2020-06-19

Aggregate results from bioinformatics analyses across many samples into a single report.
##### [arq5x](https://github.com/arq5x)/[bedtools2](https://github.com/arq5x/bedtools2)

ID = 14, Stars = 611, Language = C, Update = 2020-06-05

bedtools - the swiss army knife for genome arithmetic
##### [broadinstitute](https://github.com/broadinstitute)/[picard](https://github.com/broadinstitute/picard)

ID = 15, Stars = 573, Language = Java, Update = 2020-06-22

A set of command line tools (in Java) for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.
##### [broadinstitute](https://github.com/broadinstitute)/[cromwell](https://github.com/broadinstitute/cromwell)

ID = 16, Stars = 558, Language = Scala, Update = 2020-06-23

Scientific workflow engine designed for simplicity & scalability. Trivially transition between one off use cases to massive scale production environments
##### [biocore](https://github.com/biocore)/[scikit-bio](https://github.com/biocore/scikit-bio)

ID = 17, Stars = 557, Language = Python, Update = 2020-06-20

scikit-bio is an open-source, BSD-licensed, Python package providing data structures, algorithms, and educational resources for bioinformatics.
##### [quinlan-lab](https://github.com/quinlan-lab)/[applied-computational-genomics](https://github.com/quinlan-lab/applied-computational-genomics)

ID = 18, Stars = 556, Language = None, Update = 2020-04-21

Applied Computational Genomics Course at UU: Spring 2020
##### [broadinstitute](https://github.com/broadinstitute)/[keras-rcnn](https://github.com/broadinstitute/keras-rcnn)

ID = 19, Stars = 502, Language = Python, Update = 2020-05-20

Keras package for region-based convolutional neural networks (RCNNs)
##### [crazyhottommy](https://github.com/crazyhottommy)/[RNA-seq-analysis](https://github.com/crazyhottommy/RNA-seq-analysis)

ID = 20, Stars = 483, Language = Python, Update = 2020-04-27

RNAseq analysis notes from Ming Tang
##### [samtools](https://github.com/samtools)/[htslib](https://github.com/samtools/htslib)

ID = 21, Stars = 480, Language = C, Update = 2020-06-22

C library for high-throughput sequencing data formats
##### [ekg](https://github.com/ekg)/[freebayes](https://github.com/ekg/freebayes)

ID = 22, Stars = 474, Language = C++, Update = 2020-05-26

Bayesian haplotype-based genetic polymorphism discovery and genotyping.
##### [crazyhottommy](https://github.com/crazyhottommy)/[getting-started-with-genomics-tools-and-resources](https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources)

ID = 23, Stars = 446, Language = Shell, Update = 2020-06-04

Unix, R and python tools for genomics and data science
##### [pysam-developers](https://github.com/pysam-developers)/[pysam](https://github.com/pysam-developers/pysam)

ID = 24, Stars = 440, Language = C, Update = 2020-06-16

Pysam is a Python module for reading and manipulating SAM/BAM/VCF/BCF files. It's a lightweight wrapper of the htslib C-API, the same one that powers samtools, bcftools, and tabix.
##### [lh3](https://github.com/lh3)/[bioawk](https://github.com/lh3/bioawk)

ID = 25, Stars = 429, Language = C, Update = 2019-09-15

BWK awk modified for biological data
##### [crazyhottommy](https://github.com/crazyhottommy)/[ChIP-seq-analysis](https://github.com/crazyhottommy/ChIP-seq-analysis)

ID = 26, Stars = 424, Language = Python, Update = 2020-04-25

ChIP-seq analysis notes from Ming Tang
##### [ncbi](https://github.com/ncbi)/[sra-tools](https://github.com/ncbi/sra-tools)

ID = 27, Stars = 423, Language = C, Update = 2020-06-22

SRA Tools
##### [deeptools](https://github.com/deeptools)/[deepTools](https://github.com/deeptools/deepTools)

ID = 28, Stars = 402, Language = Python, Update = 2020-05-20

Tools to process and analyze deep sequencing data.
##### [bbuchfink](https://github.com/bbuchfink)/[diamond](https://github.com/bbuchfink/diamond)

ID = 29, Stars = 395, Language = C++, Update = 2020-06-22

Accelerated BLAST compatible local sequence aligner.
##### [samtools](https://github.com/samtools)/[hts-specs](https://github.com/samtools/hts-specs)

ID = 30, Stars = 375, Language = TeX, Update = 2020-06-22

Specifications of SAM/BAM and related high-throughput sequencing file formats
##### [biod](https://github.com/biod)/[sambamba](https://github.com/biod/sambamba)

ID = 31, Stars = 374, Language = D, Update = 2019-12-17

Tools for working with SAM/BAM/CRAM data
##### [tseemann](https://github.com/tseemann)/[prokka](https://github.com/tseemann/prokka)

ID = 32, Stars = 364, Language = Perl, Update = 2020-05-17

‚ö°
‚ôí
Rapid prokaryotic genome annotation
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[Bioinformatics-Training](https://github.com/PacificBiosciences/Bioinformatics-Training)

ID = 33, Stars = 359, Language = R, Update = 2017-09-14

Bioinformatics training resources
##### [deeptools](https://github.com/deeptools)/[pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks)

ID = 34, Stars = 353, Language = Python, Update = 2020-06-09

python module to plot beautiful and highly customizable genome browser tracks
##### [tanghaibao](https://github.com/tanghaibao)/[goatools](https://github.com/tanghaibao/goatools)

ID = 35, Stars = 336, Language = Python, Update = 2020-06-20

Python library to handle Gene Ontology (GO) terms
##### [tanghaibao](https://github.com/tanghaibao)/[jcvi](https://github.com/tanghaibao/jcvi)

ID = 36, Stars = 332, Language = Python, Update = 2020-06-17

Python library to facilitate genome assembly, annotation and comparative genomics
##### [samtools](https://github.com/samtools)/[bcftools](https://github.com/samtools/bcftools)

ID = 37, Stars = 309, Language = C, Update = 2020-06-18

This is the official development repository for BCFtools. To compile, the develop branch of htslib is needed: git clone --branch=develop git://github.com/samtools/htslib.git htslib
##### [brentp](https://github.com/brentp)/[mosdepth](https://github.com/brentp/mosdepth)

ID = 38, Stars = 308, Language = Nim, Update = 2020-06-22

fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing
##### [benlangmead](https://github.com/benlangmead)/[bowtie2](https://github.com/BenLangmead/bowtie2)

ID = 39, Stars = 303, Language = C++, Update = 2020-06-21

A fast and sensitive gapped read aligner
##### [bwa-mem2](https://github.com/bwa-mem2)/[bwa-mem2](https://github.com/bwa-mem2/bwa-mem2)

ID = 40, Stars = 281, Language = C++, Update = 2020-06-17

The next version of bwa-mem (WIP; not recommended for production uses at the moment)
##### [vcftools](https://github.com/vcftools)/[vcftools](https://github.com/vcftools/vcftools)

ID = 41, Stars = 279, Language = C++, Update = 2020-03-10

A set of tools written in Perl and C++ for working with VCF files, such as those generated by the 1000 Genomes Project.
##### [arq5x](https://github.com/arq5x)/[gemini](https://github.com/arq5x/gemini)

ID = 42, Stars = 274, Language = Python, Update = 2020-04-28

a lightweight db framework for exploring genetic variation.
##### [mdshw5](https://github.com/mdshw5)/[pyfaidx](https://github.com/mdshw5/pyfaidx)

ID = 43, Stars = 272, Language = Python, Update = 2020-06-22

Efficient pythonic random access to fasta subsequences
##### [hall-lab](https://github.com/hall-lab)/[speedseq](https://github.com/hall-lab/speedseq)

ID = 44, Stars = 266, Language = C, Update = 2018-02-08

A flexible framework for rapid genome analysis and interpretation
##### [biocore](https://github.com/biocore)/[qiime](https://github.com/biocore/qiime)

ID = 45, Stars = 265, Language = Python, Update = 2018-01-23

Official QIIME 1 software repository. QIIME 2 (https://qiime2.org) has succeeded QIIME 1 as of January 2018.
##### [crazyhottommy](https://github.com/crazyhottommy)/[bioinformatics-one-liners](https://github.com/crazyhottommy/bioinformatics-one-liners)

ID = 46, Stars = 261, Language = None, Update = 2020-02-27

Bioinformatics one liners from Ming Tang
##### [cgatoxford](https://github.com/cgatoxford)/[UMI-tools](https://github.com/CGATOxford/UMI-tools)

ID = 47, Stars = 261, Language = Python, Update = 2020-03-02

Tools for handling Unique Molecular Identifiers in NGS data sets
##### [nanopore-wgs-consortium](https://github.com/nanopore-wgs-consortium)/[NA12878](https://github.com/nanopore-wgs-consortium/NA12878)

ID = 48, Stars = 250, Language = Python, Update = 2020-01-25

Data and analysis for NA12878 genome on nanopore
##### [etal](https://github.com/etal)/[cnvkit](https://github.com/etal/cnvkit)

ID = 49, Stars = 244, Language = Python, Update = 2020-05-29

Copy number variant detection from targeted DNA sequencing
##### [daehwankimlab](https://github.com/daehwankimlab)/[hisat2](https://github.com/DaehwanKimLab/hisat2)

ID = 50, Stars = 241, Language = C++, Update = 2020-06-22

Graph-based alignment (Hierarchical Graph FM index)
##### [brentp](https://github.com/brentp)/[vcfanno](https://github.com/brentp/vcfanno)

ID = 51, Stars = 233, Language = Go, Update = 2020-04-15

annotate a VCF with other VCFs/BEDs/tabixed files
##### [benjjneb](https://github.com/benjjneb)/[dada2](https://github.com/benjjneb/dada2)

ID = 52, Stars = 231, Language = R, Update = 2020-05-18

Accurate sample inference from amplicon data with single nucleotide resolution
##### [broadinstitute](https://github.com/broadinstitute)/[keras-resnet](https://github.com/broadinstitute/keras-resnet)

ID = 53, Stars = 224, Language = Python, Update = 2020-06-23

Keras package for deep residual networks
##### [dzerbino](https://github.com/dzerbino)/[velvet](https://github.com/dzerbino/velvet)

ID = 54, Stars = 223, Language = C, Update = 2018-07-05

Short read de novo assembler using de Bruijn graphs
##### [illumina](https://github.com/illumina)/[manta](https://github.com/Illumina/manta)

ID = 55, Stars = 223, Language = C++, Update = 2019-07-17

Structural variant and indel caller for mapped sequencing data
##### [arq5x](https://github.com/arq5x)/[poretools](https://github.com/arq5x/poretools)

ID = 56, Stars = 221, Language = Jupyter Notebook, Update = 2017-12-16

a toolkit for working with Oxford nanopore data
##### [qiime2](https://github.com/qiime2)/[qiime2](https://github.com/qiime2/qiime2)

ID = 57, Stars = 219, Language = Python, Update = 2020-06-22

Official repository for the QIIME 2 framework.
##### [openvax](https://github.com/openvax)/[pyensembl](https://github.com/openvax/pyensembl)

ID = 58, Stars = 216, Language = Python, Update = 2020-05-27

Python interface to access reference genome features (such as genes, transcripts, and exons) from Ensembl
##### [illumina](https://github.com/illumina)/[hap.py](https://github.com/Illumina/hap.py)

ID = 59, Stars = 214, Language = C++, Update = 2020-02-27

Haplotype VCF comparison tools
##### [ensembl](https://github.com/ensembl)/[ensembl-vep](https://github.com/Ensembl/ensembl-vep)

ID = 60, Stars = 212, Language = Perl, Update = 2020-06-17

The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
##### [benlangmead](https://github.com/benlangmead)/[comp-genomics-class](https://github.com/BenLangmead/comp-genomics-class)

ID = 61, Stars = 211, Language = Jupyter Notebook, Update = 2019-10-29

Code and examples for JHU Computational Genomics class
##### [illumina](https://github.com/illumina)/[strelka](https://github.com/Illumina/strelka)

ID = 62, Stars = 211, Language = C++, Update = 2019-09-26

Strelka2 germline and somatic small variant caller
##### [samtools](https://github.com/samtools)/[htsjdk](https://github.com/samtools/htsjdk)

ID = 63, Stars = 210, Language = Java, Update = 2020-06-12

A Java API for high-throughput sequencing data (HTS) formats.
##### [benlangmead](https://github.com/benlangmead)/[c-cpp-notes](https://github.com/BenLangmead/c-cpp-notes)

ID = 64, Stars = 207, Language = Python, Update = 2020-05-09

Lecture notes and example code for teaching C & C++
##### [benlangmead](https://github.com/benlangmead)/[bowtie](https://github.com/BenLangmead/bowtie)

ID = 65, Stars = 206, Language = C++, Update = 2020-06-16

An ultrafast memory-efficient short read aligner
##### [brentp](https://github.com/brentp)/[cyvcf2](https://github.com/brentp/cyvcf2)

ID = 66, Stars = 204, Language = Python, Update = 2020-06-15

cython + htslib == fast VCF and BCF processing
##### [daler](https://github.com/daler)/[pybedtools](https://github.com/daler/pybedtools)

ID = 67, Stars = 200, Language = Python, Update = 2020-06-02

Python wrapper -- and more -- for Aaron Quinlan's BEDTools (bioinformatics tools)
##### [dellytools](https://github.com/dellytools)/[delly](https://github.com/dellytools/delly)

ID = 68, Stars = 199, Language = C++, Update = 2020-06-16

DELLY2: Structural variant discovery by integrated paired-end and split-read analysis
##### [lh3](https://github.com/lh3)/[miniasm](https://github.com/lh3/miniasm)

ID = 69, Stars = 198, Language = TeX, Update = 2020-04-03

Ultrafast de novo assembly for long noisy reads (though having no consensus step)
##### [arq5x](https://github.com/arq5x)/[lumpy-sv](https://github.com/arq5x/lumpy-sv)

ID = 70, Stars = 197, Language = C, Update = 2019-04-16

lumpy: a general probabilistic framework for structural variant discovery
##### [nf-core](https://github.com/nf-core)/[rnaseq](https://github.com/nf-core/rnaseq)

ID = 71, Stars = 197, Language = Nextflow, Update = 2020-06-16

RNA sequencing analysis pipeline using STAR, HISAT2 and Salmon with gene counts and quality control
##### [broadinstitute](https://github.com/broadinstitute)/[pilon](https://github.com/broadinstitute/pilon)

ID = 72, Stars = 197, Language = Scala, Update = 2020-02-20

Pilon is an automated genome assembly improvement and variant detection tool
##### [felixkrueger](https://github.com/felixkrueger)/[TrimGalore](https://github.com/FelixKrueger/TrimGalore)

ID = 73, Stars = 196, Language = Perl, Update = 2020-05-11

A wrapper around Cutadapt and FastQC to consistently apply adapter and quality trimming to FastQ files, with extra functionality for RRBS data
##### [tseemann](https://github.com/tseemann)/[snippy](https://github.com/tseemann/snippy)

ID = 74, Stars = 194, Language = Perl, Update = 2020-03-28

‚úÇÔ∏è
‚ö°
Rapid haploid variant calling and core genome alignment
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[FALCON](https://github.com/PacificBiosciences/FALCON)

ID = 75, Stars = 185, Language = Python, Update = 2018-10-01

FALCON: experimental PacBio diploid assembler -- Out-of-date -- Please use a binary release: https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries
##### [benlangmead](https://github.com/benlangmead)/[ads1-notebooks](https://github.com/BenLangmead/ads1-notebooks)

ID = 76, Stars = 183, Language = None, Update = 2015-10-09

Copies of notebooks used in the practical sessions for Algorithms for DNA Sequencing
##### [seandavi](https://github.com/seandavi)/[awesome-cancer-variant-databases](https://github.com/seandavi/awesome-cancer-variant-databases)

ID = 77, Stars = 177, Language = None, Update = 2019-03-13

A community-maintained repository of cancer clinical knowledge bases and databases focused on cancer variants.
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[snakepipes](https://github.com/maxplanck-ie/snakepipes)

ID = 78, Stars = 173, Language = Python, Update = 2020-06-22

Customizable workflows based on snakemake and python for the analysis of NGS data
##### [felixkrueger](https://github.com/felixkrueger)/[Bismark](https://github.com/FelixKrueger/Bismark)

ID = 79, Stars = 167, Language = HTML, Update = 2020-04-28

A tool to map bisulfite converted sequence reads and determine cytosine methylation states
##### [griffithlab](https://github.com/griffithlab)/[GenVisR](https://github.com/griffithlab/GenVisR)

ID = 80, Stars = 166, Language = R, Update = 2020-06-08

Genome data visualizations
##### [ekg](https://github.com/ekg)/[intervaltree](https://github.com/ekg/intervaltree)

ID = 81, Stars = 164, Language = C++, Update = 2020-06-18

a minimal C++ interval tree implementation
##### [lh3](https://github.com/lh3)/[wgsim](https://github.com/lh3/wgsim)

ID = 82, Stars = 164, Language = C, Update = 2019-10-02

Reads simulator
##### [brentp](https://github.com/brentp)/[goleft](https://github.com/brentp/goleft)

ID = 83, Stars = 162, Language = Go, Update = 2020-02-04

goleft is a collection of bioinformatics tools distributed under MIT license in a single static binary
##### [eweitz](https://github.com/eweitz)/[ideogram](https://github.com/eweitz/ideogram)

ID = 84, Stars = 160, Language = JavaScript, Update = 2020-06-14

Chromosome visualization for the web
##### [genome](https://github.com/genome)/[bam-readcount](https://github.com/genome/bam-readcount)

ID = 85, Stars = 159, Language = Python, Update = 2020-02-10

count DNA sequence reads in BAM files
##### [benlangmead](https://github.com/benlangmead)/[ads1-slides](https://github.com/BenLangmead/ads1-slides)

ID = 86, Stars = 158, Language = None, Update = 2015-09-15

Slides for Algorithms for DNA Sequencing Coursera class
##### [opengene](https://github.com/opengene)/[AfterQC](https://github.com/OpenGene/AfterQC)

ID = 87, Stars = 158, Language = Python, Update = 2020-05-14

Automatic Filtering, Trimming, Error Removing and Quality Control for fastq data
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-pipeline](https://github.com/broadinstitute/gtex-pipeline)

ID = 88, Stars = 157, Language = Python, Update = 2020-05-26

GTEx & TOPMed data production and analysis pipelines
##### [ncbi](https://github.com/ncbi)/[PubReader](https://github.com/ncbi/PubReader)

ID = 89, Stars = 157, Language = JavaScript, Update = 2018-06-21

A new way to view journal articles
##### [dpryan79](https://github.com/dpryan79)/[ChromosomeMappings](https://github.com/dpryan79/ChromosomeMappings)

ID = 90, Stars = 156, Language = None, Update = 2020-04-16

This repository contains chromosome/contig name mappings between UCSC <-> Ensembl <-> Gencode for a variety of genomes.
##### [sigven](https://github.com/sigven)/[pcgr](https://github.com/sigven/pcgr)

ID = 91, Stars = 149, Language = R, Update = 2020-02-25

Personal Cancer Genome Reporter (PCGR)
##### [atks](https://github.com/atks)/[vt](https://github.com/atks/vt)

ID = 92, Stars = 147, Language = C, Update = 2020-03-31

A tool set for short variant discovery in genetic sequence data.
##### [broadinstitute](https://github.com/broadinstitute)/[infercnv](https://github.com/broadinstitute/infercnv)

ID = 93, Stars = 147, Language = R, Update = 2020-06-04

Inferring CNV from Single-Cell RNA-Seq
##### [brentp](https://github.com/brentp)/[bio-playground](https://github.com/brentp/bio-playground)

ID = 94, Stars = 146, Language = C, Update = 2018-11-26

miscellaneous scripts for bioinformatics/genomics that dont merit their own repo.
##### [crazyhottommy](https://github.com/crazyhottommy)/[scRNAseq-analysis-notes](https://github.com/crazyhottommy/scRNAseq-analysis-notes)

ID = 95, Stars = 146, Language = None, Update = 2020-06-11

scRNAseq analysis notes from Ming Tang
##### [gregoryfaust](https://github.com/gregoryfaust)/[samblaster](https://github.com/GregoryFaust/samblaster)

ID = 96, Stars = 144, Language = C++, Update = 2020-06-04

samblaster: a tool to mark duplicates and extract discordant and split reads from sam files.
##### [bioinformaticsfmrp](https://github.com/bioinformaticsfmrp)/[TCGAbiolinks](https://github.com/BioinformaticsFMRP/TCGAbiolinks)

ID = 97, Stars = 144, Language = R, Update = 2020-06-18

TCGAbiolinks
##### [daler](https://github.com/daler)/[gffutils](https://github.com/daler/gffutils)

ID = 98, Stars = 142, Language = Python, Update = 2020-04-07

GFF and GTF file manipulation and interconversion
##### [lh3](https://github.com/lh3)/[minigraph](https://github.com/lh3/minigraph)

ID = 99, Stars = 141, Language = C, Update = 2020-05-28

Proof-of-concept seq-to-graph mapper and graph generator
##### [daehwankimlab](https://github.com/daehwankimlab)/[centrifuge](https://github.com/DaehwanKimLab/centrifuge)

ID = 100, Stars = 135, Language = C++, Update = 2020-06-11

Classifier for metagenomic sequences
##### [marianattestad](https://github.com/marianattestad)/[Ribbon](https://github.com/MariaNattestad/Ribbon)

ID = 101, Stars = 134, Language = JavaScript, Update = 2020-05-02

A genome browser that shows long reads and complex variants better
##### [ekg](https://github.com/ekg)/[alignment-and-variant-calling-tutorial](https://github.com/ekg/alignment-and-variant-calling-tutorial)

ID = 102, Stars = 132, Language = None, Update = 2019-06-12

basic walk-throughs for alignment and variant calling from NGS sequencing data
##### [ekg](https://github.com/ekg)/[fraction.js](https://github.com/ekg/fraction.js)

ID = 103, Stars = 132, Language = JavaScript, Update = 2018-07-25

A fraction math library in javascript.
##### [broadinstitute](https://github.com/broadinstitute)/[viral-ngs](https://github.com/broadinstitute/viral-ngs)

ID = 104, Stars = 132, Language = Python, Update = 2020-05-20

Viral genomics analysis pipelines
##### [arq5x](https://github.com/arq5x)/[bedtools](https://github.com/arq5x/bedtools)

ID = 105, Stars = 131, Language = C++, Update = 2019-03-10

A powerful toolset for genome arithmetic.
##### [illumina](https://github.com/illumina)/[SpliceAI](https://github.com/Illumina/SpliceAI)

ID = 106, Stars = 131, Language = Python, Update = 2020-05-15

A deep learning-based tool to identify splice variants
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[VarDict](https://github.com/AstraZeneca-NGS/VarDict)

ID = 107, Stars = 127, Language = Perl, Update = 2020-05-08

VarDict
##### [wglab](https://github.com/wglab)/[dragonstar2019](https://github.com/WGLab/dragonstar2019)

ID = 108, Stars = 124, Language = None, Update = 2019-08-30

None
##### [brentp](https://github.com/brentp)/[hts-nim](https://github.com/brentp/hts-nim)

ID = 109, Stars = 122, Language = Nim, Update = 2020-06-01

nim wrapper for htslib for parsing genomics data files
##### [s-andrews](https://github.com/s-andrews)/[FastQC](https://github.com/s-andrews/FastQC)

ID = 110, Stars = 121, Language = Java, Update = 2020-02-28

A quality control analysis tool for high throughput sequencing data
##### [brentp](https://github.com/brentp)/[smoove](https://github.com/brentp/smoove)

ID = 111, Stars = 120, Language = Go, Update = 2019-11-11

structural variant calling and genotyping with existing tools, but, smoothly.
##### [genetalks](https://github.com/genetalks)/[gtz](https://github.com/Genetalks/gtz)

ID = 112, Stars = 119, Language = None, Update = 2020-05-15

A high performance and compression ratio compressor for genomic data, powered by GTXLab of Genetalks.
##### [abyzovlab](https://github.com/abyzovlab)/[CNVnator](https://github.com/abyzovlab/CNVnator)

ID = 113, Stars = 116, Language = C++, Update = 2020-06-11

a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[blasr](https://github.com/PacificBiosciences/blasr)

ID = 114, Stars = 116, Language = C++, Update = 2020-06-04

BLASR: The PacBio¬Æ long read aligner
##### [cgatoxford](https://github.com/cgatoxford)/[cgat](https://github.com/CGATOxford/cgat)

ID = 115, Stars = 116, Language = Python, Update = 2018-11-08

Do not use - please refer to our newest code: https://github.com/cgat-developers/cgat-apps
##### [tseemann](https://github.com/tseemann)/[abricate](https://github.com/tseemann/abricate)

ID = 116, Stars = 115, Language = Perl, Update = 2020-05-12

üîé
üíä
Mass screening of contigs for antimicrobial and virulence genes
##### [stephenturner](https://github.com/stephenturner)/[annotables](https://github.com/stephenturner/annotables)

ID = 117, Stars = 115, Language = R, Update = 2019-01-28

R data package for annotating/converting Gene IDs
##### [deeptools](https://github.com/deeptools)/[pyBigWig](https://github.com/deeptools/pyBigWig)

ID = 118, Stars = 115, Language = C, Update = 2020-05-02

A python extension for quick access to bigWig and bigBed files
##### [ga4gh](https://github.com/ga4gh)/[benchmarking-tools](https://github.com/ga4gh/benchmarking-tools)

ID = 119, Stars = 114, Language = HTML, Update = 2020-04-07

Repository for the GA4GH Benchmarking Team work developing standardized benchmarking methods for germline small variant calls
##### [bioinform](https://github.com/bioinform)/[neusomatic](https://github.com/bioinform/neusomatic)

ID = 120, Stars = 114, Language = Python, Update = 2020-06-20

NeuSomatic: Deep convolutional neural networks for accurate somatic mutation detection
##### [ocxtal](https://github.com/ocxtal)/[minialign](https://github.com/ocxtal/minialign)

ID = 121, Stars = 113, Language = C, Update = 2018-09-10

fast and accurate alignment tool for PacBio and Nanopore long reads
##### [brentp](https://github.com/brentp)/[cruzdb](https://github.com/brentp/cruzdb)

ID = 122, Stars = 112, Language = Python, Update = 2020-03-12

python access to UCSC genomes database
##### [lh3](https://github.com/lh3)/[readfq](https://github.com/lh3/readfq)

ID = 123, Stars = 112, Language = C, Update = 2019-08-09

Fast multi-line FASTA/Q reader in several programming languages
##### [lh3](https://github.com/lh3)/[kmer-cnt](https://github.com/lh3/kmer-cnt)

ID = 124, Stars = 107, Language = C++, Update = 2020-03-10

Code examples of fast and simple k-mer counters for tutorial purposes
##### [opengene](https://github.com/opengene)/[awesome-bio-datasets](https://github.com/OpenGene/awesome-bio-datasets)

ID = 125, Stars = 106, Language = None, Update = 2017-10-28

awesome-bio-datasets
##### [papenfusslab](https://github.com/papenfusslab)/[gridss](https://github.com/PapenfussLab/gridss)

ID = 126, Stars = 105, Language = Java, Update = 2020-06-21

GRIDSS: the Genomic Rearrangement IDentification Software Suite
##### [stephenturner](https://github.com/stephenturner)/[qqman](https://github.com/stephenturner/qqman)

ID = 127, Stars = 104, Language = R, Update = 2018-08-01

An R package for creating Q-Q and manhattan plots from GWAS results
##### [wglab](https://github.com/wglab)/[doc-ANNOVAR](https://github.com/WGLab/doc-ANNOVAR)

ID = 128, Stars = 104, Language = None, Update = 2020-06-08

Documentation for the ANNOVAR software
##### [bioinform](https://github.com/bioinform)/[somaticseq](https://github.com/bioinform/somaticseq)

ID = 129, Stars = 104, Language = Shell, Update = 2020-06-18

An ensemble approach to accurately detect somatic mutations using SomaticSeq
##### [opengene](https://github.com/opengene)/[MutScan](https://github.com/OpenGene/MutScan)

ID = 130, Stars = 104, Language = C, Update = 2020-01-21

Detect and visualize target mutations by scanning FastQ files directly
##### [biocore](https://github.com/biocore)/[emp](https://github.com/biocore/emp)

ID = 131, Stars = 104, Language = Jupyter Notebook, Update = 2020-05-09

Code repository of the Earth Microbiome Project.
##### [vals](https://github.com/vals)/[umis](https://github.com/vals/umis)

ID = 132, Stars = 103, Language = Python, Update = 2020-03-28

Tools for processing UMI RNA-tag data
##### [refresh-bio](https://github.com/refresh-bio)/[KMC](https://github.com/refresh-bio/KMC)

ID = 133, Stars = 103, Language = C++, Update = 2020-05-14

Fast and frugal disk based k-mer counter
##### [vibansal](https://github.com/vibansal)/[HapCUT2](https://github.com/vibansal/HapCUT2)

ID = 134, Stars = 102, Language = C, Update = 2020-05-12

software tools for haplotype assembly from sequence data
##### [wglab](https://github.com/wglab)/[InterVar](https://github.com/WGLab/InterVar)

ID = 135, Stars = 102, Language = Python, Update = 2020-01-02

A bioinformatics software tool for clinical interpretation of genetic variants by the 2015 ACMG-AMP guideline
##### [brentp](https://github.com/brentp)/[somalier](https://github.com/brentp/somalier)

ID = 136, Stars = 101, Language = Nim, Update = 2020-06-04

fast sample-swap and relatedness checks on BAMs/CRAMs/VCFs/GVCFs... "like damn that is one smart wine guy"
##### [lh3](https://github.com/lh3)/[minimap](https://github.com/lh3/minimap)

ID = 137, Stars = 101, Language = C, Update = 2017-09-20

This repo is DEPRECATED. Please use minimap2, the successor of minimap.
##### [tseemann](https://github.com/tseemann)/[shovill](https://github.com/tseemann/shovill)

ID = 138, Stars = 100, Language = Perl, Update = 2020-06-04

‚ö°
‚ô†Ô∏è
Assemble bacterial isolate genomes from Illumina paired-end reads
##### [lh3](https://github.com/lh3)/[cgranges](https://github.com/lh3/cgranges)

ID = 139, Stars = 100, Language = C, Update = 2020-05-20

A C/C++ library for fast interval overlap queries (with a "bedtools coverage" example)
##### [alimanfoo](https://github.com/alimanfoo)/[pysamstats](https://github.com/alimanfoo/pysamstats)

ID = 140, Stars = 99, Language = Python, Update = 2020-06-01

A fast Python and command-line utility for extracting simple statistics against genome positions based on sequence alignments from a SAM or BAM file.
##### [broadinstitute](https://github.com/broadinstitute)/[xtermcolor](https://github.com/broadinstitute/xtermcolor)

ID = 141, Stars = 99, Language = Python, Update = 2017-10-08

Python library for terminal color support (including 256-color support)
##### [ncbi](https://github.com/ncbi)/[ngs](https://github.com/ncbi/ngs)

ID = 142, Stars = 99, Language = C++, Update = 2020-06-22

NGS Language Bindings
##### [ncbi](https://github.com/ncbi)/[pgap](https://github.com/ncbi/pgap)

ID = 143, Stars = 99, Language = Common Workflow Language, Update = 2020-06-22

NCBI Prokaryotic Genome Annotation Pipeline
##### [illumina](https://github.com/illumina)/[pyflow](https://github.com/Illumina/pyflow)

ID = 144, Stars = 97, Language = Python, Update = 2018-03-02

A lightweight parallel task engine
##### [pcingola](https://github.com/pcingola)/[SnpEff](https://github.com/pcingola/SnpEff)

ID = 145, Stars = 96, Language = Java, Update = 2020-05-27

None
##### [deeptools](https://github.com/deeptools)/[HiCExplorer](https://github.com/deeptools/HiCExplorer)

ID = 146, Stars = 96, Language = Python, Update = 2020-06-22

HiCExplorer is a powerful and easy to use set of tools to process, normalize and visualize Hi-C data.
##### [ensembl](https://github.com/ensembl)/[ensembl-rest](https://github.com/Ensembl/ensembl-rest)

ID = 147, Stars = 95, Language = Perl, Update = 2020-06-17

Language agnostic RESTful data access to Ensembl data over HTTP
##### [ensembl](https://github.com/ensembl)/[WiggleTools](https://github.com/Ensembl/WiggleTools)

ID = 148, Stars = 95, Language = C, Update = 2020-06-02

Basic operations on the space of numerical functions defined on the genome using lazy evaluators for flexibility and efficiency
##### [dkoboldt](https://github.com/dkoboldt)/[varscan](https://github.com/dkoboldt/varscan)

ID = 149, Stars = 94, Language = None, Update = 2019-07-25

Variant calling and somatic mutation/CNV detection for next-generation sequencing data
##### [sunbeam-labs](https://github.com/sunbeam-labs)/[sunbeam](https://github.com/sunbeam-labs/sunbeam)

ID = 150, Stars = 94, Language = Python, Update = 2020-05-20

A robust, extensible metagenomics pipeline
##### [brentp](https://github.com/brentp)/[peddy](https://github.com/brentp/peddy)

ID = 151, Stars = 93, Language = Jupyter Notebook, Update = 2020-04-22

genotype :: ped correspondence check, ancestry check, sex check. directly, quickly on VCF
##### [ming-lian](https://github.com/ming-lian)/[NGS-analysis](https://github.com/Ming-Lian/NGS-analysis)

ID = 152, Stars = 91, Language = None, Update = 2020-04-12

‰∫å‰ª£ÊµãÂ∫èÊï∞ÊçÆÂàÜÊûê
##### [vgteam](https://github.com/vgteam)/[sequenceTubeMap](https://github.com/vgteam/sequenceTubeMap)

ID = 153, Stars = 91, Language = JavaScript, Update = 2020-06-06

displays multiple genomic sequences in the form of a tube map
##### [biocore](https://github.com/biocore)/[American-Gut](https://github.com/biocore/American-Gut)

ID = 154, Stars = 91, Language = Jupyter Notebook, Update = 2017-03-17

American Gut open-access data and IPython notebooks
##### [ding-lab](https://github.com/ding-lab)/[msisensor](https://github.com/ding-lab/msisensor)

ID = 155, Stars = 91, Language = C++, Update = 2020-04-22

microsatellite instability detection using tumor only or paired tumor-normal data
##### [brentp](https://github.com/brentp)/[nim-plotly](https://github.com/brentp/nim-plotly)

ID = 156, Stars = 90, Language = Nim, Update = 2020-05-02

plotting library for nim-lang
##### [crazyhottommy](https://github.com/crazyhottommy)/[DNA-seq-analysis](https://github.com/crazyhottommy/DNA-seq-analysis)

ID = 157, Stars = 88, Language = Shell, Update = 2020-05-11

DNA sequencing analysis notes from Ming Tang
##### [a-slide](https://github.com/a-slide)/[pycoQC](https://github.com/a-slide/pycoQC)

ID = 158, Stars = 88, Language = Python, Update = 2020-04-03

pycoQC computes metrics and generates Interactive QC plots from the sequencing summary report generated by Oxford Nanopore technologies basecaller (Albacore/Guppy)
##### [mdshw5](https://github.com/mdshw5)/[fastqp](https://github.com/mdshw5/fastqp)

ID = 159, Stars = 88, Language = Python, Update = 2018-05-29

Simple FASTQ quality assessment using Python
##### [biocore](https://github.com/biocore)/[qiita](https://github.com/biocore/qiita)

ID = 160, Stars = 88, Language = Python, Update = 2020-06-20

Qiita - A multi-omics databasing effort
##### [lh3](https://github.com/lh3)/[fermikit](https://github.com/lh3/fermikit)

ID = 161, Stars = 87, Language = TeX, Update = 2017-12-20

De novo assembly based variant calling pipeline for Illumina short reads
##### [fhcrc](https://github.com/fhcrc)/[seqmagick](https://github.com/fhcrc/seqmagick)

ID = 162, Stars = 87, Language = Python, Update = 2019-12-13

An imagemagick-like frontend to Biopython SeqIO
##### [cschin](https://github.com/cschin)/[ipython_d3_mashup](https://github.com/cschin/ipython_d3_mashup)

ID = 163, Stars = 86, Language = JavaScript, Update = 2012-09-10

None
##### [ewels](https://github.com/ewels)/[ClusterFlow](https://github.com/ewels/ClusterFlow)

ID = 164, Stars = 85, Language = Perl, Update = 2019-06-08

A pipelining tool to automate and standardise bioinformatics analyses on cluster environments.
##### [pcingola](https://github.com/pcingola)/[BigDataScript](https://github.com/pcingola/BigDataScript)

ID = 165, Stars = 85, Language = Java, Update = 2020-06-08

BigDataScript: Scirpting language for big data
##### [biocore](https://github.com/biocore)/[sortmerna](https://github.com/biocore/sortmerna)

ID = 166, Stars = 85, Language = C++, Update = 2020-06-11

SortMeRNA: next-generation sequence filtering and alignment tool
##### [statgen](https://github.com/statgen)/[locuszoom](https://github.com/statgen/locuszoom)

ID = 167, Stars = 85, Language = JavaScript, Update = 2020-06-22

A Javascript/d3 embeddable plugin for interactively visualizing statistical genetic data from customizable sources.
##### [illumina](https://github.com/illumina)/[paragraph](https://github.com/Illumina/paragraph)

ID = 168, Stars = 85, Language = C++, Update = 2019-09-24

Graph realignment tools for structural variants
##### [brentp](https://github.com/brentp)/[slivar](https://github.com/brentp/slivar)

ID = 169, Stars = 84, Language = Nim, Update = 2020-06-15

variant expressions, annotation, and filtering for great good.
##### [daehwankimlab](https://github.com/daehwankimlab)/[tophat](https://github.com/DaehwanKimLab/tophat)

ID = 170, Stars = 84, Language = C++, Update = 2018-05-24

Spliced read mapper for RNA-Seq
##### [openvax](https://github.com/openvax)/[mhcflurry](https://github.com/openvax/mhcflurry)

ID = 171, Stars = 84, Language = Python, Update = 2020-06-16

Peptide-MHC I binding affinity prediction
##### [danielecook](https://github.com/danielecook)/[Quiver-alfred](https://github.com/danielecook/Quiver-alfred)

ID = 172, Stars = 83, Language = Python, Update = 2018-08-22

Search Quiver from Alfred
##### [broadinstitute](https://github.com/broadinstitute)/[mutect](https://github.com/broadinstitute/mutect)

ID = 173, Stars = 83, Language = Java, Update = 2015-08-26

MuTect -- Accurate and sensitive cancer mutation detection
##### [hall-lab](https://github.com/hall-lab)/[svtools](https://github.com/hall-lab/svtools)

ID = 174, Stars = 83, Language = Python, Update = 2019-10-24

Tools for processing and analyzing structural variants.
##### [lh3](https://github.com/lh3)/[biofast](https://github.com/lh3/biofast)

ID = 175, Stars = 82, Language = C, Update = 2020-06-12

Benchmarking programming languages/implementations for common tasks in Bioinformatics
##### [illumina](https://github.com/illumina)/[Nirvana](https://github.com/Illumina/Nirvana)

ID = 176, Stars = 82, Language = C#, Update = 2020-06-22

The nimble & robust variant annotator
##### [lh3](https://github.com/lh3)/[psmc](https://github.com/lh3/psmc)

ID = 177, Stars = 81, Language = C, Update = 2016-01-21

Implementation of the Pairwise Sequentially Markovian Coalescent (PSMC) model
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[GenomicConsensus](https://github.com/PacificBiosciences/GenomicConsensus)

ID = 178, Stars = 81, Language = Python, Update = 2019-04-25

PacBio¬Æ variant and consensus caller
##### [gt1](https://github.com/gt1)/[biobambam2](https://github.com/gt1/biobambam2)

ID = 179, Stars = 80, Language = C++, Update = 2019-03-12

Tools for early stage alignment file processing
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[VarDictJava](https://github.com/AstraZeneca-NGS/VarDictJava)

ID = 180, Stars = 80, Language = Java, Update = 2020-06-10

VarDict Java port
##### [hall-lab](https://github.com/hall-lab)/[svtyper](https://github.com/hall-lab/svtyper)

ID = 181, Stars = 80, Language = Python, Update = 2020-02-27

Bayesian genotyper for structural variants
##### [genome](https://github.com/genome)/[sciclone](https://github.com/genome/sciclone)

ID = 182, Stars = 79, Language = R, Update = 2017-03-08

An R package for inferring the subclonal architecture of tumors
##### [tseemann](https://github.com/tseemann)/[nullarbor](https://github.com/tseemann/nullarbor)

ID = 183, Stars = 78, Language = Perl, Update = 2020-03-16

üíæ
üìÉ
"Reads to report" for public health and clinical microbiology
##### [tseemann](https://github.com/tseemann)/[barrnap](https://github.com/tseemann/barrnap)

ID = 184, Stars = 78, Language = Perl, Update = 2018-10-29

üî¨
‚ôå
Bacterial ribosomal RNA predictor
##### [dpryan79](https://github.com/dpryan79)/[MethylDackel](https://github.com/dpryan79/MethylDackel)

ID = 185, Stars = 78, Language = C, Update = 2020-05-23

A (mostly) universal methylation extractor for BS-seq experiments.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pb-assembly](https://github.com/PacificBiosciences/pb-assembly)

ID = 186, Stars = 78, Language = Python, Update = 2019-11-17

PacBio Assembly Tool Suite: Reads in ‚á® Assembly out
##### [kamilsjaron](https://github.com/kamilsjaron)/[smudgeplot](https://github.com/KamilSJaron/smudgeplot)

ID = 187, Stars = 77, Language = R, Update = 2020-06-11

Inference of ploidy and heterozygosity structure using whole genome sequencing data
##### [brentp](https://github.com/brentp)/[pyfasta](https://github.com/brentp/pyfasta)

ID = 188, Stars = 77, Language = Python, Update = 2017-12-11

fast, memory-efficient, pythonic (and command-line) access to fasta sequence files
##### [lh3](https://github.com/lh3)/[bgt](https://github.com/lh3/bgt)

ID = 189, Stars = 77, Language = C, Update = 2019-09-04

Flexible genotype query among 30,000+ samples whole-genome
##### [lh3](https://github.com/lh3)/[ksw2](https://github.com/lh3/ksw2)

ID = 190, Stars = 77, Language = C, Update = 2019-04-29

Global alignment and alignment extension
##### [arq5x](https://github.com/arq5x)/[grabix](https://github.com/arq5x/grabix)

ID = 191, Stars = 77, Language = C, Update = 2018-05-10

a wee tool for random access into BGZF files.
##### [danielecook](https://github.com/danielecook)/[gist-alfred](https://github.com/danielecook/gist-alfred)

ID = 192, Stars = 77, Language = Python, Update = 2019-11-14

An alfred workflow for accessing gists
##### [benjjneb](https://github.com/benjjneb)/[decontam](https://github.com/benjjneb/decontam)

ID = 193, Stars = 76, Language = R, Update = 2019-10-07

Simple statistical identification and removal of contaminants in marker-gene and metagenomics sequencing data
##### [richarddurbin](https://github.com/richarddurbin)/[pbwt](https://github.com/richarddurbin/pbwt)

ID = 194, Stars = 75, Language = C, Update = 2020-06-10

Implementation of Positional Burrows-Wheeler Transform for genetic data
##### [brentp](https://github.com/brentp)/[combat.py](https://github.com/brentp/combat.py)

ID = 195, Stars = 74, Language = Python, Update = 2018-12-31

python / numpy / pandas / patsy version of ComBat for removing batch effects.
##### [broadinstitute](https://github.com/broadinstitute)/[wot](https://github.com/broadinstitute/wot)

ID = 196, Stars = 74, Language = Jupyter Notebook, Update = 2020-06-05

A software package for analyzing snapshots of developmental processes
##### [broadinstitute](https://github.com/broadinstitute)/[ssGSEA2.0](https://github.com/broadinstitute/ssGSEA2.0)

ID = 197, Stars = 74, Language = R, Update = 2020-06-01

Single sample Gene Set Enrichment analysis (ssGSEA) and PTM Enrichment Analysis (PTM-SEA)
##### [ming-lian](https://github.com/ming-lian)/[Machine-Learning-Course-in-UCAS](https://github.com/Ming-Lian/Machine-Learning-Course-in-UCAS)

ID = 198, Stars = 73, Language = None, Update = 2019-09-24

ÂõΩÁßëÂ§ßÊú∫Âô®Â≠¶‰π†ËØæÁ®ã&Êú∫Âô®Â≠¶‰π†Á¨îËÆ∞
##### [brentp](https://github.com/brentp)/[bwa-meth](https://github.com/brentp/bwa-meth)

ID = 199, Stars = 73, Language = PostScript, Update = 2018-08-07

fast and accurate alignment of BS-Seq reads
##### [samtools](https://github.com/samtools)/[tabix](https://github.com/samtools/tabix)

ID = 200, Stars = 73, Language = C, Update = 2019-11-14

Note: tabix and bgzip binaries are now part of the HTSlib project.
##### [broadinstitute](https://github.com/broadinstitute)/[ichorCNA](https://github.com/broadinstitute/ichorCNA)

ID = 201, Stars = 73, Language = R, Update = 2020-05-06

Estimating tumor fraction in cell-free DNA from ultra-low-pass whole genome sequencing.
##### [andersenlab](https://github.com/andersenlab)/[VCF-kit](https://github.com/AndersenLab/VCF-kit)

ID = 202, Stars = 73, Language = Python, Update = 2020-02-11

VCF-kit: Assorted utilities for the variant call format
##### [mhammell-laboratory](https://github.com/mhammell-laboratory)/[TEtranscripts](https://github.com/mhammell-laboratory/TEtranscripts)

ID = 203, Stars = 73, Language = Python, Update = 2020-06-18

A package for including transposable elements in differential enrichment analysis of sequencing datasets.
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-data-processing](https://github.com/gatk-workflows/gatk4-data-processing)

ID = 204, Stars = 73, Language = wdl, Update = 2020-04-02

Workflows for processing high-throughput sequencing data for variant discovery with GATK4 and related tools
##### [ewels](https://github.com/ewels)/[sra-explorer](https://github.com/ewels/sra-explorer)

ID = 205, Stars = 72, Language = HTML, Update = 2020-01-20

Web application to explore the Sequence Read Archive.
##### [ncbi](https://github.com/ncbi)/[robotframework-pageobjects](https://github.com/ncbi/robotframework-pageobjects)

ID = 206, Stars = 72, Language = Python, Update = 2018-11-30

Implementation of the Page Object pattern with Robot Framework and selenium. Also facilitates page object pattern independent of Robot Framework
##### [opengene](https://github.com/opengene)/[repaq](https://github.com/OpenGene/repaq)

ID = 207, Stars = 72, Language = C, Update = 2020-06-14

A fast lossless FASTQ compressor with ultra-high compression ratio
##### [umcugenetics](https://github.com/umcugenetics)/[MutationalPatterns](https://github.com/UMCUGenetics/MutationalPatterns)

ID = 208, Stars = 71, Language = R, Update = 2020-04-01

R package for extracting and visualizing mutational patterns in base substitution catalogues
##### [tseemann](https://github.com/tseemann)/[mlst](https://github.com/tseemann/mlst)

ID = 209, Stars = 70, Language = Perl, Update = 2020-02-24

üÜî
Scan contig files against PubMLST typing schemes
##### [marianattestad](https://github.com/marianattestad)/[Assemblytics](https://github.com/MariaNattestad/Assemblytics)

ID = 210, Stars = 70, Language = JavaScript, Update = 2020-01-20

Assemblytics is a bioinformatics tool to detect and analyze structural variants from a genome assembly by comparing it to a reference genome.
##### [biocore](https://github.com/biocore)/[deblur](https://github.com/biocore/deblur)

ID = 211, Stars = 70, Language = Python, Update = 2020-04-14

Deblur is a greedy deconvolution algorithm based on known read error profiles.
##### [cschin](https://github.com/cschin)/[Peregrine](https://github.com/cschin/Peregrine)

ID = 212, Stars = 69, Language = Jupyter Notebook, Update = 2020-05-03

Peregrine: Fast Genome Assembler Using SHIMMER Index
##### [daler](https://github.com/daler)/[metaseq](https://github.com/daler/metaseq)

ID = 213, Stars = 69, Language = Python, Update = 2017-01-10

Framework for integrated analysis and plotting of ChIP/RIP/RNA/*-seq data
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbbioconda](https://github.com/PacificBiosciences/pbbioconda)

ID = 214, Stars = 69, Language = None, Update = 2020-02-16

PacBio Secondary Analysis Tools on Bioconda
##### [ensembl](https://github.com/ensembl)/[VEP_plugins](https://github.com/Ensembl/VEP_plugins)

ID = 215, Stars = 69, Language = Perl, Update = 2020-06-16

Plugins for the Ensembl Variant Effect Predictor (VEP)
##### [statgen](https://github.com/statgen)/[demuxlet](https://github.com/statgen/demuxlet)

ID = 216, Stars = 69, Language = C++, Update = 2020-03-25

Genetic multiplexing of barcoded single cell RNA-seq
##### [lh3](https://github.com/lh3)/[fermi](https://github.com/lh3/fermi)

ID = 217, Stars = 68, Language = C, Update = 2013-12-06

A WGS de novo assembler based on the FMD-index for large genomes
##### [genome](https://github.com/genome)/[analysis-workflows](https://github.com/genome/analysis-workflows)

ID = 218, Stars = 68, Language = Common Workflow Language, Update = 2020-06-18

Open workflow definitions for genomic analysis from MGI at WUSM.
##### [illumina](https://github.com/illumina)/[ExpansionHunter](https://github.com/Illumina/ExpansionHunter)

ID = 219, Stars = 68, Language = C++, Update = 2020-02-25

A tool for estimating repeat sizes
##### [arq5x](https://github.com/arq5x)/[bedtools-protocols](https://github.com/arq5x/bedtools-protocols)

ID = 220, Stars = 67, Language = CSS, Update = 2014-03-06

None
##### [griffithlab](https://github.com/griffithlab)/[pVACtools](https://github.com/griffithlab/pVACtools)

ID = 221, Stars = 67, Language = Python, Update = 2020-06-08

None
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-germline-snps-indels](https://github.com/gatk-workflows/gatk4-germline-snps-indels)

ID = 222, Stars = 67, Language = wdl, Update = 2020-06-22

Workflows for germline short variant discovery with GATK4
##### [nf-core](https://github.com/nf-core)/[tools](https://github.com/nf-core/tools)

ID = 223, Stars = 66, Language = Python, Update = 2020-06-22

Python package with helper tools for the nf-core community.
##### [brentp](https://github.com/brentp)/[gargs](https://github.com/brentp/gargs)

ID = 224, Stars = 65, Language = Go, Update = 2018-01-26

better(?) xargs in go
##### [lh3](https://github.com/lh3)/[gfatools](https://github.com/lh3/gfatools)

ID = 225, Stars = 65, Language = C, Update = 2020-06-10

Tools for manipulating sequence graphs in the GFA and rGFA formats
##### [opengene](https://github.com/opengene)/[gencore](https://github.com/OpenGene/gencore)

ID = 226, Stars = 65, Language = C++, Update = 2020-03-18

Generate consensus reads to reduce sequencing noises and remove duplications
##### [brentp](https://github.com/brentp)/[duphold](https://github.com/brentp/duphold)

ID = 227, Stars = 64, Language = Nim, Update = 2020-05-27

don't get DUP'ed or DEL'ed by your putative SVs.
##### [mroosmalen](https://github.com/mroosmalen)/[nanosv](https://github.com/mroosmalen/nanosv)

ID = 228, Stars = 64, Language = Python, Update = 2020-06-07

SV caller for nanopore data
##### [broadinstitute](https://github.com/broadinstitute)/[Drop-seq](https://github.com/broadinstitute/Drop-seq)

ID = 229, Stars = 64, Language = Java, Update = 2020-06-09

Java tools for analyzing Drop-seq data
##### [griffithlab](https://github.com/griffithlab)/[dgi-db](https://github.com/griffithlab/dgi-db)

ID = 230, Stars = 63, Language = Ruby, Update = 2020-06-22

Rails frontend to The Genome Institute's drug gene interaction database.
##### [ncbi](https://github.com/ncbi)/[ncbi-vdb](https://github.com/ncbi/ncbi-vdb)

ID = 231, Stars = 63, Language = C, Update = 2020-06-22

ncbi-vdb
##### [bioinform](https://github.com/bioinform)/[varsim](https://github.com/bioinform/varsim)

ID = 232, Stars = 63, Language = Java, Update = 2020-05-15

VarSim: A high-fidelity simulation validation framework for high-throughput genome sequencing with cancer applications
##### [ding-lab](https://github.com/ding-lab)/[CharGer](https://github.com/ding-lab/CharGer)

ID = 233, Stars = 63, Language = Python, Update = 2020-04-10

Characterization of Germline variants
##### [statgen](https://github.com/statgen)/[bamUtil](https://github.com/statgen/bamUtil)

ID = 234, Stars = 63, Language = C++, Update = 2020-06-02

None
##### [brentp](https://github.com/brentp)/[slurmpy](https://github.com/brentp/slurmpy)

ID = 235, Stars = 62, Language = Python, Update = 2020-04-08

submit jobs to slurm with quick-and-dirty python
##### [dzerbino](https://github.com/dzerbino)/[ascii_plots](https://github.com/dzerbino/ascii_plots)

ID = 236, Stars = 62, Language = Python, Update = 2017-10-30

Convenience function for quick and dirty data analysis
##### [alimanfoo](https://github.com/alimanfoo)/[csvvalidator](https://github.com/alimanfoo/csvvalidator)

ID = 237, Stars = 61, Language = Python, Update = 2020-03-09

A small CSV validator library for Python.
##### [lh3](https://github.com/lh3)/[fermi-lite](https://github.com/lh3/fermi-lite)

ID = 238, Stars = 61, Language = C, Update = 2020-03-30

Standalone C library for assembling Illumina short reads in small regions
##### [nanopore-wgs-consortium](https://github.com/nanopore-wgs-consortium)/[CHM13](https://github.com/nanopore-wgs-consortium/CHM13)

ID = 239, Stars = 61, Language = None, Update = 2020-06-04

Ultra-long reads for CHM13 genome assembly
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbmm2](https://github.com/PacificBiosciences/pbmm2)

ID = 240, Stars = 61, Language = C++, Update = 2020-06-03

A minimap2 frontend for PacBio native data formats
##### [illumina](https://github.com/illumina)/[Pisces](https://github.com/Illumina/Pisces)

ID = 241, Stars = 61, Language = C#, Update = 2019-08-09

Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows.
##### [ekg](https://github.com/ekg)/[seqwish](https://github.com/ekg/seqwish)

ID = 242, Stars = 60, Language = C++, Update = 2020-06-22

alignment to variation graph inducer
##### [lh3](https://github.com/lh3)/[hickit](https://github.com/lh3/hickit)

ID = 243, Stars = 60, Language = C, Update = 2019-05-13

TAD calling, phase imputation, 3D modeling and more for diploid single-cell Hi-C (Dip-C) and general Hi-C
##### [ncbi](https://github.com/ncbi)/[TPMCalculator](https://github.com/ncbi/TPMCalculator)

ID = 244, Stars = 60, Language = C++, Update = 2020-05-01

TPMCalculator quantifies mRNA abundance directly from the alignments by parsing BAM files
##### [bioinform](https://github.com/bioinform)/[rnacocktail](https://github.com/bioinform/rnacocktail)

ID = 245, Stars = 60, Language = Jupyter Notebook, Update = 2019-11-23

None
##### [biocore](https://github.com/biocore)/[biom-format](https://github.com/biocore/biom-format)

ID = 246, Stars = 60, Language = Python, Update = 2020-03-30

The Biological Observation Matrix (BIOM) Format Project
##### [parklab](https://github.com/parklab)/[NGSCheckMate](https://github.com/parklab/NGSCheckMate)

ID = 247, Stars = 60, Language = Python, Update = 2020-05-28

Software program for checking sample matching for NGS data
##### [mozack](https://github.com/mozack)/[abra](https://github.com/mozack/abra)

ID = 248, Stars = 59, Language = Java, Update = 2018-05-24

Assembly Based ReAligner
##### [humanlongevity](https://github.com/humanlongevity)/[HLA](https://github.com/humanlongevity/HLA)

ID = 249, Stars = 59, Language = Jupyter Notebook, Update = 2018-10-11

xHLA: Fast and accurate HLA typing from short read sequence data
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[IsoSeq_SA3nUP](https://github.com/PacificBiosciences/IsoSeq_SA3nUP)

ID = 250, Stars = 59, Language = None, Update = 2019-12-05

Documentation and Developers Code Base for Iso-Seq SMRTAnalysis 3.x and beyond
##### [opengene](https://github.com/opengene)/[OpenGene.jl](https://github.com/OpenGene/OpenGene.jl)

ID = 251, Stars = 59, Language = Julia, Update = 2017-10-08

(No maintenance) OpenGene, core libraries for NGS data analysis and bioinformatics in Julia
##### [statgen](https://github.com/statgen)/[pheweb](https://github.com/statgen/pheweb)

ID = 252, Stars = 59, Language = Python, Update = 2020-06-20

A tool to build multi-phenotype GWAS browsers
##### [crazyhottommy](https://github.com/crazyhottommy)/[pyflow-ChIPseq](https://github.com/crazyhottommy/pyflow-ChIPseq)

ID = 253, Stars = 58, Language = Python, Update = 2020-04-17

a snakemake pipeline to process ChIP-seq files from GEO or in-house
##### [lh3](https://github.com/lh3)/[bfc](https://github.com/lh3/bfc)

ID = 254, Stars = 58, Language = TeX, Update = 2016-05-31

High-performance error correction for Illumina resequencing data
##### [bioconda](https://github.com/bioconda)/[bioconda-utils](https://github.com/bioconda/bioconda-utils)

ID = 255, Stars = 58, Language = Python, Update = 2020-06-22

Utilities for building and managing bioconda recipes
##### [ncbi](https://github.com/ncbi)/[dbsnp](https://github.com/ncbi/dbsnp)

ID = 256, Stars = 58, Language = Jupyter Notebook, Update = 2020-04-22

dbSNP
##### [yongxinliu](https://github.com/yongxinliu)/[Note](https://github.com/YongxinLiu/Note)

ID = 257, Stars = 57, Language = HTML, Update = 2020-05-10

None
##### [griffithlab](https://github.com/griffithlab)/[regtools](https://github.com/griffithlab/regtools)

ID = 258, Stars = 57, Language = C++, Update = 2020-05-21

Integrate DNA-seq and RNA-seq data to identify mutations that are associated with regulatory effects on gene expression.
##### [opengene](https://github.com/opengene)/[GeneFuse](https://github.com/OpenGene/GeneFuse)

ID = 259, Stars = 57, Language = C, Update = 2020-05-21

Gene fusion detection and visualization
##### [statgen](https://github.com/statgen)/[SLURM-examples](https://github.com/statgen/SLURM-examples)

ID = 260, Stars = 57, Language = Python, Update = 2019-06-26

None
##### [broadinstitute](https://github.com/broadinstitute)/[oncotator](https://github.com/broadinstitute/oncotator)

ID = 261, Stars = 56, Language = Python, Update = 2020-02-26

None
##### [tseemann](https://github.com/tseemann)/[any2fasta](https://github.com/tseemann/any2fasta)

ID = 262, Stars = 55, Language = Perl, Update = 2019-05-17

Convert various sequence formats to FASTA
##### [yongxinliu](https://github.com/yongxinliu)/[QIIME2ChineseManual](https://github.com/YongxinLiu/QIIME2ChineseManual)

ID = 263, Stars = 55, Language = JavaScript, Update = 2020-05-15

QIIME 2‰∏≠ÊñáÊñáÊ°£(QIIME 2 Chinese Manual)
##### [dzerbino](https://github.com/dzerbino)/[oases](https://github.com/dzerbino/oases)

ID = 264, Stars = 55, Language = C, Update = 2018-03-12

De novo transcriptome assembler for short reads
##### [parklab](https://github.com/parklab)/[nozzle](https://github.com/parklab/nozzle)

ID = 265, Stars = 55, Language = R, Update = 2016-04-02

Nozzle is a report generation toolkit for data analysis pipelines implemented in R.
##### [ekg](https://github.com/ekg)/[fastahack](https://github.com/ekg/fastahack)

ID = 266, Stars = 54, Language = C++, Update = 2020-01-06

utilities for indexing and sequence extraction from FASTA files
##### [lh3](https://github.com/lh3)/[bedtk](https://github.com/lh3/bedtk)

ID = 267, Stars = 54, Language = C, Update = 2020-01-27

A simple toolset for BED files (warning: CLI may change before bedtk becomes stable)
##### [cschin](https://github.com/cschin)/[DCNet](https://github.com/cschin/DCNet)

ID = 268, Stars = 54, Language = Jupyter Notebook, Update = 2018-04-16

DCNet‚Ää‚Äî‚ÄäDenoising (DNA) Sequence With a LSTM-RNN and¬†PyTorch & Neural DBG
##### [griffithlab](https://github.com/griffithlab)/[pVAC-Seq](https://github.com/griffithlab/pVAC-Seq)

ID = 269, Stars = 54, Language = Python, Update = 2019-04-16

DEPRECATED. This tool has been superseded by https://github.com/griffithlab/pVACtools
##### [benlangmead](https://github.com/benlangmead)/[langmead-lab](https://github.com/BenLangmead/langmead-lab)

ID = 270, Stars = 53, Language = None, Update = 2015-05-28

Publicly-visible Langmead Lab resources
##### [ewels](https://github.com/ewels)/[AWS-iGenomes](https://github.com/ewels/AWS-iGenomes)

ID = 271, Stars = 53, Language = HTML, Update = 2019-11-06

Documentation and description of AWS iGenomes S3 resource.
##### [mozack](https://github.com/mozack)/[abra2](https://github.com/mozack/abra2)

ID = 272, Stars = 53, Language = Java, Update = 2020-04-23

ABRA2
##### [nf-core](https://github.com/nf-core)/[chipseq](https://github.com/nf-core/chipseq)

ID = 273, Stars = 53, Language = Nextflow, Update = 2020-06-03

ChIP-seq peak-calling, QC and differential analysis pipeline.
##### [wglab](https://github.com/wglab)/[DeepMod](https://github.com/WGLab/DeepMod)

ID = 274, Stars = 53, Language = Python, Update = 2020-01-02

DeepMod: a deep-learning tool for genomic-scale, strand-sensitive and single-nucleotide based detection of DNA modifications
##### [illumina](https://github.com/illumina)/[PrimateAI](https://github.com/Illumina/PrimateAI)

ID = 275, Stars = 53, Language = Python, Update = 2019-02-26

deep residual neural network for classifying the pathogenicity of missense mutations.
##### [lh3](https://github.com/lh3)/[dna-nn](https://github.com/lh3/dna-nn)

ID = 276, Stars = 52, Language = C, Update = 2019-06-16

Model and predict short DNA sequence features with neural networks
##### [ocxtal](https://github.com/ocxtal)/[libgaba](https://github.com/ocxtal/libgaba)

ID = 277, Stars = 52, Language = C, Update = 2018-05-04

Adaptive semi-global banded alignment on string graphs
##### [ncbi](https://github.com/ncbi)/[ngs-tools](https://github.com/ncbi/ngs-tools)

ID = 278, Stars = 52, Language = C++, Update = 2020-06-22

None
##### [dpryan79](https://github.com/dpryan79)/[libBigWig](https://github.com/dpryan79/libBigWig)

ID = 279, Stars = 51, Language = C, Update = 2020-03-04

A C library for handling bigWig files
##### [arq5x](https://github.com/arq5x)/[cyvcf](https://github.com/arq5x/cyvcf)

ID = 280, Stars = 51, Language = Python, Update = 2018-03-27

A fast Python library for VCF files leveraging Cython for speed.
##### [dutilh](https://github.com/dutilh)/[CAT](https://github.com/dutilh/CAT)

ID = 281, Stars = 51, Language = Python, Update = 2020-06-22

CAT/BAT: tool for taxonomic classification of contigs and metagenome-assembled genomes (MAGs)
##### [ncbi](https://github.com/ncbi)/[SKESA](https://github.com/ncbi/SKESA)

ID = 282, Stars = 51, Language = C++, Update = 2020-05-17

SKESA assembler
##### [ensembl](https://github.com/ensembl)/[ensembl](https://github.com/Ensembl/ensembl)

ID = 283, Stars = 51, Language = Perl, Update = 2020-06-15

The Ensembl Core Perl API and SQL schema
##### [brentp](https://github.com/brentp)/[xopen](https://github.com/brentp/xopen)

ID = 284, Stars = 50, Language = Go, Update = 2018-11-16

open files for buffered reading and writing in #golang
##### [quinlan-lab](https://github.com/quinlan-lab)/[vcf2db](https://github.com/quinlan-lab/vcf2db)

ID = 285, Stars = 50, Language = Python, Update = 2020-02-24

create a gemini-compatible database from a VCF
##### [openvax](https://github.com/openvax)/[varcode](https://github.com/openvax/varcode)

ID = 286, Stars = 50, Language = Python, Update = 2019-11-08

Library for manipulating genomic variants and predicting their effects
##### [gavinha](https://github.com/gavinha)/[TitanCNA](https://github.com/gavinha/TitanCNA)

ID = 287, Stars = 49, Language = R, Update = 2020-06-08

Analysis of subclonal copy number alterations (CNA) and loss of heterozygosity (LOH) in cancer
##### [lh3](https://github.com/lh3)/[ropebwt2](https://github.com/lh3/ropebwt2)

ID = 288, Stars = 49, Language = TeX, Update = 2020-04-17

Incremental construction of FM-index for DNA sequences
##### [lh3](https://github.com/lh3)/[tabtk](https://github.com/lh3/tabtk)

ID = 289, Stars = 49, Language = C, Update = 2016-09-08

Toolkit for processing TAB-delimited format
##### [marianattestad](https://github.com/marianattestad)/[SplitThreader](https://github.com/MariaNattestad/SplitThreader)

ID = 290, Stars = 49, Language = JavaScript, Update = 2017-02-03

Explore rearrangements and copy-number amplifications in a cancer genome
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[ccs](https://github.com/PacificBiosciences/ccs)

ID = 291, Stars = 49, Language = None, Update = 2020-04-20

CCS: Generate Highly Accurate Single-Molecule Consensus Reads (HiFi Reads)
##### [biocore](https://github.com/biocore)/[emperor](https://github.com/biocore/emperor)

ID = 292, Stars = 49, Language = JavaScript, Update = 2020-06-22

Emperor a tool for the analysis and visualization of large microbial ecology datasets
##### [ewels](https://github.com/ewels)/[CPT-Bootstrap-Carousel](https://github.com/ewels/CPT-Bootstrap-Carousel)

ID = 293, Stars = 48, Language = PHP, Update = 2020-04-17

WordPress plugin which generates a custom post type for choosing images and content. Outputs Bootstrap Image Carousel (slider) HTML from a shortcode.
##### [ensembl](https://github.com/ensembl)/[ensembl-pipeline](https://github.com/Ensembl/ensembl-pipeline)

ID = 294, Stars = 48, Language = Perl, Update = 2020-01-30

*DEPRECATED* Job management for the Ensembl Genebuild pipeline
##### [yanglab](https://github.com/yanglab)/[CIRCexplorer2](https://github.com/YangLab/CIRCexplorer2)

ID = 295, Stars = 48, Language = Python, Update = 2020-02-25

circular RNA analysis toolset
##### [yanglab](https://github.com/yanglab)/[CIRCexplorer](https://github.com/YangLab/CIRCexplorer)

ID = 296, Stars = 48, Language = Python, Update = 2019-06-01

A combined strategy to identify circular RNAs (circRNAs and ciRNAs) (Zhang et al., Complementary Sequence-Mediated Exon Circularization, Cell (2014), 159:134-147)
##### [gt1](https://github.com/gt1)/[biobambam](https://github.com/gt1/biobambam)

ID = 297, Stars = 47, Language = C++, Update = 2015-04-20

Tools for bam file processing
##### [genome](https://github.com/genome)/[genome](https://github.com/genome/genome)

ID = 298, Stars = 47, Language = Perl, Update = 2020-06-22

Core modules used by the GMS
##### [ga4gh](https://github.com/ga4gh)/[workflow-execution-service-schemas](https://github.com/ga4gh/workflow-execution-service-schemas)

ID = 299, Stars = 47, Language = Python, Update = 2020-06-22

The WES API is a standard way to run and manage portable workflows.
##### [nf-core](https://github.com/nf-core)/[sarek](https://github.com/nf-core/sarek)

ID = 300, Stars = 47, Language = Nextflow, Update = 2020-06-22

Analysis pipeline to detect germline or somatic variants from WGS / targeted sequencing
##### [wglab](https://github.com/wglab)/[PennCNV](https://github.com/WGLab/PennCNV)

ID = 301, Stars = 47, Language = C, Update = 2019-12-05

Copy number vaiation detection from SNP arrays
##### [illumina](https://github.com/illumina)/[akt](https://github.com/Illumina/akt)

ID = 302, Stars = 47, Language = C++, Update = 2020-05-22

Ancestry and Kinship Tools
##### [ncbi](https://github.com/ncbi)/[amr](https://github.com/ncbi/amr)

ID = 303, Stars = 46, Language = C++, Update = 2020-05-26

amr_finder
##### [illumina](https://github.com/illumina)/[interop](https://github.com/Illumina/interop)

ID = 304, Stars = 46, Language = C++, Update = 2020-04-18

C++ Library to parse Illumina InterOp files
##### [tanghaibao](https://github.com/tanghaibao)/[bio-pipeline](https://github.com/tanghaibao/bio-pipeline)

ID = 305, Stars = 45, Language = C, Update = 2020-06-14

My collection of light bioinformatics analysis pipelines for specific tasks
##### [tanghaibao](https://github.com/tanghaibao)/[quota-alignment](https://github.com/tanghaibao/quota-alignment)

ID = 306, Stars = 45, Language = Python, Update = 2017-09-05

Guided synteny alignment between duplicated genomes (within specified quota constraint)
##### [pcingola](https://github.com/pcingola)/[jFuzzyLogic](https://github.com/pcingola/jFuzzyLogic)

ID = 307, Stars = 45, Language = Java, Update = 2018-07-17

The most complete fuzzy logic library in Java. The de-facto standard for research and industrial applications.
##### [arq5x](https://github.com/arq5x)/[ggd](https://github.com/arq5x/ggd)

ID = 308, Stars = 45, Language = Python, Update = 2016-04-20

None
##### [daler](https://github.com/daler)/[matplotlibrc](https://github.com/daler/matplotlibrc)

ID = 309, Stars = 45, Language = Python, Update = 2013-11-05

some example matplotlibrc files, and a script display their effects
##### [griffithlab](https://github.com/griffithlab)/[civic-client](https://github.com/griffithlab/civic-client)

ID = 310, Stars = 45, Language = JavaScript, Update = 2020-06-21

Web client for CIViC: Clinical Interpretations of Variants in Cancer
##### [griffithlab](https://github.com/griffithlab)/[DeepSVR](https://github.com/griffithlab/DeepSVR)

ID = 311, Stars = 45, Language = Jupyter Notebook, Update = 2019-06-27

None
##### [refresh-bio](https://github.com/refresh-bio)/[kmer-db](https://github.com/refresh-bio/kmer-db)

ID = 312, Stars = 45, Language = C++, Update = 2020-03-30

None
##### [vgteam](https://github.com/vgteam)/[odgi](https://github.com/vgteam/odgi)

ID = 313, Stars = 45, Language = C++, Update = 2020-06-22

optimized dynamic genome graph implementation
##### [nf-core](https://github.com/nf-core)/[rnafusion](https://github.com/nf-core/rnafusion)

ID = 314, Stars = 45, Language = Nextflow, Update = 2020-06-16

RNA-seq analysis pipeline for detection gene-fusions
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[IsoSeq](https://github.com/PacificBiosciences/IsoSeq)

ID = 315, Stars = 45, Language = None, Update = 2020-03-18

IsoSeq3 - Scalable De Novo Isoform Discovery from Single-Molecule PacBio Reads
##### [opengene](https://github.com/opengene)/[CfdnaPattern](https://github.com/OpenGene/CfdnaPattern)

ID = 316, Stars = 45, Language = Python, Update = 2018-08-03

Pattern Recognition for Cell-free DNA
##### [illumina](https://github.com/illumina)/[PlatinumGenomes](https://github.com/Illumina/PlatinumGenomes)

ID = 317, Stars = 45, Language = None, Update = 2017-11-08

The Platinum Genomes Truthset
##### [brentp](https://github.com/brentp)/[vcfgo](https://github.com/brentp/vcfgo)

ID = 318, Stars = 44, Language = Go, Update = 2020-03-14

a golang library to read, write and manipulate files in the variant call format.
##### [brentp](https://github.com/brentp)/[hts-python](https://github.com/brentp/hts-python)

ID = 319, Stars = 44, Language = Python, Update = 2017-03-03

pythonic wrapper for libhts (moved to: https://github.com/quinlan-lab/hts-python)
##### [nf-core](https://github.com/nf-core)/[methylseq](https://github.com/nf-core/methylseq)

ID = 320, Stars = 44, Language = Nextflow, Update = 2020-04-28

Methylation (Bisulfite-Sequencing) analysis pipeline using Bismark or bwa-meth + MethylDackel
##### [broadinstitute](https://github.com/broadinstitute)/[tensorqtl](https://github.com/broadinstitute/tensorqtl)

ID = 321, Stars = 44, Language = Python, Update = 2020-06-18

Ultrafast GPU-based QTL mapper
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad-browser](https://github.com/broadinstitute/gnomad-browser)

ID = 322, Stars = 44, Language = JavaScript, Update = 2020-06-11

JavaScript tools for exploring genomic data
##### [openvax](https://github.com/openvax)/[mhctools](https://github.com/openvax/mhctools)

ID = 323, Stars = 44, Language = Python, Update = 2020-05-01

Python interface to running command-line and web-based MHC binding predictors
##### [biocore](https://github.com/biocore)/[mmvec](https://github.com/biocore/mmvec)

ID = 324, Stars = 44, Language = Python, Update = 2020-04-25

Neural networks for microbe-metabolite interaction analysis
##### [ding-lab](https://github.com/ding-lab)/[MuSiC2](https://github.com/ding-lab/MuSiC2)

ID = 325, Stars = 44, Language = Perl, Update = 2019-05-19

identifying mutational significance in cancer genomes
##### [seandavi](https://github.com/seandavi)/[GEOquery](https://github.com/seandavi/GEOquery)

ID = 326, Stars = 43, Language = R, Update = 2020-05-30

The bridge between the NCBI Gene Expression Omnibus and Bioconductor
##### [tseemann](https://github.com/tseemann)/[snp-dists](https://github.com/tseemann/snp-dists)

ID = 327, Stars = 43, Language = C, Update = 2020-03-01

Pairwise SNP distance matrix from a FASTA sequence alignment
##### [lh3](https://github.com/lh3)/[samtools](https://github.com/lh3/samtools)

ID = 328, Stars = 43, Language = C, Update = 2017-06-09

This is *NOT* the official repository of samtools.
##### [arq5x](https://github.com/arq5x)/[filo](https://github.com/arq5x/filo)

ID = 329, Stars = 43, Language = C++, Update = 2015-12-21

Useful FILe and stream Operations
##### [ncbi](https://github.com/ncbi)/[icn3d](https://github.com/ncbi/icn3d)

ID = 330, Stars = 43, Language = JavaScript, Update = 2020-06-20

WebGL-based protein structure viewer
##### [biod](https://github.com/biod)/[BioD](https://github.com/biod/BioD)

ID = 331, Stars = 43, Language = D, Update = 2019-12-17

A D library for computational biology and bioinformatics
##### [ewels](https://github.com/ewels)/[MegaQC](https://github.com/ewels/MegaQC)

ID = 332, Stars = 42, Language = Python, Update = 2020-06-07

Web application to collect and visualise data across multiple MultiQC runs.
##### [brentp](https://github.com/brentp)/[interlap](https://github.com/brentp/interlap)

ID = 333, Stars = 42, Language = Python, Update = 2018-08-31

fast, pure-python interval overlap testing
##### [brentp](https://github.com/brentp)/[go-chartjs](https://github.com/brentp/go-chartjs)

ID = 334, Stars = 42, Language = Go, Update = 2017-09-01

golang library to make https://chartjs.org/ plots (this is vanilla #golang, not gopherjs)
##### [crazyhottommy](https://github.com/crazyhottommy)/[DNA-methylation-analysis](https://github.com/crazyhottommy/DNA-methylation-analysis)

ID = 335, Stars = 42, Language = None, Update = 2019-09-19

DNA methylation analysis notes from Ming Tang
##### [quinlan-lab](https://github.com/quinlan-lab)/[sllobs-biostats](https://github.com/quinlan-lab/sllobs-biostats)

ID = 336, Stars = 42, Language = HTML, Update = 2020-04-28

None
##### [wglab](https://github.com/wglab)/[biocluster](https://github.com/WGLab/biocluster)

ID = 337, Stars = 42, Language = None, Update = 2019-04-14

Tutorial on building a computing cluster for bioinformatics
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[reference_data](https://github.com/AstraZeneca-NGS/reference_data)

ID = 338, Stars = 42, Language = Python, Update = 2017-11-28

Reference data: BED files, genes, transcripts, variations.
##### [papenfusslab](https://github.com/papenfusslab)/[bioshake](https://github.com/PapenfussLab/bioshake)

ID = 339, Stars = 42, Language = Haskell, Update = 2019-08-13

Bioinformatics pipelines with Haskell and Shake
##### [freeseek](https://github.com/freeseek)/[gtc2vcf](https://github.com/freeseek/gtc2vcf)

ID = 340, Stars = 41, Language = C, Update = 2020-06-10

Tools to convert Illumina IDAT/BPM/EGT/GTC and Affymetrix CEL/CHP files to VCF
##### [ming-lian](https://github.com/ming-lian)/[Bioinfo_LeetCode](https://github.com/Ming-Lian/Bioinfo_LeetCode)

ID = 341, Stars = 41, Language = Python, Update = 2020-04-11

ËÆ©Êàë‰ª¨‰ª•Âà∑LeetCodeÈ¢òÁõÆÁöÑÂΩ¢ÂºèÔºåÊù•ÁºñÁ®ãËß£ÂÜ≥ÁîüÁâ©‰ø°ÊÅØÂ≠¶ÈóÆÈ¢òÂêßÔºåCome on
##### [brentp](https://github.com/brentp)/[fishers_exact_test](https://github.com/brentp/fishers_exact_test)

ID = 342, Stars = 41, Language = Python, Update = 2019-10-02

Fishers Exact Test for Python (Cython)
##### [refresh-bio](https://github.com/refresh-bio)/[DSRC](https://github.com/refresh-bio/DSRC)

ID = 343, Stars = 41, Language = C++, Update = 2020-04-19

DSRC - DNA Sequence Reads Compressor
##### [broadinstitute](https://github.com/broadinstitute)/[docker-terraform](https://github.com/broadinstitute/docker-terraform)

ID = 344, Stars = 41, Language = Dockerfile, Update = 2020-05-22

Docker container for running the Terraform application
##### [andersenlab](https://github.com/andersenlab)/[IBiS-Bootcamp](https://github.com/AndersenLab/IBiS-Bootcamp)

ID = 345, Stars = 41, Language = Shell, Update = 2016-09-14

None
##### [bioinform](https://github.com/bioinform)/[metasv](https://github.com/bioinform/metasv)

ID = 346, Stars = 41, Language = Python, Update = 2017-06-30

MetaSV: An accurate and integrative structural-variant caller for next generation sequencing
##### [brentp](https://github.com/brentp)/[irelate](https://github.com/brentp/irelate)

ID = 347, Stars = 40, Language = Go, Update = 2018-08-01

Streaming relation (overlap, distance, KNN) of (any number of) sorted genomic interval sets. #golang
##### [brentp](https://github.com/brentp)/[indelope](https://github.com/brentp/indelope)

ID = 348, Stars = 40, Language = Nim, Update = 2017-12-03

find large indels (in the blind spot between GATK/freebayes and SV callers)
##### [zhou-lab](https://github.com/zhou-lab)/[biscuit](https://github.com/zhou-lab/biscuit)

ID = 349, Stars = 40, Language = C, Update = 2020-05-13

BISulfite-seq CUI Toolkit
##### [ga4gh](https://github.com/ga4gh)/[vr-spec](https://github.com/ga4gh/vr-spec)

ID = 350, Stars = 40, Language = Makefile, Update = 2020-06-22

Extensible specification for representing and uniquely identifying biological sequence variation
##### [ga4gh](https://github.com/ga4gh)/[task-execution-schemas](https://github.com/ga4gh/task-execution-schemas)

ID = 351, Stars = 40, Language = None, Update = 2020-02-05

None
##### [nf-core](https://github.com/nf-core)/[ampliseq](https://github.com/nf-core/ampliseq)

ID = 352, Stars = 40, Language = Nextflow, Update = 2020-06-17

16S rRNA amplicon sequencing analysis workflow using QIIME2
##### [broadinstitute](https://github.com/broadinstitute)/[ABC-Enhancer-Gene-Prediction](https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction)

ID = 353, Stars = 40, Language = Python, Update = 2020-06-13

Cell type specific enhancer-gene predictions using ABC model (Fulco, Nasser et al, Nature Genetics 2019)
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbsv](https://github.com/PacificBiosciences/pbsv)

ID = 354, Stars = 40, Language = None, Update = 2020-05-05

pbsv - PacBio structural variant (SV) calling and analysis tools
##### [ensembl](https://github.com/ensembl)/[ensembl-hive](https://github.com/Ensembl/ensembl-hive)

ID = 355, Stars = 40, Language = Perl, Update = 2020-06-19

EnsEMBL Hive - a system for creating and running pipelines on a distributed compute resource
##### [wglab](https://github.com/wglab)/[phenolyzer](https://github.com/WGLab/phenolyzer)

ID = 356, Stars = 40, Language = Perl, Update = 2019-12-04

phenotype-based prioritization of candidate genes for human diseases
##### [opengene](https://github.com/opengene)/[fastv](https://github.com/OpenGene/fastv)

ID = 357, Stars = 40, Language = C++, Update = 2020-06-01

An ultra-fast tool for identification of SARS-CoV-2 and other microbes from sequencing data. This tool can be used to detect viral infectious diseases, like COVID-19.
##### [statgen](https://github.com/statgen)/[libStatGen](https://github.com/statgen/libStatGen)

ID = 358, Stars = 40, Language = C++, Update = 2020-05-12

Useful set of classes for creating statistical genetic programs.
##### [tseemann](https://github.com/tseemann)/[VelvetOptimiser](https://github.com/tseemann/VelvetOptimiser)

ID = 359, Stars = 39, Language = Perl, Update = 2017-12-21

üìà
Automatically optimise three of Velvet's assembly parameters.
##### [brentp](https://github.com/brentp)/[hts-nim-tools](https://github.com/brentp/hts-nim-tools)

ID = 360, Stars = 39, Language = Nim, Update = 2020-04-21

useful command-line tools written to show-case hts-nim
##### [broadinstitute](https://github.com/broadinstitute)/[gamgee](https://github.com/broadinstitute/gamgee)

ID = 361, Stars = 39, Language = C++, Update = 2019-08-05

A C++14 library for NGS data formats
##### [openvax](https://github.com/openvax)/[neoantigen-vaccine-pipeline](https://github.com/openvax/neoantigen-vaccine-pipeline)

ID = 362, Stars = 39, Language = Jupyter Notebook, Update = 2020-06-01

Bioinformatics pipeline for selecting patient-specific cancer neoantigen vaccines
##### [openvax](https://github.com/openvax)/[pepdata](https://github.com/openvax/pepdata)

ID = 363, Stars = 39, Language = Python, Update = 2018-02-26

Python interface to amino acid properties and IEDB
##### [ewels](https://github.com/ewels)/[nf-core-rnaseq](https://github.com/ewels/nf-core-rnaseq)

ID = 364, Stars = 38, Language = Nextflow, Update = 2020-06-17

See the main fork of this repository here >>>
##### [crazyhottommy](https://github.com/crazyhottommy)/[pyflow-ATACseq](https://github.com/crazyhottommy/pyflow-ATACseq)

ID = 365, Stars = 38, Language = Python, Update = 2020-02-13

ATAC-seq snakemake pipeline
##### [daler](https://github.com/daler)/[trackhub](https://github.com/daler/trackhub)

ID = 366, Stars = 38, Language = Python, Update = 2019-12-24

create, manage, and upload track hubs for use in the UCSC genome browser
##### [danielecook](https://github.com/danielecook)/[iex-api-python](https://github.com/danielecook/iex-api-python)

ID = 367, Stars = 38, Language = Python, Update = 2018-10-23

A python wrapper for the IEX API
##### [mathonco](https://github.com/mathonco)/[NeoPredPipe](https://github.com/MathOnco/NeoPredPipe)

ID = 368, Stars = 38, Language = Python, Update = 2020-06-17

Neoantigens prediction pipeline for multi- or single-region vcf files using ANNOVAR and netMHCpan.
##### [broadinstitute](https://github.com/broadinstitute)/[dsde-deep-learning](https://github.com/broadinstitute/dsde-deep-learning)

ID = 369, Stars = 38, Language = Python, Update = 2019-03-28

DSDE Deep Learning Club
##### [ncbi](https://github.com/ncbi)/[JATSPreviewStylesheets](https://github.com/ncbi/JATSPreviewStylesheets)

ID = 370, Stars = 38, Language = XSLT, Update = 2014-05-13

JATS Preview Stylesheets
##### [openvax](https://github.com/openvax)/[gtfparse](https://github.com/openvax/gtfparse)

ID = 371, Stars = 38, Language = Python, Update = 2018-10-09

Parsing tools for GTF (gene transfer format) files
##### [gatk-workflows](https://github.com/gatk-workflows)/[broad-prod-wgs-germline-snps-indels](https://github.com/gatk-workflows/broad-prod-wgs-germline-snps-indels)

ID = 372, Stars = 38, Language = wdl, Update = 2020-03-19

Workflows for germline short variant discovery in WGS data. PairedEndSingleSampleWf has been superseded in Broad production by https://github.com/gatk-workflows/five-dollar-genome-analysis-pipeline
##### [benlangmead](https://github.com/benlangmead)/[crossbow](https://github.com/BenLangmead/crossbow)

ID = 373, Stars = 37, Language = Perl, Update = 2014-01-14

Variant calling from sequence reads using cloud computing
##### [brentp](https://github.com/brentp)/[intintmap](https://github.com/brentp/intintmap)

ID = 374, Stars = 37, Language = Go, Update = 2019-02-11

fast int64-int64 map for go
##### [lh3](https://github.com/lh3)/[htsbox](https://github.com/lh3/htsbox)

ID = 375, Stars = 37, Language = C, Update = 2018-03-29

My experimental tools on top of htslib. NOT OFFICIAL!!!
##### [nf-core](https://github.com/nf-core)/[atacseq](https://github.com/nf-core/atacseq)

ID = 376, Stars = 37, Language = Nextflow, Update = 2020-06-17

ATAC-seq peak-calling, QC and differential analysis pipeline
##### [daehwankimlab](https://github.com/daehwankimlab)/[hisat](https://github.com/DaehwanKimLab/hisat)

ID = 377, Stars = 37, Language = C++, Update = 2015-09-08

Fast spliced aligner with low memory requirements
##### [cgatoxford](https://github.com/cgatoxford)/[CGATPipelines](https://github.com/CGATOxford/CGATPipelines)

ID = 378, Stars = 37, Language = Python, Update = 2018-11-08

Collection of CGAT NGS Pipelines
##### [seandavi](https://github.com/seandavi)/[sars2pack](https://github.com/seandavi/sars2pack)

ID = 379, Stars = 36, Language = R, Update = 2020-06-18

An R package with nearly 40 highly cited, read-to-use, up-to-date COVID-19 pandemic data resources
##### [brentp](https://github.com/brentp)/[jigv](https://github.com/brentp/jigv)

ID = 380, Stars = 36, Language = Nim, Update = 2020-04-07

igv.js server and automatic configuration to view bam/cram/vcf/bed. "working in under 1 minute"
##### [crazyhottommy](https://github.com/crazyhottommy)/[scATACseq-analysis-notes](https://github.com/crazyhottommy/scATACseq-analysis-notes)

ID = 381, Stars = 36, Language = None, Update = 2020-06-19

my notes for scATACseq analysis
##### [lh3](https://github.com/lh3)/[CHM-eval](https://github.com/lh3/CHM-eval)

ID = 382, Stars = 36, Language = TeX, Update = 2019-10-03

None
##### [broadinstitute](https://github.com/broadinstitute)/[catch](https://github.com/broadinstitute/catch)

ID = 383, Stars = 36, Language = Python, Update = 2020-06-12

A package for designing compact and comprehensive capture probe sets.
##### [ncbi](https://github.com/ncbi)/[BAMscale](https://github.com/ncbi/BAMscale)

ID = 384, Stars = 36, Language = C, Update = 2020-06-02

BAMscale is a one-step tool for either 1) quantifying and normalizing the coverage of peaks or 2) generated scaled BigWig files for easy visualization of commonly used DNA-seq capture based methods.
##### [niu-lab](https://github.com/niu-lab)/[msisensor2](https://github.com/niu-lab/msisensor2)

ID = 385, Stars = 36, Language = R, Update = 2019-11-21

Microsatellite instability (MSI) detection for tumor only data.
##### [mdshw5](https://github.com/mdshw5)/[simplesam](https://github.com/mdshw5/simplesam)

ID = 386, Stars = 35, Language = Python, Update = 2020-06-02

Simple pure Python SAM parser and objects for working with SAM records
##### [stephenturner](https://github.com/stephenturner)/[twitterchive](https://github.com/stephenturner/twitterchive)

ID = 387, Stars = 35, Language = R, Update = 2015-03-25

Archive and analyze results from a Twitter search (**no longer maintained**)
##### [griffithlab](https://github.com/griffithlab)/[civic-server](https://github.com/griffithlab/civic-server)

ID = 388, Stars = 35, Language = HTML, Update = 2020-06-22

Backend Server for CIViC Project
##### [vgteam](https://github.com/vgteam)/[graph-genomics-review](https://github.com/vgteam/graph-genomics-review)

ID = 389, Stars = 35, Language = TeX, Update = 2020-04-15

Pangenome graphs (review article on graph-based pangenomic methods)
##### [ensembl](https://github.com/ensembl)/[ensembl-compara](https://github.com/Ensembl/ensembl-compara)

ID = 390, Stars = 35, Language = Perl, Update = 2020-06-22

The Ensembl Compara Perl API and SQL schema
##### [biocore](https://github.com/biocore)/[gneiss](https://github.com/biocore/gneiss)

ID = 391, Stars = 35, Language = Jupyter Notebook, Update = 2019-10-11

compositional data analysis toolbox
##### [ekg](https://github.com/ekg)/[bamaddrg](https://github.com/ekg/bamaddrg)

ID = 392, Stars = 34, Language = C++, Update = 2017-09-21

adds sample names and read-group (RG) tags to BAM alignments
##### [ekg](https://github.com/ekg)/[glia](https://github.com/ekg/glia)

ID = 393, Stars = 34, Language = C++, Update = 2016-07-05

a string to graph aligner
##### [ming-lian](https://github.com/ming-lian)/[Bioinformatics-skills](https://github.com/Ming-Lian/Bioinformatics-skills)

ID = 394, Stars = 34, Language = None, Update = 2020-04-23

ÁîüÁâ©‰ø°ÊÅØÂ≠¶ÊäÄËÉΩÊèêÂçáÔºå‰∏ìÊ≥®‰∫éÂ∑•ÂÖ∑‰ΩøÁî®‰∏éÁºñÁ®ã
##### [brentp](https://github.com/brentp)/[bigly](https://github.com/brentp/bigly)

ID = 395, Stars = 34, Language = Go, Update = 2018-01-17

a pileup library that embraces the huge
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-viz](https://github.com/broadinstitute/gtex-viz)

ID = 396, Stars = 34, Language = JavaScript, Update = 2020-01-28

GTEx Visualizations
##### [pharmgkb](https://github.com/pharmgkb)/[PharmCAT](https://github.com/PharmGKB/PharmCAT)

ID = 397, Stars = 34, Language = Java, Update = 2020-06-14

The Pharmacogenomic Clinical Annotation Tool
##### [openvax](https://github.com/openvax)/[vaxrank](https://github.com/openvax/vaxrank)

ID = 398, Stars = 34, Language = Python, Update = 2020-02-22

Ranked vaccine peptides for personalized cancer immunotherapy
##### [parklab](https://github.com/parklab)/[SigMA](https://github.com/parklab/SigMA)

ID = 399, Stars = 34, Language = R, Update = 2020-04-30

Mutational signature analysis for low statistics SNV data
##### [lh3](https://github.com/lh3)/[etrf](https://github.com/lh3/etrf)

ID = 400, Stars = 33, Language = C, Update = 2019-10-22

Exact Tandem Repeat Finder (not a TRF replacement)
##### [nf-core](https://github.com/nf-core)/[eager](https://github.com/nf-core/eager)

ID = 401, Stars = 33, Language = Nextflow, Update = 2020-06-22

A fully reproducible and state of the art ancient DNA analysis pipeline
##### [ncbi](https://github.com/ncbi)/[dbvar](https://github.com/ncbi/dbvar)

ID = 402, Stars = 33, Language = None, Update = 2020-04-22

dbVar
##### [ding-lab](https://github.com/ding-lab)/[hotspot3d](https://github.com/ding-lab/hotspot3d)

ID = 403, Stars = 33, Language = Perl, Update = 2019-08-22

3D hotspot mutation proximity analysis tool
##### [papenfusslab](https://github.com/papenfusslab)/[StructuralVariantAnnotation](https://github.com/PapenfussLab/StructuralVariantAnnotation)

ID = 404, Stars = 33, Language = R, Update = 2020-04-15

R package designed to simplify structural variant analysis
##### [seandavi](https://github.com/seandavi)/[wdlRunR](https://github.com/seandavi/wdlRunR)

ID = 405, Stars = 32, Language = R, Update = 2019-01-30

Elastic, reproducible, and reusable genomic data science tools from R backed by cloud resources
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_portal_core](https://github.com/broadinstitute/single_cell_portal_core)

ID = 406, Stars = 32, Language = Ruby, Update = 2020-06-23

Rails/Docker application for the Broad Institute's single cell RNA-seq data portal
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[barcoding](https://github.com/PacificBiosciences/barcoding)

ID = 407, Stars = 32, Language = R, Update = 2020-02-18

Lima - Demultiplex Barcoded PacBio Samples
##### [ensembl](https://github.com/ensembl)/[ensembl-tools](https://github.com/Ensembl/ensembl-tools)

ID = 408, Stars = 32, Language = Perl, Update = 2020-06-09

Ensembl tools
##### [biocore](https://github.com/biocore)/[scikit-bio-cookbook](https://github.com/biocore/scikit-bio-cookbook)

ID = 409, Stars = 32, Language = Jupyter Notebook, Update = 2016-06-15

Recipes for bioinformatics analyses with scikit-bio
##### [statgen](https://github.com/statgen)/[locuszoom-standalone](https://github.com/statgen/locuszoom-standalone)

ID = 410, Stars = 32, Language = Python, Update = 2020-01-21

Create regional association plots from GWAS or meta-analysis
##### [brentp](https://github.com/brentp)/[450k-analysis-guide](https://github.com/brentp/450k-analysis-guide)

ID = 411, Stars = 31, Language = TeX, Update = 2014-10-02

A Practical (And Opinionated) Guide To Analyzing 450K Data
##### [lh3](https://github.com/lh3)/[klib.nim](https://github.com/lh3/klib.nim)

ID = 412, Stars = 31, Language = Nim, Update = 2020-04-20

Experimental getopt, gzip reader, FASTA/Q parser and interval queries in nim-lang
##### [lh3](https://github.com/lh3)/[bioseq-js](https://github.com/lh3/bioseq-js)

ID = 413, Stars = 31, Language = HTML, Update = 2019-08-08

For live demo, see http://lh3lh3.users.sourceforge.net/bioseq.shtml
##### [arq5x](https://github.com/arq5x)/[nanopore-scripts](https://github.com/arq5x/nanopore-scripts)

ID = 414, Stars = 31, Language = Python, Update = 2016-04-17

Various scripts and recipes for working with nanopore data
##### [bioinformaticsfmrp](https://github.com/bioinformaticsfmrp)/[TCGAbiolinksGUI](https://github.com/BioinformaticsFMRP/TCGAbiolinksGUI)

ID = 415, Stars = 31, Language = R, Update = 2019-12-15

TCGAbiolinksGUI code development for R/Bioconductor submission http://tcgabiolinks.fmrp.usp.br:3838/
##### [ga4gh](https://github.com/ga4gh)/[data-repository-service-schemas](https://github.com/ga4gh/data-repository-service-schemas)

ID = 416, Stars = 31, Language = Python, Update = 2020-06-22

A repository for the schemas used for the Data Repository Service.
##### [broadinstitute](https://github.com/broadinstitute)/[CellBender](https://github.com/broadinstitute/CellBender)

ID = 417, Stars = 31, Language = Python, Update = 2020-06-19

CellBender is a software package for eliminating technical artifacts from high-throughput single-cell RNA sequencing (scRNA-seq) data.
##### [freeseek](https://github.com/freeseek)/[getmyancestors](https://github.com/freeseek/getmyancestors)

ID = 418, Stars = 30, Language = None, Update = 2018-06-11

None
##### [crazyhottommy](https://github.com/crazyhottommy)/[The-world-of-faculty](https://github.com/crazyhottommy/The-world-of-faculty)

ID = 419, Stars = 30, Language = None, Update = 2020-05-26

resources for faculty
##### [lh3](https://github.com/lh3)/[yak](https://github.com/lh3/yak)

ID = 420, Stars = 30, Language = C, Update = 2020-03-21

Yet another k-mer analyzer
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbdagcon](https://github.com/PacificBiosciences/pbdagcon)

ID = 421, Stars = 30, Language = C++, Update = 2018-04-12

A sequence consensus algorithm implementation based on using directed acyclic graphs to encode multiple sequence alignment
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[FALCON_unzip](https://github.com/PacificBiosciences/FALCON_unzip)

ID = 422, Stars = 30, Language = Python, Update = 2017-10-02

Making diploid assembly becomes common practice for genomic study
##### [biowdl](https://github.com/biowdl)/[tasks](https://github.com/biowdl/tasks)

ID = 423, Stars = 30, Language = wdl, Update = 2020-06-22

A collection of reusable WDL tasks. Category:Other
##### [statgen](https://github.com/statgen)/[gotcloud](https://github.com/statgen/gotcloud)

ID = 424, Stars = 30, Language = C++, Update = 2017-03-24

Genomes on the Cloud, Mapping & Variant Calling Pipelines
##### [s-andrews](https://github.com/s-andrews)/[BamQC](https://github.com/s-andrews/BamQC)

ID = 425, Stars = 29, Language = Java, Update = 2018-05-16

Mapped QC analysis program
##### [ga4gh](https://github.com/ga4gh)/[mme-apis](https://github.com/ga4gh/mme-apis)

ID = 426, Stars = 29, Language = HTML, Update = 2020-01-10

Documentation for the MatchmakerExchange APIs
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_portal](https://github.com/broadinstitute/single_cell_portal)

ID = 427, Stars = 29, Language = Python, Update = 2020-06-22

Tutorials, workflows, and convenience scripts for Single Cell Portal
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbcore](https://github.com/PacificBiosciences/pbcore)

ID = 428, Stars = 29, Language = Python, Update = 2020-06-18

A Python library for reading and writing PacBio¬Æ data files
##### [umcugenetics](https://github.com/umcugenetics)/[RNASeq](https://github.com/UMCUGenetics/RNASeq)

ID = 429, Stars = 29, Language = Perl, Update = 2019-10-28

RNASeq pipeline
##### [parklab](https://github.com/parklab)/[MosaicForecast](https://github.com/parklab/MosaicForecast)

ID = 430, Stars = 29, Language = Python, Update = 2020-05-27

A mosaic detecting software based on phasing and random forest
##### [illumina](https://github.com/illumina)/[Polaris](https://github.com/Illumina/Polaris)

ID = 431, Stars = 29, Language = None, Update = 2019-11-18

Data and information about the Polaris study
##### [ekg](https://github.com/ekg)/[thesis](https://github.com/ekg/thesis)

ID = 432, Stars = 28, Language = TeX, Update = 2019-07-10

my PhD thesis
##### [tanghaibao](https://github.com/tanghaibao)/[allhic](https://github.com/tanghaibao/allhic)

ID = 433, Stars = 28, Language = Jupyter Notebook, Update = 2020-02-27

Genome scaffolding based on HiC data in heterozygous and high ploidy genomes
##### [ewels](https://github.com/ewels)/[Labrador](https://github.com/ewels/Labrador)

ID = 434, Stars = 28, Language = PHP, Update = 2018-11-16

A web based tool to manage and automate the processing of publicly available datasets.
##### [brentp](https://github.com/brentp)/[combined-pvalues](https://github.com/brentp/combined-pvalues)

ID = 435, Stars = 28, Language = Python, Update = 2019-12-18

combining p-values using modified stouffer-liptak for spatially correlated results (probes)
##### [brentp](https://github.com/brentp)/[quicksect](https://github.com/brentp/quicksect)

ID = 436, Stars = 28, Language = Python, Update = 2019-06-04

a cythonized, extended version of the interval search tree in bx
##### [brentp](https://github.com/brentp)/[align](https://github.com/brentp/align)

ID = 437, Stars = 28, Language = Python, Update = 2016-12-21

sequence alignment. global, local, glocal.
##### [broadinstitute](https://github.com/broadinstitute)/[SignatureAnalyzer-GPU](https://github.com/broadinstitute/SignatureAnalyzer-GPU)

ID = 438, Stars = 28, Language = Python, Update = 2019-12-09

GPU implementation of ARD NMF
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[FALCON-integrate](https://github.com/PacificBiosciences/FALCON-integrate)

ID = 439, Stars = 28, Language = Shell, Update = 2018-04-06

Mostly deprecated. See https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries
##### [ding-lab](https://github.com/ding-lab)/[PanCanAtlasGermline](https://github.com/ding-lab/PanCanAtlasGermline)

ID = 440, Stars = 28, Language = R, Update = 2018-10-27

None
##### [statgen](https://github.com/statgen)/[fastQValidator](https://github.com/statgen/fastQValidator)

ID = 441, Stars = 28, Language = C++, Update = 2018-10-30

Validate FastQ Files
##### [freeseek](https://github.com/freeseek)/[gwaspipeline](https://github.com/freeseek/gwaspipeline)

ID = 442, Stars = 27, Language = R, Update = 2018-01-24

None
##### [brentp](https://github.com/brentp)/[gsort](https://github.com/brentp/gsort)

ID = 443, Stars = 27, Language = Go, Update = 2020-05-26

sort genomic data
##### [brentp](https://github.com/brentp)/[bsub](https://github.com/brentp/bsub)

ID = 444, Stars = 27, Language = Python, Update = 2020-02-27

python wrapper to submit jobs to bsub (and later qsub)
##### [lh3](https://github.com/lh3)/[hapdip](https://github.com/lh3/hapdip)

ID = 445, Stars = 27, Language = JavaScript, Update = 2018-02-20

The CHM1-NA12878 benchmark for single-sample SNP/INDEL calling from WGS Illumina data
##### [richarddurbin](https://github.com/richarddurbin)/[modimizer](https://github.com/richarddurbin/modimizer)

ID = 446, Stars = 27, Language = C, Update = 2020-06-03

a toolset for fast DNA read set matching and assembly using a new type of reduced kmer
##### [broadinstitute](https://github.com/broadinstitute)/[long-read-pipelines](https://github.com/broadinstitute/long-read-pipelines)

ID = 447, Stars = 27, Language = Jupyter Notebook, Update = 2020-06-16

Long read production pipelines
##### [tseemann](https://github.com/tseemann)/[homebrew-bioinformatics-linux](https://github.com/tseemann/homebrew-bioinformatics-linux)

ID = 448, Stars = 26, Language = Ruby, Update = 2019-08-23

üç∫
üêß
Homebrew formulae for bioinformatics software only available for Linux
##### [brentp](https://github.com/brentp)/[poverlap](https://github.com/brentp/poverlap)

ID = 449, Stars = 26, Language = Python, Update = 2020-06-18

significance testing over interval overlaps
##### [brentp](https://github.com/brentp)/[geneimpacts](https://github.com/brentp/geneimpacts)

ID = 450, Stars = 26, Language = Python, Update = 2019-10-29

prioritize effects of variant annotations from VEP, SnpEff, et al.
##### [lh3](https://github.com/lh3)/[pre-pe](https://github.com/lh3/pre-pe)

ID = 451, Stars = 26, Language = C, Update = 2018-06-28

Preprocessing paired-end reads produced with experiment-specific protocols
##### [stephenturner](https://github.com/stephenturner)/[adapters](https://github.com/stephenturner/adapters)

ID = 452, Stars = 26, Language = R, Update = 2019-01-30

Adapters for trimming
##### [stephenturner](https://github.com/stephenturner)/[solarplot](https://github.com/stephenturner/solarplot)

ID = 453, Stars = 26, Language = R, Update = 2017-02-25

Mitochondrial Solar Plot
##### [quinlan-lab](https://github.com/quinlan-lab)/[STRling](https://github.com/quinlan-lab/STRling)

ID = 454, Stars = 26, Language = Nim, Update = 2020-06-08

Detect novel (and reference) STR expansions from short-read data
##### [biocore](https://github.com/biocore)/[tcga](https://github.com/biocore/tcga)

ID = 455, Stars = 26, Language = Jupyter Notebook, Update = 2020-01-11

Microbial analysis in TCGA data
##### [alimanfoo](https://github.com/alimanfoo)/[vcfnp](https://github.com/alimanfoo/vcfnp)

ID = 456, Stars = 25, Language = Python, Update = 2017-06-27

Load numpy arrays and HDF5 files from VCF (variant call format)
##### [jodyphelan](https://github.com/jodyphelan)/[TBProfiler](https://github.com/jodyphelan/TBProfiler)

ID = 457, Stars = 25, Language = Python, Update = 2020-06-15

Profiling tool for Mycobacterium tuberculosis to detect ressistance and strain type from WGS data
##### [mozack](https://github.com/mozack)/[vdjer](https://github.com/mozack/vdjer)

ID = 458, Stars = 25, Language = C, Update = 2017-04-11

V'DJer - B Cell Receptor Repertoire Reconstruction from short read mRNA-Seq data
##### [daler](https://github.com/daler)/[sphinxdoc-test](https://github.com/daler/sphinxdoc-test)

ID = 459, Stars = 25, Language = None, Update = 2015-06-12

experimenting with the best way to push sphinx-generated docs to gh-pages
##### [ncbi](https://github.com/ncbi)/[docker](https://github.com/ncbi/docker)

ID = 460, Stars = 25, Language = C++, Update = 2020-06-15

None
##### [ensembl](https://github.com/ensembl)/[postgap](https://github.com/Ensembl/postgap)

ID = 461, Stars = 25, Language = Jupyter Notebook, Update = 2020-05-18

Linking GWAS studies to genes through cis-regulatory datasets
##### [wglab](https://github.com/wglab)/[VirTect](https://github.com/WGLab/VirTect)

ID = 462, Stars = 25, Language = Python, Update = 2020-06-02

Detection of viruses from RNA-Seq on human samples
##### [opengene](https://github.com/opengene)/[ctdna-pipeline](https://github.com/OpenGene/ctdna-pipeline)

ID = 463, Stars = 25, Language = Shell, Update = 2017-09-23

A simplified pipeline for ctDNA sequencing data analysis
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-exome-analysis-pipeline](https://github.com/gatk-workflows/gatk4-exome-analysis-pipeline)

ID = 464, Stars = 25, Language = wdl, Update = 2020-06-08

This WDL pipeline implements data pre-processing and initial variant calling according to the GATK Best Practices for germline SNP and Indel discovery in human exome sequencing data.
##### [statgen](https://github.com/statgen)/[EPACTS](https://github.com/statgen/EPACTS)

ID = 465, Stars = 25, Language = C++, Update = 2019-07-30

None
##### [illumina](https://github.com/illumina)/[BeadArrayFiles](https://github.com/Illumina/BeadArrayFiles)

ID = 466, Stars = 25, Language = Python, Update = 2020-02-17

Python library to parse file formats related to Illumina bead arrays
##### [felixkrueger](https://github.com/felixkrueger)/[SNPsplit](https://github.com/FelixKrueger/SNPsplit)

ID = 467, Stars = 24, Language = Perl, Update = 2020-06-17

Allele-specific alignment sorting
##### [brentp](https://github.com/brentp)/[fastbit-python](https://github.com/brentp/fastbit-python)

ID = 468, Stars = 24, Language = Python, Update = 2018-08-10

pythonic access to fastbit
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad_methods](https://github.com/broadinstitute/gnomad_methods)

ID = 469, Stars = 24, Language = Python, Update = 2020-06-12

Hail helper functions for gnomAD/MacArthur lab
##### [broadinstitute](https://github.com/broadinstitute)/[cromshell](https://github.com/broadinstitute/cromshell)

ID = 470, Stars = 24, Language = Shell, Update = 2020-06-09

CLI for interacting with Cromwell servers
##### [broadinstitute](https://github.com/broadinstitute)/[gdctools](https://github.com/broadinstitute/gdctools)

ID = 471, Stars = 24, Language = Python, Update = 2018-10-11

Python and UNIX CLI utilities to simplify interaction with the NIH/NCI Genomics Data Commons
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_analysis](https://github.com/broadinstitute/single_cell_analysis)

ID = 472, Stars = 24, Language = HTML, Update = 2017-02-05

Documents used for workshops on single cell analysis
##### [wglab](https://github.com/wglab)/[EHR-Phenolyzer](https://github.com/WGLab/EHR-Phenolyzer)

ID = 473, Stars = 24, Language = Python, Update = 2019-10-30

A pipeline that takes clinical notes from EHRs and generate phenotype terms, then generate ranked gene list
##### [biocore](https://github.com/biocore)/[songbird](https://github.com/biocore/songbird)

ID = 474, Stars = 24, Language = Python, Update = 2020-05-19

Vanilla regression methods for microbiome differential abundance analysis
##### [seandavi](https://github.com/seandavi)/[ci4cc-informatics-resources](https://github.com/seandavi/ci4cc-informatics-resources)

ID = 475, Stars = 23, Language = None, Update = 2018-09-24

Community-maintained list of resources that the CI4CC organization and the larger cancer informatics community have found useful or are developing.
##### [benlangmead](https://github.com/benlangmead)/[qtip](https://github.com/BenLangmead/qtip)

ID = 476, Stars = 23, Language = C++, Update = 2019-10-31

Qtip: a tandem simulation approach for accurately predicting read alignment mapping qualities
##### [tanghaibao](https://github.com/tanghaibao)/[treecut](https://github.com/tanghaibao/treecut)

ID = 477, Stars = 23, Language = Python, Update = 2017-10-15

Find nodes in hierarchical clustering that are statistically significant
##### [lh3](https://github.com/lh3)/[asub](https://github.com/lh3/asub)

ID = 478, Stars = 23, Language = Perl, Update = 2019-11-29

A unified array job submitter for LSF, SGE/UGE and Slurm
##### [lh3](https://github.com/lh3)/[fermi2](https://github.com/lh3/fermi2)

ID = 479, Stars = 23, Language = C, Update = 2018-02-07

None
##### [s-andrews](https://github.com/s-andrews)/[SeqMonk](https://github.com/s-andrews/SeqMonk)

ID = 480, Stars = 23, Language = Java, Update = 2020-06-15

SeqMonk NGS visualisation and analysis tool
##### [yongxinliu](https://github.com/yongxinliu)/[Metagenome](https://github.com/YongxinLiu/Metagenome)

ID = 481, Stars = 23, Language = Shell, Update = 2019-06-28

Metagenome analysis pipeline
##### [quinlan-lab](https://github.com/quinlan-lab)/[ccrhtml](https://github.com/quinlan-lab/ccrhtml)

ID = 482, Stars = 23, Language = HTML, Update = 2019-10-22

A small repo for storing the code for making the files and html for CCRs.
##### [broadinstitute](https://github.com/broadinstitute)/[genomics-in-the-cloud](https://github.com/broadinstitute/genomics-in-the-cloud)

ID = 483, Stars = 23, Language = Jupyter Notebook, Update = 2020-05-29

Source code and related materials for the O'Reilly book
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[PacBioFileFormats](https://github.com/PacificBiosciences/PacBioFileFormats)

ID = 484, Stars = 23, Language = Python, Update = 2020-04-07

Specifications for PacBio¬Æ native file formats
##### [wglab](https://github.com/wglab)/[NanoMod](https://github.com/WGLab/NanoMod)

ID = 485, Stars = 23, Language = Python, Update = 2020-06-22

NanoMod: a computational tool to detect DNA modifications using Nanopore long-read sequencing data
##### [opengene](https://github.com/opengene)/[FusionDirect.jl](https://github.com/OpenGene/FusionDirect.jl)

ID = 486, Stars = 23, Language = Julia, Update = 2017-08-13

(No maintenance) Detect gene fusion directly from raw fastq files
##### [illumina](https://github.com/illumina)/[GTCtoVCF](https://github.com/Illumina/GTCtoVCF)

ID = 487, Stars = 23, Language = Python, Update = 2020-06-11

Script to convert GTC/BPM files to VCF
##### [illumina](https://github.com/illumina)/[gvcfgenotyper](https://github.com/Illumina/gvcfgenotyper)

ID = 488, Stars = 23, Language = C++, Update = 2019-02-26

A utility for merging and genotyping Illumina-style GVCFs.
##### [hall-lab](https://github.com/hall-lab)/[sv-pipeline](https://github.com/hall-lab/sv-pipeline)

ID = 489, Stars = 23, Language = wdl, Update = 2020-06-02

Pipeline for structural variation detection in cohorts
##### [tseemann](https://github.com/tseemann)/[seeka](https://github.com/tseemann/seeka)

ID = 490, Stars = 22, Language = Perl, Update = 2019-11-07

Get microbial sequence data easier and faster
##### [brentp](https://github.com/brentp)/[nim-lapper](https://github.com/brentp/nim-lapper)

ID = 491, Stars = 22, Language = Nim, Update = 2020-06-09

fast easy interval overlapping for nim-lang
##### [brentp](https://github.com/brentp)/[cigar](https://github.com/brentp/cigar)

ID = 492, Stars = 22, Language = Python, Update = 2019-09-26

simple library for dealing with SAM cigar strings
##### [vibansal](https://github.com/vibansal)/[hapcut](https://github.com/vibansal/hapcut)

ID = 493, Stars = 22, Language = C, Update = 2018-11-29

program for haplotype phasing from sequence reads and related tools
##### [arq5x](https://github.com/arq5x)/[bits](https://github.com/arq5x/bits)

ID = 494, Stars = 22, Language = C++, Update = 2017-04-26

BITS: Binary Interval Search
##### [marianattestad](https://github.com/marianattestad)/[R_for_biologists](https://github.com/MariaNattestad/R_for_biologists)

ID = 495, Stars = 22, Language = R, Update = 2018-11-25

Scripts and data for Plotting in R for Biologists, a video course on YouTube
##### [bioinformaticsfmrp](https://github.com/bioinformaticsfmrp)/[TCGAWorkflow](https://github.com/BioinformaticsFMRP/TCGAWorkflow)

ID = 496, Stars = 22, Language = R, Update = 2020-04-20

TCGA Workflow: Analyze cancer genomics and epigenomics data using Bioconductor packages
##### [vgteam](https://github.com/vgteam)/[gssw](https://github.com/vgteam/gssw)

ID = 497, Stars = 22, Language = C, Update = 2020-04-29

efficient alignment of strings to partially ordered string graphs
##### [broadinstitute](https://github.com/broadinstitute)/[PhylogicNDT](https://github.com/broadinstitute/PhylogicNDT)

ID = 498, Stars = 22, Language = HTML, Update = 2020-06-12

None
##### [opengene](https://github.com/opengene)/[defq](https://github.com/OpenGene/defq)

ID = 499, Stars = 22, Language = C, Update = 2018-07-28

Ultra-fast Multi-threaded FASTQ Demultiplexing
##### [statgen](https://github.com/statgen)/[emeraLD](https://github.com/statgen/emeraLD)

ID = 500, Stars = 22, Language = C++, Update = 2019-08-27

tools to efficiently retrieve and calculate LD
##### [statgen](https://github.com/statgen)/[verifyBamID](https://github.com/statgen/verifyBamID)

ID = 501, Stars = 22, Language = C++, Update = 2017-03-15

None
##### [crazyhottommy](https://github.com/crazyhottommy)/[scRNA-seq-workshop-Fall-2019](https://github.com/crazyhottommy/scRNA-seq-workshop-Fall-2019)

ID = 502, Stars = 21, Language = R, Update = 2019-08-08

Harvard FAS informatics scRNAseq workshop website
##### [lh3](https://github.com/lh3)/[dipcall](https://github.com/lh3/dipcall)

ID = 503, Stars = 21, Language = JavaScript, Update = 2020-05-04

Reference-based variant calling pipeline for a pair of phased haplotype assemblies
##### [pcingola](https://github.com/pcingola)/[SnpSift](https://github.com/pcingola/SnpSift)

ID = 504, Stars = 21, Language = Java, Update = 2020-05-17

None
##### [vibansal](https://github.com/vibansal)/[ancestry](https://github.com/vibansal/ancestry)

ID = 505, Stars = 21, Language = C, Update = 2018-04-26

program to estimate admixture coefficients from individual genotype or sequence data
##### [cschin](https://github.com/cschin)/[Write_A_Genome_Assembler_With_IPython](https://github.com/cschin/Write_A_Genome_Assembler_With_IPython)

ID = 506, Stars = 21, Language = Python, Update = 2018-05-10

The repository keeps the files for an IPython notebook on about how to make a simple genome assembler using python
##### [gregoryfaust](https://github.com/gregoryfaust)/[yaha](https://github.com/GregoryFaust/yaha)

ID = 507, Stars = 21, Language = C, Update = 2017-12-06

yaha: a flexible, sensitive and accurate DNA alignment tool designed to find optimal split-read mappings on single-end queries from 100bp to 32kb.
##### [refresh-bio](https://github.com/refresh-bio)/[FAMSA](https://github.com/refresh-bio/FAMSA)

ID = 508, Stars = 21, Language = C++, Update = 2020-06-21

Algorithm for large-scale multiple sequence alignments (400k proteins in 2 hours and 8BG of RAM)
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-orchestration](https://github.com/broadinstitute/firecloud-orchestration)

ID = 509, Stars = 21, Language = Scala, Update = 2020-06-12

None
##### [broadinstitute](https://github.com/broadinstitute)/[2019_scWorkshop](https://github.com/broadinstitute/2019_scWorkshop)

ID = 510, Stars = 21, Language = TeX, Update = 2019-11-09

Repo for Physalia course Analysis of Single Cell RNA-Seq data
##### [bioconda](https://github.com/bioconda)/[bioconda.github.io](https://github.com/bioconda/bioconda.github.io)

ID = 511, Stars = 21, Language = HTML, Update = 2020-06-23

Bioconda is a distribution of bioinformatics software realized as a channel for the versatile package manager Conda.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbh5tools](https://github.com/PacificBiosciences/pbh5tools)

ID = 512, Stars = 21, Language = Python, Update = 2018-12-12

A swiss-army knife for interrogating PacBio¬Æ HDF5 files (cmp.h5, bas.h5)
##### [freeseek](https://github.com/freeseek)/[getmydnamatches](https://github.com/freeseek/getmydnamatches)

ID = 513, Stars = 20, Language = Python, Update = 2019-01-21

None
##### [seandavi](https://github.com/seandavi)/[Bioc2017BigDataWorkshopSession](https://github.com/seandavi/Bioc2017BigDataWorkshopSession)

ID = 514, Stars = 20, Language = None, Update = 2017-07-26

Tutorial for working with cloud infrastructure and AWS from R
##### [crazyhottommy](https://github.com/crazyhottommy)/[scclusteval](https://github.com/crazyhottommy/scclusteval)

ID = 515, Stars = 20, Language = R, Update = 2020-05-24

Single Cell Cluster Evaluation
##### [sigven](https://github.com/sigven)/[cpsr](https://github.com/sigven/cpsr)

ID = 516, Stars = 20, Language = R, Update = 2019-12-01

Cancer Predisposition Sequencing Reporter (CPSR)
##### [fhcrc](https://github.com/fhcrc)/[nestly](https://github.com/fhcrc/nestly)

ID = 517, Stars = 20, Language = Python, Update = 2017-04-23

Make and use nested directory trees corresponding to combinatorial choices of parameters
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbsmrtpipe](https://github.com/PacificBiosciences/pbsmrtpipe)

ID = 518, Stars = 20, Language = Python, Update = 2018-10-24

PacBio Workflow Engine
##### [ncbi](https://github.com/ncbi)/[consul-announcer](https://github.com/ncbi/consul-announcer)

ID = 519, Stars = 20, Language = Python, Update = 2016-12-12

Service announcer for Consul (https://www.consul.io/).
##### [ensembl](https://github.com/ensembl)/[ensembl-variation](https://github.com/Ensembl/ensembl-variation)

ID = 520, Stars = 20, Language = Perl, Update = 2020-06-19

The Ensembl Variation Perl API and SQL schema
##### [wglab](https://github.com/wglab)/[SeqMule](https://github.com/WGLab/SeqMule)

ID = 521, Stars = 20, Language = C++, Update = 2020-01-05

Automated human exome/genome variants detection from FASTQ files
##### [opengene](https://github.com/opengene)/[VisualMSI](https://github.com/OpenGene/VisualMSI)

ID = 522, Stars = 20, Language = C++, Update = 2019-06-04

Detect and visualize microsatellite instability(MSI) from NGS data
##### [biocore](https://github.com/biocore)/[unifrac](https://github.com/biocore/unifrac)

ID = 523, Stars = 20, Language = C++, Update = 2020-06-18

None
##### [biocore](https://github.com/biocore)/[redbiom](https://github.com/biocore/redbiom)

ID = 524, Stars = 20, Language = Python, Update = 2020-05-29

Sample search by metadata and features
##### [ekg](https://github.com/ekg)/[hhga](https://github.com/ekg/hhga)

ID = 525, Stars = 19, Language = C++, Update = 2019-06-13

haplotypes genotypes and alleles example decision synthesizer
##### [ekg](https://github.com/ekg)/[1000G-integration](https://github.com/ekg/1000G-integration)

ID = 526, Stars = 19, Language = Shell, Update = 2018-01-16

variant integration methods for the 1000 Genomes Project
##### [seandavi](https://github.com/seandavi)/[SRAdbV2](https://github.com/seandavi/SRAdbV2)

ID = 527, Stars = 19, Language = R, Update = 2018-11-19

R Interface to the NCBI SRA metadata
##### [tseemann](https://github.com/tseemann)/[ekidna](https://github.com/tseemann/ekidna)

ID = 528, Stars = 19, Language = Perl, Update = 2019-10-23

Assembly based core genome SNP alignments for bacteria
##### [crazyhottommy](https://github.com/crazyhottommy)/[scATACutils](https://github.com/crazyhottommy/scATACutils)

ID = 529, Stars = 19, Language = R, Update = 2020-06-14

R/Bioconductor package for working with 10x scATACseq data
##### [refresh-bio](https://github.com/refresh-bio)/[Whisper](https://github.com/refresh-bio/Whisper)

ID = 530, Stars = 19, Language = C++, Update = 2020-04-20

None
##### [ga4gh](https://github.com/ga4gh)/[tool-registry-service-schemas](https://github.com/ga4gh/tool-registry-service-schemas)

ID = 531, Stars = 19, Language = Shell, Update = 2020-04-06

APIs for discovering genomics tools, their metadata and their containers
##### [broadinstitute](https://github.com/broadinstitute)/[BipolarCell2016](https://github.com/broadinstitute/BipolarCell2016)

ID = 532, Stars = 19, Language = R, Update = 2020-02-07

None
##### [broadinstitute](https://github.com/broadinstitute)/[BARD](https://github.com/broadinstitute/BARD)

ID = 533, Stars = 19, Language = Groovy, Update = 2019-01-29

BioAssay Research Database
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[sv-benchmark](https://github.com/PacificBiosciences/sv-benchmark)

ID = 534, Stars = 19, Language = None, Update = 2019-05-01

Public Benchmark of Long-Read Structural Variant Caller on PacBio CCS HG002 Data
##### [ensembl](https://github.com/ensembl)/[Bio-DB-HTS](https://github.com/Ensembl/Bio-DB-HTS)

ID = 535, Stars = 19, Language = Perl, Update = 2020-01-21

Git repo for Bio::DB::HTS module on CPAN, providing Perl links into HTSlib
##### [wglab](https://github.com/wglab)/[Phen2Gene](https://github.com/WGLab/Phen2Gene)

ID = 536, Stars = 19, Language = Python, Update = 2020-06-12

Phenotype driven gene prioritization for HPO
##### [opengene](https://github.com/opengene)/[SeqMaker.jl](https://github.com/OpenGene/SeqMaker.jl)

ID = 537, Stars = 19, Language = Julia, Update = 2017-08-10

(No maintenance) Next Generation Sequencing Simulation with SNP, Variation and Sequencing Error Integrated
##### [biocore](https://github.com/biocore)/[calour](https://github.com/biocore/calour)

ID = 538, Stars = 19, Language = Python, Update = 2020-06-17

exploratory and interactive microbiome analyses based on heatmaps
##### [parklab](https://github.com/parklab)/[ShatterSeek](https://github.com/parklab/ShatterSeek)

ID = 539, Stars = 19, Language = TeX, Update = 2019-09-12

None
##### [illumina](https://github.com/illumina)/[ExpansionHunterDenovo](https://github.com/Illumina/ExpansionHunterDenovo)

ID = 540, Stars = 19, Language = C++, Update = 2020-03-04

A suite of tools for detecting expansions of short tandem repeats
##### [crazyhottommy](https://github.com/crazyhottommy)/[machine-learning-resource](https://github.com/crazyhottommy/machine-learning-resource)

ID = 541, Stars = 18, Language = None, Update = 2019-12-11

None
##### [lh3](https://github.com/lh3)/[sdust](https://github.com/lh3/sdust)

ID = 542, Stars = 18, Language = C, Update = 2019-10-03

Symmetric DUST for finding low-complexity regions in DNA sequences
##### [yongxinliu](https://github.com/yongxinliu)/[Zhang2018SCLS](https://github.com/YongxinLiu/Zhang2018SCLS)

ID = 543, Stars = 18, Language = R, Update = 2019-03-29

Rice (Oryza Sative) root microbiome time-course analysis. (Zhang2018SCLS)
##### [griffithlab](https://github.com/griffithlab)/[rnaseq_tutorial_wiki](https://github.com/griffithlab/rnaseq_tutorial_wiki)

ID = 544, Stars = 18, Language = Perl, Update = 2020-05-07

The wiki repo, with pull request enabled, for the rnaseq_tutorial
##### [quinlan-lab](https://github.com/quinlan-lab)/[pathoscore](https://github.com/quinlan-lab/pathoscore)

ID = 545, Stars = 18, Language = Python, Update = 2019-06-29

pathoscore evaluates variant pathogenicity tools and scores.
##### [nf-core](https://github.com/nf-core)/[mag](https://github.com/nf-core/mag)

ID = 546, Stars = 18, Language = Nextflow, Update = 2020-06-22

Assembly and binning of metagenomes
##### [nf-core](https://github.com/nf-core)/[nf-co.re](https://github.com/nf-core/nf-co.re)

ID = 547, Stars = 18, Language = PHP, Update = 2020-06-22

Code and files for the main nf-core website >>>
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-ui](https://github.com/broadinstitute/firecloud-ui)

ID = 548, Stars = 18, Language = Clojure, Update = 2020-06-19

FireCloud user interface for web browsers.
##### [broadinstitute](https://github.com/broadinstitute)/[wdl-ide](https://github.com/broadinstitute/wdl-ide)

ID = 549, Stars = 18, Language = Python, Update = 2019-11-14

Rich IDE support for Workflow Description Language
##### [fhcrc](https://github.com/fhcrc)/[taxtastic](https://github.com/fhcrc/taxtastic)

ID = 550, Stars = 18, Language = Python, Update = 2019-06-24

Create and maintain phylogenetic "reference packages" of biological sequences.
##### [ncbi](https://github.com/ncbi)/[magicblast](https://github.com/ncbi/magicblast)

ID = 551, Stars = 18, Language = Python, Update = 2020-05-18

None
##### [ncbi](https://github.com/ncbi)/[JUDI](https://github.com/ncbi/JUDI)

ID = 552, Stars = 18, Language = Python, Update = 2019-12-13

This repository contains the source code of JUDI, a workflow management system for developing complex bioinformatics software with many parameter settings. Bioinformatics pipeline: Just Do It!
##### [ensembl](https://github.com/ensembl)/[ensembl-webcode](https://github.com/Ensembl/ensembl-webcode)

ID = 553, Stars = 18, Language = Perl, Update = 2020-06-22

The code to run an Ensembl website
##### [wglab](https://github.com/wglab)/[RepeatHMM](https://github.com/WGLab/RepeatHMM)

ID = 554, Stars = 18, Language = Python, Update = 2020-06-21

a hidden Markov model to infer simple repeats from genome sequences
##### [opengene](https://github.com/opengene)/[MrBam](https://github.com/OpenGene/MrBam)

ID = 555, Stars = 18, Language = Python, Update = 2018-11-26

Query Mutated Reads from a Bam
##### [openvax](https://github.com/openvax)/[topiary](https://github.com/openvax/topiary)

ID = 556, Stars = 18, Language = Python, Update = 2018-02-26

Predict mutated T-cell epitopes from sequencing data
##### [papenfusslab](https://github.com/papenfusslab)/[PathOS](https://github.com/PapenfussLab/PathOS)

ID = 557, Stars = 18, Language = Java, Update = 2020-06-07

PathOS is a clinical application for filtering, analysing and reporting on NGS variants
##### [illumina](https://github.com/illumina)/[Isaac3](https://github.com/Illumina/Isaac3)

ID = 558, Stars = 18, Language = C++, Update = 2018-02-13

Aligner for sequencing data
##### [freeseek](https://github.com/freeseek)/[mocha](https://github.com/freeseek/mocha)

ID = 559, Stars = 17, Language = C, Update = 2020-06-03

MOsaic CHromosomal Alterations (MoChA) caller
##### [gt1](https://github.com/gt1)/[daccord](https://github.com/gt1/daccord)

ID = 560, Stars = 17, Language = C++, Update = 2018-09-12

d'accord is a non hybrid long read consensus program based on local de Bruijn graph assembly
##### [brentp](https://github.com/brentp)/[toolshed](https://github.com/brentp/toolshed)

ID = 561, Stars = 17, Language = Python, Update = 2019-12-18

python stuff I use
##### [sigven](https://github.com/sigven)/[gvanno](https://github.com/sigven/gvanno)

ID = 562, Stars = 17, Language = Python, Update = 2020-05-14

Generic germline variant annotation pipeline
##### [sigven](https://github.com/sigven)/[vcf2tsv](https://github.com/sigven/vcf2tsv)

ID = 563, Stars = 17, Language = Python, Update = 2019-10-14

Genomic VCF to tab-separated values
##### [nf-core](https://github.com/nf-core)/[modules](https://github.com/nf-core/modules)

ID = 564, Stars = 17, Language = Nextflow, Update = 2020-06-19

Repository to host tool-specific module files for the Nextflow DSL2 community!
##### [humanlongevity](https://github.com/humanlongevity)/[tredparse](https://github.com/humanlongevity/tredparse)

ID = 565, Stars = 17, Language = Python, Update = 2020-04-29

TREDPARSE: HLI Short Tandem Repeat (STR) caller
##### [broadinstitute](https://github.com/broadinstitute)/[cromwell-tools](https://github.com/broadinstitute/cromwell-tools)

ID = 566, Stars = 17, Language = Python, Update = 2020-02-21

A collection of Python clients and accessory scripts for interacting with the Cromwell
##### [broadinstitute](https://github.com/broadinstitute)/[deepometry](https://github.com/broadinstitute/deepometry)

ID = 567, Stars = 17, Language = Python, Update = 2018-06-30

Image classification for imaging flow cytometry.
##### [wglab](https://github.com/wglab)/[GenGen](https://github.com/WGLab/GenGen)

ID = 568, Stars = 17, Language = C, Update = 2019-01-16

A set of software tools to facilitate GWAS analysis
##### [openvax](https://github.com/openvax)/[pepnet](https://github.com/openvax/pepnet)

ID = 569, Stars = 17, Language = Python, Update = 2018-02-26

Neural networks for amino acid sequences
##### [biocore](https://github.com/biocore)/[DEICODE](https://github.com/biocore/DEICODE)

ID = 570, Stars = 17, Language = JavaScript, Update = 2020-06-03

Robust Aitchison PCA from sparse count data
##### [biocore](https://github.com/biocore)/[micronota](https://github.com/biocore/micronota)

ID = 571, Stars = 17, Language = Python, Update = 2018-02-23

annotation pipeline for microbial genomes and metagenomes
##### [ding-lab](https://github.com/ding-lab)/[BreakPointSurveyor](https://github.com/ding-lab/BreakPointSurveyor)

ID = 572, Stars = 17, Language = Shell, Update = 2020-01-28

A comprehensive pipeline to analyze and visualize structural variants
##### [statgen](https://github.com/statgen)/[Minimac4](https://github.com/statgen/Minimac4)

ID = 573, Stars = 17, Language = C++, Update = 2020-06-05

None
##### [papenfusslab](https://github.com/papenfusslab)/[sv_benchmark](https://github.com/PapenfussLab/sv_benchmark)

ID = 574, Stars = 17, Language = R, Update = 2019-07-01

Comprehensive benchmark of structural variant callers
##### [illumina](https://github.com/illumina)/[GraphAlignmentViewer](https://github.com/Illumina/GraphAlignmentViewer)

ID = 575, Stars = 17, Language = Python, Update = 2020-06-05

None
##### [ekg](https://github.com/ekg)/[multichoose](https://github.com/ekg/multichoose)

ID = 576, Stars = 16, Language = C++, Update = 2019-12-09

generate multiset combinations (n multichoose k)
##### [ekg](https://github.com/ekg)/[drawbwt](https://github.com/ekg/drawbwt)

ID = 577, Stars = 16, Language = C++, Update = 2019-04-29

vector illustrations of BWT based searching on small strings
##### [seandavi](https://github.com/seandavi)/[BiocPkgTools](https://github.com/seandavi/BiocPkgTools)

ID = 578, Stars = 16, Language = R, Update = 2020-03-25

Computable build reports, package metadata, and download stats from the Bioconductor project
##### [brentp](https://github.com/brentp)/[genoiser](https://github.com/brentp/genoiser)

ID = 579, Stars = 16, Language = Nim, Update = 2020-04-15

use the noise
##### [lh3](https://github.com/lh3)/[lianti](https://github.com/lh3/lianti)

ID = 580, Stars = 16, Language = C, Update = 2019-02-20

Tools to process LIANTI sequence data
##### [nf-core](https://github.com/nf-core)/[test-datasets](https://github.com/nf-core/test-datasets)

ID = 581, Stars = 16, Language = None, Update = 2020-06-21

Test data to be used for automated testing with the nf-core pipelines
##### [broadinstitute](https://github.com/broadinstitute)/[sam](https://github.com/broadinstitute/sam)

ID = 582, Stars = 16, Language = Scala, Update = 2020-06-18

workbench identity and access management
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad_qc](https://github.com/broadinstitute/gnomad_qc)

ID = 583, Stars = 16, Language = Python, Update = 2020-05-19

None
##### [broadinstitute](https://github.com/broadinstitute)/[gatk-docs](https://github.com/broadinstitute/gatk-docs)

ID = 584, Stars = 16, Language = HTML, Update = 2020-01-06

Documentation archive for GATK tools and workflows
##### [broadinstitute](https://github.com/broadinstitute)/[wordpress-crowd-plugin](https://github.com/broadinstitute/wordpress-crowd-plugin)

ID = 585, Stars = 16, Language = PHP, Update = 2018-01-11

Crowd Authentication Plugin for Wordpress
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbbam](https://github.com/PacificBiosciences/pbbam)

ID = 586, Stars = 16, Language = C++, Update = 2020-06-19

PacBio BAM C++ library
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[disambiguate](https://github.com/AstraZeneca-NGS/disambiguate)

ID = 587, Stars = 16, Language = C++, Update = 2018-05-08

Disambiguation algorithm for reads aligned to human and mouse genomes using Tophat or BWA mem
##### [openvax](https://github.com/openvax)/[varlens](https://github.com/openvax/varlens)

ID = 588, Stars = 16, Language = Python, Update = 2018-02-26

commandline manipulation of genomic variants and NGS reads
##### [statgen](https://github.com/statgen)/[popscle](https://github.com/statgen/popscle)

ID = 589, Stars = 16, Language = C++, Update = 2020-06-16

A suite of population scale analysis tools for single-cell genomics data including implementation of Demuxlet / Freemuxlet methods and auxilary tools
##### [illumina](https://github.com/illumina)/[SMNCopyNumberCaller](https://github.com/Illumina/SMNCopyNumberCaller)

ID = 590, Stars = 16, Language = Python, Update = 2020-06-11

A copy number caller for SMN1 and SMN2 to enable SMA diagnosis and carrier screening with WGS
##### [ming-lian](https://github.com/ming-lian)/[Memo](https://github.com/Ming-Lian/Memo)

ID = 591, Stars = 15, Language = None, Update = 2019-12-01

Â≠¶‰π†Á¨îËÆ∞ÔºöÊàëÁöÑÁ¨¨‰∏Ä‰∏™github‰ªìÂ∫ì
##### [lh3](https://github.com/lh3)/[calN50](https://github.com/lh3/calN50)

ID = 592, Stars = 15, Language = JavaScript, Update = 2020-04-09

Compute N50/NG50 and auN/auNG
##### [sigven](https://github.com/sigven)/[cacao](https://github.com/sigven/cacao)

ID = 593, Stars = 15, Language = R, Update = 2020-05-10

Callable Cancer Loci - assessment of sequencing coverage for actionable and pathogenic loci in cancer
##### [a-slide](https://github.com/a-slide)/[pycoMeth](https://github.com/a-slide/pycoMeth)

ID = 594, Stars = 15, Language = Python, Update = 2020-05-22

DNA methylation analysis downstream to Nanopolish for Oxford Nanopore DNA sequencing datasets
##### [yongxinliu](https://github.com/yongxinliu)/[Amplicon2017](https://github.com/YongxinLiu/Amplicon2017)

ID = 595, Stars = 15, Language = HTML, Update = 2019-03-10

Data and scripts for amplicon pipeline
##### [stephenturner](https://github.com/stephenturner)/[hcop](https://github.com/stephenturner/hcop)

ID = 596, Stars = 15, Language = R, Update = 2018-03-22

HGNC Comparison of Orthology Predictions (HCOP)
##### [ga4gh](https://github.com/ga4gh)/[vr-python](https://github.com/ga4gh/vr-python)

ID = 597, Stars = 15, Language = Jupyter Notebook, Update = 2020-06-22

GA4GH Variation Representation Python Implementation
##### [vgteam](https://github.com/vgteam)/[xg-old](https://github.com/vgteam/xg-old)

ID = 598, Stars = 15, Language = C++, Update = 2018-11-18

succinct labeled graphs with collections and paths
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[parkour](https://github.com/maxplanck-ie/parkour)

ID = 599, Stars = 15, Language = JavaScript, Update = 2020-03-13

An open-source Laboratory Information Management System (LIMS) for Next-Generation Sequencing (NGS)
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[HiCAssembler](https://github.com/maxplanck-ie/HiCAssembler)

ID = 600, Stars = 15, Language = Python, Update = 2019-11-07

Software to assemble contigs/scaffolds into chromosomes using Hi-C data
##### [nf-core](https://github.com/nf-core)/[hic](https://github.com/nf-core/hic)

ID = 601, Stars = 15, Language = Python, Update = 2020-06-19

Analysis of Chromosome Conformation Capture data (Hi-C)
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[R-kinetics](https://github.com/PacificBiosciences/R-kinetics)

ID = 602, Stars = 15, Language = R, Update = 2017-10-02

A project housing some code and documentation for performing kinetics analyses using R.
##### [ncbi](https://github.com/ncbi)/[blast_plus_docs](https://github.com/ncbi/blast_plus_docs)

ID = 603, Stars = 15, Language = None, Update = 2020-06-11

None
##### [openvax](https://github.com/openvax)/[isovar](https://github.com/openvax/isovar)

ID = 604, Stars = 15, Language = Python, Update = 2020-02-11

Assembly of RNA reads to determine the effect of a cancer mutation on protein sequence
##### [parklab](https://github.com/parklab)/[MSIprofiler](https://github.com/parklab/MSIprofiler)

ID = 605, Stars = 15, Language = Python, Update = 2019-10-03

Repo that aids in the detection of microsatellite instabilities (MSI) from sequencing data
##### [ekg](https://github.com/ekg)/[viral-assembly](https://github.com/ekg/viral-assembly)

ID = 606, Stars = 14, Language = Shell, Update = 2020-05-18

exploring viral genome assembly with variation graph tools
##### [seandavi](https://github.com/seandavi)/[SRAdb](https://github.com/seandavi/SRAdb)

ID = 607, Stars = 14, Language = R, Update = 2018-02-20

Git mirror of Bioconductor SRAdb package
##### [tseemann](https://github.com/tseemann)/[samclip](https://github.com/tseemann/samclip)

ID = 608, Stars = 14, Language = Perl, Update = 2020-03-07

Filter SAM file for soft and hard clipped alignments
##### [tseemann](https://github.com/tseemann)/[phastaf](https://github.com/tseemann/phastaf)

ID = 609, Stars = 14, Language = Perl, Update = 2019-09-05

Identify phage regions in bacterial genomes for masking purposes
##### [tseemann](https://github.com/tseemann)/[sixess](https://github.com/tseemann/sixess)

ID = 610, Stars = 14, Language = Shell, Update = 2018-04-24

üî¨
üêõ
Rapid 16s rRNA identification from isolate FASTQ files
##### [vals](https://github.com/vals)/[Blog](https://github.com/vals/Blog)

ID = 611, Stars = 14, Language = Jupyter Notebook, Update = 2020-05-10

None
##### [brentp](https://github.com/brentp)/[lua-stringy](https://github.com/brentp/lua-stringy)

ID = 612, Stars = 14, Language = C, Update = 2020-03-21

fast lua string operations
##### [ocxtal](https://github.com/ocxtal)/[dozeu](https://github.com/ocxtal/dozeu)

ID = 613, Stars = 14, Language = C, Update = 2020-02-09

SIMD-parallel BLAST X-drop DP on sequence graphs
##### [richarddurbin](https://github.com/richarddurbin)/[hash10x](https://github.com/richarddurbin/hash10x)

ID = 614, Stars = 14, Language = C, Update = 2019-11-27

a toolset for efficient analysis of 10X Genomics linked read data sets, in particular for de novo assembly
##### [yongxinliu](https://github.com/yongxinliu)/[Zhang2019NBT](https://github.com/YongxinLiu/Zhang2019NBT)

ID = 615, Stars = 14, Language = HTML, Update = 2019-04-12

Scripts for stat and plot figures in rice microbiome paper
##### [qiime2](https://github.com/qiime2)/[q2cli](https://github.com/qiime2/q2cli)

ID = 616, Stars = 14, Language = Python, Update = 2020-06-23

Command line interface for QIIME 2
##### [quinlan-lab](https://github.com/quinlan-lab)/[ccr](https://github.com/quinlan-lab/ccr)

ID = 617, Stars = 14, Language = Python, Update = 2018-12-19

Building the constrained coding regions (CCR) model
##### [nf-core](https://github.com/nf-core)/[configs](https://github.com/nf-core/configs)

ID = 618, Stars = 14, Language = Nextflow, Update = 2020-06-22

Config files used to define parameters specific to compute environments at different Institutions
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbalign](https://github.com/PacificBiosciences/pbalign)

ID = 619, Stars = 14, Language = Python, Update = 2018-12-11

pbalign maps PacBio reads to reference sequences and saves alignments to a BAM file
##### [ncbi](https://github.com/ncbi)/[pipelines](https://github.com/ncbi/pipelines)

ID = 620, Stars = 14, Language = Common Workflow Language, Update = 2018-11-28

None
##### [illumina](https://github.com/illumina)/[BaseSpace_Clarity_LIMS](https://github.com/Illumina/BaseSpace_Clarity_LIMS)

ID = 621, Stars = 14, Language = Python, Update = 2019-07-23

API libraries, application examples, and custom tools for BaseSpace Clarity LIMS
##### [ekg](https://github.com/ekg)/[tabixpp](https://github.com/ekg/tabixpp)

ID = 622, Stars = 13, Language = C++, Update = 2020-05-31

C++ wrapper to tabix indexer
##### [gt1](https://github.com/gt1)/[libmaus2](https://github.com/gt1/libmaus2)

ID = 623, Stars = 13, Language = C++, Update = 2019-03-19

collection of data structures and algorithms
##### [tseemann](https://github.com/tseemann)/[berokka](https://github.com/tseemann/berokka)

ID = 624, Stars = 13, Language = Perl, Update = 2019-09-07

üçä
üí´
Trim, circularise and orient long read bacterial genome assemblies
##### [crazyhottommy](https://github.com/crazyhottommy)/[papers_with_data_to_mine](https://github.com/crazyhottommy/papers_with_data_to_mine)

ID = 625, Stars = 13, Language = None, Update = 2020-04-20

published papers with a lot of data
##### [stephenturner](https://github.com/stephenturner)/[msigdf](https://github.com/stephenturner/msigdf)

ID = 626, Stars = 13, Language = R, Update = 2018-12-03

Molecular Signatures Database (MSigDB) in a data frame
##### [refresh-bio](https://github.com/refresh-bio)/[GTC](https://github.com/refresh-bio/GTC)

ID = 627, Stars = 13, Language = C++, Update = 2020-01-13

GenoTypes Compressor
##### [vgteam](https://github.com/vgteam)/[libbdsg](https://github.com/vgteam/libbdsg)

ID = 628, Stars = 13, Language = C++, Update = 2020-06-12

Optimized sequence graph implementations for graph genomics
##### [vgteam](https://github.com/vgteam)/[sv-genotyping-paper](https://github.com/vgteam/sv-genotyping-paper)

ID = 629, Stars = 13, Language = Python, Update = 2019-11-25

None
##### [qiime2](https://github.com/qiime2)/[docs](https://github.com/qiime2/docs)

ID = 630, Stars = 13, Language = Python, Update = 2020-06-19

Source repository for https://docs.qiime2.org
##### [broadinstitute](https://github.com/broadinstitute)/[rawls](https://github.com/broadinstitute/rawls)

ID = 631, Stars = 13, Language = Scala, Update = 2020-06-23

Rawls service for DSDE
##### [broadinstitute](https://github.com/broadinstitute)/[dalmatian](https://github.com/broadinstitute/dalmatian)

ID = 632, Stars = 13, Language = Python, Update = 2020-03-27

dalmatian is a collection of high-level companion functions for Firecloud and FISS.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[apps-scripts](https://github.com/PacificBiosciences/apps-scripts)

ID = 633, Stars = 13, Language = Python, Update = 2020-06-09

Miscellaneous scripts for applications of PacBio systems
##### [wglab](https://github.com/wglab)/[icages](https://github.com/WGLab/icages)

ID = 634, Stars = 13, Language = Perl, Update = 2019-03-21

iCAGES (integrated CAncer GEnome Score) is an effective tool for prioritizing cancer driver genes for a patient
##### [yanglab](https://github.com/yanglab)/[bamTobw](https://github.com/YangLab/bamTobw)

ID = 635, Stars = 13, Language = Shell, Update = 2018-11-30

Convert BAM files to bigWig files with a simple command
##### [biocore](https://github.com/biocore)/[qurro](https://github.com/biocore/qurro)

ID = 636, Stars = 13, Language = JavaScript, Update = 2020-05-22

Visualize differentially ranked features (taxa, metabolites, ...) and their log-ratios across samples
##### [illumina](https://github.com/illumina)/[witty.er](https://github.com/Illumina/witty.er)

ID = 637, Stars = 13, Language = C#, Update = 2020-05-26

What is true, thank you, ernestly. A large variant benchmarking tool analogous to hap.py for small variants.
##### [lh3](https://github.com/lh3)/[lh3-snippets](https://github.com/lh3/lh3-snippets)

ID = 638, Stars = 12, Language = C, Update = 2020-01-23

None
##### [lh3](https://github.com/lh3)/[gfa1](https://github.com/lh3/gfa1)

ID = 639, Stars = 12, Language = C, Update = 2018-08-17

This repo is deprecated. Please use gfatools instead.
##### [vibansal](https://github.com/vibansal)/[crisp](https://github.com/vibansal/crisp)

ID = 640, Stars = 12, Language = C, Update = 2020-02-25

Code for multi-sample variant calling from sequence data of pooled or unpooled DNA samples
##### [yongxinliu](https://github.com/yongxinliu)/[MicrobiomeStatPlot](https://github.com/YongxinLiu/MicrobiomeStatPlot)

ID = 641, Stars = 12, Language = HTML, Update = 2020-06-22

Interpretation and visualization of microbiome charts
##### [griffithlab](https://github.com/griffithlab)/[rnabio.org](https://github.com/griffithlab/rnabio.org)

ID = 642, Stars = 12, Language = CSS, Update = 2020-06-19

website for the rnaseq course
##### [griffithlab](https://github.com/griffithlab)/[pmbio.org](https://github.com/griffithlab/pmbio.org)

ID = 643, Stars = 12, Language = CSS, Update = 2020-05-26

Website for the precision medicine workshop
##### [refresh-bio](https://github.com/refresh-bio)/[FaStore](https://github.com/refresh-bio/FaStore)

ID = 644, Stars = 12, Language = C++, Update = 2018-11-22

FaStore - high-performance FASTQ files compressor
##### [ga4gh](https://github.com/ga4gh)/[data-security](https://github.com/ga4gh/data-security)

ID = 645, Stars = 12, Language = None, Update = 2020-03-30

None
##### [nf-core](https://github.com/nf-core)/[nanoseq](https://github.com/nf-core/nanoseq)

ID = 646, Stars = 12, Language = Nextflow, Update = 2020-05-05

Nanopore demultiplexing, QC and alignment pipeline
##### [nf-core](https://github.com/nf-core)/[scrnaseq](https://github.com/nf-core/scrnaseq)

ID = 647, Stars = 12, Language = Nextflow, Update = 2020-03-08

A single-cell RNAseq pipeline for 10X genomics data
##### [broadinstitute](https://github.com/broadinstitute)/[fiss](https://github.com/broadinstitute/fiss)

ID = 648, Stars = 12, Language = Python, Update = 2020-06-21

FireCloud Service Selector (FISS) -- Python bindings and CLI for FireCloud execution engine
##### [broadinstitute](https://github.com/broadinstitute)/[getzlab-SignatureAnalyzer](https://github.com/broadinstitute/getzlab-SignatureAnalyzer)

ID = 649, Stars = 12, Language = Python, Update = 2020-06-02

Updated SignatureAnalyzer-GPU with mutational spectra & RNA expression compatibility.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[bam2fastx](https://github.com/PacificBiosciences/bam2fastx)

ID = 650, Stars = 12, Language = Python, Update = 2020-06-04

Converting and demultiplexing of PacBio BAM files into gzipped fasta and fastq files.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[minorseq](https://github.com/PacificBiosciences/minorseq)

ID = 651, Stars = 12, Language = None, Update = 2020-04-22

Minor Variant Calling and Phasing Tools
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[kineticsTools](https://github.com/PacificBiosciences/kineticsTools)

ID = 652, Stars = 12, Language = Python, Update = 2020-04-21

Tools for detecting DNA modifications from single molecule, real-time sequencing data
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[ANGEL](https://github.com/PacificBiosciences/ANGEL)

ID = 653, Stars = 12, Language = Python, Update = 2019-05-13

Robust Open Reading Frame prediction (ANGLE re-implementation)
##### [ensembl](https://github.com/ensembl)/[ensembl-funcgen](https://github.com/Ensembl/ensembl-funcgen)

ID = 654, Stars = 12, Language = Perl, Update = 2020-06-16

Ensembl Funcgen Perl API and SQL schema
##### [wglab](https://github.com/wglab)/[LinkedSV](https://github.com/WGLab/LinkedSV)

ID = 655, Stars = 12, Language = C, Update = 2020-04-25

None
##### [openvax](https://github.com/openvax)/[gene-lists](https://github.com/openvax/gene-lists)

ID = 656, Stars = 12, Language = None, Update = 2018-12-18

Gene lists related to cancer immunotherapy
##### [biocore](https://github.com/biocore)/[oecophylla](https://github.com/biocore/oecophylla)

ID = 657, Stars = 12, Language = Python, Update = 2019-04-03

shotgun pipeline
##### [parklab](https://github.com/parklab)/[HiNT](https://github.com/parklab/HiNT)

ID = 658, Stars = 12, Language = Python, Update = 2020-02-17

HiC for copy Number variation and Translocation detection
##### [statgen](https://github.com/statgen)/[savvy](https://github.com/statgen/savvy)

ID = 659, Stars = 12, Language = C++, Update = 2020-05-08

Interface to various variant calling formats.
##### [ekg](https://github.com/ekg)/[mmmulti](https://github.com/ekg/mmmulti)

ID = 660, Stars = 11, Language = C++, Update = 2020-06-18

memory mapped multimap, multiset, and implicit interval tree based on an in-place parallel sort
##### [vals](https://github.com/vals)/[python-gprofiler](https://github.com/vals/python-gprofiler)

ID = 661, Stars = 11, Language = Python, Update = 2019-02-16

None
##### [brentp](https://github.com/brentp)/[bigwig-nim](https://github.com/brentp/bigwig-nim)

ID = 662, Stars = 11, Language = Nim, Update = 2020-05-02

command-line querying+conversion of bigwigs and a nim wrapper for dpryan's libbigwig
##### [brentp](https://github.com/brentp)/[nim-kmer](https://github.com/brentp/nim-kmer)

ID = 663, Stars = 11, Language = Nim, Update = 2020-04-11

DNA kmer operations for nim
##### [dpryan79](https://github.com/dpryan79)/[SE-MEI](https://github.com/dpryan79/SE-MEI)

ID = 664, Stars = 11, Language = C, Update = 2019-12-14

Tools for finding mobile element insertions from single-end datasets
##### [qiime2](https://github.com/qiime2)/[q2studio](https://github.com/qiime2/q2studio)

ID = 665, Stars = 11, Language = JavaScript, Update = 2020-05-11

Prototype graphical user interface for QIIME 2
##### [nf-core](https://github.com/nf-core)/[hlatyping](https://github.com/nf-core/hlatyping)

ID = 666, Stars = 11, Language = Nextflow, Update = 2020-06-19

Precision HLA typing from next-generation sequencing data
##### [nf-core](https://github.com/nf-core)/[smrnaseq](https://github.com/nf-core/smrnaseq)

ID = 667, Stars = 11, Language = Nextflow, Update = 2020-04-15

A small-RNA sequencing analysis pipeline
##### [broadinstitute](https://github.com/broadinstitute)/[cms](https://github.com/broadinstitute/cms)

ID = 668, Stars = 11, Language = Python, Update = 2020-04-24

Composite of Multiple Signals: tests for selection in meiotically recombinant populations
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pypeFLOW](https://github.com/PacificBiosciences/pypeFLOW)

ID = 669, Stars = 11, Language = HTML, Update = 2019-09-05

a simple lightweight workflow engine for data analysis scripting
##### [ncbi](https://github.com/ncbi)/[datasets](https://github.com/ncbi/datasets)

ID = 670, Stars = 11, Language = None, Update = 2020-06-22

NCBI Datasets is an experimental resource for finding and building datasets
##### [ncbi](https://github.com/ncbi)/[cxx-toolkit](https://github.com/ncbi/cxx-toolkit)

ID = 671, Stars = 11, Language = CSS, Update = 2020-06-15

None
##### [ncbi](https://github.com/ncbi)/[osiris](https://github.com/ncbi/osiris)

ID = 672, Stars = 11, Language = C++, Update = 2020-06-04

OSIRIS is a public domain quality assurance software package that facilitates the assessment of multiplex short tandem repeat (STR) DNA profiles based on laboratory-specific protocols. OSIRIS evaluates the raw electrophoresis data contained in .fsa or .hid files using an independently derived mathematically-based sizing algorithm. OSIRIS current‚Ä¶
##### [ncbi](https://github.com/ncbi)/[gprobe](https://github.com/ncbi/gprobe)

ID = 673, Stars = 11, Language = Go, Update = 2020-01-31

client app for the gRPC health-checking protocol
##### [umcugenetics](https://github.com/umcugenetics)/[IAP](https://github.com/UMCUGenetics/IAP)

ID = 674, Stars = 11, Language = Perl, Update = 2020-04-06

Illumina analysis pipeline
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[simple_sv_annotation](https://github.com/AstraZeneca-NGS/simple_sv_annotation)

ID = 675, Stars = 11, Language = Python, Update = 2019-02-18

Simplify snpEff annotations for interesting cases
##### [gatk-workflows](https://github.com/gatk-workflows)/[seq-format-conversion](https://github.com/gatk-workflows/seq-format-conversion)

ID = 676, Stars = 11, Language = wdl, Update = 2020-06-01

Workflows for converting between sequence data formats
##### [dellytools](https://github.com/dellytools)/[svprops](https://github.com/dellytools/svprops)

ID = 677, Stars = 11, Language = C++, Update = 2020-04-14

Computes various SV statistics
##### [illumina](https://github.com/illumina)/[happyR](https://github.com/Illumina/happyR)

ID = 678, Stars = 11, Language = R, Update = 2019-07-12

R tools to interact with hap.py output
##### [benjjneb](https://github.com/benjjneb)/[LRASManuscript](https://github.com/benjjneb/LRASManuscript)

ID = 679, Stars = 10, Language = HTML, Update = 2019-06-29

Reproducible Analyses accompanying DADA2 + PacBio Manuscript
##### [ekg](https://github.com/ekg)/[gimbricate](https://github.com/ekg/gimbricate)

ID = 680, Stars = 10, Language = C++, Update = 2020-01-29

recompute GFA link overlaps
##### [ewels](https://github.com/ewels)/[MultiQC_TestData](https://github.com/ewels/MultiQC_TestData)

ID = 681, Stars = 10, Language = HTML, Update = 2020-06-19

Test data for MultiQC. See https://github.com/ewels/MultiQC
##### [danielecook](https://github.com/danielecook)/[seq-collection](https://github.com/danielecook/seq-collection)

ID = 682, Stars = 10, Language = Nim, Update = 2020-06-15

None
##### [refresh-bio](https://github.com/refresh-bio)/[fqsqueezer](https://github.com/refresh-bio/fqsqueezer)

ID = 683, Stars = 10, Language = C++, Update = 2020-06-16

FQSqueezer - FASTQ compressor based on k-mer statistics
##### [vgteam](https://github.com/vgteam)/[libhandlegraph](https://github.com/vgteam/libhandlegraph)

ID = 684, Stars = 10, Language = C++, Update = 2020-06-05

Library for the Handle Graph abstraction
##### [qiime2](https://github.com/qiime2)/[q2-dada2](https://github.com/qiime2/q2-dada2)

ID = 685, Stars = 10, Language = Python, Update = 2020-06-17

QIIME 2 plugin wrapping DADA2
##### [nf-core](https://github.com/nf-core)/[bcellmagic](https://github.com/nf-core/bcellmagic)

ID = 686, Stars = 10, Language = Nextflow, Update = 2020-02-20

B cell repertoire sequencing analysis pipeline using the immcantation framework
##### [xjtu-omics](https://github.com/xjtu-omics)/[msisensor-pro](https://github.com/xjtu-omics/msisensor-pro)

ID = 687, Stars = 10, Language = C++, Update = 2020-06-23

Microsatellite Instability (MSI) detection using high-throughput sequencing data.
##### [broadinstitute](https://github.com/broadinstitute)/[vim-wdl](https://github.com/broadinstitute/vim-wdl)

ID = 688, Stars = 10, Language = Vim script, Update = 2020-01-29

Vim syntax highlighting for WDL
##### [broadinstitute](https://github.com/broadinstitute)/[scRNA-Seq](https://github.com/broadinstitute/scRNA-Seq)

ID = 689, Stars = 10, Language = Python, Update = 2019-07-26

None
##### [ncbi](https://github.com/ncbi)/[scPopCorn](https://github.com/ncbi/scPopCorn)

ID = 690, Stars = 10, Language = Jupyter Notebook, Update = 2019-10-25

A python tool to do comparative analysis of mulitple single cell datasets.
##### [umcugenetics](https://github.com/umcugenetics)/[NextflowModules](https://github.com/UMCUGenetics/NextflowModules)

ID = 691, Stars = 10, Language = Nextflow, Update = 2020-06-19

UMCU Genetics Nextflow modules
##### [biocore](https://github.com/biocore)/[q2-qemistree](https://github.com/biocore/q2-qemistree)

ID = 692, Stars = 10, Language = Python, Update = 2020-05-29

Hierarchical orderings for mass spectrometry data. Canonically pronounced "chemis-tree".
##### [parklab](https://github.com/parklab)/[LiRA](https://github.com/parklab/LiRA)

ID = 693, Stars = 10, Language = R, Update = 2019-05-17

None
##### [statgen](https://github.com/statgen)/[encore](https://github.com/statgen/encore)

ID = 694, Stars = 10, Language = Python, Update = 2020-06-11

Encore Analysis Server
##### [seandavi](https://github.com/seandavi)/[BiocActions](https://github.com/seandavi/BiocActions)

ID = 695, Stars = 9, Language = Dockerfile, Update = 2019-12-03

Develop and host GitHub actions for Bioconductor
##### [alimanfoo](https://github.com/alimanfoo)/[alimanfoo.github.io](https://github.com/alimanfoo/alimanfoo.github.io)

ID = 696, Stars = 9, Language = Jupyter Notebook, Update = 2020-06-06

Alistair Miles' blog
##### [crazyhottommy](https://github.com/crazyhottommy)/[pyflow-scATACseq](https://github.com/crazyhottommy/pyflow-scATACseq)

ID = 697, Stars = 9, Language = Python, Update = 2020-02-03

snakemake workflow for post-processing scATACseq data
##### [lh3](https://github.com/lh3)/[unicall](https://github.com/lh3/unicall)

ID = 698, Stars = 9, Language = Perl, Update = 2019-07-21

A wrapper for calling small variants from human germline high-coverage single-sample Illumina data
##### [yongxinliu](https://github.com/yongxinliu)/[UsearchChineseManual](https://github.com/YongxinLiu/UsearchChineseManual)

ID = 699, Stars = 9, Language = None, Update = 2019-09-26

USEARCH‰∏≠ÊñáÂ∏ÆÂä©ÊñáÊ°£(USEARCH Chinese manual)
##### [griffithlab](https://github.com/griffithlab)/[genviz.org](https://github.com/griffithlab/genviz.org)

ID = 700, Stars = 9, Language = CSS, Update = 2020-05-26

Genomic data interpretation and visualization Workshop
##### [ga4gh](https://github.com/ga4gh)/[cloud-interop-testing](https://github.com/ga4gh/cloud-interop-testing)

ID = 701, Stars = 9, Language = Python, Update = 2019-10-05

Interoperable execution of workflows using GA4GH APIs
##### [qiime2](https://github.com/qiime2)/[q2-fragment-insertion](https://github.com/qiime2/q2-fragment-insertion)

ID = 702, Stars = 9, Language = Python, Update = 2020-05-15

QIIME 2 plugin for fragment insertion
##### [qiime2](https://github.com/qiime2)/[q2cwl](https://github.com/qiime2/q2cwl)

ID = 703, Stars = 9, Language = Python, Update = 2020-02-13

Prototype interface for automatically generating CWL tools from QIIME 2 actions
##### [deeptools](https://github.com/deeptools)/[HiCBrowser](https://github.com/deeptools/HiCBrowser)

ID = 704, Stars = 9, Language = JavaScript, Update = 2019-12-12

Simple web browser to visualize HiC tracks
##### [nf-core](https://github.com/nf-core)/[bacass](https://github.com/nf-core/bacass)

ID = 705, Stars = 9, Language = Nextflow, Update = 2020-03-16

Simple bacterial assembly and annotation pipeline
##### [broadinstitute](https://github.com/broadinstitute)/[agora](https://github.com/broadinstitute/agora)

ID = 706, Stars = 9, Language = Scala, Update = 2020-06-14

It's a methods repository!
##### [broadinstitute](https://github.com/broadinstitute)/[sparklespray](https://github.com/broadinstitute/sparklespray)

ID = 707, Stars = 9, Language = HTML, Update = 2020-05-20

Easy batch submission of adhoc jobs onto GCP
##### [broadinstitute](https://github.com/broadinstitute)/[2020_scWorkshop](https://github.com/broadinstitute/2020_scWorkshop)

ID = 708, Stars = 9, Language = Shell, Update = 2020-04-15

Code and data repository for the 2020 physalia course on single cell RNA sequencing.
##### [broadinstitute](https://github.com/broadinstitute)/[hdf5-java-bindings](https://github.com/broadinstitute/hdf5-java-bindings)

ID = 709, Stars = 9, Language = Java, Update = 2019-05-20

java bindings for hdf5
##### [ncbi](https://github.com/ncbi)/[cwl-ngs-workflows-cbb](https://github.com/ncbi/cwl-ngs-workflows-cbb)

ID = 710, Stars = 9, Language = Common Workflow Language, Update = 2020-06-22

A set of CWL tools and workflows used by NCBI Computational Biology Branch for NGS data analysis
##### [ensembl](https://github.com/ensembl)/[ensembl-production](https://github.com/Ensembl/ensembl-production)

ID = 711, Stars = 9, Language = Perl, Update = 2020-06-22

Ensembl Production code
##### [ensembl](https://github.com/ensembl)/[ensembl-analysis](https://github.com/Ensembl/ensembl-analysis)

ID = 712, Stars = 9, Language = Perl, Update = 2020-06-19

Modules to interface with tools used in Ensembl Gene Annotation Process and scripts to run pipelines
##### [biowdl](https://github.com/biowdl)/[RNA-seq](https://github.com/biowdl/RNA-seq)

ID = 713, Stars = 9, Language = wdl, Update = 2020-06-19

A BioWDL pipeline for processing RNA-seq data, starting with FASTQ files to produce expression measures and VCFs. Category:Multi-Sample
##### [illumina](https://github.com/illumina)/[Isaac4](https://github.com/Illumina/Isaac4) *new+*

ID = 714, Stars = 9, Language = C++, Update = 2020-06-19

Isaac aligner version 4
##### [ekg](https://github.com/ekg)/[guix-genomics](https://github.com/ekg/guix-genomics)

ID = 715, Stars = 8, Language = Scheme, Update = 2020-06-19

guix packages for bioinformatics software
##### [benlangmead](https://github.com/benlangmead)/[aws-indexes](https://github.com/BenLangmead/aws-indexes)

ID = 716, Stars = 8, Language = None, Update = 2020-06-15

Catalog of genomic indexes freely available from public clouds
##### [lh3](https://github.com/lh3)/[lh3.github.com](https://github.com/lh3/lh3.github.com)

ID = 717, Stars = 8, Language = HTML, Update = 2020-05-27

None
##### [lh3](https://github.com/lh3)/[PortableCrystal](https://github.com/lh3/PortableCrystal)

ID = 718, Stars = 8, Language = None, Update = 2020-05-16

Portable Crystal binary distributions for Linux on x86_64
##### [sigven](https://github.com/sigven)/[oncoEnrichR](https://github.com/sigven/oncoEnrichR)

ID = 719, Stars = 8, Language = R, Update = 2020-05-12

Functional analysis and prioritization of genesets in the context of cancer
##### [a-slide](https://github.com/a-slide)/[NanoCount](https://github.com/a-slide/NanoCount)

ID = 720, Stars = 8, Language = Python, Update = 2020-06-10

EM based transcript abundance from nanopore reads mapped to a transcriptome with minimap2
##### [danielecook](https://github.com/danielecook)/[tut](https://github.com/danielecook/tut)

ID = 721, Stars = 8, Language = Nim, Update = 2020-06-02

A collection of CSV/TSV Utilities
##### [griffithlab](https://github.com/griffithlab)/[VAtools](https://github.com/griffithlab/VAtools)

ID = 722, Stars = 8, Language = Python, Update = 2019-12-02

A set of tools to annotate VCF files with expression and readcount data
##### [mathonco](https://github.com/mathonco)/[EvoFreq](https://github.com/MathOnco/EvoFreq)

ID = 723, Stars = 8, Language = R, Update = 2020-04-08

Evolutionary frequency visualization tool of temporal data
##### [broadinstitute](https://github.com/broadinstitute)/[viral-pipelines](https://github.com/broadinstitute/viral-pipelines)

ID = 724, Stars = 8, Language = wdl, Update = 2020-06-23

viral-ngs: complete pipelines
##### [broadinstitute](https://github.com/broadinstitute)/[cell-health](https://github.com/broadinstitute/cell-health)

ID = 725, Stars = 8, Language = Jupyter Notebook, Update = 2020-06-21

Predicting Cell Health with Morphological Profiles
##### [ncbi](https://github.com/ncbi)/[blast-cloud](https://github.com/ncbi/blast-cloud)

ID = 726, Stars = 8, Language = CSS, Update = 2019-12-04

Documentation for NCBI BLAST AMI
##### [ensembl](https://github.com/ensembl)/[ensj-healthcheck](https://github.com/Ensembl/ensj-healthcheck)

ID = 727, Stars = 8, Language = Java, Update = 2020-06-17

Ensembl's Automated QC Framework
##### [biocore](https://github.com/biocore)/[empress](https://github.com/biocore/empress)

ID = 728, Stars = 8, Language = JavaScript, Update = 2020-06-22

A very fast and scalable phylogenetic visualization viewer
##### [biowdl](https://github.com/biowdl)/[germline-DNA](https://github.com/biowdl/germline-DNA)

ID = 729, Stars = 8, Language = wdl, Update = 2020-06-22

A BioWDL variantcalling pipeline for germline DNA data. Starting with FASTQ files to produce VCF files. Category:Multi-Sample
##### [statgen](https://github.com/statgen)/[swiss](https://github.com/statgen/swiss)

ID = 730, Stars = 8, Language = Python, Update = 2019-11-11

Software to help identify overlap between association scan results and GWAS hit catalogs.
##### [illumina](https://github.com/illumina)/[Cyrius](https://github.com/Illumina/Cyrius)

ID = 731, Stars = 8, Language = Python, Update = 2020-06-13

A tool to genotype CYP2D6 with WGS data
##### [jodyphelan](https://github.com/jodyphelan)/[tbdb](https://github.com/jodyphelan/tbdb)

ID = 732, Stars = 7, Language = Python, Update = 2020-05-28

Standard database for the TBProfiler tool
##### [richarddurbin](https://github.com/richarddurbin)/[phynder](https://github.com/richarddurbin/phynder)

ID = 733, Stars = 7, Language = C, Update = 2020-05-31

likelihood calculations over a tree to support ancient DNA Y chr analysis and more
##### [qiime2](https://github.com/qiime2)/[q2view](https://github.com/qiime2/q2view)

ID = 734, Stars = 7, Language = JavaScript, Update = 2020-06-23

A web-based "serverless" viewer of QIIME 2 Artifacts and Visualizations
##### [qiime2](https://github.com/qiime2)/[q2-longitudinal](https://github.com/qiime2/q2-longitudinal)

ID = 735, Stars = 7, Language = Python, Update = 2020-06-03

QIIME 2 plugin for paired sample comparisons
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[10X_snakepipe](https://github.com/maxplanck-ie/10X_snakepipe)

ID = 736, Stars = 7, Language = Python, Update = 2020-06-15

A snakemake pipeline for 10X genomics cellranger
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[TheWhoTheWhatTheHuh](https://github.com/maxplanck-ie/TheWhoTheWhatTheHuh)

ID = 737, Stars = 7, Language = Python, Update = 2019-10-16

"The who the what the huh?" is our pipeline for converting bcl files to fastq and performing QC.
##### [deeptools](https://github.com/deeptools)/[HiCMatrix](https://github.com/deeptools/HiCMatrix)

ID = 738, Stars = 7, Language = Python, Update = 2020-06-06

None
##### [mathonco](https://github.com/mathonco)/[HAL](https://github.com/MathOnco/HAL)

ID = 739, Stars = 7, Language = Java, Update = 2020-05-09

HAL (Hybrid Automata Library) is the one-stop shop for all of your hybrid modeling needs
##### [nf-core](https://github.com/nf-core)/[proteomicslfq](https://github.com/nf-core/proteomicslfq)

ID = 740, Stars = 7, Language = Nextflow, Update = 2020-06-09

UNDER CONSTRUCTION - Proteomics label-free quantification (LFQ) analysis pipeline
##### [abyzovlab](https://github.com/abyzovlab)/[CNVpytor](https://github.com/abyzovlab/CNVpytor)

ID = 741, Stars = 7, Language = Python, Update = 2020-06-18

a python extension of CNVnator -- a tool for CNV analysis from depth-of-coverage by mapped reads
##### [broadinstitute](https://github.com/broadinstitute)/[wdl-runner](https://github.com/broadinstitute/wdl-runner)

ID = 742, Stars = 7, Language = Python, Update = 2020-06-10

Easily run WDL workflows on GCP
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-tools](https://github.com/broadinstitute/firecloud-tools)

ID = 743, Stars = 7, Language = Python, Update = 2020-05-29

Tools for use with FireCloud
##### [broadinstitute](https://github.com/broadinstitute)/[python-cert_manager](https://github.com/broadinstitute/python-cert_manager)

ID = 744, Stars = 7, Language = Python, Update = 2020-04-10

Python interface to the Sectigo Certificate Manager REST API
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-v8](https://github.com/broadinstitute/gtex-v8)

ID = 745, Stars = 7, Language = Jupyter Notebook, Update = 2020-02-03

Notebooks and scripts for reproducing analyses and figures from the V8 GTEx Consortium paper
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbcommand](https://github.com/PacificBiosciences/pbcommand)

ID = 746, Stars = 7, Language = Python, Update = 2020-06-19

PacBio common models, CLI tool contract interface and SMRT Link Service Interface
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[hg002-ccs](https://github.com/PacificBiosciences/hg002-ccs)

ID = 747, Stars = 7, Language = Python, Update = 2020-04-16

None
##### [umcugenetics](https://github.com/umcugenetics)/[sparqling-genomics](https://github.com/UMCUGenetics/sparqling-genomics)

ID = 748, Stars = 7, Language = Scheme, Update = 2020-06-21

Ontologies, RDF-generators, and a web interface for querying it all.
##### [umcugenetics](https://github.com/umcugenetics)/[CHORD](https://github.com/UMCUGenetics/CHORD)

ID = 749, Stars = 7, Language = R, Update = 2020-05-08

An R package for predicting HR deficiency from mutation contexts
##### [ensembl](https://github.com/ensembl)/[trackhub-registry](https://github.com/Ensembl/trackhub-registry)

ID = 750, Stars = 7, Language = Perl, Update = 2020-03-18

Specifications and implementation of the TrackHub registry
##### [ensembl](https://github.com/ensembl)/[ensembl-git-tools](https://github.com/Ensembl/ensembl-git-tools)

ID = 751, Stars = 7, Language = Perl, Update = 2019-12-09

A collection of tools which Ensembl uses to work with Git
##### [statgen](https://github.com/statgen)/[bravo](https://github.com/statgen/bravo)

ID = 752, Stars = 7, Language = HTML, Update = 2020-06-12

BRowse All Variants Online
##### [statgen](https://github.com/statgen)/[METAL](https://github.com/statgen/METAL)

ID = 753, Stars = 7, Language = C++, Update = 2020-05-05

Meta-analysis of genomewide association scans
##### [ekg](https://github.com/ekg)/[gyeet](https://github.com/ekg/gyeet)

ID = 754, Stars = 6, Language = C++, Update = 2020-05-25

sequence to graph mapper
##### [felixkrueger](https://github.com/felixkrueger)/[nextflow_pipelines](https://github.com/FelixKrueger/nextflow_pipelines)

ID = 755, Stars = 6, Language = Nextflow, Update = 2020-06-19

The set of NGS processing pipelines used at Babraham
##### [danielecook](https://github.com/danielecook)/[gg](https://github.com/danielecook/gg)

ID = 756, Stars = 6, Language = Go, Update = 2020-05-04

Github Gists from the command line. On MAC OSX, it (will be) an Alfred Workflow!
##### [danielecook](https://github.com/danielecook)/[codebox-alfred](https://github.com/danielecook/codebox-alfred)

ID = 757, Stars = 6, Language = Python, Update = 2020-01-11

An alfred workflow for accessing codebox snippets.
##### [stephenturner](https://github.com/stephenturner)/[Tmisc](https://github.com/stephenturner/Tmisc)

ID = 758, Stars = 6, Language = R, Update = 2020-05-12

Turner miscellaneous
##### [stephenturner](https://github.com/stephenturner)/[devnotes](https://github.com/stephenturner/devnotes)

ID = 759, Stars = 6, Language = Shell, Update = 2020-05-05

Random programming notes and orphan scripts
##### [vgteam](https://github.com/vgteam)/[toil-vg](https://github.com/vgteam/toil-vg)

ID = 760, Stars = 6, Language = Python, Update = 2020-06-23

Distributed and cloud computing framework for vg
##### [qiime2](https://github.com/qiime2)/[vm-playbooks](https://github.com/qiime2/vm-playbooks)

ID = 761, Stars = 6, Language = Shell, Update = 2020-03-05

Ansible playbooks for QIIME 2 environments
##### [qiime2](https://github.com/qiime2)/[Keemei](https://github.com/qiime2/Keemei)

ID = 762, Stars = 6, Language = JavaScript, Update = 2020-02-28

Validate tabular bioinformatics file formats in Google Sheets
##### [broadinstitute](https://github.com/broadinstitute)/[covid19-testing](https://github.com/broadinstitute/covid19-testing)

ID = 763, Stars = 6, Language = HTML, Update = 2020-06-23

COVID-19 Diagnostic Processing Dashboard
##### [broadinstitute](https://github.com/broadinstitute)/[gatk-sv](https://github.com/broadinstitute/gatk-sv) *new+*

ID = 764, Stars = 6, Language = wdl, Update = 2020-06-18

None
##### [broadinstitute](https://github.com/broadinstitute)/[ccle_processing](https://github.com/broadinstitute/ccle_processing)

ID = 765, Stars = 6, Language = Jupyter Notebook, Update = 2020-06-18

What you need to process the Quarterly DepMap-Omics releases from Terra
##### [broadinstitute](https://github.com/broadinstitute)/[DLBCL_Nat_Med_April_2018](https://github.com/broadinstitute/DLBCL_Nat_Med_April_2018)

ID = 766, Stars = 6, Language = R, Update = 2020-06-11

None
##### [broadinstitute](https://github.com/broadinstitute)/[AwesomeGenomics](https://github.com/broadinstitute/AwesomeGenomics)

ID = 767, Stars = 6, Language = None, Update = 2020-01-10

Cancer Data Science's go to place for excellent genomics tools and packages
##### [andersenlab](https://github.com/andersenlab)/[cegwas2-nf](https://github.com/AndersenLab/cegwas2-nf)

ID = 768, Stars = 6, Language = R, Update = 2020-05-03

GWA mapping with C. elegans
##### [ncbi](https://github.com/ncbi)/[pm4ngs](https://github.com/ncbi/pm4ngs)

ID = 769, Stars = 6, Language = Jupyter Notebook, Update = 2020-05-28

Project Manager for NGS data analysis
##### [ensembl](https://github.com/ensembl)/[ensembl-io](https://github.com/Ensembl/ensembl-io)

ID = 770, Stars = 6, Language = Perl, Update = 2020-06-11

File parsing and writing code for Ensembl
##### [ensembl](https://github.com/ensembl)/[ensembl-xs](https://github.com/Ensembl/ensembl-xs)

ID = 771, Stars = 6, Language = Perl, Update = 2020-01-21

A collection of XS modules to be used with the Ensembl APIs
##### [yanglab](https://github.com/yanglab)/[CLEAR](https://github.com/YangLab/CLEAR)

ID = 772, Stars = 6, Language = Python, Update = 2020-01-30

direct comparison of circular and linear RNA expression
##### [opengene](https://github.com/opengene)/[UniqueKMER](https://github.com/OpenGene/UniqueKMER)

ID = 773, Stars = 6, Language = C, Update = 2020-05-25

Generate unique KMERs for every contig in a FASTA file
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-genome-processing-pipeline](https://github.com/gatk-workflows/gatk4-genome-processing-pipeline)

ID = 774, Stars = 6, Language = wdl, Update = 2020-06-15

Workflows used for processing whole genome sequence data + germline variant calling.
##### [ding-lab](https://github.com/ding-lab)/[somaticwrapper](https://github.com/ding-lab/somaticwrapper)

ID = 775, Stars = 6, Language = Perl, Update = 2020-06-22

Detect somatic variants from tumor and normal exome data
##### [kamilsjaron](https://github.com/kamilsjaron)/[generic_genomics](https://github.com/KamilSJaron/generic_genomics)

ID = 776, Stars = 5, Language = Python, Update = 2020-05-07

small scripts, functions or bits of code for parsing genomic data
##### [ming-lian](https://github.com/ming-lian)/[Hello-RaspberryPi](https://github.com/Ming-Lian/Hello-RaspberryPi)

ID = 777, Stars = 5, Language = None, Update = 2020-04-22

Áé©ËΩ¨ËÆ°ÁÆóÊú∫Á°¨‰ª∂
##### [vgteam](https://github.com/vgteam)/[vg_wdl](https://github.com/vgteam/vg_wdl)

ID = 778, Stars = 5, Language = wdl, Update = 2020-06-21

Workflow Description Language (WDL) scripts for common vg workflows
##### [qiime2](https://github.com/qiime2)/[q2-feature-classifier](https://github.com/qiime2/q2-feature-classifier)

ID = 779, Stars = 5, Language = Python, Update = 2020-06-17

QIIME 2 plugin supporting taxonomic classification
##### [qiime2](https://github.com/qiime2)/[q2-types](https://github.com/qiime2/q2-types)

ID = 780, Stars = 5, Language = Python, Update = 2020-06-17

Definitions of common QIIME 2 types.
##### [qiime2](https://github.com/qiime2)/[q2-sample-classifier](https://github.com/qiime2/q2-sample-classifier)

ID = 781, Stars = 5, Language = Python, Update = 2020-05-15

QIIME 2 plugin for machine learning prediction of sample data.
##### [samtools](https://github.com/samtools)/[www.htslib.org](https://github.com/samtools/www.htslib.org)

ID = 782, Stars = 5, Language = HTML, Update = 2020-04-20

The http://www.htslib.org/ web site
##### [nf-core](https://github.com/nf-core)/[imcyto](https://github.com/nf-core/imcyto)

ID = 783, Stars = 5, Language = Python, Update = 2020-05-29

Image Mass Cytometry analysis pipeline
##### [nf-core](https://github.com/nf-core)/[epitopeprediction](https://github.com/nf-core/epitopeprediction)

ID = 784, Stars = 5, Language = Python, Update = 2020-05-04

A bioinformatics best-practice analysis pipeline for epitope prediction and annotation
##### [broadinstitute](https://github.com/broadinstitute)/[dig-diabetes-portal](https://github.com/broadinstitute/dig-diabetes-portal)

ID = 785, Stars = 5, Language = JavaScript, Update = 2020-06-18

Diabetes portal for the Medical Population Genetics department at the Broad Institute
##### [broadinstitute](https://github.com/broadinstitute)/[barclay](https://github.com/broadinstitute/barclay)

ID = 786, Stars = 5, Language = Java, Update = 2020-06-16

Command line argument parser and online documentation generation utilities for java command line programs.
##### [broadinstitute](https://github.com/broadinstitute)/[gistic2](https://github.com/broadinstitute/gistic2)

ID = 787, Stars = 5, Language = MATLAB, Update = 2020-04-10

Genomic Identification of Significant Targets in Cancer (GISTIC), version 2
##### [umcugenetics](https://github.com/umcugenetics)/[guix-additions](https://github.com/UMCUGenetics/guix-additions)

ID = 788, Stars = 5, Language = Scheme, Update = 2020-06-21

None
##### [ensembl](https://github.com/ensembl)/[public-plugins](https://github.com/Ensembl/public-plugins)

ID = 789, Stars = 5, Language = HTML, Update = 2020-06-22

Plugins for an Ensembl website
##### [ensembl](https://github.com/ensembl)/[ensembl-test](https://github.com/Ensembl/ensembl-test)

ID = 790, Stars = 5, Language = Perl, Update = 2020-06-11

Test libraries and harnesses used for running the Ensembl test suite
##### [ensembl](https://github.com/ensembl)/[homebrew-ensembl](https://github.com/Ensembl/homebrew-ensembl)

ID = 791, Stars = 5, Language = Ruby, Update = 2020-04-30

Core formula for Ensembl
##### [bioinform](https://github.com/bioinform)/[ecTMB](https://github.com/bioinform/ecTMB)

ID = 792, Stars = 5, Language = R, Update = 2020-05-15

None
##### [openvax](https://github.com/openvax)/[mhcnames](https://github.com/openvax/mhcnames)

ID = 793, Stars = 5, Language = Python, Update = 2020-05-28

All the fun and adventure of MHC naming, now in Python
##### [biowdl](https://github.com/biowdl)/[somatic-variantcalling](https://github.com/biowdl/somatic-variantcalling)

ID = 794, Stars = 5, Language = wdl, Update = 2020-06-19

A pipeline for somatic variantcalling. Category:Single-Sample
##### [statgen](https://github.com/statgen)/[topmed_variant_calling](https://github.com/statgen/topmed_variant_calling)

ID = 795, Stars = 5, Language = C++, Update = 2020-05-07

None
##### [papenfusslab](https://github.com/papenfusslab)/[bionix](https://github.com/PapenfussLab/bionix)

ID = 796, Stars = 5, Language = Nix, Update = 2020-06-05

Functional highly reproducible bioinformatics pipelines
