# Bioinfomatics Repositories

Collection of bioinformatician and their repositories.

- [Top50](https://github.com/xhqsm/bioinfomatics-repositories#Top50)
- [Repositories](https://github.com/xhqsm/bioinfomatics-repositories#repositories)

**Update 2020-05-11**

## Top50

| ID | User | Type |Name | Stars |
| :-: | :-: | :-: | :-: | :-: |
| 1 | [lh3](https://github.com/lh3) | One |Heng Li | 4813 |
| 2 | [broadinstitute](https://github.com/broadinstitute) | Team |Broad Institute | 4696 |
| 3 | [brentp](https://github.com/brentp) | One |Brent Pedersen | 2977 |
| 4 | [samtools](https://github.com/samtools) | Team |samtools | 2329 |
| 5 | [crazyhottommy](https://github.com/crazyhottommy) | One |Ming Tang | 2070 |
| 6 | [arq5x](https://github.com/arq5x) | One |Aaron Quinlan | 1749 |
| 7 | [seandavi](https://github.com/seandavi) | One |Sean Davis | 1687 |
| 8 | [biocore](https://github.com/biocore) | Team |biocore | 1628 |
| 9 | [illumina](https://github.com/illumina) | Team |Illumina | 1616 |
| 10 | [opengene](https://github.com/opengene) | Team |OpenGene - Open Source Genetics Toolbox | 1543 |
| 11 | [pacificbiosciences](https://github.com/pacificbiosciences) | Team |Pacific Biosciences | 1455 |
| 12 | [ncbi](https://github.com/ncbi) | Team |NCBI - National Center for Biotechnology Information/NLM/NIH | 1453 |
| 13 | [stephenturner](https://github.com/stephenturner) | One |Stephen Turner | 1436 |
| 14 | [benlangmead](https://github.com/benlangmead) | One |Ben | 1358 |
| 15 | [bioconda](https://github.com/bioconda) | Team |Bioconda | 1240 |
| 16 | [ekg](https://github.com/ekg) | One |Erik Garrison | 1229 |
| 17 | [tseemann](https://github.com/tseemann) | One |Torsten Seemann | 1216 |
| 18 | [danielecook](https://github.com/danielecook) | One |Daniel E Cook | 1210 |
| 19 | [deeptools](https://github.com/deeptools) | Team |The deepTools ecosystem | 963 |
| 20 | [ewels](https://github.com/ewels) | One |Phil Ewels | 951 |
| 21 | [vgteam](https://github.com/vgteam) | Team |vgteam | 875 |
| 22 | [ensembl](https://github.com/ensembl) | Team |Ensembl Project | 821 |
| 23 | [tanghaibao](https://github.com/tanghaibao) | One |Haibao Tang | 780 |
| 24 | [quinlan-lab](https://github.com/quinlan-lab) | Team |quinlan-lab | 705 |
| 25 | [nf-core](https://github.com/nf-core) | Team |nf-core | 693 |
| 26 | [wglab](https://github.com/wglab) | Team |Wang Genomics Lab | 653 |
| 27 | [openvax](https://github.com/openvax) | Team |OpenVax | 618 |
| 28 | [statgen](https://github.com/statgen) | Team |Center for Statistical Genetics | 607 |
| 29 | [daler](https://github.com/daler) | One |Ryan Dale | 513 |
| 30 | [daehwankimlab](https://github.com/daehwankimlab) | Team |Kim Lab at UT Southwestern Medical Center | 485 |
| 31 | [hall-lab](https://github.com/hall-lab) | Team |Ira Hall lab | 440 |
| 32 | [biod](https://github.com/biod) | Team |BioD | 409 |
| 33 | [cgatoxford](https://github.com/cgatoxford) | Team |CGAT | 405 |
| 34 | [taoliu](https://github.com/taoliu) | One |Tao Liu (œÑŒΩ) | 397 |
| 35 | [mdshw5](https://github.com/mdshw5) | One |Matt Shirley | 391 |
| 36 | [bbuchfink](https://github.com/bbuchfink) | One |Benjamin Buchfink | 387 |
| 37 | [felixkrueger](https://github.com/felixkrueger) | One |Felix Krueger | 381 |
| 38 | [bioinform](https://github.com/bioinform) | Team |The Bioinformatics Repository | 373 |
| 39 | [ga4gh](https://github.com/ga4gh) | Team |Global Alliance for Genomics and Health | 357 |
| 40 | [genome](https://github.com/genome) | Team |The McDonnell Genome Institute | 346 |
| 41 | [dzerbino](https://github.com/dzerbino) | One |Daniel Zerbino | 338 |
| 42 | [gatk-workflows](https://github.com/gatk-workflows) | Team |GATK workflows | 310 |
| 43 | [benjjneb](https://github.com/benjjneb) | One |Benjamin Callahan | 309 |
| 44 | [qiime2](https://github.com/qiime2) | Team |QIIME 2 | 306 |
| 45 | [nanopore-wgs-consortium](https://github.com/nanopore-wgs-consortium) | Team |nanopore-wgs-consortium | 297 |
| 46 | [dpryan79](https://github.com/dpryan79) | One |Devon Ryan | 287 |
| 47 | [ding-lab](https://github.com/ding-lab) | Team |Li Ding's Lab | 276 |
| 48 | [vcftools](https://github.com/vcftools) | Team |vcftools | 274 |
| 49 | [astrazeneca-ngs](https://github.com/astrazeneca-ngs) | Team |AstraZeneca - NGS Team | 270 |
| 50 | [bwa-mem2](https://github.com/bwa-mem2) | Team |bwa-mem2 | 266 |
## Repositories

##### [seandavi](https://github.com/seandavi)/[awesome-single-cell](https://github.com/seandavi/awesome-single-cell)

ID = 1, Stars = 1314, Language = None, Update = 2020-05-05

Community-curated list of software packages and data resources for single-cell, including RNA-seq, ATAC-seq, etc.
##### [bioconda](https://github.com/bioconda)/[bioconda-recipes](https://github.com/bioconda/bioconda-recipes)

ID = 2, Stars = 1161, Language = Shell, Update = 2020-05-11

Conda recipes for the bioconda channel.
##### [stephenturner](https://github.com/stephenturner)/[oneliners](https://github.com/stephenturner/oneliners)

ID = 3, Stars = 1093, Language = None, Update = 2019-06-10

Useful bash one-liners for bioinformatics.
##### [danielecook](https://github.com/danielecook)/[Awesome-Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics)

ID = 4, Stars = 986, Language = None, Update = 2020-04-28

A curated list of awesome Bioinformatics libraries and software.
##### [samtools](https://github.com/samtools)/[samtools](https://github.com/samtools/samtools)

ID = 5, Stars = 909, Language = C, Update = 2020-05-10

Tools (written in C using htslib) for manipulating next-generation sequencing data
##### [lh3](https://github.com/lh3)/[bwa](https://github.com/lh3/bwa)

ID = 6, Stars = 842, Language = C, Update = 2020-04-23

Burrow-Wheeler Aligner for short-read alignment (see minimap2 for long-read alignment)
##### [broadinstitute](https://github.com/broadinstitute)/[gatk](https://github.com/broadinstitute/gatk)

ID = 7, Stars = 808, Language = Java, Update = 2020-05-11

Official code repository for GATK versions 4 and up
##### [opengene](https://github.com/opengene)/[fastp](https://github.com/OpenGene/fastp)

ID = 8, Stars = 747, Language = C++, Update = 2020-04-08

An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...)
##### [lh3](https://github.com/lh3)/[minimap2](https://github.com/lh3/minimap2)

ID = 9, Stars = 725, Language = C, Update = 2020-05-01

A versatile pairwise aligner for genomic and spliced nucleotide sequences
##### [lh3](https://github.com/lh3)/[seqtk](https://github.com/lh3/seqtk)

ID = 10, Stars = 670, Language = C, Update = 2020-01-20

Toolkit for processing sequences in FASTA/Q formats
##### [vgteam](https://github.com/vgteam)/[vg](https://github.com/vgteam/vg)

ID = 11, Stars = 630, Language = C++, Update = 2020-05-10

tools for working with genome variation graphs
##### [arq5x](https://github.com/arq5x)/[bedtools2](https://github.com/arq5x/bedtools2)

ID = 12, Stars = 599, Language = C, Update = 2020-04-25

bedtools - the swiss army knife for genome arithmetic
##### [ewels](https://github.com/ewels)/[MultiQC](https://github.com/ewels/MultiQC)

ID = 13, Stars = 585, Language = Python, Update = 2020-05-10

Aggregate results from bioinformatics analyses across many samples into a single report.
##### [broadinstitute](https://github.com/broadinstitute)/[picard](https://github.com/broadinstitute/picard)

ID = 14, Stars = 560, Language = Java, Update = 2020-05-07

A set of command line tools (in Java) for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.
##### [biocore](https://github.com/biocore)/[scikit-bio](https://github.com/biocore/scikit-bio)

ID = 15, Stars = 551, Language = Python, Update = 2020-03-18

scikit-bio is an open-source, BSD-licensed, Python package providing data structures, algorithms, and educational resources for bioinformatics.
##### [quinlan-lab](https://github.com/quinlan-lab)/[applied-computational-genomics](https://github.com/quinlan-lab/applied-computational-genomics)

ID = 16, Stars = 543, Language = None, Update = 2020-04-21

Applied Computational Genomics Course at UU: Spring 2020
##### [broadinstitute](https://github.com/broadinstitute)/[cromwell](https://github.com/broadinstitute/cromwell)

ID = 17, Stars = 541, Language = Scala, Update = 2020-05-10

Scientific workflow engine designed for simplicity & scalability. Trivially transition between one off use cases to massive scale production environments
##### [broadinstitute](https://github.com/broadinstitute)/[keras-rcnn](https://github.com/broadinstitute/keras-rcnn)

ID = 18, Stars = 495, Language = Python, Update = 2019-05-30

Keras package for region-based convolutional neural networks (RCNNs)
##### [crazyhottommy](https://github.com/crazyhottommy)/[RNA-seq-analysis](https://github.com/crazyhottommy/RNA-seq-analysis)

ID = 19, Stars = 473, Language = Python, Update = 2020-04-27

RNAseq analysis notes from Ming Tang
##### [samtools](https://github.com/samtools)/[htslib](https://github.com/samtools/htslib)

ID = 20, Stars = 469, Language = C, Update = 2020-05-10

C library for high-throughput sequencing data formats
##### [ekg](https://github.com/ekg)/[freebayes](https://github.com/ekg/freebayes)

ID = 21, Stars = 465, Language = C++, Update = 2020-05-08

Bayesian haplotype-based genetic polymorphism discovery and genotyping.
##### [crazyhottommy](https://github.com/crazyhottommy)/[getting-started-with-genomics-tools-and-resources](https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources)

ID = 22, Stars = 423, Language = Shell, Update = 2020-04-21

Unix, R and python tools for genomics and data science
##### [lh3](https://github.com/lh3)/[bioawk](https://github.com/lh3/bioawk)

ID = 23, Stars = 422, Language = C, Update = 2019-09-15

BWK awk modified for biological data
##### [crazyhottommy](https://github.com/crazyhottommy)/[ChIP-seq-analysis](https://github.com/crazyhottommy/ChIP-seq-analysis)

ID = 24, Stars = 415, Language = Python, Update = 2020-04-25

ChIP-seq analysis notes from Ming Tang
##### [ncbi](https://github.com/ncbi)/[sra-tools](https://github.com/ncbi/sra-tools)

ID = 25, Stars = 409, Language = C, Update = 2020-05-08

SRA Tools
##### [taoliu](https://github.com/taoliu)/[MACS](https://github.com/taoliu/MACS)

ID = 26, Stars = 397, Language = Python, Update = 2020-04-20

MACS -- Model-based Analysis of ChIP-Seq
##### [deeptools](https://github.com/deeptools)/[deepTools](https://github.com/deeptools/deepTools)

ID = 27, Stars = 391, Language = Python, Update = 2020-05-02

Tools to process and analyze deep sequencing data.
##### [bbuchfink](https://github.com/bbuchfink)/[diamond](https://github.com/bbuchfink/diamond)

ID = 28, Stars = 387, Language = C++, Update = 2020-05-08

Accelerated BLAST compatible local sequence aligner.
##### [samtools](https://github.com/samtools)/[hts-specs](https://github.com/samtools/hts-specs)

ID = 29, Stars = 366, Language = TeX, Update = 2020-05-06

Specifications of SAM/BAM and related high-throughput sequencing file formats
##### [biod](https://github.com/biod)/[sambamba](https://github.com/biod/sambamba)

ID = 30, Stars = 366, Language = D, Update = 2019-12-17

Tools for working with SAM/BAM/CRAM data
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[Bioinformatics-Training](https://github.com/PacificBiosciences/Bioinformatics-Training)

ID = 31, Stars = 352, Language = R, Update = 2017-09-14

Bioinformatics training resources
##### [tseemann](https://github.com/tseemann)/[prokka](https://github.com/tseemann/prokka)

ID = 32, Stars = 340, Language = Perl, Update = 2020-03-26

‚ö°
‚ôí
Rapid prokaryotic genome annotation
##### [deeptools](https://github.com/deeptools)/[pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks)

ID = 33, Stars = 340, Language = Python, Update = 2020-05-08

python module to plot beautiful and highly customizable genome browser tracks
##### [tanghaibao](https://github.com/tanghaibao)/[goatools](https://github.com/tanghaibao/goatools)

ID = 34, Stars = 323, Language = Python, Update = 2020-05-08

Python library to handle Gene Ontology (GO) terms
##### [tanghaibao](https://github.com/tanghaibao)/[jcvi](https://github.com/tanghaibao/jcvi)

ID = 35, Stars = 319, Language = Python, Update = 2020-05-09

Python library to facilitate genome assembly, annotation and comparative genomics
##### [samtools](https://github.com/samtools)/[bcftools](https://github.com/samtools/bcftools)

ID = 36, Stars = 300, Language = C, Update = 2020-05-05

This is the official development repository for BCFtools. To compile, the develop branch of htslib is needed: git clone --branch=develop git://github.com/samtools/htslib.git htslib
##### [benlangmead](https://github.com/benlangmead)/[bowtie2](https://github.com/BenLangmead/bowtie2)

ID = 37, Stars = 297, Language = C++, Update = 2020-05-03

A fast and sensitive gapped read aligner
##### [brentp](https://github.com/brentp)/[mosdepth](https://github.com/brentp/mosdepth)

ID = 38, Stars = 297, Language = Nim, Update = 2020-05-02

fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing
##### [vcftools](https://github.com/vcftools)/[vcftools](https://github.com/vcftools/vcftools)

ID = 39, Stars = 274, Language = C++, Update = 2020-03-10

A set of tools written in Perl and C++ for working with VCF files, such as those generated by the 1000 Genomes Project.
##### [arq5x](https://github.com/arq5x)/[gemini](https://github.com/arq5x/gemini)

ID = 40, Stars = 272, Language = Python, Update = 2020-04-28

a lightweight db framework for exploring genetic variation.
##### [mdshw5](https://github.com/mdshw5)/[pyfaidx](https://github.com/mdshw5/pyfaidx)

ID = 41, Stars = 268, Language = Python, Update = 2020-03-07

Efficient pythonic random access to fasta subsequences
##### [bwa-mem2](https://github.com/bwa-mem2)/[bwa-mem2](https://github.com/bwa-mem2/bwa-mem2)

ID = 42, Stars = 266, Language = C++, Update = 2020-05-09

The next version of bwa-mem (WIP; not recommended for production uses at the moment)
##### [biocore](https://github.com/biocore)/[qiime](https://github.com/biocore/qiime)

ID = 43, Stars = 264, Language = Python, Update = 2018-01-23

Official QIIME 1 software repository. QIIME 2 (https://qiime2.org) has succeeded QIIME 1 as of January 2018.
##### [hall-lab](https://github.com/hall-lab)/[speedseq](https://github.com/hall-lab/speedseq)

ID = 44, Stars = 262, Language = C, Update = 2018-02-08

A flexible framework for rapid genome analysis and interpretation
##### [crazyhottommy](https://github.com/crazyhottommy)/[bioinformatics-one-liners](https://github.com/crazyhottommy/bioinformatics-one-liners)

ID = 45, Stars = 256, Language = None, Update = 2020-02-27

Bioinformatics one liners from Ming Tang
##### [cgatoxford](https://github.com/cgatoxford)/[UMI-tools](https://github.com/CGATOxford/UMI-tools)

ID = 46, Stars = 253, Language = Python, Update = 2020-03-02

Tools for handling Unique Molecular Identifiers in NGS data sets
##### [nanopore-wgs-consortium](https://github.com/nanopore-wgs-consortium)/[NA12878](https://github.com/nanopore-wgs-consortium/NA12878)

ID = 47, Stars = 243, Language = Python, Update = 2020-01-25

Data and analysis for NA12878 genome on nanopore
##### [etal](https://github.com/etal)/[cnvkit](https://github.com/etal/cnvkit)

ID = 48, Stars = 241, Language = Python, Update = 2020-02-10

Copy number variant detection from targeted DNA sequencing
##### [daehwankimlab](https://github.com/daehwankimlab)/[hisat2](https://github.com/DaehwanKimLab/hisat2)

ID = 49, Stars = 235, Language = C++, Update = 2020-04-25

Graph-based alignment (Hierarchical Graph FM index)
##### [brentp](https://github.com/brentp)/[vcfanno](https://github.com/brentp/vcfanno)

ID = 50, Stars = 229, Language = Go, Update = 2020-04-15

annotate a VCF with other VCFs/BEDs/tabixed files
##### [benjjneb](https://github.com/benjjneb)/[dada2](https://github.com/benjjneb/dada2)

ID = 51, Stars = 226, Language = R, Update = 2020-04-29

Accurate sample inference from amplicon data with single nucleotide resolution
##### [dzerbino](https://github.com/dzerbino)/[velvet](https://github.com/dzerbino/velvet)

ID = 52, Stars = 222, Language = C, Update = 2018-07-05

Short read de novo assembler using de Bruijn graphs
##### [arq5x](https://github.com/arq5x)/[poretools](https://github.com/arq5x/poretools)

ID = 53, Stars = 221, Language = Jupyter Notebook, Update = 2017-12-16

a toolkit for working with Oxford nanopore data
##### [broadinstitute](https://github.com/broadinstitute)/[keras-resnet](https://github.com/broadinstitute/keras-resnet)

ID = 54, Stars = 219, Language = Python, Update = 2019-08-21

Keras package for deep residual networks
##### [openvax](https://github.com/openvax)/[pyensembl](https://github.com/openvax/pyensembl)

ID = 55, Stars = 217, Language = Python, Update = 2020-01-21

Python interface to access reference genome features (such as genes, transcripts, and exons) from Ensembl
##### [illumina](https://github.com/illumina)/[manta](https://github.com/Illumina/manta)

ID = 56, Stars = 216, Language = C++, Update = 2019-07-17

Structural variant and indel caller for mapped sequencing data
##### [qiime2](https://github.com/qiime2)/[qiime2](https://github.com/qiime2/qiime2)

ID = 57, Stars = 211, Language = Python, Update = 2020-04-10

Official repository for the QIIME 2 framework.
##### [samtools](https://github.com/samtools)/[htsjdk](https://github.com/samtools/htsjdk)

ID = 58, Stars = 208, Language = Java, Update = 2020-05-06

A Java API for high-throughput sequencing data (HTS) formats.
##### [benlangmead](https://github.com/benlangmead)/[comp-genomics-class](https://github.com/BenLangmead/comp-genomics-class)

ID = 59, Stars = 207, Language = Jupyter Notebook, Update = 2019-10-29

Code and examples for JHU Computational Genomics class
##### [illumina](https://github.com/illumina)/[hap.py](https://github.com/Illumina/hap.py)

ID = 60, Stars = 207, Language = C++, Update = 2020-02-27

Haplotype VCF comparison tools
##### [ensembl](https://github.com/ensembl)/[ensembl-vep](https://github.com/Ensembl/ensembl-vep)

ID = 61, Stars = 206, Language = Perl, Update = 2020-04-29

The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
##### [illumina](https://github.com/illumina)/[strelka](https://github.com/Illumina/strelka)

ID = 62, Stars = 206, Language = C++, Update = 2019-09-26

Strelka2 germline and somatic small variant caller
##### [benlangmead](https://github.com/benlangmead)/[c-cpp-notes](https://github.com/BenLangmead/c-cpp-notes)

ID = 63, Stars = 205, Language = Python, Update = 2020-05-09

Lecture notes and example code for teaching C & C++
##### [benlangmead](https://github.com/benlangmead)/[bowtie](https://github.com/BenLangmead/bowtie)

ID = 64, Stars = 204, Language = C++, Update = 2020-04-24

An ultrafast memory-efficient short read aligner
##### [brentp](https://github.com/brentp)/[cyvcf2](https://github.com/brentp/cyvcf2)

ID = 65, Stars = 198, Language = Python, Update = 2020-05-02

cython + htslib == fast VCF and BCF processing
##### [daler](https://github.com/daler)/[pybedtools](https://github.com/daler/pybedtools)

ID = 66, Stars = 197, Language = Python, Update = 2020-04-27

Python wrapper -- and more -- for Aaron Quinlan's BEDTools (bioinformatics tools)
##### [lh3](https://github.com/lh3)/[miniasm](https://github.com/lh3/miniasm)

ID = 67, Stars = 196, Language = TeX, Update = 2020-04-03

Ultrafast de novo assembly for long noisy reads (though having no consensus step)
##### [broadinstitute](https://github.com/broadinstitute)/[pilon](https://github.com/broadinstitute/pilon)

ID = 68, Stars = 196, Language = Scala, Update = 2020-02-20

Pilon is an automated genome assembly improvement and variant detection tool
##### [arq5x](https://github.com/arq5x)/[lumpy-sv](https://github.com/arq5x/lumpy-sv)

ID = 69, Stars = 194, Language = C, Update = 2019-04-16

lumpy: a general probabilistic framework for structural variant discovery
##### [dellytools](https://github.com/dellytools)/[delly](https://github.com/dellytools/delly)

ID = 70, Stars = 192, Language = C++, Update = 2020-05-09

DELLY2: Structural variant discovery by integrated paired-end and split-read analysis
##### [felixkrueger](https://github.com/felixkrueger)/[TrimGalore](https://github.com/FelixKrueger/TrimGalore)

ID = 71, Stars = 189, Language = Perl, Update = 2020-05-06

A wrapper around Cutadapt and FastQC to consistently apply adapter and quality trimming to FastQ files, with extra functionality for RRBS data
##### [tseemann](https://github.com/tseemann)/[snippy](https://github.com/tseemann/snippy)

ID = 72, Stars = 185, Language = Perl, Update = 2020-03-28

‚úÇÔ∏è
‚ö°
Rapid haploid variant calling and core genome alignment
##### [nf-core](https://github.com/nf-core)/[rnaseq](https://github.com/nf-core/rnaseq)

ID = 73, Stars = 184, Language = Nextflow, Update = 2020-05-10

RNA sequencing analysis pipeline using STAR, HISAT2 and Salmon with gene counts and quality control
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[FALCON](https://github.com/PacificBiosciences/FALCON)

ID = 74, Stars = 184, Language = Python, Update = 2018-10-01

FALCON: experimental PacBio diploid assembler -- Out-of-date -- Please use a binary release: https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries
##### [benlangmead](https://github.com/benlangmead)/[ads1-notebooks](https://github.com/BenLangmead/ads1-notebooks)

ID = 75, Stars = 179, Language = None, Update = 2015-10-09

Copies of notebooks used in the practical sessions for Algorithms for DNA Sequencing
##### [seandavi](https://github.com/seandavi)/[awesome-cancer-variant-databases](https://github.com/seandavi/awesome-cancer-variant-databases)

ID = 76, Stars = 173, Language = None, Update = 2019-03-13

A community-maintained repository of cancer clinical knowledge bases and databases focused on cancer variants.
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[snakepipes](https://github.com/maxplanck-ie/snakepipes)

ID = 77, Stars = 166, Language = Python, Update = 2020-05-05

Customizable workflows based on snakemake and python for the analysis of NGS data
##### [felixkrueger](https://github.com/felixkrueger)/[Bismark](https://github.com/FelixKrueger/Bismark)

ID = 78, Stars = 163, Language = HTML, Update = 2020-04-28

A tool to map bisulfite converted sequence reads and determine cytosine methylation states
##### [ekg](https://github.com/ekg)/[intervaltree](https://github.com/ekg/intervaltree)

ID = 79, Stars = 162, Language = C++, Update = 2019-12-10

a minimal C++ interval tree implementation
##### [brentp](https://github.com/brentp)/[goleft](https://github.com/brentp/goleft)

ID = 80, Stars = 162, Language = Go, Update = 2020-02-04

goleft is a collection of bioinformatics tools distributed under MIT license in a single static binary
##### [genome](https://github.com/genome)/[bam-readcount](https://github.com/genome/bam-readcount)

ID = 81, Stars = 159, Language = Python, Update = 2020-02-10

count DNA sequence reads in BAM files
##### [eweitz](https://github.com/eweitz)/[ideogram](https://github.com/eweitz/ideogram)

ID = 82, Stars = 158, Language = JavaScript, Update = 2020-05-10

Chromosome visualization for the web
##### [lh3](https://github.com/lh3)/[wgsim](https://github.com/lh3/wgsim)

ID = 83, Stars = 157, Language = C, Update = 2019-10-02

Reads simulator
##### [opengene](https://github.com/opengene)/[AfterQC](https://github.com/OpenGene/AfterQC)

ID = 84, Stars = 157, Language = Python, Update = 2017-12-29

Automatic Filtering, Trimming, Error Removing and Quality Control for fastq data
##### [benlangmead](https://github.com/benlangmead)/[ads1-slides](https://github.com/BenLangmead/ads1-slides)

ID = 85, Stars = 153, Language = None, Update = 2015-09-15

Slides for Algorithms for DNA Sequencing Coursera class
##### [ncbi](https://github.com/ncbi)/[PubReader](https://github.com/ncbi/PubReader)

ID = 86, Stars = 153, Language = JavaScript, Update = 2018-06-21

A new way to view journal articles
##### [dpryan79](https://github.com/dpryan79)/[ChromosomeMappings](https://github.com/dpryan79/ChromosomeMappings)

ID = 87, Stars = 152, Language = None, Update = 2020-04-16

This repository contains chromosome/contig name mappings between UCSC <-> Ensembl <-> Gencode for a variety of genomes.
##### [sigven](https://github.com/sigven)/[pcgr](https://github.com/sigven/pcgr)

ID = 88, Stars = 147, Language = R, Update = 2020-02-25

Personal Cancer Genome Reporter (PCGR)
##### [atks](https://github.com/atks)/[vt](https://github.com/atks/vt)

ID = 89, Stars = 146, Language = C, Update = 2020-03-31

A tool set for short variant discovery in genetic sequence data.
##### [brentp](https://github.com/brentp)/[bio-playground](https://github.com/brentp/bio-playground)

ID = 90, Stars = 146, Language = C, Update = 2018-11-26

miscellaneous scripts for bioinformatics/genomics that dont merit their own repo.
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-pipeline](https://github.com/broadinstitute/gtex-pipeline)

ID = 91, Stars = 146, Language = Python, Update = 2020-03-27

GTEx & TOPMed data production and analysis pipelines
##### [gregoryfaust](https://github.com/gregoryfaust)/[samblaster](https://github.com/GregoryFaust/samblaster) *new+*

ID = 92, Stars = 142, Language = C++, Update = 2020-03-18

samblaster: a tool to mark duplicates and extract discordant and split reads from sam files.
##### [daler](https://github.com/daler)/[gffutils](https://github.com/daler/gffutils)

ID = 93, Stars = 142, Language = Python, Update = 2020-04-07

GFF and GTF file manipulation and interconversion
##### [bioinformaticsfmrp](https://github.com/bioinformaticsfmrp)/[TCGAbiolinks](https://github.com/BioinformaticsFMRP/TCGAbiolinks)

ID = 94, Stars = 141, Language = R, Update = 2020-05-07

TCGAbiolinks
##### [crazyhottommy](https://github.com/crazyhottommy)/[scRNAseq-analysis-notes](https://github.com/crazyhottommy/scRNAseq-analysis-notes)

ID = 95, Stars = 139, Language = None, Update = 2020-04-09

scRNAseq analysis notes from Ming Tang
##### [broadinstitute](https://github.com/broadinstitute)/[infercnv](https://github.com/broadinstitute/infercnv)

ID = 96, Stars = 137, Language = R, Update = 2020-05-08

Inferring CNV from Single-Cell RNA-Seq
##### [lh3](https://github.com/lh3)/[minigraph](https://github.com/lh3/minigraph)

ID = 97, Stars = 132, Language = C, Update = 2020-03-31

Proof-of-concept seq-to-graph mapper and graph generator
##### [ekg](https://github.com/ekg)/[alignment-and-variant-calling-tutorial](https://github.com/ekg/alignment-and-variant-calling-tutorial)

ID = 98, Stars = 130, Language = None, Update = 2019-06-12

basic walk-throughs for alignment and variant calling from NGS sequencing data
##### [ekg](https://github.com/ekg)/[fraction.js](https://github.com/ekg/fraction.js)

ID = 99, Stars = 130, Language = JavaScript, Update = 2018-07-25

A fraction math library in javascript.
##### [arq5x](https://github.com/arq5x)/[bedtools](https://github.com/arq5x/bedtools)

ID = 100, Stars = 130, Language = C++, Update = 2019-03-10

A powerful toolset for genome arithmetic.
##### [daehwankimlab](https://github.com/daehwankimlab)/[centrifuge](https://github.com/DaehwanKimLab/centrifuge)

ID = 101, Stars = 130, Language = C++, Update = 2020-05-10

Classifier for metagenomic sequences
##### [broadinstitute](https://github.com/broadinstitute)/[viral-ngs](https://github.com/broadinstitute/viral-ngs)

ID = 102, Stars = 129, Language = Python, Update = 2020-04-10

Viral genomics analysis pipelines
##### [marianattestad](https://github.com/marianattestad)/[Ribbon](https://github.com/MariaNattestad/Ribbon)

ID = 103, Stars = 128, Language = JavaScript, Update = 2020-05-02

A genome browser that shows long reads and complex variants better
##### [illumina](https://github.com/illumina)/[SpliceAI](https://github.com/Illumina/SpliceAI)

ID = 104, Stars = 127, Language = Python, Update = 2020-05-07

A deep learning-based tool to identify splice variants
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[VarDict](https://github.com/AstraZeneca-NGS/VarDict)

ID = 105, Stars = 126, Language = Perl, Update = 2020-05-08

VarDict
##### [wglab](https://github.com/wglab)/[dragonstar2019](https://github.com/WGLab/dragonstar2019)

ID = 106, Stars = 122, Language = None, Update = 2019-08-30

None
##### [brentp](https://github.com/brentp)/[hts-nim](https://github.com/brentp/hts-nim)

ID = 107, Stars = 121, Language = Nim, Update = 2020-05-08

nim wrapper for htslib for parsing genomics data files
##### [brentp](https://github.com/brentp)/[smoove](https://github.com/brentp/smoove)

ID = 108, Stars = 118, Language = Go, Update = 2019-11-11

structural variant calling and genotyping with existing tools, but, smoothly.
##### [genetalks](https://github.com/genetalks)/[gtz](https://github.com/Genetalks/gtz)

ID = 109, Stars = 118, Language = None, Update = 2020-04-01

A high performance and compression ratio compressor for genomic data, powered by GTXLab of Genetalks.
##### [s-andrews](https://github.com/s-andrews)/[FastQC](https://github.com/s-andrews/FastQC)

ID = 110, Stars = 115, Language = Java, Update = 2020-02-28

A quality control analysis tool for high throughput sequencing data
##### [cgatoxford](https://github.com/cgatoxford)/[cgat](https://github.com/CGATOxford/cgat)

ID = 111, Stars = 115, Language = Python, Update = 2018-11-08

Do not use - please refer to our newest code: https://github.com/cgat-developers/cgat-apps
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[blasr](https://github.com/PacificBiosciences/blasr)

ID = 112, Stars = 115, Language = C++, Update = 2019-09-13

BLASR: The PacBio¬Æ long read aligner
##### [deeptools](https://github.com/deeptools)/[pyBigWig](https://github.com/deeptools/pyBigWig)

ID = 113, Stars = 114, Language = C, Update = 2020-05-02

A python extension for quick access to bigWig and bigBed files
##### [stephenturner](https://github.com/stephenturner)/[annotables](https://github.com/stephenturner/annotables)

ID = 114, Stars = 113, Language = R, Update = 2019-01-28

R data package for annotating/converting Gene IDs
##### [abyzovlab](https://github.com/abyzovlab)/[CNVnator](https://github.com/abyzovlab/CNVnator)

ID = 115, Stars = 113, Language = C++, Update = 2019-12-17

a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads
##### [bioinform](https://github.com/bioinform)/[neusomatic](https://github.com/bioinform/neusomatic)

ID = 116, Stars = 112, Language = Python, Update = 2020-05-10

NeuSomatic: Deep convolutional neural networks for accurate somatic mutation detection
##### [ocxtal](https://github.com/ocxtal)/[minialign](https://github.com/ocxtal/minialign)

ID = 117, Stars = 111, Language = C, Update = 2018-09-10

fast and accurate alignment tool for PacBio and Nanopore long reads
##### [brentp](https://github.com/brentp)/[cruzdb](https://github.com/brentp/cruzdb)

ID = 118, Stars = 111, Language = Python, Update = 2020-03-12

python access to UCSC genomes database
##### [ga4gh](https://github.com/ga4gh)/[benchmarking-tools](https://github.com/ga4gh/benchmarking-tools)

ID = 119, Stars = 110, Language = HTML, Update = 2020-04-07

Repository for the GA4GH Benchmarking Team work developing standardized benchmarking methods for germline small variant calls
##### [tseemann](https://github.com/tseemann)/[abricate](https://github.com/tseemann/abricate)

ID = 120, Stars = 108, Language = Perl, Update = 2020-04-19

üîé
üíä
Mass screening of contigs for antimicrobial and virulence genes
##### [lh3](https://github.com/lh3)/[kmer-cnt](https://github.com/lh3/kmer-cnt)

ID = 121, Stars = 107, Language = C++, Update = 2020-03-10

Code examples of fast and simple k-mer counters for tutorial purposes
##### [lh3](https://github.com/lh3)/[readfq](https://github.com/lh3/readfq)

ID = 122, Stars = 106, Language = C, Update = 2019-08-09

Fast multi-line FASTA/Q reader in several programming languages
##### [opengene](https://github.com/opengene)/[awesome-bio-datasets](https://github.com/OpenGene/awesome-bio-datasets)

ID = 123, Stars = 105, Language = None, Update = 2017-10-28

awesome-bio-datasets
##### [stephenturner](https://github.com/stephenturner)/[qqman](https://github.com/stephenturner/qqman)

ID = 124, Stars = 103, Language = R, Update = 2018-08-01

An R package for creating Q-Q and manhattan plots from GWAS results
##### [vals](https://github.com/vals)/[umis](https://github.com/vals/umis)

ID = 125, Stars = 103, Language = Python, Update = 2020-03-28

Tools for processing UMI RNA-tag data
##### [opengene](https://github.com/opengene)/[MutScan](https://github.com/OpenGene/MutScan)

ID = 126, Stars = 103, Language = C, Update = 2020-01-21

Detect and visualize target mutations by scanning FastQ files directly
##### [illumina](https://github.com/illumina)/[canvas](https://github.com/Illumina/canvas)

ID = 127, Stars = 103, Language = C#, Update = 2019-09-03

Canvas - Copy number variant (CNV) calling from DNA sequencing data
##### [lh3](https://github.com/lh3)/[minimap](https://github.com/lh3/minimap)

ID = 128, Stars = 101, Language = C, Update = 2017-09-20

This repo is DEPRECATED. Please use minimap2, the successor of minimap.
##### [bioinform](https://github.com/bioinform)/[somaticseq](https://github.com/bioinform/somaticseq)

ID = 129, Stars = 101, Language = Shell, Update = 2020-05-09

An ensemble approach to accurately detect somatic mutations using SomaticSeq
##### [biocore](https://github.com/biocore)/[emp](https://github.com/biocore/emp)

ID = 130, Stars = 101, Language = Jupyter Notebook, Update = 2020-05-09

Code repository of the Earth Microbiome Project.
##### [broadinstitute](https://github.com/broadinstitute)/[xtermcolor](https://github.com/broadinstitute/xtermcolor)

ID = 131, Stars = 99, Language = Python, Update = 2017-10-08

Python library for terminal color support (including 256-color support)
##### [refresh-bio](https://github.com/refresh-bio)/[KMC](https://github.com/refresh-bio/KMC)

ID = 132, Stars = 99, Language = C++, Update = 2019-08-13

Fast and frugal disk based k-mer counter
##### [tseemann](https://github.com/tseemann)/[shovill](https://github.com/tseemann/shovill)

ID = 133, Stars = 98, Language = Perl, Update = 2020-03-15

‚ö°
‚ô†Ô∏è
Assemble bacterial isolate genomes from Illumina paired-end reads
##### [wglab](https://github.com/wglab)/[InterVar](https://github.com/WGLab/InterVar)

ID = 134, Stars = 98, Language = Python, Update = 2020-01-02

A bioinformatics software tool for clinical interpretation of genetic variants by the 2015 ACMG-AMP guideline
##### [wglab](https://github.com/wglab)/[doc-ANNOVAR](https://github.com/WGLab/doc-ANNOVAR)

ID = 135, Stars = 97, Language = None, Update = 2020-04-29

Documentation for the ANNOVAR software
##### [ncbi](https://github.com/ncbi)/[ngs](https://github.com/ncbi/ngs)

ID = 136, Stars = 97, Language = C++, Update = 2020-05-07

NGS Language Bindings
##### [papenfusslab](https://github.com/papenfusslab)/[gridss](https://github.com/PapenfussLab/gridss)

ID = 137, Stars = 96, Language = Java, Update = 2020-05-10

GRIDSS: the Genomic Rearrangement IDentification Software Suite
##### [illumina](https://github.com/illumina)/[pyflow](https://github.com/Illumina/pyflow)

ID = 138, Stars = 95, Language = Python, Update = 2018-03-02

A lightweight parallel task engine
##### [deeptools](https://github.com/deeptools)/[HiCExplorer](https://github.com/deeptools/HiCExplorer)

ID = 139, Stars = 94, Language = Python, Update = 2020-05-06

HiCExplorer is a powerful and easy to use set of tools to process, normalize and visualize Hi-C data.
##### [ensembl](https://github.com/ensembl)/[ensembl-rest](https://github.com/Ensembl/ensembl-rest)

ID = 140, Stars = 94, Language = Perl, Update = 2020-05-01

Language agnostic RESTful data access to Ensembl data over HTTP
##### [lh3](https://github.com/lh3)/[cgranges](https://github.com/lh3/cgranges)

ID = 141, Stars = 93, Language = C, Update = 2020-04-19

A C/C++ library for fast interval overlap queries (with a "bedtools coverage" example)
##### [vibansal](https://github.com/vibansal)/[HapCUT2](https://github.com/vibansal/HapCUT2)

ID = 142, Stars = 93, Language = C, Update = 2020-05-07

software tools for haplotype assembly from sequence data
##### [sunbeam-labs](https://github.com/sunbeam-labs)/[sunbeam](https://github.com/sunbeam-labs/sunbeam)

ID = 143, Stars = 93, Language = Python, Update = 2020-03-11

A robust, extensible metagenomics pipeline
##### [ncbi](https://github.com/ncbi)/[pgap](https://github.com/ncbi/pgap)

ID = 144, Stars = 93, Language = Common Workflow Language, Update = 2020-04-30

NCBI Prokaryotic Genome Annotation Pipeline
##### [ensembl](https://github.com/ensembl)/[WiggleTools](https://github.com/Ensembl/WiggleTools)

ID = 145, Stars = 92, Language = C, Update = 2020-03-11

Basic operations on the space of numerical functions defined on the genome using lazy evaluators for flexibility and efficiency
##### [brentp](https://github.com/brentp)/[peddy](https://github.com/brentp/peddy)

ID = 146, Stars = 91, Language = Jupyter Notebook, Update = 2020-04-22

genotype :: ped correspondence check, ancestry check, sex check. directly, quickly on VCF
##### [biocore](https://github.com/biocore)/[American-Gut](https://github.com/biocore/American-Gut)

ID = 147, Stars = 91, Language = Jupyter Notebook, Update = 2017-03-17

American Gut open-access data and IPython notebooks
##### [dkoboldt](https://github.com/dkoboldt)/[varscan](https://github.com/dkoboldt/varscan)

ID = 148, Stars = 90, Language = None, Update = 2019-07-25

Variant calling and somatic mutation/CNV detection for next-generation sequencing data
##### [pcingola](https://github.com/pcingola)/[SnpEff](https://github.com/pcingola/SnpEff)

ID = 149, Stars = 90, Language = Java, Update = 2020-05-10

None
##### [brentp](https://github.com/brentp)/[somalier](https://github.com/brentp/somalier)

ID = 150, Stars = 90, Language = Nim, Update = 2020-05-01

fast sample-swap and relatedness checks on BAMs/CRAMs/VCFs/GVCFs... "like damn that is one smart wine guy"
##### [mdshw5](https://github.com/mdshw5)/[fastqp](https://github.com/mdshw5/fastqp)

ID = 151, Stars = 89, Language = Python, Update = 2018-05-29

Simple FASTQ quality assessment using Python
##### [vgteam](https://github.com/vgteam)/[sequenceTubeMap](https://github.com/vgteam/sequenceTubeMap)

ID = 152, Stars = 89, Language = JavaScript, Update = 2020-03-14

displays multiple genomic sequences in the form of a tube map
##### [ding-lab](https://github.com/ding-lab)/[msisensor](https://github.com/ding-lab/msisensor)

ID = 153, Stars = 89, Language = C++, Update = 2020-04-22

microsatellite instability detection using tumor only or paired tumor-normal data
##### [fhcrc](https://github.com/fhcrc)/[seqmagick](https://github.com/fhcrc/seqmagick)

ID = 154, Stars = 87, Language = Python, Update = 2019-12-13

An imagemagick-like frontend to Biopython SeqIO
##### [ming-lian](https://github.com/ming-lian)/[NGS-analysis](https://github.com/Ming-Lian/NGS-analysis)

ID = 155, Stars = 86, Language = None, Update = 2020-04-12

‰∫å‰ª£ÊµãÂ∫èÊï∞ÊçÆÂàÜÊûê
##### [cschin](https://github.com/cschin)/[ipython_d3_mashup](https://github.com/cschin/ipython_d3_mashup)

ID = 156, Stars = 86, Language = JavaScript, Update = 2012-09-10

None
##### [brentp](https://github.com/brentp)/[nim-plotly](https://github.com/brentp/nim-plotly)

ID = 157, Stars = 86, Language = Nim, Update = 2020-05-02

plotting library for nim-lang
##### [a-slide](https://github.com/a-slide)/[pycoQC](https://github.com/a-slide/pycoQC)

ID = 158, Stars = 85, Language = Python, Update = 2020-04-03

pycoQC computes metrics and generates Interactive QC plots from the sequencing summary report generated by Oxford Nanopore technologies basecaller (Albacore/Guppy)
##### [lh3](https://github.com/lh3)/[fermikit](https://github.com/lh3/fermikit)

ID = 159, Stars = 85, Language = TeX, Update = 2017-12-20

De novo assembly based variant calling pipeline for Illumina short reads
##### [crazyhottommy](https://github.com/crazyhottommy)/[DNA-seq-analysis](https://github.com/crazyhottommy/DNA-seq-analysis)

ID = 160, Stars = 85, Language = Shell, Update = 2020-02-13

DNA sequencing analysis notes from Ming Tang
##### [ewels](https://github.com/ewels)/[ClusterFlow](https://github.com/ewels/ClusterFlow)

ID = 161, Stars = 85, Language = Perl, Update = 2019-06-08

A pipelining tool to automate and standardise bioinformatics analyses on cluster environments.
##### [illumina](https://github.com/illumina)/[paragraph](https://github.com/Illumina/paragraph)

ID = 162, Stars = 85, Language = C++, Update = 2019-09-24

Graph realignment tools for structural variants
##### [pcingola](https://github.com/pcingola)/[BigDataScript](https://github.com/pcingola/BigDataScript)

ID = 163, Stars = 84, Language = Java, Update = 2020-03-31

BigDataScript: Scirpting language for big data
##### [statgen](https://github.com/statgen)/[locuszoom](https://github.com/statgen/locuszoom)

ID = 164, Stars = 84, Language = JavaScript, Update = 2020-05-08

A Javascript/d3 embeddable plugin for interactively visualizing statistical genetic data from customizable sources.
##### [broadinstitute](https://github.com/broadinstitute)/[mutect](https://github.com/broadinstitute/mutect)

ID = 165, Stars = 84, Language = Java, Update = 2015-08-26

MuTect -- Accurate and sensitive cancer mutation detection
##### [openvax](https://github.com/openvax)/[mhcflurry](https://github.com/openvax/mhcflurry)

ID = 166, Stars = 83, Language = Python, Update = 2020-05-06

Peptide-MHC I binding affinity prediction
##### [daehwankimlab](https://github.com/daehwankimlab)/[tophat](https://github.com/DaehwanKimLab/tophat)

ID = 167, Stars = 83, Language = C++, Update = 2018-05-24

Spliced read mapper for RNA-Seq
##### [biocore](https://github.com/biocore)/[qiita](https://github.com/biocore/qiita)

ID = 168, Stars = 83, Language = Python, Update = 2020-05-08

Qiita - A multi-omics databasing effort
##### [danielecook](https://github.com/danielecook)/[Quiver-alfred](https://github.com/danielecook/Quiver-alfred)

ID = 169, Stars = 81, Language = Python, Update = 2018-08-22

Search Quiver from Alfred
##### [brentp](https://github.com/brentp)/[slivar](https://github.com/brentp/slivar)

ID = 170, Stars = 80, Language = Nim, Update = 2020-05-06

variant expressions, annotation, and filtering for great good.
##### [lh3](https://github.com/lh3)/[psmc](https://github.com/lh3/psmc)

ID = 171, Stars = 79, Language = C, Update = 2016-01-21

Implementation of the Pairwise Sequentially Markovian Coalescent (PSMC) model
##### [gt1](https://github.com/gt1)/[biobambam2](https://github.com/gt1/biobambam2)

ID = 172, Stars = 79, Language = C++, Update = 2019-03-12

Tools for early stage alignment file processing
##### [hall-lab](https://github.com/hall-lab)/[svtools](https://github.com/hall-lab/svtools)

ID = 173, Stars = 79, Language = Python, Update = 2019-10-24

Tools for processing and analyzing structural variants.
##### [biocore](https://github.com/biocore)/[sortmerna](https://github.com/biocore/sortmerna)

ID = 174, Stars = 79, Language = C++, Update = 2020-04-20

SortMeRNA: next-generation sequence filtering and alignment tool
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[GenomicConsensus](https://github.com/PacificBiosciences/GenomicConsensus)

ID = 175, Stars = 79, Language = Python, Update = 2019-04-25

PacBio¬Æ variant and consensus caller
##### [tseemann](https://github.com/tseemann)/[nullarbor](https://github.com/tseemann/nullarbor)

ID = 176, Stars = 78, Language = Perl, Update = 2020-03-16

üíæ
üìÉ
"Reads to report" for public health and clinical microbiology
##### [hall-lab](https://github.com/hall-lab)/[svtyper](https://github.com/hall-lab/svtyper)

ID = 177, Stars = 78, Language = Python, Update = 2020-02-27

Bayesian genotyper for structural variants
##### [lh3](https://github.com/lh3)/[bgt](https://github.com/lh3/bgt)

ID = 178, Stars = 77, Language = C, Update = 2019-09-04

Flexible genotype query among 30,000+ samples whole-genome
##### [lh3](https://github.com/lh3)/[ksw2](https://github.com/lh3/ksw2)

ID = 179, Stars = 77, Language = C, Update = 2019-04-29

Global alignment and alignment extension
##### [brentp](https://github.com/brentp)/[pyfasta](https://github.com/brentp/pyfasta)

ID = 180, Stars = 77, Language = Python, Update = 2017-12-11

fast, memory-efficient, pythonic (and command-line) access to fasta sequence files
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[VarDictJava](https://github.com/AstraZeneca-NGS/VarDictJava)

ID = 181, Stars = 77, Language = Java, Update = 2020-05-08

VarDict Java port
##### [genome](https://github.com/genome)/[sciclone](https://github.com/genome/sciclone)

ID = 182, Stars = 77, Language = R, Update = 2017-03-08

An R package for inferring the subclonal architecture of tumors
##### [danielecook](https://github.com/danielecook)/[gist-alfred](https://github.com/danielecook/gist-alfred)

ID = 183, Stars = 76, Language = Python, Update = 2019-11-14

An alfred workflow for accessing gists
##### [arq5x](https://github.com/arq5x)/[grabix](https://github.com/arq5x/grabix)

ID = 184, Stars = 76, Language = C, Update = 2018-05-10

a wee tool for random access into BGZF files.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pb-assembly](https://github.com/PacificBiosciences/pb-assembly)

ID = 185, Stars = 76, Language = Python, Update = 2019-11-17

PacBio Assembly Tool Suite: Reads in ‚á® Assembly out
##### [broadinstitute](https://github.com/broadinstitute)/[wot](https://github.com/broadinstitute/wot)

ID = 186, Stars = 75, Language = Jupyter Notebook, Update = 2020-04-22

A software package for analyzing snapshots of developmental processes
##### [dpryan79](https://github.com/dpryan79)/[MethylDackel](https://github.com/dpryan79/MethylDackel)

ID = 187, Stars = 74, Language = C, Update = 2020-01-30

A (mostly) universal methylation extractor for BS-seq experiments.
##### [benjjneb](https://github.com/benjjneb)/[decontam](https://github.com/benjjneb/decontam)

ID = 188, Stars = 74, Language = R, Update = 2019-10-07

Simple statistical identification and removal of contaminants in marker-gene and metagenomics sequencing data
##### [kamilsjaron](https://github.com/kamilsjaron)/[smudgeplot](https://github.com/KamilSJaron/smudgeplot)

ID = 189, Stars = 74, Language = R, Update = 2020-04-24

Inference of ploidy and heterozygosity structure using whole genome sequencing data
##### [tseemann](https://github.com/tseemann)/[barrnap](https://github.com/tseemann/barrnap)

ID = 190, Stars = 74, Language = Perl, Update = 2018-10-29

üî¨
‚ôå
Bacterial ribosomal RNA predictor
##### [andersenlab](https://github.com/andersenlab)/[VCF-kit](https://github.com/AndersenLab/VCF-kit)

ID = 191, Stars = 73, Language = Python, Update = 2020-02-11

VCF-kit: Assorted utilities for the variant call format
##### [broadinstitute](https://github.com/broadinstitute)/[ichorCNA](https://github.com/broadinstitute/ichorCNA)

ID = 192, Stars = 73, Language = R, Update = 2020-05-06

Estimating tumor fraction in cell-free DNA from ultra-low-pass whole genome sequencing.
##### [ncbi](https://github.com/ncbi)/[robotframework-pageobjects](https://github.com/ncbi/robotframework-pageobjects)

ID = 193, Stars = 73, Language = Python, Update = 2018-11-30

Implementation of the Page Object pattern with Robot Framework and selenium. Also facilitates page object pattern independent of Robot Framework
##### [ming-lian](https://github.com/ming-lian)/[Machine-Learning-Course-in-UCAS](https://github.com/Ming-Lian/Machine-Learning-Course-in-UCAS)

ID = 194, Stars = 72, Language = None, Update = 2019-09-24

ÂõΩÁßëÂ§ßÊú∫Âô®Â≠¶‰π†ËØæÁ®ã&Êú∫Âô®Â≠¶‰π†Á¨îËÆ∞
##### [samtools](https://github.com/samtools)/[tabix](https://github.com/samtools/tabix)

ID = 195, Stars = 72, Language = C, Update = 2019-11-14

Note: tabix and bgzip binaries are now part of the HTSlib project.
##### [richarddurbin](https://github.com/richarddurbin)/[pbwt](https://github.com/richarddurbin/pbwt)

ID = 196, Stars = 71, Language = C, Update = 2019-07-14

Implementation of Positional Burrows-Wheeler Transform for genetic data
##### [brentp](https://github.com/brentp)/[combat.py](https://github.com/brentp/combat.py)

ID = 197, Stars = 71, Language = Python, Update = 2018-12-31

python / numpy / pandas / patsy version of ComBat for removing batch effects.
##### [brentp](https://github.com/brentp)/[bwa-meth](https://github.com/brentp/bwa-meth)

ID = 198, Stars = 71, Language = PostScript, Update = 2018-08-07

fast and accurate alignment of BS-Seq reads
##### [illumina](https://github.com/illumina)/[Nirvana](https://github.com/Illumina/Nirvana)

ID = 199, Stars = 71, Language = C#, Update = 2020-01-16

The nimble & robust variant annotator
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-data-processing](https://github.com/gatk-workflows/gatk4-data-processing)

ID = 200, Stars = 70, Language = wdl, Update = 2020-04-02

Workflows for processing high-throughput sequencing data for variant discovery with GATK4 and related tools
##### [marianattestad](https://github.com/marianattestad)/[Assemblytics](https://github.com/MariaNattestad/Assemblytics)

ID = 201, Stars = 69, Language = JavaScript, Update = 2020-01-20

Assemblytics is a bioinformatics tool to detect and analyze structural variants from a genome assembly by comparing it to a reference genome.
##### [ensembl](https://github.com/ensembl)/[VEP_plugins](https://github.com/Ensembl/VEP_plugins)

ID = 202, Stars = 69, Language = Perl, Update = 2020-05-04

Plugins for the Ensembl Variant Effect Predictor (VEP)
##### [lh3](https://github.com/lh3)/[fermi](https://github.com/lh3/fermi)

ID = 203, Stars = 68, Language = C, Update = 2013-12-06

A WGS de novo assembler based on the FMD-index for large genomes
##### [daler](https://github.com/daler)/[metaseq](https://github.com/daler/metaseq)

ID = 204, Stars = 68, Language = Python, Update = 2017-01-10

Framework for integrated analysis and plotting of ChIP/RIP/RNA/*-seq data
##### [cschin](https://github.com/cschin)/[Peregrine](https://github.com/cschin/Peregrine)

ID = 205, Stars = 68, Language = Jupyter Notebook, Update = 2020-05-03

Peregrine: Fast Genome Assembler Using SHIMMER Index
##### [ewels](https://github.com/ewels)/[sra-explorer](https://github.com/ewels/sra-explorer)

ID = 206, Stars = 68, Language = HTML, Update = 2020-01-20

Web application to explore the Sequence Read Archive.
##### [statgen](https://github.com/statgen)/[demuxlet](https://github.com/statgen/demuxlet)

ID = 207, Stars = 68, Language = C++, Update = 2020-03-25

Genetic multiplexing of barcoded single cell RNA-seq
##### [opengene](https://github.com/opengene)/[repaq](https://github.com/OpenGene/repaq)

ID = 208, Stars = 68, Language = C, Update = 2020-03-16

A fast lossless FASTQ compressor with ultra-high compression ratio
##### [biocore](https://github.com/biocore)/[deblur](https://github.com/biocore/deblur)

ID = 209, Stars = 68, Language = Python, Update = 2020-04-14

Deblur is a greedy deconvolution algorithm based on known read error profiles.
##### [arq5x](https://github.com/arq5x)/[bedtools-protocols](https://github.com/arq5x/bedtools-protocols)

ID = 210, Stars = 67, Language = CSS, Update = 2014-03-06

None
##### [tseemann](https://github.com/tseemann)/[mlst](https://github.com/tseemann/mlst)

ID = 211, Stars = 67, Language = Perl, Update = 2020-02-24

üÜî
Scan contig files against PubMLST typing schemes
##### [broadinstitute](https://github.com/broadinstitute)/[ssGSEA2.0](https://github.com/broadinstitute/ssGSEA2.0)

ID = 212, Stars = 67, Language = R, Update = 2020-03-20

Single sample Gene Set Enrichment analysis (ssGSEA) and PTM Enrichment Analysis (PTM-SEA)
##### [umcugenetics](https://github.com/umcugenetics)/[MutationalPatterns](https://github.com/UMCUGenetics/MutationalPatterns)

ID = 213, Stars = 67, Language = R, Update = 2020-04-01

R package for extracting and visualizing mutational patterns in base substitution catalogues
##### [genome](https://github.com/genome)/[analysis-workflows](https://github.com/genome/analysis-workflows)

ID = 214, Stars = 67, Language = Common Workflow Language, Update = 2020-04-24

Open workflow definitions for genomic analysis from MGI at WUSM.
##### [illumina](https://github.com/illumina)/[ExpansionHunter](https://github.com/Illumina/ExpansionHunter)

ID = 215, Stars = 66, Language = C++, Update = 2020-02-25

A tool for estimating repeat sizes
##### [brentp](https://github.com/brentp)/[gargs](https://github.com/brentp/gargs)

ID = 216, Stars = 64, Language = Go, Update = 2018-01-26

better(?) xargs in go
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-germline-snps-indels](https://github.com/gatk-workflows/gatk4-germline-snps-indels)

ID = 217, Stars = 64, Language = wdl, Update = 2020-03-06

Workflows for germline short variant discovery with GATK4
##### [mroosmalen](https://github.com/mroosmalen)/[nanosv](https://github.com/mroosmalen/nanosv)

ID = 218, Stars = 63, Language = Python, Update = 2019-04-09

SV caller for nanopore data
##### [nf-core](https://github.com/nf-core)/[tools](https://github.com/nf-core/tools)

ID = 219, Stars = 63, Language = Python, Update = 2020-05-04

Python package with helper tools for the nf-core community.
##### [bioinform](https://github.com/bioinform)/[varsim](https://github.com/bioinform/varsim)

ID = 220, Stars = 63, Language = Java, Update = 2020-03-04

VarSim: A high-fidelity simulation validation framework for high-throughput genome sequencing with cancer applications
##### [opengene](https://github.com/opengene)/[gencore](https://github.com/OpenGene/gencore)

ID = 221, Stars = 63, Language = C++, Update = 2020-03-18

Generate consensus reads to reduce sequencing noises and remove duplications
##### [lh3](https://github.com/lh3)/[fermi-lite](https://github.com/lh3/fermi-lite)

ID = 222, Stars = 62, Language = C, Update = 2020-03-30

Standalone C library for assembling Illumina short reads in small regions
##### [dzerbino](https://github.com/dzerbino)/[ascii_plots](https://github.com/dzerbino/ascii_plots)

ID = 223, Stars = 62, Language = Python, Update = 2017-10-30

Convenience function for quick and dirty data analysis
##### [statgen](https://github.com/statgen)/[bamUtil](https://github.com/statgen/bamUtil)

ID = 224, Stars = 62, Language = C++, Update = 2019-04-20

None
##### [broadinstitute](https://github.com/broadinstitute)/[Drop-seq](https://github.com/broadinstitute/Drop-seq)

ID = 225, Stars = 62, Language = Java, Update = 2020-05-05

Java tools for analyzing Drop-seq data
##### [ding-lab](https://github.com/ding-lab)/[CharGer](https://github.com/ding-lab/CharGer)

ID = 226, Stars = 61, Language = Python, Update = 2020-04-10

Characterization of Germline variants
##### [ncbi](https://github.com/ncbi)/[ncbi-vdb](https://github.com/ncbi/ncbi-vdb)

ID = 227, Stars = 61, Language = C, Update = 2020-05-08

ncbi-vdb
##### [brentp](https://github.com/brentp)/[slurmpy](https://github.com/brentp/slurmpy)

ID = 228, Stars = 60, Language = Python, Update = 2020-04-08

submit jobs to slurm with quick-and-dirty python
##### [mhammell-laboratory](https://github.com/mhammell-laboratory)/[TEtranscripts](https://github.com/mhammell-laboratory/TEtranscripts)

ID = 229, Stars = 60, Language = Python, Update = 2020-04-24

A package for including transposable elements in differential enrichment analysis of sequencing datasets.
##### [biocore](https://github.com/biocore)/[biom-format](https://github.com/biocore/biom-format)

ID = 230, Stars = 60, Language = Python, Update = 2020-03-30

The Biological Observation Matrix (BIOM) Format Project
##### [mozack](https://github.com/mozack)/[abra](https://github.com/mozack/abra)

ID = 231, Stars = 59, Language = Java, Update = 2018-05-24

Assembly Based ReAligner
##### [lh3](https://github.com/lh3)/[gfatools](https://github.com/lh3/gfatools)

ID = 232, Stars = 59, Language = C, Update = 2020-03-09

Tools for manipulating sequence graphs in the GFA and rGFA formats
##### [ekg](https://github.com/ekg)/[seqwish](https://github.com/ekg/seqwish)

ID = 233, Stars = 59, Language = C++, Update = 2020-04-29

alignment to variation graph inducer
##### [brentp](https://github.com/brentp)/[duphold](https://github.com/brentp/duphold)

ID = 234, Stars = 59, Language = Nim, Update = 2020-04-13

don't get DUP'ed or DEL'ed by your putative SVs.
##### [bioinform](https://github.com/bioinform)/[rnacocktail](https://github.com/bioinform/rnacocktail)

ID = 235, Stars = 59, Language = Jupyter Notebook, Update = 2019-11-23

None
##### [opengene](https://github.com/opengene)/[OpenGene.jl](https://github.com/OpenGene/OpenGene.jl)

ID = 236, Stars = 59, Language = Julia, Update = 2017-10-08

(No maintenance) OpenGene, core libraries for NGS data analysis and bioinformatics in Julia
##### [illumina](https://github.com/illumina)/[Pisces](https://github.com/Illumina/Pisces)

ID = 237, Stars = 59, Language = C#, Update = 2019-08-09

Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows.
##### [lh3](https://github.com/lh3)/[hickit](https://github.com/lh3/hickit)

ID = 238, Stars = 58, Language = C, Update = 2019-05-13

TAD calling, phase imputation, 3D modeling and more for diploid single-cell Hi-C (Dip-C) and general Hi-C
##### [lh3](https://github.com/lh3)/[bfc](https://github.com/lh3/bfc)

ID = 239, Stars = 58, Language = TeX, Update = 2016-05-31

High-performance error correction for Illumina resequencing data
##### [bioconda](https://github.com/bioconda)/[bioconda-utils](https://github.com/bioconda/bioconda-utils)

ID = 240, Stars = 58, Language = Python, Update = 2020-05-09

Utilities for building and managing bioconda recipes
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbmm2](https://github.com/PacificBiosciences/pbmm2)

ID = 241, Stars = 58, Language = C++, Update = 2020-05-04

A minimap2 frontend for PacBio native data formats
##### [humanlongevity](https://github.com/humanlongevity)/[HLA](https://github.com/humanlongevity/HLA)

ID = 242, Stars = 57, Language = Jupyter Notebook, Update = 2018-10-11

xHLA: Fast and accurate HLA typing from short read sequence data
##### [broadinstitute](https://github.com/broadinstitute)/[oncotator](https://github.com/broadinstitute/oncotator)

ID = 243, Stars = 56, Language = Python, Update = 2020-02-26

None
##### [parklab](https://github.com/parklab)/[NGSCheckMate](https://github.com/parklab/NGSCheckMate)

ID = 244, Stars = 56, Language = Python, Update = 2020-03-20

Software program for checking sample matching for NGS data
##### [opengene](https://github.com/opengene)/[GeneFuse](https://github.com/OpenGene/GeneFuse)

ID = 245, Stars = 56, Language = C, Update = 2020-01-21

Gene fusion detection and visualization
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[IsoSeq_SA3nUP](https://github.com/PacificBiosciences/IsoSeq_SA3nUP)

ID = 246, Stars = 56, Language = None, Update = 2019-12-05

Documentation and Developers Code Base for Iso-Seq SMRTAnalysis 3.x and beyond
##### [ncbi](https://github.com/ncbi)/[TPMCalculator](https://github.com/ncbi/TPMCalculator)

ID = 247, Stars = 56, Language = C++, Update = 2020-05-01

TPMCalculator quantifies mRNA abundance directly from the alignments by parsing BAM files
##### [crazyhottommy](https://github.com/crazyhottommy)/[pyflow-ChIPseq](https://github.com/crazyhottommy/pyflow-ChIPseq)

ID = 248, Stars = 55, Language = Python, Update = 2020-04-17

a snakemake pipeline to process ChIP-seq files from GEO or in-house
##### [cschin](https://github.com/cschin)/[DCNet](https://github.com/cschin/DCNet)

ID = 249, Stars = 55, Language = Jupyter Notebook, Update = 2018-04-16

DCNet‚Ää‚Äî‚ÄäDenoising (DNA) Sequence With a LSTM-RNN and¬†PyTorch & Neural DBG
##### [parklab](https://github.com/parklab)/[nozzle](https://github.com/parklab/nozzle)

ID = 250, Stars = 55, Language = R, Update = 2016-04-02

Nozzle is a report generation toolkit for data analysis pipelines implemented in R.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbbioconda](https://github.com/PacificBiosciences/pbbioconda)

ID = 251, Stars = 55, Language = None, Update = 2020-02-16

PacBio Secondary Analysis Tools on Bioconda
##### [dzerbino](https://github.com/dzerbino)/[oases](https://github.com/dzerbino/oases)

ID = 252, Stars = 54, Language = C, Update = 2018-03-12

De novo transcriptome assembler for short reads
##### [ekg](https://github.com/ekg)/[fastahack](https://github.com/ekg/fastahack)

ID = 253, Stars = 54, Language = C++, Update = 2020-01-06

utilities for indexing and sequence extraction from FASTA files
##### [tseemann](https://github.com/tseemann)/[any2fasta](https://github.com/tseemann/any2fasta)

ID = 254, Stars = 54, Language = Perl, Update = 2019-05-17

Convert various sequence formats to FASTA
##### [statgen](https://github.com/statgen)/[SLURM-examples](https://github.com/statgen/SLURM-examples)

ID = 255, Stars = 54, Language = Python, Update = 2019-06-26

None
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-somatic-snvs-indels](https://github.com/gatk-workflows/gatk4-somatic-snvs-indels)

ID = 256, Stars = 54, Language = wdl, Update = 2020-03-09

This repo will be archived soon, these workflows will be housed in the GATK repository under the scripts directory. These workflows are also organized in Dockstore in the GATK Best Practices Workflows collection.
##### [nanopore-wgs-consortium](https://github.com/nanopore-wgs-consortium)/[CHM13](https://github.com/nanopore-wgs-consortium/CHM13)

ID = 257, Stars = 54, Language = None, Update = 2020-05-08

Ultra-long reads for CHM13 genome assembly
##### [ncbi](https://github.com/ncbi)/[dbsnp](https://github.com/ncbi/dbsnp)

ID = 258, Stars = 54, Language = Jupyter Notebook, Update = 2020-04-22

dbSNP
##### [benlangmead](https://github.com/benlangmead)/[langmead-lab](https://github.com/BenLangmead/langmead-lab)

ID = 259, Stars = 53, Language = None, Update = 2015-05-28

Publicly-visible Langmead Lab resources
##### [statgen](https://github.com/statgen)/[pheweb](https://github.com/statgen/pheweb)

ID = 260, Stars = 53, Language = Python, Update = 2020-05-09

A tool to build multi-phenotype GWAS browsers
##### [yongxinliu](https://github.com/yongxinliu)/[Note](https://github.com/YongxinLiu/Note)

ID = 261, Stars = 52, Language = HTML, Update = 2020-05-10

None
##### [mozack](https://github.com/mozack)/[abra2](https://github.com/mozack/abra2)

ID = 262, Stars = 52, Language = Java, Update = 2020-04-23

ABRA2
##### [lh3](https://github.com/lh3)/[bedtk](https://github.com/lh3/bedtk)

ID = 263, Stars = 52, Language = C, Update = 2020-01-27

A simple toolset for BED files (warning: CLI may change before bedtk becomes stable)
##### [ocxtal](https://github.com/ocxtal)/[libgaba](https://github.com/ocxtal/libgaba)

ID = 264, Stars = 52, Language = C, Update = 2018-05-04

Adaptive semi-global banded alignment on string graphs
##### [illumina](https://github.com/illumina)/[PrimateAI](https://github.com/Illumina/PrimateAI)

ID = 265, Stars = 52, Language = Python, Update = 2019-02-26

deep residual neural network for classifying the pathogenicity of missense mutations.
##### [lh3](https://github.com/lh3)/[dna-nn](https://github.com/lh3/dna-nn)

ID = 266, Stars = 51, Language = C, Update = 2019-06-16

Model and predict short DNA sequence features with neural networks
##### [nf-core](https://github.com/nf-core)/[chipseq](https://github.com/nf-core/chipseq)

ID = 267, Stars = 51, Language = Nextflow, Update = 2020-04-20

ChIP-seq peak-calling, QC and differential analysis pipeline.
##### [ensembl](https://github.com/ensembl)/[ensembl](https://github.com/Ensembl/ensembl)

ID = 268, Stars = 51, Language = Perl, Update = 2020-05-06

The Ensembl Core Perl API and SQL schema
##### [dpryan79](https://github.com/dpryan79)/[libBigWig](https://github.com/dpryan79/libBigWig)

ID = 269, Stars = 50, Language = C, Update = 2020-03-04

A C library for handling bigWig files
##### [arq5x](https://github.com/arq5x)/[cyvcf](https://github.com/arq5x/cyvcf)

ID = 270, Stars = 50, Language = Python, Update = 2018-03-27

A fast Python library for VCF files leveraging Cython for speed.
##### [openvax](https://github.com/openvax)/[varcode](https://github.com/openvax/varcode)

ID = 271, Stars = 50, Language = Python, Update = 2019-11-08

Library for manipulating genomic variants and predicting their effects
##### [wglab](https://github.com/wglab)/[DeepMod](https://github.com/WGLab/DeepMod)

ID = 272, Stars = 50, Language = Python, Update = 2020-01-02

DeepMod: a deep-learning tool for genomic-scale, strand-sensitive and single-nucleotide based detection of DNA modifications
##### [ncbi](https://github.com/ncbi)/[ngs-tools](https://github.com/ncbi/ngs-tools)

ID = 273, Stars = 50, Language = C++, Update = 2020-04-22

None
##### [ewels](https://github.com/ewels)/[AWS-iGenomes](https://github.com/ewels/AWS-iGenomes)

ID = 274, Stars = 49, Language = HTML, Update = 2019-11-06

Documentation and description of AWS iGenomes S3 resource.
##### [brentp](https://github.com/brentp)/[xopen](https://github.com/brentp/xopen)

ID = 275, Stars = 49, Language = Go, Update = 2018-11-16

open files for buffered reading and writing in #golang
##### [biocore](https://github.com/biocore)/[emperor](https://github.com/biocore/emperor)

ID = 276, Stars = 49, Language = JavaScript, Update = 2020-05-04

Emperor a tool for the analysis and visualization of large microbial ecology datasets
##### [gavinha](https://github.com/gavinha)/[TitanCNA](https://github.com/gavinha/TitanCNA)

ID = 277, Stars = 48, Language = R, Update = 2020-03-02

Analysis of subclonal copy number alterations (CNA) and loss of heterozygosity (LOH) in cancer
##### [lh3](https://github.com/lh3)/[ropebwt2](https://github.com/lh3/ropebwt2)

ID = 278, Stars = 48, Language = TeX, Update = 2020-04-17

Incremental construction of FM-index for DNA sequences
##### [lh3](https://github.com/lh3)/[tabtk](https://github.com/lh3/tabtk)

ID = 279, Stars = 48, Language = C, Update = 2016-09-08

Toolkit for processing TAB-delimited format
##### [quinlan-lab](https://github.com/quinlan-lab)/[vcf2db](https://github.com/quinlan-lab/vcf2db)

ID = 280, Stars = 48, Language = Python, Update = 2020-02-24

create a gemini-compatible database from a VCF
##### [yanglab](https://github.com/yanglab)/[CIRCexplorer](https://github.com/YangLab/CIRCexplorer)

ID = 281, Stars = 48, Language = Python, Update = 2019-06-01

A combined strategy to identify circular RNAs (circRNAs and ciRNAs) (Zhang et al., Complementary Sequence-Mediated Exon Circularization, Cell (2014), 159:134-147)
##### [ensembl](https://github.com/ensembl)/[ensembl-pipeline](https://github.com/Ensembl/ensembl-pipeline)

ID = 282, Stars = 48, Language = Perl, Update = 2020-01-30

*DEPRECATED* Job management for the Ensembl Genebuild pipeline
##### [dutilh](https://github.com/dutilh)/[CAT](https://github.com/dutilh/CAT)

ID = 283, Stars = 47, Language = Python, Update = 2020-04-08

CAT/BAT: tool for taxonomic classification of contigs and metagenome-assembled genomes (MAGs)
##### [marianattestad](https://github.com/marianattestad)/[SplitThreader](https://github.com/MariaNattestad/SplitThreader)

ID = 284, Stars = 47, Language = JavaScript, Update = 2017-02-03

Explore rearrangements and copy-number amplifications in a cancer genome
##### [ewels](https://github.com/ewels)/[CPT-Bootstrap-Carousel](https://github.com/ewels/CPT-Bootstrap-Carousel)

ID = 285, Stars = 47, Language = PHP, Update = 2020-04-17

WordPress plugin which generates a custom post type for choosing images and content. Outputs Bootstrap Image Carousel (slider) HTML from a shortcode.
##### [yanglab](https://github.com/yanglab)/[CIRCexplorer2](https://github.com/YangLab/CIRCexplorer2)

ID = 286, Stars = 47, Language = Python, Update = 2020-02-25

circular RNA analysis toolset
##### [ga4gh](https://github.com/ga4gh)/[workflow-execution-service-schemas](https://github.com/ga4gh/workflow-execution-service-schemas)

ID = 287, Stars = 47, Language = Python, Update = 2020-05-10

The WES API is a standard way to run and manage portable workflows.
##### [gt1](https://github.com/gt1)/[biobambam](https://github.com/gt1/biobambam)

ID = 288, Stars = 46, Language = C++, Update = 2015-04-20

Tools for bam file processing
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[ccs](https://github.com/PacificBiosciences/ccs)

ID = 289, Stars = 46, Language = None, Update = 2020-04-20

CCS: Generate Highly Accurate Single-Molecule Consensus Reads (HiFi Reads)
##### [ncbi](https://github.com/ncbi)/[SKESA](https://github.com/ncbi/SKESA)

ID = 290, Stars = 46, Language = C++, Update = 2020-03-14

SKESA assembler
##### [daler](https://github.com/daler)/[matplotlibrc](https://github.com/daler/matplotlibrc)

ID = 291, Stars = 45, Language = Python, Update = 2013-11-05

some example matplotlibrc files, and a script display their effects
##### [arq5x](https://github.com/arq5x)/[ggd](https://github.com/arq5x/ggd)

ID = 292, Stars = 45, Language = Python, Update = 2016-04-20

None
##### [tanghaibao](https://github.com/tanghaibao)/[bio-pipeline](https://github.com/tanghaibao/bio-pipeline)

ID = 293, Stars = 45, Language = C, Update = 2020-04-05

My collection of light bioinformatics analysis pipelines for specific tasks
##### [wglab](https://github.com/wglab)/[PennCNV](https://github.com/WGLab/PennCNV)

ID = 294, Stars = 45, Language = C, Update = 2019-12-05

Copy number vaiation detection from SNP arrays
##### [illumina](https://github.com/illumina)/[interop](https://github.com/Illumina/interop)

ID = 295, Stars = 45, Language = C++, Update = 2020-04-18

C++ Library to parse Illumina InterOp files
##### [pcingola](https://github.com/pcingola)/[jFuzzyLogic](https://github.com/pcingola/jFuzzyLogic)

ID = 296, Stars = 44, Language = Java, Update = 2018-07-17

The most complete fuzzy logic library in Java. The de-facto standard for research and industrial applications.
##### [tanghaibao](https://github.com/tanghaibao)/[quota-alignment](https://github.com/tanghaibao/quota-alignment)

ID = 297, Stars = 44, Language = Python, Update = 2017-09-05

Guided synteny alignment between duplicated genomes (within specified quota constraint)
##### [brentp](https://github.com/brentp)/[vcfgo](https://github.com/brentp/vcfgo)

ID = 298, Stars = 44, Language = Go, Update = 2020-03-14

a golang library to read, write and manipulate files in the variant call format.
##### [brentp](https://github.com/brentp)/[hts-python](https://github.com/brentp/hts-python)

ID = 299, Stars = 44, Language = Python, Update = 2017-03-03

pythonic wrapper for libhts (moved to: https://github.com/quinlan-lab/hts-python)
##### [broadinstitute](https://github.com/broadinstitute)/[tensorqtl](https://github.com/broadinstitute/tensorqtl)

ID = 300, Stars = 44, Language = Python, Update = 2020-04-15

Ultrafast GPU-based QTL mapper
##### [refresh-bio](https://github.com/refresh-bio)/[kmer-db](https://github.com/refresh-bio/kmer-db)

ID = 301, Stars = 44, Language = C++, Update = 2020-03-30

None
##### [ncbi](https://github.com/ncbi)/[amr](https://github.com/ncbi/amr)

ID = 302, Stars = 44, Language = C++, Update = 2020-05-01

amr_finder
##### [lh3](https://github.com/lh3)/[samtools](https://github.com/lh3/samtools)

ID = 303, Stars = 43, Language = C, Update = 2017-06-09

This is *NOT* the official repository of samtools.
##### [arq5x](https://github.com/arq5x)/[filo](https://github.com/arq5x/filo)

ID = 304, Stars = 43, Language = C++, Update = 2015-12-21

Useful FILe and stream Operations
##### [tseemann](https://github.com/tseemann)/[snp-dists](https://github.com/tseemann/snp-dists)

ID = 305, Stars = 43, Language = C, Update = 2020-03-01

Pairwise SNP distance matrix from a FASTA sequence alignment
##### [nf-core](https://github.com/nf-core)/[rnafusion](https://github.com/nf-core/rnafusion)

ID = 306, Stars = 43, Language = Nextflow, Update = 2020-05-08

RNA-seq analysis pipeline for detection gene-fusions
##### [nf-core](https://github.com/nf-core)/[methylseq](https://github.com/nf-core/methylseq)

ID = 307, Stars = 43, Language = Nextflow, Update = 2020-04-28

Methylation (Bisulfite-Sequencing) analysis pipeline using Bismark or bwa-meth + MethylDackel
##### [biod](https://github.com/biod)/[BioD](https://github.com/biod/BioD)

ID = 308, Stars = 43, Language = D, Update = 2019-12-17

A D library for computational biology and bioinformatics
##### [genome](https://github.com/genome)/[genome](https://github.com/genome/genome)

ID = 309, Stars = 43, Language = Perl, Update = 2020-04-06

Core modules used by the GMS
##### [illumina](https://github.com/illumina)/[akt](https://github.com/Illumina/akt)

ID = 310, Stars = 43, Language = C++, Update = 2018-07-16

Ancestry and Kinship Tools
##### [illumina](https://github.com/illumina)/[PlatinumGenomes](https://github.com/Illumina/PlatinumGenomes)

ID = 311, Stars = 43, Language = None, Update = 2017-11-08

The Platinum Genomes Truthset
##### [crazyhottommy](https://github.com/crazyhottommy)/[DNA-methylation-analysis](https://github.com/crazyhottommy/DNA-methylation-analysis)

ID = 312, Stars = 42, Language = None, Update = 2019-09-19

DNA methylation analysis notes from Ming Tang
##### [seandavi](https://github.com/seandavi)/[GEOquery](https://github.com/seandavi/GEOquery)

ID = 313, Stars = 42, Language = R, Update = 2019-11-18

The bridge between the NCBI Gene Expression Omnibus and Bioconductor
##### [ewels](https://github.com/ewels)/[MegaQC](https://github.com/ewels/MegaQC)

ID = 314, Stars = 42, Language = Python, Update = 2020-03-17

Web application to collect and visualise data across multiple MultiQC runs.
##### [brentp](https://github.com/brentp)/[go-chartjs](https://github.com/brentp/go-chartjs)

ID = 315, Stars = 42, Language = Go, Update = 2017-09-01

golang library to make https://chartjs.org/ plots (this is vanilla #golang, not gopherjs)
##### [openvax](https://github.com/openvax)/[mhctools](https://github.com/openvax/mhctools)

ID = 316, Stars = 42, Language = Python, Update = 2020-05-01

Python interface to running command-line and web-based MHC binding predictors
##### [vgteam](https://github.com/vgteam)/[odgi](https://github.com/vgteam/odgi)

ID = 317, Stars = 42, Language = C++, Update = 2020-05-09

optimized dynamic genome graph implementation
##### [ding-lab](https://github.com/ding-lab)/[MuSiC2](https://github.com/ding-lab/MuSiC2)

ID = 318, Stars = 42, Language = Perl, Update = 2019-05-19

identifying mutational significance in cancer genomes
##### [papenfusslab](https://github.com/papenfusslab)/[bioshake](https://github.com/PapenfussLab/bioshake)

ID = 319, Stars = 42, Language = Haskell, Update = 2019-08-13

Bioinformatics pipelines with Haskell and Shake
##### [opengene](https://github.com/opengene)/[CfdnaPattern](https://github.com/OpenGene/CfdnaPattern)

ID = 320, Stars = 42, Language = Python, Update = 2018-08-03

Pattern Recognition for Cell-free DNA
##### [biocore](https://github.com/biocore)/[mmvec](https://github.com/biocore/mmvec)

ID = 321, Stars = 42, Language = Python, Update = 2020-04-25

Neural networks for microbe-metabolite interaction analysis
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[IsoSeq](https://github.com/PacificBiosciences/IsoSeq)

ID = 322, Stars = 42, Language = None, Update = 2020-03-18

IsoSeq3 - Scalable De Novo Isoform Discovery from Single-Molecule PacBio Reads
##### [yongxinliu](https://github.com/yongxinliu)/[QIIME2ChineseManual](https://github.com/YongxinLiu/QIIME2ChineseManual)

ID = 323, Stars = 41, Language = JavaScript, Update = 2020-05-07

QIIME 2‰∏≠ÊñáÊñáÊ°£(QIIME 2 Chinese Manual)
##### [brentp](https://github.com/brentp)/[fishers_exact_test](https://github.com/brentp/fishers_exact_test)

ID = 324, Stars = 41, Language = Python, Update = 2019-10-02

Fishers Exact Test for Python (Cython)
##### [brentp](https://github.com/brentp)/[interlap](https://github.com/brentp/interlap)

ID = 325, Stars = 41, Language = Python, Update = 2018-08-31

fast, pure-python interval overlap testing
##### [andersenlab](https://github.com/andersenlab)/[IBiS-Bootcamp](https://github.com/AndersenLab/IBiS-Bootcamp)

ID = 326, Stars = 41, Language = Shell, Update = 2016-09-14

None
##### [refresh-bio](https://github.com/refresh-bio)/[DSRC](https://github.com/refresh-bio/DSRC)

ID = 327, Stars = 41, Language = C++, Update = 2020-04-19

DSRC - DNA Sequence Reads Compressor
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[reference_data](https://github.com/AstraZeneca-NGS/reference_data)

ID = 328, Stars = 41, Language = Python, Update = 2017-11-28

Reference data: BED files, genes, transcripts, variations.
##### [wglab](https://github.com/wglab)/[biocluster](https://github.com/WGLab/biocluster)

ID = 329, Stars = 41, Language = None, Update = 2019-04-14

Tutorial on building a computing cluster for bioinformatics
##### [brentp](https://github.com/brentp)/[irelate](https://github.com/brentp/irelate)

ID = 330, Stars = 40, Language = Go, Update = 2018-08-01

Streaming relation (overlap, distance, KNN) of (any number of) sorted genomic interval sets. #golang
##### [quinlan-lab](https://github.com/quinlan-lab)/[sllobs-biostats](https://github.com/quinlan-lab/sllobs-biostats)

ID = 331, Stars = 40, Language = HTML, Update = 2020-04-28

None
##### [ensembl](https://github.com/ensembl)/[ensembl-hive](https://github.com/Ensembl/ensembl-hive)

ID = 332, Stars = 40, Language = Perl, Update = 2020-05-07

EnsEMBL Hive - a system for creating and running pipelines on a distributed compute resource
##### [ga4gh](https://github.com/ga4gh)/[task-execution-schemas](https://github.com/ga4gh/task-execution-schemas)

ID = 333, Stars = 40, Language = None, Update = 2020-02-05

None
##### [ncbi](https://github.com/ncbi)/[icn3d](https://github.com/ncbi/icn3d)

ID = 334, Stars = 40, Language = JavaScript, Update = 2020-05-08

WebGL-based protein structure viewer
##### [ming-lian](https://github.com/ming-lian)/[Bioinfo_LeetCode](https://github.com/Ming-Lian/Bioinfo_LeetCode)

ID = 335, Stars = 39, Language = Python, Update = 2020-04-11

ËÆ©Êàë‰ª¨‰ª•Âà∑LeetCodeÈ¢òÁõÆÁöÑÂΩ¢ÂºèÔºåÊù•ÁºñÁ®ãËß£ÂÜ≥ÁîüÁâ©‰ø°ÊÅØÂ≠¶ÈóÆÈ¢òÂêßÔºåCome on
##### [freeseek](https://github.com/freeseek)/[gtc2vcf](https://github.com/freeseek/gtc2vcf)

ID = 336, Stars = 39, Language = C, Update = 2020-05-09

Tools to convert Illumina IDAT/BPM/EGT/GTC and Affymetrix CEL/CHP files to VCF
##### [brentp](https://github.com/brentp)/[hts-nim-tools](https://github.com/brentp/hts-nim-tools)

ID = 337, Stars = 39, Language = Nim, Update = 2020-04-21

useful command-line tools written to show-case hts-nim
##### [brentp](https://github.com/brentp)/[indelope](https://github.com/brentp/indelope)

ID = 338, Stars = 39, Language = Nim, Update = 2017-12-03

find large indels (in the blind spot between GATK/freebayes and SV callers)
##### [statgen](https://github.com/statgen)/[libStatGen](https://github.com/statgen/libStatGen)

ID = 339, Stars = 39, Language = C++, Update = 2019-05-22

Useful set of classes for creating statistical genetic programs.
##### [openvax](https://github.com/openvax)/[neoantigen-vaccine-pipeline](https://github.com/openvax/neoantigen-vaccine-pipeline)

ID = 340, Stars = 39, Language = Jupyter Notebook, Update = 2019-12-17

Bioinformatics pipeline for selecting patient-specific cancer neoantigen vaccines
##### [openvax](https://github.com/openvax)/[pepdata](https://github.com/openvax/pepdata)

ID = 341, Stars = 39, Language = Python, Update = 2018-02-26

Python interface to amino acid properties and IEDB
##### [zhou-lab](https://github.com/zhou-lab)/[biscuit](https://github.com/zhou-lab/biscuit)

ID = 342, Stars = 39, Language = C, Update = 2020-04-24

BISulfite-seq CUI Toolkit
##### [nf-core](https://github.com/nf-core)/[ampliseq](https://github.com/nf-core/ampliseq)

ID = 343, Stars = 39, Language = Nextflow, Update = 2020-05-05

16S rRNA amplicon sequencing analysis workflow using QIIME2
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad-browser](https://github.com/broadinstitute/gnomad-browser)

ID = 344, Stars = 39, Language = JavaScript, Update = 2020-05-07

JavaScript tools for exploring genomic data
##### [broadinstitute](https://github.com/broadinstitute)/[docker-terraform](https://github.com/broadinstitute/docker-terraform)

ID = 345, Stars = 39, Language = Dockerfile, Update = 2020-04-17

Docker container for running the Terraform application
##### [broadinstitute](https://github.com/broadinstitute)/[gamgee](https://github.com/broadinstitute/gamgee)

ID = 346, Stars = 39, Language = C++, Update = 2019-08-05

A C++14 library for NGS data formats
##### [gatk-workflows](https://github.com/gatk-workflows)/[broad-prod-wgs-germline-snps-indels](https://github.com/gatk-workflows/broad-prod-wgs-germline-snps-indels)

ID = 347, Stars = 39, Language = wdl, Update = 2020-03-19

Workflows for germline short variant discovery in WGS data. PairedEndSingleSampleWf has been superseded in Broad production by https://github.com/gatk-workflows/five-dollar-genome-analysis-pipeline
##### [wglab](https://github.com/wglab)/[phenolyzer](https://github.com/WGLab/phenolyzer)

ID = 348, Stars = 39, Language = Perl, Update = 2019-12-04

phenotype-based prioritization of candidate genes for human diseases
##### [danielecook](https://github.com/danielecook)/[iex-api-python](https://github.com/danielecook/iex-api-python)

ID = 349, Stars = 38, Language = Python, Update = 2018-10-23

A python wrapper for the IEX API
##### [ewels](https://github.com/ewels)/[nf-core-rnaseq](https://github.com/ewels/nf-core-rnaseq)

ID = 350, Stars = 38, Language = Nextflow, Update = 2020-02-19

See the main fork of this repository here >>>
##### [tseemann](https://github.com/tseemann)/[VelvetOptimiser](https://github.com/tseemann/VelvetOptimiser)

ID = 351, Stars = 38, Language = Perl, Update = 2017-12-21

üìà
Automatically optimise three of Velvet's assembly parameters.
##### [bioinform](https://github.com/bioinform)/[metasv](https://github.com/bioinform/metasv)

ID = 352, Stars = 38, Language = Python, Update = 2017-06-30

MetaSV: An accurate and integrative structural-variant caller for next generation sequencing
##### [mathonco](https://github.com/mathonco)/[NeoPredPipe](https://github.com/MathOnco/NeoPredPipe)

ID = 353, Stars = 38, Language = Python, Update = 2020-04-17

Neoantigens prediction pipeline for multi- or single-region vcf files using ANNOVAR and netMHCpan.
##### [ga4gh](https://github.com/ga4gh)/[vr-spec](https://github.com/ga4gh/vr-spec)

ID = 354, Stars = 38, Language = Makefile, Update = 2020-05-08

Extensible specification for representing and uniquely identifying biological sequence variation
##### [crazyhottommy](https://github.com/crazyhottommy)/[pyflow-ATACseq](https://github.com/crazyhottommy/pyflow-ATACseq)

ID = 355, Stars = 37, Language = Python, Update = 2020-02-13

ATAC-seq snakemake pipeline
##### [benlangmead](https://github.com/benlangmead)/[crossbow](https://github.com/BenLangmead/crossbow)

ID = 356, Stars = 37, Language = Perl, Update = 2014-01-14

Variant calling from sequence reads using cloud computing
##### [daler](https://github.com/daler)/[trackhub](https://github.com/daler/trackhub)

ID = 357, Stars = 37, Language = Python, Update = 2019-12-24

create, manage, and upload track hubs for use in the UCSC genome browser
##### [daehwankimlab](https://github.com/daehwankimlab)/[hisat](https://github.com/DaehwanKimLab/hisat)

ID = 358, Stars = 37, Language = C++, Update = 2015-09-08

Fast spliced aligner with low memory requirements
##### [broadinstitute](https://github.com/broadinstitute)/[dsde-deep-learning](https://github.com/broadinstitute/dsde-deep-learning)

ID = 359, Stars = 37, Language = Python, Update = 2019-03-28

DSDE Deep Learning Club
##### [cgatoxford](https://github.com/cgatoxford)/[CGATPipelines](https://github.com/CGATOxford/CGATPipelines)

ID = 360, Stars = 37, Language = Python, Update = 2018-11-08

Collection of CGAT NGS Pipelines
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbsv](https://github.com/PacificBiosciences/pbsv)

ID = 361, Stars = 37, Language = None, Update = 2020-05-05

pbsv - PacBio structural variant (SV) calling and analysis tools
##### [ncbi](https://github.com/ncbi)/[JATSPreviewStylesheets](https://github.com/ncbi/JATSPreviewStylesheets)

ID = 362, Stars = 37, Language = XSLT, Update = 2014-05-13

JATS Preview Stylesheets
##### [lh3](https://github.com/lh3)/[htsbox](https://github.com/lh3/htsbox)

ID = 363, Stars = 36, Language = C, Update = 2018-03-29

My experimental tools on top of htslib. NOT OFFICIAL!!!
##### [openvax](https://github.com/openvax)/[gtfparse](https://github.com/openvax/gtfparse)

ID = 364, Stars = 36, Language = Python, Update = 2018-10-09

Parsing tools for GTF (gene transfer format) files
##### [nf-core](https://github.com/nf-core)/[atacseq](https://github.com/nf-core/atacseq)

ID = 365, Stars = 36, Language = Nextflow, Update = 2020-03-29

ATAC-seq peak-calling, QC and differential analysis pipeline
##### [stephenturner](https://github.com/stephenturner)/[twitterchive](https://github.com/stephenturner/twitterchive)

ID = 366, Stars = 35, Language = R, Update = 2015-03-25

Archive and analyze results from a Twitter search (**no longer maintained**)
##### [brentp](https://github.com/brentp)/[jigv](https://github.com/brentp/jigv)

ID = 367, Stars = 35, Language = Nim, Update = 2020-04-07

igv.js server and automatic configuration to view bam/cram/vcf/bed. "working in under 1 minute"
##### [vgteam](https://github.com/vgteam)/[graph-genomics-review](https://github.com/vgteam/graph-genomics-review)

ID = 368, Stars = 35, Language = TeX, Update = 2020-04-15

Pangenome graphs (review article on graph-based pangenomic methods)
##### [ensembl](https://github.com/ensembl)/[ensembl-compara](https://github.com/Ensembl/ensembl-compara)

ID = 369, Stars = 35, Language = Perl, Update = 2020-05-07

The Ensembl Compara Perl API and SQL schema
##### [ncbi](https://github.com/ncbi)/[BAMscale](https://github.com/ncbi/BAMscale)

ID = 370, Stars = 35, Language = C, Update = 2020-04-23

BAMscale is a one-step tool for either 1) quantifying and normalizing the coverage of peaks or 2) generated scaled BigWig files for easy visualization of commonly used DNA-seq capture based methods.
##### [lh3](https://github.com/lh3)/[CHM-eval](https://github.com/lh3/CHM-eval)

ID = 371, Stars = 34, Language = TeX, Update = 2019-10-03

None
##### [ekg](https://github.com/ekg)/[bamaddrg](https://github.com/ekg/bamaddrg)

ID = 372, Stars = 34, Language = C++, Update = 2017-09-21

adds sample names and read-group (RG) tags to BAM alignments
##### [ekg](https://github.com/ekg)/[glia](https://github.com/ekg/glia)

ID = 373, Stars = 34, Language = C++, Update = 2016-07-05

a string to graph aligner
##### [mdshw5](https://github.com/mdshw5)/[simplesam](https://github.com/mdshw5/simplesam)

ID = 374, Stars = 34, Language = Python, Update = 2019-09-17

Simple pure Python SAM parser and objects for working with SAM records
##### [brentp](https://github.com/brentp)/[intintmap](https://github.com/brentp/intintmap)

ID = 375, Stars = 34, Language = Go, Update = 2019-02-11

fast int64-int64 map for go
##### [brentp](https://github.com/brentp)/[bigly](https://github.com/brentp/bigly)

ID = 376, Stars = 34, Language = Go, Update = 2018-01-17

a pileup library that embraces the huge
##### [openvax](https://github.com/openvax)/[vaxrank](https://github.com/openvax/vaxrank)

ID = 377, Stars = 34, Language = Python, Update = 2020-02-22

Ranked vaccine peptides for personalized cancer immunotherapy
##### [nf-core](https://github.com/nf-core)/[sarek](https://github.com/nf-core/sarek)

ID = 378, Stars = 34, Language = Nextflow, Update = 2020-05-08

Analysis pipeline to detect germline or somatic variants from WGS / targeted sequencing
##### [pharmgkb](https://github.com/pharmgkb)/[PharmCAT](https://github.com/PharmGKB/PharmCAT)

ID = 379, Stars = 33, Language = Java, Update = 2020-05-08

The Pharmacogenomic Clinical Annotation Tool
##### [broadinstitute](https://github.com/broadinstitute)/[catch](https://github.com/broadinstitute/catch)

ID = 380, Stars = 33, Language = Python, Update = 2020-03-28

A package for designing compact and comprehensive capture probe sets.
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-viz](https://github.com/broadinstitute/gtex-viz)

ID = 381, Stars = 33, Language = JavaScript, Update = 2020-01-28

GTEx Visualizations
##### [niu-lab](https://github.com/niu-lab)/[msisensor2](https://github.com/niu-lab/msisensor2)

ID = 382, Stars = 33, Language = R, Update = 2019-11-21

Microsatellite instability (MSI) detection for tumor only data.
##### [ding-lab](https://github.com/ding-lab)/[hotspot3d](https://github.com/ding-lab/hotspot3d)

ID = 383, Stars = 33, Language = Perl, Update = 2019-08-22

3D hotspot mutation proximity analysis tool
##### [papenfusslab](https://github.com/papenfusslab)/[StructuralVariantAnnotation](https://github.com/PapenfussLab/StructuralVariantAnnotation)

ID = 384, Stars = 33, Language = R, Update = 2020-04-15

R package designed to simplify structural variant analysis
##### [parklab](https://github.com/parklab)/[SigMA](https://github.com/parklab/SigMA)

ID = 385, Stars = 33, Language = R, Update = 2020-04-30

Mutational signature analysis for low statistics SNV data
##### [biocore](https://github.com/biocore)/[gneiss](https://github.com/biocore/gneiss)

ID = 386, Stars = 33, Language = Jupyter Notebook, Update = 2019-10-11

compositional data analysis toolbox
##### [ncbi](https://github.com/ncbi)/[dbvar](https://github.com/ncbi/dbvar)

ID = 387, Stars = 33, Language = None, Update = 2020-04-22

dbVar
##### [seandavi](https://github.com/seandavi)/[wdlRunR](https://github.com/seandavi/wdlRunR)

ID = 388, Stars = 32, Language = R, Update = 2019-01-30

Elastic, reproducible, and reusable genomic data science tools from R backed by cloud resources
##### [nf-core](https://github.com/nf-core)/[eager](https://github.com/nf-core/eager)

ID = 389, Stars = 32, Language = Nextflow, Update = 2020-05-10

A fully reproducible and state of the art ancient DNA analysis pipeline
##### [biocore](https://github.com/biocore)/[scikit-bio-cookbook](https://github.com/biocore/scikit-bio-cookbook)

ID = 390, Stars = 32, Language = Jupyter Notebook, Update = 2016-06-15

Recipes for bioinformatics analyses with scikit-bio
##### [lh3](https://github.com/lh3)/[bioseq-js](https://github.com/lh3/bioseq-js)

ID = 391, Stars = 31, Language = HTML, Update = 2019-08-08

For live demo, see http://lh3lh3.users.sourceforge.net/bioseq.shtml
##### [broadinstitute](https://github.com/broadinstitute)/[ABC-Enhancer-Gene-Prediction](https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction)

ID = 392, Stars = 31, Language = Python, Update = 2020-05-08

Cell type specific enhancer-gene predictions using ABC model (Fulco, Nasser et al, Nature Genetics 2019)
##### [bioinformaticsfmrp](https://github.com/bioinformaticsfmrp)/[TCGAbiolinksGUI](https://github.com/BioinformaticsFMRP/TCGAbiolinksGUI)

ID = 393, Stars = 31, Language = R, Update = 2019-12-15

TCGAbiolinksGUI code development for R/Bioconductor submission http://tcgabiolinks.fmrp.usp.br:3838/
##### [ensembl](https://github.com/ensembl)/[ensembl-tools](https://github.com/Ensembl/ensembl-tools)

ID = 394, Stars = 31, Language = Perl, Update = 2020-02-24

Ensembl tools
##### [ga4gh](https://github.com/ga4gh)/[data-repository-service-schemas](https://github.com/ga4gh/data-repository-service-schemas)

ID = 395, Stars = 31, Language = Python, Update = 2020-05-11

A repository for the schemas used for the Data Repository Service.
##### [lh3](https://github.com/lh3)/[klib.nim](https://github.com/lh3/klib.nim)

ID = 396, Stars = 30, Language = Nim, Update = 2020-04-20

Experimental getopt, gzip reader, FASTA/Q parser and interval queries in nim-lang
##### [lh3](https://github.com/lh3)/[yak](https://github.com/lh3/yak)

ID = 397, Stars = 30, Language = C, Update = 2020-03-21

Yet another k-mer analyzer
##### [lh3](https://github.com/lh3)/[etrf](https://github.com/lh3/etrf)

ID = 398, Stars = 30, Language = C, Update = 2019-10-22

Exact Tandem Repeat Finder (not a TRF replacement)
##### [crazyhottommy](https://github.com/crazyhottommy)/[The-world-of-faculty](https://github.com/crazyhottommy/The-world-of-faculty)

ID = 399, Stars = 30, Language = None, Update = 2020-04-23

resources for faculty
##### [arq5x](https://github.com/arq5x)/[nanopore-scripts](https://github.com/arq5x/nanopore-scripts)

ID = 400, Stars = 30, Language = Python, Update = 2016-04-17

Various scripts and recipes for working with nanopore data
##### [freeseek](https://github.com/freeseek)/[getmyancestors](https://github.com/freeseek/getmyancestors)

ID = 401, Stars = 30, Language = None, Update = 2018-06-11

None
##### [statgen](https://github.com/statgen)/[locuszoom-standalone](https://github.com/statgen/locuszoom-standalone)

ID = 402, Stars = 30, Language = Python, Update = 2020-01-21

Create regional association plots from GWAS or meta-analysis
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_portal_core](https://github.com/broadinstitute/single_cell_portal_core)

ID = 403, Stars = 30, Language = Ruby, Update = 2020-05-10

Rails/Docker application for the Broad Institute's single cell RNA-seq data portal
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-somatic-cnvs](https://github.com/gatk-workflows/gatk4-somatic-cnvs)

ID = 404, Stars = 30, Language = wdl, Update = 2020-03-10

This repo will be archived soon, these workflows will be housed in the GATK repository under the scripts directory. These workflows are also organized in Dockstore in the GATK Best Practices Workflows collection.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[barcoding](https://github.com/PacificBiosciences/barcoding)

ID = 405, Stars = 30, Language = R, Update = 2020-02-18

Lima - Demultiplex Barcoded PacBio Samples
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[FALCON_unzip](https://github.com/PacificBiosciences/FALCON_unzip)

ID = 406, Stars = 30, Language = Python, Update = 2017-10-02

Making diploid assembly becomes common practice for genomic study
##### [ming-lian](https://github.com/ming-lian)/[Bioinformatics-skills](https://github.com/Ming-Lian/Bioinformatics-skills)

ID = 407, Stars = 29, Language = None, Update = 2020-04-23

ÁîüÁâ©‰ø°ÊÅØÂ≠¶ÊäÄËÉΩÊèêÂçáÔºå‰∏ìÊ≥®‰∫éÂ∑•ÂÖ∑‰ΩøÁî®‰∏éÁºñÁ®ã
##### [s-andrews](https://github.com/s-andrews)/[BamQC](https://github.com/s-andrews/BamQC)

ID = 408, Stars = 29, Language = Java, Update = 2018-05-16

Mapped QC analysis program
##### [brentp](https://github.com/brentp)/[450k-analysis-guide](https://github.com/brentp/450k-analysis-guide)

ID = 409, Stars = 29, Language = TeX, Update = 2014-10-02

A Practical (And Opinionated) Guide To Analyzing 450K Data
##### [statgen](https://github.com/statgen)/[gotcloud](https://github.com/statgen/gotcloud)

ID = 410, Stars = 29, Language = C++, Update = 2017-03-24

Genomes on the Cloud, Mapping & Variant Calling Pipelines
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbcore](https://github.com/PacificBiosciences/pbcore)

ID = 411, Stars = 29, Language = Python, Update = 2020-05-07

A Python library for reading and writing PacBio¬Æ data files
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbdagcon](https://github.com/PacificBiosciences/pbdagcon)

ID = 412, Stars = 29, Language = C++, Update = 2018-04-12

A sequence consensus algorithm implementation based on using directed acyclic graphs to encode multiple sequence alignment
##### [ga4gh](https://github.com/ga4gh)/[mme-apis](https://github.com/ga4gh/mme-apis)

ID = 413, Stars = 29, Language = HTML, Update = 2020-01-10

Documentation for the MatchmakerExchange APIs
##### [crazyhottommy](https://github.com/crazyhottommy)/[scATACseq-analysis-notes](https://github.com/crazyhottommy/scATACseq-analysis-notes)

ID = 414, Stars = 28, Language = None, Update = 2020-02-24

my notes for scATACseq analysis
##### [ekg](https://github.com/ekg)/[thesis](https://github.com/ekg/thesis)

ID = 415, Stars = 28, Language = TeX, Update = 2019-07-10

my PhD thesis
##### [ewels](https://github.com/ewels)/[Labrador](https://github.com/ewels/Labrador)

ID = 416, Stars = 28, Language = PHP, Update = 2018-11-16

A web based tool to manage and automate the processing of publicly available datasets.
##### [brentp](https://github.com/brentp)/[quicksect](https://github.com/brentp/quicksect)

ID = 417, Stars = 28, Language = Python, Update = 2019-06-04

a cythonized, extended version of the interval search tree in bx
##### [brentp](https://github.com/brentp)/[align](https://github.com/brentp/align)

ID = 418, Stars = 28, Language = Python, Update = 2016-12-21

sequence alignment. global, local, glocal.
##### [ding-lab](https://github.com/ding-lab)/[PanCanAtlasGermline](https://github.com/ding-lab/PanCanAtlasGermline)

ID = 419, Stars = 28, Language = R, Update = 2018-10-27

None
##### [umcugenetics](https://github.com/umcugenetics)/[RNASeq](https://github.com/UMCUGenetics/RNASeq)

ID = 420, Stars = 28, Language = Perl, Update = 2019-10-28

RNASeq pipeline
##### [parklab](https://github.com/parklab)/[MosaicForecast](https://github.com/parklab/MosaicForecast)

ID = 421, Stars = 28, Language = Python, Update = 2020-05-06

A mosaic detecting software based on phasing and random forest
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[FALCON-integrate](https://github.com/PacificBiosciences/FALCON-integrate)

ID = 422, Stars = 28, Language = Shell, Update = 2018-04-06

Mostly deprecated. See https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries
##### [lh3](https://github.com/lh3)/[hapdip](https://github.com/lh3/hapdip)

ID = 423, Stars = 27, Language = JavaScript, Update = 2018-02-20

The CHM1-NA12878 benchmark for single-sample SNP/INDEL calling from WGS Illumina data
##### [richarddurbin](https://github.com/richarddurbin)/[modimizer](https://github.com/richarddurbin/modimizer)

ID = 424, Stars = 27, Language = C, Update = 2019-02-01

a toolset for fast DNA read set matching and assembly using a new type of reduced kmer
##### [brentp](https://github.com/brentp)/[combined-pvalues](https://github.com/brentp/combined-pvalues)

ID = 425, Stars = 27, Language = Python, Update = 2019-12-18

combining p-values using modified stouffer-liptak for spatially correlated results (probes)
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_portal](https://github.com/broadinstitute/single_cell_portal)

ID = 426, Stars = 27, Language = Python, Update = 2020-04-22

Tutorials, workflows, and convenience scripts for Single Cell Portal
##### [illumina](https://github.com/illumina)/[Polaris](https://github.com/Illumina/Polaris)

ID = 427, Stars = 27, Language = None, Update = 2019-11-18

Data and information about the Polaris study
##### [lh3](https://github.com/lh3)/[pre-pe](https://github.com/lh3/pre-pe)

ID = 428, Stars = 26, Language = C, Update = 2018-06-28

Preprocessing paired-end reads produced with experiment-specific protocols
##### [stephenturner](https://github.com/stephenturner)/[adapters](https://github.com/stephenturner/adapters)

ID = 429, Stars = 26, Language = R, Update = 2019-01-30

Adapters for trimming
##### [stephenturner](https://github.com/stephenturner)/[solarplot](https://github.com/stephenturner/solarplot)

ID = 430, Stars = 26, Language = R, Update = 2017-02-25

Mitochondrial Solar Plot
##### [seandavi](https://github.com/seandavi)/[sars2pack](https://github.com/seandavi/sars2pack)

ID = 431, Stars = 26, Language = R, Update = 2020-05-10

Up-to-date data resources, analysis tools, and visualizations for COVID-19 pandemic data
##### [tanghaibao](https://github.com/tanghaibao)/[allhic](https://github.com/tanghaibao/allhic)

ID = 432, Stars = 26, Language = Jupyter Notebook, Update = 2020-02-27

Genome scaffolding based on HiC data in heterozygous and high ploidy genomes
##### [freeseek](https://github.com/freeseek)/[gwaspipeline](https://github.com/freeseek/gwaspipeline)

ID = 433, Stars = 26, Language = R, Update = 2018-01-24

None
##### [tseemann](https://github.com/tseemann)/[homebrew-bioinformatics-linux](https://github.com/tseemann/homebrew-bioinformatics-linux)

ID = 434, Stars = 26, Language = Ruby, Update = 2019-08-23

üç∫
üêß
Homebrew formulae for bioinformatics software only available for Linux
##### [brentp](https://github.com/brentp)/[bsub](https://github.com/brentp/bsub)

ID = 435, Stars = 26, Language = Python, Update = 2020-02-27

python wrapper to submit jobs to bsub (and later qsub)
##### [statgen](https://github.com/statgen)/[fastQValidator](https://github.com/statgen/fastQValidator)

ID = 436, Stars = 26, Language = C++, Update = 2018-10-30

Validate FastQ Files
##### [broadinstitute](https://github.com/broadinstitute)/[SignatureAnalyzer-GPU](https://github.com/broadinstitute/SignatureAnalyzer-GPU)

ID = 437, Stars = 26, Language = Python, Update = 2019-12-09

GPU implementation of ARD NMF
##### [biowdl](https://github.com/biowdl)/[tasks](https://github.com/biowdl/tasks)

ID = 438, Stars = 26, Language = wdl, Update = 2020-05-08

A collection of reusable WDL tasks. Category:Other
##### [jodyphelan](https://github.com/jodyphelan)/[TBProfiler](https://github.com/jodyphelan/TBProfiler)

ID = 439, Stars = 25, Language = Python, Update = 2020-05-05

Profiling tool for Mycobacterium tuberculosis to detect ressistance and strain type from WGS data
##### [brentp](https://github.com/brentp)/[gsort](https://github.com/brentp/gsort)

ID = 440, Stars = 25, Language = Go, Update = 2019-11-11

sort genomic data
##### [brentp](https://github.com/brentp)/[geneimpacts](https://github.com/brentp/geneimpacts)

ID = 441, Stars = 25, Language = Python, Update = 2019-10-29

prioritize effects of variant annotations from VEP, SnpEff, et al.
##### [brentp](https://github.com/brentp)/[poverlap](https://github.com/brentp/poverlap)

ID = 442, Stars = 25, Language = Python, Update = 2016-08-16

significance testing over interval overlaps
##### [broadinstitute](https://github.com/broadinstitute)/[long-read-pipelines](https://github.com/broadinstitute/long-read-pipelines)

ID = 443, Stars = 25, Language = wdl, Update = 2020-05-08

Long read production pipelines
##### [quinlan-lab](https://github.com/quinlan-lab)/[STRling](https://github.com/quinlan-lab/STRling)

ID = 444, Stars = 25, Language = Nim, Update = 2020-05-07

Detect novel (and reference) STR expansions from short-read data
##### [ensembl](https://github.com/ensembl)/[postgap](https://github.com/Ensembl/postgap)

ID = 445, Stars = 25, Language = Jupyter Notebook, Update = 2020-05-03

Linking GWAS studies to genes through cis-regulatory datasets
##### [ncbi](https://github.com/ncbi)/[docker](https://github.com/ncbi/docker)

ID = 446, Stars = 25, Language = C++, Update = 2020-03-24

None
##### [mozack](https://github.com/mozack)/[vdjer](https://github.com/mozack/vdjer)

ID = 447, Stars = 24, Language = C, Update = 2017-04-11

V'DJer - B Cell Receptor Repertoire Reconstruction from short read mRNA-Seq data
##### [daler](https://github.com/daler)/[sphinxdoc-test](https://github.com/daler/sphinxdoc-test)

ID = 448, Stars = 24, Language = None, Update = 2015-06-12

experimenting with the best way to push sphinx-generated docs to gh-pages
##### [brentp](https://github.com/brentp)/[fastbit-python](https://github.com/brentp/fastbit-python)

ID = 449, Stars = 24, Language = Python, Update = 2018-08-10

pythonic access to fastbit
##### [broadinstitute](https://github.com/broadinstitute)/[gdctools](https://github.com/broadinstitute/gdctools)

ID = 450, Stars = 24, Language = Python, Update = 2018-10-11

Python and UNIX CLI utilities to simplify interaction with the NIH/NCI Genomics Data Commons
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_analysis](https://github.com/broadinstitute/single_cell_analysis)

ID = 451, Stars = 24, Language = HTML, Update = 2017-02-05

Documents used for workshops on single cell analysis
##### [opengene](https://github.com/opengene)/[ctdna-pipeline](https://github.com/OpenGene/ctdna-pipeline)

ID = 452, Stars = 24, Language = Shell, Update = 2017-09-23

A simplified pipeline for ctDNA sequencing data analysis
##### [wglab](https://github.com/wglab)/[NanoMod](https://github.com/WGLab/NanoMod)

ID = 453, Stars = 24, Language = Python, Update = 2019-12-05

NanoMod: a computational tool to detect DNA modifications using Nanopore long-read sequencing data
##### [wglab](https://github.com/wglab)/[VirTect](https://github.com/WGLab/VirTect)

ID = 454, Stars = 24, Language = Python, Update = 2019-12-04

Detection of viruses from RNA-Seq on human samples
##### [wglab](https://github.com/wglab)/[EHR-Phenolyzer](https://github.com/WGLab/EHR-Phenolyzer)

ID = 455, Stars = 24, Language = Python, Update = 2019-10-30

A pipeline that takes clinical notes from EHRs and generate phenotype terms, then generate ranked gene list
##### [illumina](https://github.com/illumina)/[BeadArrayFiles](https://github.com/Illumina/BeadArrayFiles)

ID = 456, Stars = 24, Language = Python, Update = 2020-02-17

Python library to parse file formats related to Illumina bead arrays
##### [lh3](https://github.com/lh3)/[fermi2](https://github.com/lh3/fermi2)

ID = 457, Stars = 23, Language = C, Update = 2018-02-07

None
##### [benlangmead](https://github.com/benlangmead)/[qtip](https://github.com/BenLangmead/qtip)

ID = 458, Stars = 23, Language = C++, Update = 2019-10-31

Qtip: a tandem simulation approach for accurately predicting read alignment mapping qualities
##### [seandavi](https://github.com/seandavi)/[ci4cc-informatics-resources](https://github.com/seandavi/ci4cc-informatics-resources)

ID = 459, Stars = 23, Language = None, Update = 2018-09-24

Community-maintained list of resources that the CI4CC organization and the larger cancer informatics community have found useful or are developing.
##### [felixkrueger](https://github.com/felixkrueger)/[SNPsplit](https://github.com/FelixKrueger/SNPsplit)

ID = 460, Stars = 23, Language = Perl, Update = 2019-12-03

Allele-specific alignment sorting
##### [tanghaibao](https://github.com/tanghaibao)/[treecut](https://github.com/tanghaibao/treecut)

ID = 461, Stars = 23, Language = Python, Update = 2017-10-15

Find nodes in hierarchical clustering that are statistically significant
##### [statgen](https://github.com/statgen)/[EPACTS](https://github.com/statgen/EPACTS)

ID = 462, Stars = 23, Language = C++, Update = 2019-07-30

None
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad_methods](https://github.com/broadinstitute/gnomad_methods)

ID = 463, Stars = 23, Language = Python, Update = 2020-05-08

Hail helper functions for gnomAD/MacArthur lab
##### [broadinstitute](https://github.com/broadinstitute)/[CellBender](https://github.com/broadinstitute/CellBender)

ID = 464, Stars = 23, Language = Python, Update = 2020-03-31

CellBender is a software package for eliminating technical artifacts from high-throughput single-cell RNA sequencing (scRNA-seq) data.
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-exome-analysis-pipeline](https://github.com/gatk-workflows/gatk4-exome-analysis-pipeline)

ID = 465, Stars = 23, Language = wdl, Update = 2020-03-03

This WDL pipeline implements data pre-processing and initial variant calling according to the GATK Best Practices for germline SNP and Indel discovery in human exome sequencing data.
##### [opengene](https://github.com/opengene)/[FusionDirect.jl](https://github.com/OpenGene/FusionDirect.jl)

ID = 466, Stars = 23, Language = Julia, Update = 2017-08-13

(No maintenance) Detect gene fusion directly from raw fastq files
##### [biocore](https://github.com/biocore)/[songbird](https://github.com/biocore/songbird)

ID = 467, Stars = 23, Language = Python, Update = 2020-03-31

Vanilla regression methods for microbiome differential abundance analysis
##### [biocore](https://github.com/biocore)/[tcga](https://github.com/biocore/tcga)

ID = 468, Stars = 23, Language = Jupyter Notebook, Update = 2020-01-11

Microbial analysis in TCGA data
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[PacBioFileFormats](https://github.com/PacificBiosciences/PacBioFileFormats)

ID = 469, Stars = 23, Language = Python, Update = 2020-04-07

Specifications for PacBio¬Æ native file formats
##### [illumina](https://github.com/illumina)/[gvcfgenotyper](https://github.com/Illumina/gvcfgenotyper)

ID = 470, Stars = 23, Language = C++, Update = 2019-02-26

A utility for merging and genotyping Illumina-style GVCFs.
##### [lh3](https://github.com/lh3)/[asub](https://github.com/lh3/asub)

ID = 471, Stars = 22, Language = Perl, Update = 2019-11-29

A unified array job submitter for LSF, SGE/UGE and Slurm
##### [arq5x](https://github.com/arq5x)/[bits](https://github.com/arq5x/bits)

ID = 472, Stars = 22, Language = C++, Update = 2017-04-26

BITS: Binary Interval Search
##### [vibansal](https://github.com/vibansal)/[hapcut](https://github.com/vibansal/hapcut)

ID = 473, Stars = 22, Language = C, Update = 2018-11-29

program for haplotype phasing from sequence reads and related tools
##### [s-andrews](https://github.com/s-andrews)/[SeqMonk](https://github.com/s-andrews/SeqMonk)

ID = 474, Stars = 22, Language = Java, Update = 2020-05-07

SeqMonk NGS visualisation and analysis tool
##### [tseemann](https://github.com/tseemann)/[seeka](https://github.com/tseemann/seeka)

ID = 475, Stars = 22, Language = Perl, Update = 2019-11-07

Get microbial sequence data easier and faster
##### [statgen](https://github.com/statgen)/[emeraLD](https://github.com/statgen/emeraLD)

ID = 476, Stars = 22, Language = C++, Update = 2019-08-27

tools to efficiently retrieve and calculate LD
##### [vgteam](https://github.com/vgteam)/[gssw](https://github.com/vgteam/gssw)

ID = 477, Stars = 22, Language = C, Update = 2020-04-29

efficient alignment of strings to partially ordered string graphs
##### [broadinstitute](https://github.com/broadinstitute)/[firepony](https://github.com/broadinstitute/firepony)

ID = 478, Stars = 22, Language = Cuda, Update = 2015-11-28

Efficient base quality score recalibrator for NGS data
##### [quinlan-lab](https://github.com/quinlan-lab)/[ccrhtml](https://github.com/quinlan-lab/ccrhtml)

ID = 479, Stars = 22, Language = HTML, Update = 2019-10-22

A small repo for storing the code for making the files and html for CCRs.
##### [bioinformaticsfmrp](https://github.com/bioinformaticsfmrp)/[TCGAWorkflow](https://github.com/BioinformaticsFMRP/TCGAWorkflow)

ID = 480, Stars = 22, Language = R, Update = 2020-04-20

TCGA Workflow: Analyze cancer genomics and epigenomics data using Bioconductor packages
##### [illumina](https://github.com/illumina)/[GTCtoVCF](https://github.com/Illumina/GTCtoVCF)

ID = 481, Stars = 22, Language = Python, Update = 2020-04-07

Script to convert GTC/BPM files to VCF
##### [pcingola](https://github.com/pcingola)/[SnpSift](https://github.com/pcingola/SnpSift)

ID = 482, Stars = 21, Language = Java, Update = 2020-04-16

None
##### [marianattestad](https://github.com/marianattestad)/[R_for_biologists](https://github.com/MariaNattestad/R_for_biologists)

ID = 483, Stars = 21, Language = R, Update = 2018-11-25

Scripts and data for Plotting in R for Biologists, a video course on YouTube
##### [cschin](https://github.com/cschin)/[Write_A_Genome_Assembler_With_IPython](https://github.com/cschin/Write_A_Genome_Assembler_With_IPython)

ID = 484, Stars = 21, Language = Python, Update = 2018-05-10

The repository keeps the files for an IPython notebook on about how to make a simple genome assembler using python
##### [vibansal](https://github.com/vibansal)/[ancestry](https://github.com/vibansal/ancestry)

ID = 485, Stars = 21, Language = C, Update = 2018-04-26

program to estimate admixture coefficients from individual genotype or sequence data
##### [brentp](https://github.com/brentp)/[cigar](https://github.com/brentp/cigar)

ID = 486, Stars = 21, Language = Python, Update = 2019-09-26

simple library for dealing with SAM cigar strings
##### [statgen](https://github.com/statgen)/[verifyBamID](https://github.com/statgen/verifyBamID)

ID = 487, Stars = 21, Language = C++, Update = 2017-03-15

None
##### [bioconda](https://github.com/bioconda)/[bioconda.github.io](https://github.com/bioconda/bioconda.github.io)

ID = 488, Stars = 21, Language = HTML, Update = 2020-05-11

Bioconda is a distribution of bioinformatics software realized as a channel for the versatile package manager Conda.
##### [broadinstitute](https://github.com/broadinstitute)/[cromshell](https://github.com/broadinstitute/cromshell)

ID = 489, Stars = 21, Language = Shell, Update = 2020-05-08

CLI for interacting with Cromwell servers
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-orchestration](https://github.com/broadinstitute/firecloud-orchestration)

ID = 490, Stars = 21, Language = Scala, Update = 2020-05-04

None
##### [hall-lab](https://github.com/hall-lab)/[sv-pipeline](https://github.com/hall-lab/sv-pipeline)

ID = 491, Stars = 21, Language = wdl, Update = 2020-05-09

Pipeline for structural variation detection in cohorts
##### [opengene](https://github.com/opengene)/[defq](https://github.com/OpenGene/defq)

ID = 492, Stars = 21, Language = C, Update = 2018-07-28

Ultra-fast Multi-threaded FASTQ Demultiplexing
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbh5tools](https://github.com/PacificBiosciences/pbh5tools)

ID = 493, Stars = 21, Language = Python, Update = 2018-12-12

A swiss-army knife for interrogating PacBio¬Æ HDF5 files (cmp.h5, bas.h5)
##### [gregoryfaust](https://github.com/gregoryfaust)/[yaha](https://github.com/GregoryFaust/yaha) *new+*

ID = 494, Stars = 20, Language = C, Update = 2017-12-06

yaha: a flexible, sensitive and accurate DNA alignment tool designed to find optimal split-read mappings on single-end queries from 100bp to 32kb.
##### [seandavi](https://github.com/seandavi)/[Bioc2017BigDataWorkshopSession](https://github.com/seandavi/Bioc2017BigDataWorkshopSession)

ID = 495, Stars = 20, Language = None, Update = 2017-07-26

Tutorial for working with cloud infrastructure and AWS from R
##### [freeseek](https://github.com/freeseek)/[getmydnamatches](https://github.com/freeseek/getmydnamatches)

ID = 496, Stars = 20, Language = Python, Update = 2019-01-21

None
##### [broadinstitute](https://github.com/broadinstitute)/[PhylogicNDT](https://github.com/broadinstitute/PhylogicNDT)

ID = 497, Stars = 20, Language = HTML, Update = 2020-04-02

None
##### [fhcrc](https://github.com/fhcrc)/[nestly](https://github.com/fhcrc/nestly)

ID = 498, Stars = 20, Language = Python, Update = 2017-04-23

Make and use nested directory trees corresponding to combinatorial choices of parameters
##### [refresh-bio](https://github.com/refresh-bio)/[FAMSA](https://github.com/refresh-bio/FAMSA)

ID = 499, Stars = 20, Language = C++, Update = 2020-05-08

Algorithm for large-scale multiple sequence alignments (400k proteins in 2 hours and 8BG of RAM)
##### [ensembl](https://github.com/ensembl)/[ensembl-variation](https://github.com/Ensembl/ensembl-variation)

ID = 500, Stars = 20, Language = Perl, Update = 2020-05-07

The Ensembl Variation Perl API and SQL schema
##### [opengene](https://github.com/opengene)/[VisualMSI](https://github.com/OpenGene/VisualMSI)

ID = 501, Stars = 20, Language = C++, Update = 2019-06-04

Detect and visualize microsatellite instability(MSI) from NGS data
##### [wglab](https://github.com/wglab)/[SeqMule](https://github.com/WGLab/SeqMule)

ID = 502, Stars = 20, Language = C++, Update = 2020-01-05

Automated human exome/genome variants detection from FASTQ files
##### [biocore](https://github.com/biocore)/[unifrac](https://github.com/biocore/unifrac)

ID = 503, Stars = 20, Language = C++, Update = 2020-05-08

None
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbsmrtpipe](https://github.com/PacificBiosciences/pbsmrtpipe)

ID = 504, Stars = 20, Language = Python, Update = 2018-10-24

PacBio Workflow Engine
##### [yongxinliu](https://github.com/yongxinliu)/[Metagenome](https://github.com/YongxinLiu/Metagenome)

ID = 505, Stars = 19, Language = Shell, Update = 2019-06-28

Metagenome analysis pipeline
##### [lh3](https://github.com/lh3)/[dipcall](https://github.com/lh3/dipcall)

ID = 506, Stars = 19, Language = JavaScript, Update = 2020-05-04

Reference-based variant calling pipeline for a pair of phased haplotype assemblies
##### [sigven](https://github.com/sigven)/[cpsr](https://github.com/sigven/cpsr)

ID = 507, Stars = 19, Language = R, Update = 2019-12-01

Cancer Predisposition Sequencing Reporter (CPSR)
##### [crazyhottommy](https://github.com/crazyhottommy)/[scRNA-seq-workshop-Fall-2019](https://github.com/crazyhottommy/scRNA-seq-workshop-Fall-2019)

ID = 508, Stars = 19, Language = R, Update = 2019-08-08

Harvard FAS informatics scRNAseq workshop website
##### [ekg](https://github.com/ekg)/[1000G-integration](https://github.com/ekg/1000G-integration)

ID = 509, Stars = 19, Language = Shell, Update = 2018-01-16

variant integration methods for the 1000 Genomes Project
##### [seandavi](https://github.com/seandavi)/[SRAdbV2](https://github.com/seandavi/SRAdbV2)

ID = 510, Stars = 19, Language = R, Update = 2018-11-19

R Interface to the NCBI SRA metadata
##### [tseemann](https://github.com/tseemann)/[ekidna](https://github.com/tseemann/ekidna)

ID = 511, Stars = 19, Language = Perl, Update = 2019-10-23

Assembly based core genome SNP alignments for bacteria
##### [brentp](https://github.com/brentp)/[nim-lapper](https://github.com/brentp/nim-lapper)

ID = 512, Stars = 19, Language = Nim, Update = 2020-02-28

fast easy interval overlapping for nim-lang
##### [broadinstitute](https://github.com/broadinstitute)/[BipolarCell2016](https://github.com/broadinstitute/BipolarCell2016)

ID = 513, Stars = 19, Language = R, Update = 2020-02-07

None
##### [broadinstitute](https://github.com/broadinstitute)/[BARD](https://github.com/broadinstitute/BARD)

ID = 514, Stars = 19, Language = Groovy, Update = 2019-01-29

BioAssay Research Database
##### [refresh-bio](https://github.com/refresh-bio)/[Whisper](https://github.com/refresh-bio/Whisper)

ID = 515, Stars = 19, Language = C++, Update = 2020-04-20

None
##### [ensembl](https://github.com/ensembl)/[Bio-DB-HTS](https://github.com/Ensembl/Bio-DB-HTS)

ID = 516, Stars = 19, Language = Perl, Update = 2020-01-21

Git repo for Bio::DB::HTS module on CPAN, providing Perl links into HTSlib
##### [parklab](https://github.com/parklab)/[ShatterSeek](https://github.com/parklab/ShatterSeek)

ID = 517, Stars = 19, Language = TeX, Update = 2019-09-12

None
##### [opengene](https://github.com/opengene)/[SeqMaker.jl](https://github.com/OpenGene/SeqMaker.jl)

ID = 518, Stars = 19, Language = Julia, Update = 2017-08-10

(No maintenance) Next Generation Sequencing Simulation with SNP, Variation and Sequencing Error Integrated
##### [biocore](https://github.com/biocore)/[calour](https://github.com/biocore/calour)

ID = 519, Stars = 19, Language = Python, Update = 2020-04-16

exploratory and interactive microbiome analyses based on heatmaps
##### [biocore](https://github.com/biocore)/[redbiom](https://github.com/biocore/redbiom)

ID = 520, Stars = 19, Language = Python, Update = 2019-11-06

Sample search by metadata and features
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[sv-benchmark](https://github.com/PacificBiosciences/sv-benchmark)

ID = 521, Stars = 19, Language = None, Update = 2019-05-01

Public Benchmark of Long-Read Structural Variant Caller on PacBio CCS HG002 Data
##### [ga4gh](https://github.com/ga4gh)/[tool-registry-service-schemas](https://github.com/ga4gh/tool-registry-service-schemas)

ID = 522, Stars = 19, Language = Shell, Update = 2020-04-06

APIs for discovering genomics tools, their metadata and their containers
##### [ncbi](https://github.com/ncbi)/[consul-announcer](https://github.com/ncbi/consul-announcer)

ID = 523, Stars = 19, Language = Python, Update = 2016-12-12

Service announcer for Consul (https://www.consul.io/).
##### [lh3](https://github.com/lh3)/[sdust](https://github.com/lh3/sdust)

ID = 524, Stars = 18, Language = C, Update = 2019-10-03

Symmetric DUST for finding low-complexity regions in DNA sequences
##### [crazyhottommy](https://github.com/crazyhottommy)/[machine-learning-resource](https://github.com/crazyhottommy/machine-learning-resource)

ID = 525, Stars = 18, Language = None, Update = 2019-12-11

None
##### [ekg](https://github.com/ekg)/[hhga](https://github.com/ekg/hhga)

ID = 526, Stars = 18, Language = C++, Update = 2019-06-13

haplotypes genotypes and alleles example decision synthesizer
##### [statgen](https://github.com/statgen)/[Minimac4](https://github.com/statgen/Minimac4)

ID = 527, Stars = 18, Language = C++, Update = 2019-08-30

None
##### [openvax](https://github.com/openvax)/[topiary](https://github.com/openvax/topiary)

ID = 528, Stars = 18, Language = Python, Update = 2018-02-26

Predict mutated T-cell epitopes from sequencing data
##### [nf-core](https://github.com/nf-core)/[nf-co.re](https://github.com/nf-core/nf-co.re)

ID = 529, Stars = 18, Language = PHP, Update = 2020-05-08

Code and files for the main nf-core website >>>
##### [nf-core](https://github.com/nf-core)/[mag](https://github.com/nf-core/mag)

ID = 530, Stars = 18, Language = Nextflow, Update = 2020-04-28

Assembly and binning of metagenomes
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-ui](https://github.com/broadinstitute/firecloud-ui)

ID = 531, Stars = 18, Language = Clojure, Update = 2020-05-04

FireCloud user interface for web browsers.
##### [broadinstitute](https://github.com/broadinstitute)/[wdl-ide](https://github.com/broadinstitute/wdl-ide)

ID = 532, Stars = 18, Language = Python, Update = 2019-11-14

Rich IDE support for Workflow Description Language
##### [broadinstitute](https://github.com/broadinstitute)/[2019_scWorkshop](https://github.com/broadinstitute/2019_scWorkshop)

ID = 533, Stars = 18, Language = TeX, Update = 2019-11-09

Repo for Physalia course Analysis of Single Cell RNA-Seq data
##### [papenfusslab](https://github.com/papenfusslab)/[PathOS](https://github.com/PapenfussLab/PathOS)

ID = 534, Stars = 18, Language = Java, Update = 2020-04-06

PathOS is a clinical application for filtering, analysing and reporting on NGS variants
##### [fhcrc](https://github.com/fhcrc)/[taxtastic](https://github.com/fhcrc/taxtastic)

ID = 535, Stars = 18, Language = Python, Update = 2019-06-24

Create and maintain phylogenetic "reference packages" of biological sequences.
##### [ensembl](https://github.com/ensembl)/[ensembl-webcode](https://github.com/Ensembl/ensembl-webcode)

ID = 536, Stars = 18, Language = Perl, Update = 2020-05-07

The code to run an Ensembl website
##### [opengene](https://github.com/opengene)/[fastv](https://github.com/OpenGene/fastv)

ID = 537, Stars = 18, Language = C++, Update = 2020-05-11

An ultra-fast tool for identification of SARS-CoV-2 and other microbes from sequencing data. This tool can be used to detect viral infectious diseases, like COVID-19.
##### [opengene](https://github.com/opengene)/[MrBam](https://github.com/OpenGene/MrBam)

ID = 538, Stars = 18, Language = Python, Update = 2018-11-26

Query Mutated Reads from a Bam
##### [illumina](https://github.com/illumina)/[Isaac3](https://github.com/Illumina/Isaac3)

ID = 539, Stars = 18, Language = C++, Update = 2018-02-13

Aligner for sequencing data
##### [yongxinliu](https://github.com/yongxinliu)/[Zhang2018SCLS](https://github.com/YongxinLiu/Zhang2018SCLS)

ID = 540, Stars = 17, Language = R, Update = 2019-03-29

Rice (Oryza Sative) root microbiome time-course analysis. (Zhang2018SCLS)
##### [gt1](https://github.com/gt1)/[daccord](https://github.com/gt1/daccord)

ID = 541, Stars = 17, Language = C++, Update = 2018-09-12

d'accord is a non hybrid long read consensus program based on local de Bruijn graph assembly
##### [freeseek](https://github.com/freeseek)/[mocha](https://github.com/freeseek/mocha)

ID = 542, Stars = 17, Language = C, Update = 2020-04-08

MOsaic CHromosomal Alterations (MoChA) caller
##### [brentp](https://github.com/brentp)/[toolshed](https://github.com/brentp/toolshed)

ID = 543, Stars = 17, Language = Python, Update = 2019-12-18

python stuff I use
##### [openvax](https://github.com/openvax)/[pepnet](https://github.com/openvax/pepnet)

ID = 544, Stars = 17, Language = Python, Update = 2018-02-26

Neural networks for amino acid sequences
##### [humanlongevity](https://github.com/humanlongevity)/[tredparse](https://github.com/humanlongevity/tredparse)

ID = 545, Stars = 17, Language = Python, Update = 2020-04-29

TREDPARSE: HLI Short Tandem Repeat (STR) caller
##### [broadinstitute](https://github.com/broadinstitute)/[deepometry](https://github.com/broadinstitute/deepometry)

ID = 546, Stars = 17, Language = Python, Update = 2018-06-30

Image classification for imaging flow cytometry.
##### [ding-lab](https://github.com/ding-lab)/[BreakPointSurveyor](https://github.com/ding-lab/BreakPointSurveyor)

ID = 547, Stars = 17, Language = Shell, Update = 2020-01-28

A comprehensive pipeline to analyze and visualize structural variants
##### [wglab](https://github.com/wglab)/[GenGen](https://github.com/WGLab/GenGen)

ID = 548, Stars = 17, Language = C, Update = 2019-01-16

A set of software tools to facilitate GWAS analysis
##### [biocore](https://github.com/biocore)/[DEICODE](https://github.com/biocore/DEICODE)

ID = 549, Stars = 17, Language = JavaScript, Update = 2020-01-10

Robust Aitchison PCA from sparse count data
##### [illumina](https://github.com/illumina)/[ExpansionHunterDenovo](https://github.com/Illumina/ExpansionHunterDenovo)

ID = 550, Stars = 17, Language = C++, Update = 2020-03-04

A suite of tools for detecting expansions of short tandem repeats
##### [lh3](https://github.com/lh3)/[lianti](https://github.com/lh3/lianti)

ID = 551, Stars = 16, Language = C, Update = 2019-02-20

Tools to process LIANTI sequence data
##### [sigven](https://github.com/sigven)/[gvanno](https://github.com/sigven/gvanno)

ID = 552, Stars = 16, Language = Python, Update = 2020-05-08

Generic germline variant annotation pipeline
##### [sigven](https://github.com/sigven)/[vcf2tsv](https://github.com/sigven/vcf2tsv)

ID = 553, Stars = 16, Language = Python, Update = 2019-10-14

Genomic VCF to tab-separated values
##### [crazyhottommy](https://github.com/crazyhottommy)/[scclusteval](https://github.com/crazyhottommy/scclusteval)

ID = 554, Stars = 16, Language = R, Update = 2020-04-02

Single Cell Cluster Evaluation
##### [ekg](https://github.com/ekg)/[multichoose](https://github.com/ekg/multichoose)

ID = 555, Stars = 16, Language = C++, Update = 2019-12-09

generate multiset combinations (n multichoose k)
##### [ekg](https://github.com/ekg)/[drawbwt](https://github.com/ekg/drawbwt)

ID = 556, Stars = 16, Language = C++, Update = 2019-04-29

vector illustrations of BWT based searching on small strings
##### [seandavi](https://github.com/seandavi)/[BiocPkgTools](https://github.com/seandavi/BiocPkgTools)

ID = 557, Stars = 16, Language = R, Update = 2020-03-25

Computable build reports, package metadata, and download stats from the Bioconductor project
##### [brentp](https://github.com/brentp)/[genoiser](https://github.com/brentp/genoiser)

ID = 558, Stars = 16, Language = Nim, Update = 2020-04-15

use the noise
##### [openvax](https://github.com/openvax)/[varlens](https://github.com/openvax/varlens)

ID = 559, Stars = 16, Language = Python, Update = 2018-02-26

commandline manipulation of genomic variants and NGS reads
##### [nf-core](https://github.com/nf-core)/[test-datasets](https://github.com/nf-core/test-datasets)

ID = 560, Stars = 16, Language = None, Update = 2020-05-02

Test data to be used for automated testing with the nf-core pipelines
##### [nf-core](https://github.com/nf-core)/[modules](https://github.com/nf-core/modules)

ID = 561, Stars = 16, Language = Nextflow, Update = 2020-03-20

Repository to host tool-specific module files for the Nextflow DSL2 community!
##### [broadinstitute](https://github.com/broadinstitute)/[sam](https://github.com/broadinstitute/sam)

ID = 562, Stars = 16, Language = Scala, Update = 2020-05-07

workbench identity and access management
##### [broadinstitute](https://github.com/broadinstitute)/[cromwell-tools](https://github.com/broadinstitute/cromwell-tools)

ID = 563, Stars = 16, Language = Python, Update = 2020-02-21

A collection of Python clients and accessory scripts for interacting with the Cromwell
##### [broadinstitute](https://github.com/broadinstitute)/[wordpress-crowd-plugin](https://github.com/broadinstitute/wordpress-crowd-plugin)

ID = 564, Stars = 16, Language = PHP, Update = 2018-01-11

Crowd Authentication Plugin for Wordpress
##### [quinlan-lab](https://github.com/quinlan-lab)/[pathoscore](https://github.com/quinlan-lab/pathoscore)

ID = 565, Stars = 16, Language = Python, Update = 2019-06-29

pathoscore evaluates variant pathogenicity tools and scores.
##### [papenfusslab](https://github.com/papenfusslab)/[sv_benchmark](https://github.com/PapenfussLab/sv_benchmark)

ID = 566, Stars = 16, Language = R, Update = 2019-07-01

Comprehensive benchmark of structural variant callers
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[disambiguate](https://github.com/AstraZeneca-NGS/disambiguate)

ID = 567, Stars = 16, Language = C++, Update = 2018-05-08

Disambiguation algorithm for reads aligned to human and mouse genomes using Tophat or BWA mem
##### [wglab](https://github.com/wglab)/[RepeatHMM](https://github.com/WGLab/RepeatHMM)

ID = 568, Stars = 16, Language = Python, Update = 2020-02-25

a hidden Markov model to infer simple repeats from genome sequences
##### [ncbi](https://github.com/ncbi)/[magicblast](https://github.com/ncbi/magicblast)

ID = 569, Stars = 16, Language = Python, Update = 2020-02-27

None
##### [ncbi](https://github.com/ncbi)/[JUDI](https://github.com/ncbi/JUDI)

ID = 570, Stars = 16, Language = Python, Update = 2019-12-13

This repository contains the source code of JUDI, a workflow management system for developing complex bioinformatics software with many parameter settings. Bioinformatics pipeline: Just Do It!
##### [yongxinliu](https://github.com/yongxinliu)/[Amplicon2017](https://github.com/YongxinLiu/Amplicon2017)

ID = 571, Stars = 15, Language = HTML, Update = 2019-03-10

Data and scripts for amplicon pipeline
##### [lh3](https://github.com/lh3)/[calN50](https://github.com/lh3/calN50)

ID = 572, Stars = 15, Language = JavaScript, Update = 2020-04-09

Compute N50/NG50 and auN/auNG
##### [sigven](https://github.com/sigven)/[cacao](https://github.com/sigven/cacao)

ID = 573, Stars = 15, Language = R, Update = 2020-05-10

Callable Cancer Loci - assessment of sequencing coverage for actionable and pathogenic loci in cancer
##### [stephenturner](https://github.com/stephenturner)/[hcop](https://github.com/stephenturner/hcop)

ID = 574, Stars = 15, Language = R, Update = 2018-03-22

HGNC Comparison of Orthology Predictions (HCOP)
##### [statgen](https://github.com/statgen)/[popscle](https://github.com/statgen/popscle)

ID = 575, Stars = 15, Language = C++, Update = 2020-03-02

A suite of population scale analysis tools for single-cell genomics data including implementation of Demuxlet / Freemuxlet methods and auxilary tools
##### [openvax](https://github.com/openvax)/[isovar](https://github.com/openvax/isovar)

ID = 576, Stars = 15, Language = Python, Update = 2020-02-11

Assembly of RNA reads to determine the effect of a cancer mutation on protein sequence
##### [vgteam](https://github.com/vgteam)/[xg-old](https://github.com/vgteam/xg-old)

ID = 577, Stars = 15, Language = C++, Update = 2018-11-18

succinct labeled graphs with collections and paths
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[HiCAssembler](https://github.com/maxplanck-ie/HiCAssembler)

ID = 578, Stars = 15, Language = Python, Update = 2019-11-07

Software to assemble contigs/scaffolds into chromosomes using Hi-C data
##### [biocore](https://github.com/biocore)/[micronota](https://github.com/biocore/micronota)

ID = 579, Stars = 15, Language = Python, Update = 2018-02-23

annotation pipeline for microbial genomes and metagenomes
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbbam](https://github.com/PacificBiosciences/pbbam)

ID = 580, Stars = 15, Language = C++, Update = 2020-05-10

PacBio BAM C++ library
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[R-kinetics](https://github.com/PacificBiosciences/R-kinetics)

ID = 581, Stars = 15, Language = R, Update = 2017-10-02

A project housing some code and documentation for performing kinetics analyses using R.
##### [illumina](https://github.com/illumina)/[GraphAlignmentViewer](https://github.com/Illumina/GraphAlignmentViewer)

ID = 582, Stars = 15, Language = Python, Update = 2020-03-30

None
##### [illumina](https://github.com/illumina)/[SMNCopyNumberCaller](https://github.com/Illumina/SMNCopyNumberCaller)

ID = 583, Stars = 15, Language = Python, Update = 2020-03-13

A copy number caller for SMN1 and SMN2 to enable SMA diagnosis and carrier screening with WGS
##### [illumina](https://github.com/illumina)/[BaseSpace_Clarity_LIMS](https://github.com/Illumina/BaseSpace_Clarity_LIMS)

ID = 584, Stars = 15, Language = Python, Update = 2019-07-23

API libraries, application examples, and custom tools for BaseSpace Clarity LIMS
##### [yongxinliu](https://github.com/yongxinliu)/[Zhang2019NBT](https://github.com/YongxinLiu/Zhang2019NBT)

ID = 585, Stars = 14, Language = HTML, Update = 2019-04-12

Scripts for stat and plot figures in rice microbiome paper
##### [gregoryfaust](https://github.com/gregoryfaust)/[SVsim](https://github.com/GregoryFaust/SVsim) *new+*

ID = 586, Stars = 14, Language = Python, Update = 2017-12-04

SVsim: a tool that generates synthetic Structural Variant calls as benchmarks to test/evaluate SV calling pipelines.
##### [vals](https://github.com/vals)/[Blog](https://github.com/vals/Blog)

ID = 587, Stars = 14, Language = Jupyter Notebook, Update = 2020-05-10

None
##### [seandavi](https://github.com/seandavi)/[SRAdb](https://github.com/seandavi/SRAdb)

ID = 588, Stars = 14, Language = R, Update = 2018-02-20

Git mirror of Bioconductor SRAdb package
##### [richarddurbin](https://github.com/richarddurbin)/[hash10x](https://github.com/richarddurbin/hash10x)

ID = 589, Stars = 14, Language = C, Update = 2019-11-27

a toolset for efficient analysis of 10X Genomics linked read data sets, in particular for de novo assembly
##### [tseemann](https://github.com/tseemann)/[phastaf](https://github.com/tseemann/phastaf)

ID = 590, Stars = 14, Language = Perl, Update = 2019-09-05

Identify phage regions in bacterial genomes for masking purposes
##### [tseemann](https://github.com/tseemann)/[sixess](https://github.com/tseemann/sixess)

ID = 591, Stars = 14, Language = Shell, Update = 2018-04-24

üî¨
üêõ
Rapid 16s rRNA identification from isolate FASTQ files
##### [brentp](https://github.com/brentp)/[lua-stringy](https://github.com/brentp/lua-stringy)

ID = 592, Stars = 14, Language = C, Update = 2020-03-21

fast lua string operations
##### [nf-core](https://github.com/nf-core)/[hic](https://github.com/nf-core/hic)

ID = 593, Stars = 14, Language = Python, Update = 2020-05-04

Analysis of Chromosome Conformation Capture data (Hi-C)
##### [qiime2](https://github.com/qiime2)/[q2cli](https://github.com/qiime2/q2cli)

ID = 594, Stars = 14, Language = Python, Update = 2020-04-29

Command line interface for QIIME 2
##### [parklab](https://github.com/parklab)/[MSIprofiler](https://github.com/parklab/MSIprofiler)

ID = 595, Stars = 14, Language = Python, Update = 2019-10-03

Repo that aids in the detection of microsatellite instabilities (MSI) from sequencing data
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[parkour](https://github.com/maxplanck-ie/parkour)

ID = 596, Stars = 14, Language = JavaScript, Update = 2020-03-13

An open-source Laboratory Information Management System (LIMS) for Next-Generation Sequencing (NGS)
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbalign](https://github.com/PacificBiosciences/pbalign)

ID = 597, Stars = 14, Language = Python, Update = 2018-12-11

pbalign maps PacBio reads to reference sequences and saves alignments to a BAM file
##### [ga4gh](https://github.com/ga4gh)/[vr-python](https://github.com/ga4gh/vr-python)

ID = 598, Stars = 14, Language = Jupyter Notebook, Update = 2020-04-27

GA4GH Variation Representation Python Implementation
##### [ncbi](https://github.com/ncbi)/[pipelines](https://github.com/ncbi/pipelines)

ID = 599, Stars = 14, Language = Common Workflow Language, Update = 2018-11-28

None
##### [crazyhottommy](https://github.com/crazyhottommy)/[papers_with_data_to_mine](https://github.com/crazyhottommy/papers_with_data_to_mine)

ID = 600, Stars = 13, Language = None, Update = 2020-04-20

published papers with a lot of data
##### [crazyhottommy](https://github.com/crazyhottommy)/[scATACutils](https://github.com/crazyhottommy/scATACutils)

ID = 601, Stars = 13, Language = R, Update = 2020-01-10

R/Bioconductor package for working with 10x scATACseq data
##### [ekg](https://github.com/ekg)/[tabixpp](https://github.com/ekg/tabixpp)

ID = 602, Stars = 13, Language = C++, Update = 2019-12-13

C++ wrapper to tabix indexer
##### [ming-lian](https://github.com/ming-lian)/[Memo](https://github.com/Ming-Lian/Memo)

ID = 603, Stars = 13, Language = None, Update = 2019-12-01

Â≠¶‰π†Á¨îËÆ∞ÔºöÊàëÁöÑÁ¨¨‰∏Ä‰∏™github‰ªìÂ∫ì
##### [gt1](https://github.com/gt1)/[libmaus2](https://github.com/gt1/libmaus2)

ID = 604, Stars = 13, Language = C++, Update = 2019-03-19

collection of data structures and algorithms
##### [ocxtal](https://github.com/ocxtal)/[dozeu](https://github.com/ocxtal/dozeu)

ID = 605, Stars = 13, Language = C, Update = 2020-02-09

SIMD-parallel BLAST X-drop DP on sequence graphs
##### [tseemann](https://github.com/tseemann)/[samclip](https://github.com/tseemann/samclip)

ID = 606, Stars = 13, Language = Perl, Update = 2020-03-07

Filter SAM file for soft and hard clipped alignments
##### [tseemann](https://github.com/tseemann)/[berokka](https://github.com/tseemann/berokka)

ID = 607, Stars = 13, Language = Perl, Update = 2019-09-07

üçä
üí´
Trim, circularise and orient long read bacterial genome assemblies
##### [brentp](https://github.com/brentp)/[python-giggle](https://github.com/brentp/python-giggle)

ID = 608, Stars = 13, Language = Python, Update = 2018-02-13

python (cython) wrapper for https://github.com/ryanlayer/giggle for fast interval searching of huge datasets.
##### [broadinstitute](https://github.com/broadinstitute)/[rawls](https://github.com/broadinstitute/rawls)

ID = 609, Stars = 13, Language = Scala, Update = 2020-05-08

Rawls service for DSDE
##### [broadinstitute](https://github.com/broadinstitute)/[genomics-in-the-cloud](https://github.com/broadinstitute/genomics-in-the-cloud)

ID = 610, Stars = 13, Language = Jupyter Notebook, Update = 2020-04-07

Source code and related materials for the O'Reilly book
##### [broadinstitute](https://github.com/broadinstitute)/[dalmatian](https://github.com/broadinstitute/dalmatian)

ID = 611, Stars = 13, Language = Python, Update = 2020-03-27

dalmatian is a collection of high-level companion functions for Firecloud and FISS.
##### [broadinstitute](https://github.com/broadinstitute)/[gatk-docs](https://github.com/broadinstitute/gatk-docs)

ID = 612, Stars = 13, Language = HTML, Update = 2020-01-06

Documentation archive for GATK tools and workflows
##### [yanglab](https://github.com/yanglab)/[bamTobw](https://github.com/YangLab/bamTobw)

ID = 613, Stars = 13, Language = Shell, Update = 2018-11-30

Convert BAM files to bigWig files with a simple command
##### [refresh-bio](https://github.com/refresh-bio)/[GTC](https://github.com/refresh-bio/GTC)

ID = 614, Stars = 13, Language = C++, Update = 2020-01-13

GenoTypes Compressor
##### [wglab](https://github.com/wglab)/[Phen2Gene](https://github.com/WGLab/Phen2Gene)

ID = 615, Stars = 13, Language = Python, Update = 2020-05-01

Phenotype driven gene prioritization for HPO
##### [wglab](https://github.com/wglab)/[icages](https://github.com/WGLab/icages)

ID = 616, Stars = 13, Language = Perl, Update = 2019-03-21

iCAGES (integrated CAncer GEnome Score) is an effective tool for prioritizing cancer driver genes for a patient
##### [biocore](https://github.com/biocore)/[qurro](https://github.com/biocore/qurro)

ID = 617, Stars = 13, Language = JavaScript, Update = 2020-05-06

Visualize differentially ranked features (taxa, metabolites, ...) and their log-ratios across samples
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[apps-scripts](https://github.com/PacificBiosciences/apps-scripts)

ID = 618, Stars = 13, Language = Python, Update = 2020-04-08

Miscellaneous scripts for applications of PacBio systems
##### [illumina](https://github.com/illumina)/[witty.er](https://github.com/Illumina/witty.er)

ID = 619, Stars = 13, Language = C#, Update = 2020-04-13

What is true, thank you, ernestly. A large variant benchmarking tool analogous to hap.py for small variants.
##### [lh3](https://github.com/lh3)/[lh3-snippets](https://github.com/lh3/lh3-snippets)

ID = 620, Stars = 12, Language = C, Update = 2020-01-23

None
##### [lh3](https://github.com/lh3)/[gfa1](https://github.com/lh3/gfa1)

ID = 621, Stars = 12, Language = C, Update = 2018-08-17

This repo is deprecated. Please use gfatools instead.
##### [ekg](https://github.com/ekg)/[viral-assembly](https://github.com/ekg/viral-assembly)

ID = 622, Stars = 12, Language = None, Update = 2020-03-19

exploring viral genome assembly with variation graph tools
##### [stephenturner](https://github.com/stephenturner)/[msigdf](https://github.com/stephenturner/msigdf)

ID = 623, Stars = 12, Language = R, Update = 2018-12-03

Molecular Signatures Database (MSigDB) in a data frame
##### [statgen](https://github.com/statgen)/[savvy](https://github.com/statgen/savvy)

ID = 624, Stars = 12, Language = C++, Update = 2020-05-08

Interface to various variant calling formats.
##### [openvax](https://github.com/openvax)/[gene-lists](https://github.com/openvax/gene-lists)

ID = 625, Stars = 12, Language = None, Update = 2018-12-18

Gene lists related to cancer immunotherapy
##### [nf-core](https://github.com/nf-core)/[configs](https://github.com/nf-core/configs)

ID = 626, Stars = 12, Language = Nextflow, Update = 2020-05-07

Config files used to define parameters specific to compute environments at different Institutions
##### [nf-core](https://github.com/nf-core)/[scrnaseq](https://github.com/nf-core/scrnaseq)

ID = 627, Stars = 12, Language = Nextflow, Update = 2020-03-08

A single-cell RNAseq pipeline for 10X genomics data
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad_qc](https://github.com/broadinstitute/gnomad_qc)

ID = 628, Stars = 12, Language = Python, Update = 2020-05-07

None
##### [broadinstitute](https://github.com/broadinstitute)/[fiss](https://github.com/broadinstitute/fiss)

ID = 629, Stars = 12, Language = Python, Update = 2020-03-02

FireCloud Service Selector (FISS) -- Python bindings and CLI for FireCloud execution engine
##### [refresh-bio](https://github.com/refresh-bio)/[FaStore](https://github.com/refresh-bio/FaStore)

ID = 630, Stars = 12, Language = C++, Update = 2018-11-22

FaStore - high-performance FASTQ files compressor
##### [qiime2](https://github.com/qiime2)/[docs](https://github.com/qiime2/docs)

ID = 631, Stars = 12, Language = Python, Update = 2020-04-09

Source repository for https://docs.qiime2.org
##### [ensembl](https://github.com/ensembl)/[ensembl-funcgen](https://github.com/Ensembl/ensembl-funcgen)

ID = 632, Stars = 12, Language = Perl, Update = 2020-03-18

Ensembl Funcgen Perl API and SQL schema
##### [parklab](https://github.com/parklab)/[HiNT](https://github.com/parklab/HiNT)

ID = 633, Stars = 12, Language = Python, Update = 2020-02-17

HiC for copy Number variation and Translocation detection
##### [biocore](https://github.com/biocore)/[oecophylla](https://github.com/biocore/oecophylla)

ID = 634, Stars = 12, Language = Python, Update = 2019-04-03

shotgun pipeline
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[minorseq](https://github.com/PacificBiosciences/minorseq)

ID = 635, Stars = 12, Language = None, Update = 2020-04-22

Minor Variant Calling and Phasing Tools
##### [a-slide](https://github.com/a-slide)/[pycoMeth](https://github.com/a-slide/pycoMeth)

ID = 636, Stars = 11, Language = Python, Update = 2020-04-21

DNA methylation analysis downstream to Nanopolish for Oxford Nanopore DNA sequencing datasets
##### [dpryan79](https://github.com/dpryan79)/[SE-MEI](https://github.com/dpryan79/SE-MEI)

ID = 637, Stars = 11, Language = C, Update = 2019-12-14

Tools for finding mobile element insertions from single-end datasets
##### [vibansal](https://github.com/vibansal)/[crisp](https://github.com/vibansal/crisp)

ID = 638, Stars = 11, Language = C, Update = 2020-02-25

Code for multi-sample variant calling from sequence data of pooled or unpooled DNA samples
##### [brentp](https://github.com/brentp)/[bigwig-nim](https://github.com/brentp/bigwig-nim)

ID = 639, Stars = 11, Language = Nim, Update = 2020-05-02

command-line querying+conversion of bigwigs and a nim wrapper for dpryan's libbigwig
##### [vgteam](https://github.com/vgteam)/[libbdsg](https://github.com/vgteam/libbdsg)

ID = 640, Stars = 11, Language = C++, Update = 2020-05-05

Optimized sequence graph implementations for graph genomics
##### [nf-core](https://github.com/nf-core)/[nanoseq](https://github.com/nf-core/nanoseq)

ID = 641, Stars = 11, Language = Nextflow, Update = 2020-05-05

Nanopore demultiplexing, QC and alignment pipeline
##### [nf-core](https://github.com/nf-core)/[hlatyping](https://github.com/nf-core/hlatyping)

ID = 642, Stars = 11, Language = Nextflow, Update = 2020-02-12

Precision HLA typing from next-generation sequencing data
##### [broadinstitute](https://github.com/broadinstitute)/[cms](https://github.com/broadinstitute/cms)

ID = 643, Stars = 11, Language = Python, Update = 2020-04-24

Composite of Multiple Signals: tests for selection in meiotically recombinant populations
##### [dellytools](https://github.com/dellytools)/[svprops](https://github.com/dellytools/svprops)

ID = 644, Stars = 11, Language = C++, Update = 2020-04-14

Computes various SV statistics
##### [quinlan-lab](https://github.com/quinlan-lab)/[ccr](https://github.com/quinlan-lab/ccr)

ID = 645, Stars = 11, Language = Python, Update = 2018-12-19

Building the constrained coding regions (CCR) model
##### [umcugenetics](https://github.com/umcugenetics)/[IAP](https://github.com/UMCUGenetics/IAP)

ID = 646, Stars = 11, Language = Perl, Update = 2020-04-06

Illumina analysis pipeline
##### [qiime2](https://github.com/qiime2)/[q2studio](https://github.com/qiime2/q2studio)

ID = 647, Stars = 11, Language = JavaScript, Update = 2020-05-06

Prototype graphical user interface for QIIME 2
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[bam2fastx](https://github.com/PacificBiosciences/bam2fastx)

ID = 648, Stars = 11, Language = Python, Update = 2020-04-23

Converting and demultiplexing of PacBio BAM files into gzipped fasta and fastq files.
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[kineticsTools](https://github.com/PacificBiosciences/kineticsTools)

ID = 649, Stars = 11, Language = Python, Update = 2020-04-21

Tools for detecting DNA modifications from single molecule, real-time sequencing data
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pypeFLOW](https://github.com/PacificBiosciences/pypeFLOW)

ID = 650, Stars = 11, Language = HTML, Update = 2019-09-05

a simple lightweight workflow engine for data analysis scripting
##### [ga4gh](https://github.com/ga4gh)/[data-security](https://github.com/ga4gh/data-security)

ID = 651, Stars = 11, Language = None, Update = 2020-03-30

None
##### [ncbi](https://github.com/ncbi)/[blast_plus_docs](https://github.com/ncbi/blast_plus_docs)

ID = 652, Stars = 11, Language = None, Update = 2020-04-27

None
##### [ncbi](https://github.com/ncbi)/[osiris](https://github.com/ncbi/osiris)

ID = 653, Stars = 11, Language = C++, Update = 2020-04-17

OSIRIS is a public domain quality assurance software package that facilitates the assessment of multiplex short tandem repeat (STR) DNA profiles based on laboratory-specific protocols. OSIRIS evaluates the raw electrophoresis data contained in .fsa or .hid files using an independently derived mathematically-based sizing algorithm. OSIRIS current‚Ä¶
##### [ekg](https://github.com/ekg)/[ACAD18](https://github.com/ekg/ACAD18)

ID = 654, Stars = 10, Language = None, Update = 2018-11-27

Day 2 of ACAD's 2018 Advanced Bioinformatics Workshop
##### [vals](https://github.com/vals)/[python-gprofiler](https://github.com/vals/python-gprofiler)

ID = 655, Stars = 10, Language = Python, Update = 2019-02-16

None
##### [brentp](https://github.com/brentp)/[nim-kmer](https://github.com/brentp/nim-kmer)

ID = 656, Stars = 10, Language = Nim, Update = 2020-04-11

DNA kmer operations for nim
##### [statgen](https://github.com/statgen)/[encore](https://github.com/statgen/encore)

ID = 657, Stars = 10, Language = Python, Update = 2020-01-14

Encore Analysis Server
##### [statgen](https://github.com/statgen)/[topmed_freeze3_calling](https://github.com/statgen/topmed_freeze3_calling)

ID = 658, Stars = 10, Language = C, Update = 2018-12-05

TOPMed Freeze 3 variant calling pipeline
##### [vgteam](https://github.com/vgteam)/[libhandlegraph](https://github.com/vgteam/libhandlegraph)

ID = 659, Stars = 10, Language = C++, Update = 2020-05-05

Library for the Handle Graph abstraction
##### [vgteam](https://github.com/vgteam)/[sv-genotyping-paper](https://github.com/vgteam/sv-genotyping-paper)

ID = 660, Stars = 10, Language = Python, Update = 2019-11-25

None
##### [nf-core](https://github.com/nf-core)/[smrnaseq](https://github.com/nf-core/smrnaseq)

ID = 661, Stars = 10, Language = Nextflow, Update = 2020-04-15

A small-RNA sequencing analysis pipeline
##### [nf-core](https://github.com/nf-core)/[bcellmagic](https://github.com/nf-core/bcellmagic)

ID = 662, Stars = 10, Language = Nextflow, Update = 2020-02-20

B cell repertoire sequencing analysis pipeline using the immcantation framework
##### [gatk-workflows](https://github.com/gatk-workflows)/[seq-format-conversion](https://github.com/gatk-workflows/seq-format-conversion)

ID = 663, Stars = 10, Language = wdl, Update = 2020-03-09

Workflows for converting between sequence data formats
##### [umcugenetics](https://github.com/umcugenetics)/[NextflowModules](https://github.com/UMCUGenetics/NextflowModules)

ID = 664, Stars = 10, Language = Nextflow, Update = 2020-05-07

UMCU Genetics Nextflow modules
##### [refresh-bio](https://github.com/refresh-bio)/[fqsqueezer](https://github.com/refresh-bio/fqsqueezer)

ID = 665, Stars = 10, Language = C++, Update = 2019-05-17

FQSqueezer - FASTQ compressor based on k-mer statistics
##### [qiime2](https://github.com/qiime2)/[q2-dada2](https://github.com/qiime2/q2-dada2)

ID = 666, Stars = 10, Language = Python, Update = 2020-04-10

QIIME 2 plugin wrapping DADA2
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[simple_sv_annotation](https://github.com/AstraZeneca-NGS/simple_sv_annotation)

ID = 667, Stars = 10, Language = Python, Update = 2019-02-18

Simplify snpEff annotations for interesting cases
##### [parklab](https://github.com/parklab)/[LiRA](https://github.com/parklab/LiRA)

ID = 668, Stars = 10, Language = R, Update = 2019-05-17

None
##### [wglab](https://github.com/wglab)/[LinkedSV](https://github.com/WGLab/LinkedSV)

ID = 669, Stars = 10, Language = C, Update = 2020-04-25

None
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[ANGEL](https://github.com/PacificBiosciences/ANGEL)

ID = 670, Stars = 10, Language = Python, Update = 2019-05-13

Robust Open Reading Frame prediction (ANGLE re-implementation)
##### [ncbi](https://github.com/ncbi)/[cxx-toolkit](https://github.com/ncbi/cxx-toolkit)

ID = 671, Stars = 10, Language = HTML, Update = 2020-05-08

None
##### [ncbi](https://github.com/ncbi)/[scPopCorn](https://github.com/ncbi/scPopCorn)

ID = 672, Stars = 10, Language = Jupyter Notebook, Update = 2019-10-25

A python tool to do comparative analysis of mulitple single cell datasets.
##### [ekg](https://github.com/ekg)/[mmmulti](https://github.com/ekg/mmmulti)

ID = 673, Stars = 9, Language = C++, Update = 2020-04-28

memory mapped multimap, multiset, and implicit interval tree based on an in-place parallel sort
##### [benjjneb](https://github.com/benjjneb)/[LRASManuscript](https://github.com/benjjneb/LRASManuscript)

ID = 674, Stars = 9, Language = HTML, Update = 2019-06-29

Reproducible Analyses accompanying DADA2 + PacBio Manuscript
##### [danielecook](https://github.com/danielecook)/[seq-collection](https://github.com/danielecook/seq-collection)

ID = 675, Stars = 9, Language = Nim, Update = 2020-05-10

None
##### [ewels](https://github.com/ewels)/[MultiQC_TestData](https://github.com/ewels/MultiQC_TestData)

ID = 676, Stars = 9, Language = HTML, Update = 2020-05-08

Test data for MultiQC. See https://github.com/ewels/MultiQC
##### [broadinstitute](https://github.com/broadinstitute)/[agora](https://github.com/broadinstitute/agora)

ID = 677, Stars = 9, Language = Scala, Update = 2020-02-04

It's a methods repository!
##### [broadinstitute](https://github.com/broadinstitute)/[vim-wdl](https://github.com/broadinstitute/vim-wdl)

ID = 678, Stars = 9, Language = Vim script, Update = 2020-01-29

Vim syntax highlighting for WDL
##### [broadinstitute](https://github.com/broadinstitute)/[scRNA-Seq](https://github.com/broadinstitute/scRNA-Seq)

ID = 679, Stars = 9, Language = Python, Update = 2019-07-26

None
##### [broadinstitute](https://github.com/broadinstitute)/[hdf5-java-bindings](https://github.com/broadinstitute/hdf5-java-bindings)

ID = 680, Stars = 9, Language = Java, Update = 2019-05-20

java bindings for hdf5
##### [deeptools](https://github.com/deeptools)/[HiCBrowser](https://github.com/deeptools/HiCBrowser)

ID = 681, Stars = 9, Language = JavaScript, Update = 2019-12-12

Simple web browser to visualize HiC tracks
##### [deeptools](https://github.com/deeptools)/[docker-galaxy-hicexplorer](https://github.com/deeptools/docker-galaxy-hicexplorer)

ID = 682, Stars = 9, Language = HTML, Update = 2019-03-03

Galaxy Docker Image for the HiCExplorer
##### [qiime2](https://github.com/qiime2)/[q2-fragment-insertion](https://github.com/qiime2/q2-fragment-insertion)

ID = 683, Stars = 9, Language = Python, Update = 2020-03-05

QIIME 2 plugin for fragment insertion
##### [qiime2](https://github.com/qiime2)/[q2cwl](https://github.com/qiime2/q2cwl)

ID = 684, Stars = 9, Language = Python, Update = 2020-02-13

Prototype interface for automatically generating CWL tools from QIIME 2 actions
##### [ensembl](https://github.com/ensembl)/[ensembl-production](https://github.com/Ensembl/ensembl-production)

ID = 685, Stars = 9, Language = Perl, Update = 2020-05-07

Ensembl Production code
##### [ensembl](https://github.com/ensembl)/[ensembl-analysis](https://github.com/Ensembl/ensembl-analysis)

ID = 686, Stars = 9, Language = Perl, Update = 2020-04-30

Modules to interface with tools used in Ensembl Gene Annotation Process and scripts to run pipelines
##### [ga4gh](https://github.com/ga4gh)/[cloud-interop-testing](https://github.com/ga4gh/cloud-interop-testing)

ID = 687, Stars = 9, Language = Python, Update = 2019-10-05

Interoperable execution of workflows using GA4GH APIs
##### [ga4gh](https://github.com/ga4gh)/[metadata-team](https://github.com/ga4gh/metadata-team)

ID = 688, Stars = 9, Language = None, Update = 2019-03-20

None
##### [ncbi](https://github.com/ncbi)/[gprobe](https://github.com/ncbi/gprobe)

ID = 689, Stars = 9, Language = Go, Update = 2020-01-31

client app for the gRPC health-checking protocol
##### [illumina](https://github.com/illumina)/[happyR](https://github.com/Illumina/happyR)

ID = 690, Stars = 9, Language = R, Update = 2019-07-12

R tools to interact with hap.py output
##### [yongxinliu](https://github.com/yongxinliu)/[UsearchChineseManual](https://github.com/YongxinLiu/UsearchChineseManual)

ID = 691, Stars = 8, Language = None, Update = 2019-09-26

USEARCH‰∏≠ÊñáÂ∏ÆÂä©ÊñáÊ°£(USEARCH Chinese manual)
##### [lh3](https://github.com/lh3)/[lh3.github.com](https://github.com/lh3/lh3.github.com)

ID = 692, Stars = 8, Language = HTML, Update = 2020-04-08

None
##### [lh3](https://github.com/lh3)/[unicall](https://github.com/lh3/unicall)

ID = 693, Stars = 8, Language = Perl, Update = 2019-07-21

A wrapper for calling small variants from human germline high-coverage single-sample Illumina data
##### [crazyhottommy](https://github.com/crazyhottommy)/[pyflow-scATACseq](https://github.com/crazyhottommy/pyflow-scATACseq)

ID = 694, Stars = 8, Language = Python, Update = 2020-02-03

snakemake workflow for post-processing scATACseq data
##### [ekg](https://github.com/ekg)/[gimbricate](https://github.com/ekg/gimbricate)

ID = 695, Stars = 8, Language = C++, Update = 2020-01-29

recompute GFA link overlaps
##### [danielecook](https://github.com/danielecook)/[tut](https://github.com/danielecook/tut)

ID = 696, Stars = 8, Language = Nim, Update = 2020-05-01

A collection of CSV/TSV Utilities
##### [seandavi](https://github.com/seandavi)/[BiocActions](https://github.com/seandavi/BiocActions)

ID = 697, Stars = 8, Language = Dockerfile, Update = 2019-12-03

Develop and host GitHub actions for Bioconductor
##### [statgen](https://github.com/statgen)/[swiss](https://github.com/statgen/swiss)

ID = 698, Stars = 8, Language = Python, Update = 2019-11-11

Software to help identify overlap between association scan results and GWAS hit catalogs.
##### [nf-core](https://github.com/nf-core)/[bacass](https://github.com/nf-core/bacass)

ID = 699, Stars = 8, Language = Nextflow, Update = 2020-03-16

Simple bacterial assembly and annotation pipeline
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-mitochondria-pipeline](https://github.com/gatk-workflows/gatk4-mitochondria-pipeline)

ID = 700, Stars = 8, Language = wdl, Update = 2020-03-09

Workflow for SNP and INDEL variant calling on mitochondria.
##### [ensembl](https://github.com/ensembl)/[ensj-healthcheck](https://github.com/Ensembl/ensj-healthcheck)

ID = 701, Stars = 8, Language = Java, Update = 2020-04-22

Ensembl's Automated QC Framework
##### [ncbi](https://github.com/ncbi)/[blast-cloud](https://github.com/ncbi/blast-cloud)

ID = 702, Stars = 8, Language = CSS, Update = 2019-12-04

Documentation for NCBI BLAST AMI
##### [a-slide](https://github.com/a-slide)/[NanoCount](https://github.com/a-slide/NanoCount)

ID = 703, Stars = 7, Language = Python, Update = 2020-05-05

EM based transcript abundance from nanopore reads mapped to a transcriptome with minimap2
##### [a-slide](https://github.com/a-slide)/[NanopolishComp](https://github.com/a-slide/NanopolishComp)

ID = 704, Stars = 7, Language = Python, Update = 2019-09-10

NanopolishComp is a Python3 package for downstream analyses of Nanopolish output files
##### [sigven](https://github.com/sigven)/[oncoEnrichR](https://github.com/sigven/oncoEnrichR)

ID = 705, Stars = 7, Language = R, Update = 2020-05-10

Functional analysis and prioritization of genesets in the context of cancer
##### [stephenturner](https://github.com/stephenturner)/[Tmisc](https://github.com/stephenturner/Tmisc)

ID = 706, Stars = 7, Language = R, Update = 2020-03-26

Turner miscellaneous
##### [richarddurbin](https://github.com/richarddurbin)/[phynder](https://github.com/richarddurbin/phynder)

ID = 707, Stars = 7, Language = C, Update = 2020-05-11

likelihood calculations over a tree to support ancient DNA Y chr analysis and more
##### [statgen](https://github.com/statgen)/[METAL](https://github.com/statgen/METAL)

ID = 708, Stars = 7, Language = C++, Update = 2020-05-05

Meta-analysis of genomewide association scans
##### [nf-core](https://github.com/nf-core)/[proteomicslfq](https://github.com/nf-core/proteomicslfq)

ID = 709, Stars = 7, Language = Nextflow, Update = 2020-05-06

UNDER CONSTRUCTION - Proteomics label-free quantification (LFQ) analysis pipeline
##### [broadinstitute](https://github.com/broadinstitute)/[viral-pipelines](https://github.com/broadinstitute/viral-pipelines)

ID = 710, Stars = 7, Language = wdl, Update = 2020-05-10

viral-ngs: complete pipelines
##### [broadinstitute](https://github.com/broadinstitute)/[cell-health](https://github.com/broadinstitute/cell-health)

ID = 711, Stars = 7, Language = Jupyter Notebook, Update = 2020-05-07

Predicting Cell Health with Morphological Profiles
##### [broadinstitute](https://github.com/broadinstitute)/[getzlab-SignatureAnalyzer](https://github.com/broadinstitute/getzlab-SignatureAnalyzer)

ID = 712, Stars = 7, Language = Python, Update = 2020-04-29

Updated SignatureAnalyzer-GPU with mutational spectra & RNA expression compatibility.
##### [broadinstitute](https://github.com/broadinstitute)/[sparklespray](https://github.com/broadinstitute/sparklespray)

ID = 713, Stars = 7, Language = HTML, Update = 2020-04-24

Easy batch submission of adhoc jobs onto GCP
##### [broadinstitute](https://github.com/broadinstitute)/[2020_scWorkshop](https://github.com/broadinstitute/2020_scWorkshop)

ID = 714, Stars = 7, Language = Shell, Update = 2020-04-15

Code and data repository for the 2020 physalia course on single cell RNA sequencing.
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-tools](https://github.com/broadinstitute/firecloud-tools)

ID = 715, Stars = 7, Language = Python, Update = 2020-04-13

Tools for use with FireCloud
##### [broadinstitute](https://github.com/broadinstitute)/[python-cert_manager](https://github.com/broadinstitute/python-cert_manager)

ID = 716, Stars = 7, Language = Python, Update = 2020-04-10

Python interface to the Sectigo Certificate Manager REST API
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-v8](https://github.com/broadinstitute/gtex-v8)

ID = 717, Stars = 7, Language = Jupyter Notebook, Update = 2020-02-03

Notebooks and scripts for reproducing analyses and figures from the V8 GTEx Consortium paper
##### [mathonco](https://github.com/mathonco)/[EvoFreq](https://github.com/MathOnco/EvoFreq)

ID = 718, Stars = 7, Language = R, Update = 2020-04-08

Evolutionary frequency visualization tool of temporal data
##### [qiime2](https://github.com/qiime2)/[q2-longitudinal](https://github.com/qiime2/q2-longitudinal)

ID = 719, Stars = 7, Language = Python, Update = 2020-04-08

QIIME 2 plugin for paired sample comparisons
##### [qiime2](https://github.com/qiime2)/[q2view](https://github.com/qiime2/q2view)

ID = 720, Stars = 7, Language = JavaScript, Update = 2020-02-28

A web-based "serverless" viewer of QIIME 2 Artifacts and Visualizations
##### [ensembl](https://github.com/ensembl)/[trackhub-registry](https://github.com/Ensembl/trackhub-registry)

ID = 721, Stars = 7, Language = Perl, Update = 2020-03-18

Specifications and implementation of the TrackHub registry
##### [ensembl](https://github.com/ensembl)/[ensembl-git-tools](https://github.com/Ensembl/ensembl-git-tools)

ID = 722, Stars = 7, Language = Perl, Update = 2019-12-09

A collection of tools which Ensembl uses to work with Git
##### [maxplanck-ie](https://github.com/maxplanck-ie)/[TheWhoTheWhatTheHuh](https://github.com/maxplanck-ie/TheWhoTheWhatTheHuh)

ID = 723, Stars = 7, Language = Python, Update = 2019-10-16

"The who the what the huh?" is our pipeline for converting bcl files to fastq and performing QC.
##### [biocore](https://github.com/biocore)/[empress](https://github.com/biocore/empress)

ID = 724, Stars = 7, Language = JavaScript, Update = 2020-05-09

A very fast and scalable phylogenetic visualization viewer
##### [biocore](https://github.com/biocore)/[q2-qemistree](https://github.com/biocore/q2-qemistree)

ID = 725, Stars = 7, Language = Python, Update = 2020-04-30

Hierarchical orderings for mass spectrometry data. Canonically pronounced "chemis-tree".
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[pbcommand](https://github.com/PacificBiosciences/pbcommand)

ID = 726, Stars = 7, Language = Python, Update = 2020-05-06

PacBio common models, CLI tool contract interface and SMRT Link Service Interface
##### [pacificbiosciences](https://github.com/pacificbiosciences)/[hg002-ccs](https://github.com/PacificBiosciences/hg002-ccs)

ID = 727, Stars = 7, Language = Python, Update = 2020-04-16

None
##### [biowdl](https://github.com/biowdl)/[RNA-seq](https://github.com/biowdl/RNA-seq)

ID = 728, Stars = 7, Language = wdl, Update = 2020-05-08

A BioWDL pipeline for processing RNA-seq data, starting with FASTQ files to produce expression measures and VCFs. Category:Multi-Sample
##### [biowdl](https://github.com/biowdl)/[germline-DNA](https://github.com/biowdl/germline-DNA)

ID = 729, Stars = 7, Language = wdl, Update = 2020-04-21

A BioWDL variantcalling pipeline for germline DNA data. Starting with FASTQ files to produce VCF files. Category:Multi-Sample
##### [ncbi](https://github.com/ncbi)/[cwl-ngs-workflows-cbb](https://github.com/ncbi/cwl-ngs-workflows-cbb)

ID = 730, Stars = 7, Language = Common Workflow Language, Update = 2020-02-27

A set of CWL tools and workflows used by NCBI Computational Biology Branch for NGS data analysis
##### [ekg](https://github.com/ekg)/[guix-genomics](https://github.com/ekg/guix-genomics)

ID = 731, Stars = 6, Language = Scheme, Update = 2020-05-08

guix packages for bioinformatics software
##### [ekg](https://github.com/ekg)/[gyeet](https://github.com/ekg/gyeet)

ID = 732, Stars = 6, Language = C++, Update = 2020-04-22

sequence to graph mapper
##### [jodyphelan](https://github.com/jodyphelan)/[tbdb](https://github.com/jodyphelan/tbdb)

ID = 733, Stars = 6, Language = Python, Update = 2020-04-02

Standard database for the TBProfiler tool
##### [stephenturner](https://github.com/stephenturner)/[devnotes](https://github.com/stephenturner/devnotes)

ID = 734, Stars = 6, Language = Shell, Update = 2020-05-05

Random programming notes and orphan scripts
##### [ocxtal](https://github.com/ocxtal)/[tinymasker](https://github.com/ocxtal/tinymasker)

ID = 735, Stars = 6, Language = C, Update = 2019-12-09

ultrasensitive alignment for masking repeats on assembly contigs
##### [danielecook](https://github.com/danielecook)/[gg](https://github.com/danielecook/gg)

ID = 736, Stars = 6, Language = Go, Update = 2020-05-04

Github Gists from the command line. On MAC OSX, it (will be) an Alfred Workflow!
##### [danielecook](https://github.com/danielecook)/[codebox-alfred](https://github.com/danielecook/codebox-alfred)

ID = 737, Stars = 6, Language = Python, Update = 2020-01-11

An alfred workflow for accessing codebox snippets.
##### [felixkrueger](https://github.com/felixkrueger)/[nextflow_pipelines](https://github.com/FelixKrueger/nextflow_pipelines)

ID = 738, Stars = 6, Language = Nextflow, Update = 2020-04-07

The set of NGS processing pipelines used at Babraham
##### [brentp](https://github.com/brentp)/[hileup](https://github.com/brentp/hileup)

ID = 739, Stars = 6, Language = C, Update = 2019-12-06

horizontal pileup
##### [statgen](https://github.com/statgen)/[bravo](https://github.com/statgen/bravo)

ID = 740, Stars = 6, Language = HTML, Update = 2020-03-12

BRowse All Variants Online
##### [abyzovlab](https://github.com/abyzovlab)/[CNVpytor](https://github.com/abyzovlab/CNVpytor)

ID = 741, Stars = 6, Language = Python, Update = 2020-05-08

a python extension of CNVnator -- a tool for CNV analysis from depth-of-coverage by mapped reads
##### [vgteam](https://github.com/vgteam)/[toil-vg](https://github.com/vgteam/toil-vg)

ID = 742, Stars = 6, Language = Python, Update = 2020-04-30

Distributed and cloud computing framework for vg
##### [andersenlab](https://github.com/andersenlab)/[cegwas2-nf](https://github.com/AndersenLab/cegwas2-nf)

ID = 743, Stars = 6, Language = R, Update = 2020-05-03

GWA mapping with C. elegans
##### [broadinstitute](https://github.com/broadinstitute)/[covid19-testing](https://github.com/broadinstitute/covid19-testing)

ID = 744, Stars = 6, Language = HTML, Update = 2020-05-11

COVID-19 Diagnostic Processing Dashboard
##### [broadinstitute](https://github.com/broadinstitute)/[ccle_processing](https://github.com/broadinstitute/ccle_processing)

ID = 745, Stars = 6, Language = Jupyter Notebook, Update = 2020-05-05

What you need to process the Quarterly DepMap-Omics releases from Terra
##### [ding-lab](https://github.com/ding-lab)/[somaticwrapper](https://github.com/ding-lab/somaticwrapper)

ID = 746, Stars = 6, Language = Perl, Update = 2020-04-28

Detect somatic variants from tumor and normal exome data
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-genome-processing-pipeline](https://github.com/gatk-workflows/gatk4-genome-processing-pipeline)

ID = 747, Stars = 6, Language = wdl, Update = 2020-04-23

Workflows used for processing whole genome sequence data + germline variant calling.
##### [gatk-workflows](https://github.com/gatk-workflows)/[gatk4-jupyter-notebook-tutorials](https://github.com/gatk-workflows/gatk4-jupyter-notebook-tutorials)

ID = 748, Stars = 6, Language = Jupyter Notebook, Update = 2019-12-19

This repository contains Jupyter Notebooks containing GATK Best Practices Workflows
##### [deeptools](https://github.com/deeptools)/[HiCMatrix](https://github.com/deeptools/HiCMatrix)

ID = 749, Stars = 6, Language = Python, Update = 2020-05-08

None
##### [umcugenetics](https://github.com/umcugenetics)/[CHORD](https://github.com/UMCUGenetics/CHORD)

ID = 750, Stars = 6, Language = R, Update = 2020-05-08

An R package for predicting HR deficiency from mutation contexts
##### [umcugenetics](https://github.com/umcugenetics)/[sparqling-genomics](https://github.com/UMCUGenetics/sparqling-genomics)

ID = 751, Stars = 6, Language = Scheme, Update = 2020-05-07

Ontologies, RDF-generators, and a web interface for querying it all.
##### [mathonco](https://github.com/mathonco)/[HAL](https://github.com/MathOnco/HAL)

ID = 752, Stars = 6, Language = Java, Update = 2020-05-09

HAL (Hybrid Automata Library) is the one-stop shop for all of your hybrid modeling needs
##### [qiime2](https://github.com/qiime2)/[vm-playbooks](https://github.com/qiime2/vm-playbooks)

ID = 753, Stars = 6, Language = Shell, Update = 2020-03-05

Ansible playbooks for QIIME 2 environments
##### [ensembl](https://github.com/ensembl)/[ensembl-io](https://github.com/Ensembl/ensembl-io)

ID = 754, Stars = 6, Language = Perl, Update = 2020-05-07

File parsing and writing code for Ensembl
##### [xjtu-omics](https://github.com/xjtu-omics)/[msisensor-pro](https://github.com/xjtu-omics/msisensor-pro)

ID = 755, Stars = 6, Language = C++, Update = 2020-04-23

Microsatellite Instability (MSI) detection using high-throughput sequencing data.
##### [ncbi](https://github.com/ncbi)/[datasets](https://github.com/ncbi/datasets)

ID = 756, Stars = 6, Language = None, Update = 2020-04-29

NCBI Datasets is an experimental resource for finding and building datasets
##### [ming-lian](https://github.com/ming-lian)/[Hello-RaspberryPi](https://github.com/Ming-Lian/Hello-RaspberryPi)

ID = 757, Stars = 5, Language = None, Update = 2020-04-22

Áé©ËΩ¨ËÆ°ÁÆóÊú∫Á°¨‰ª∂
##### [tseemann](https://github.com/tseemann)/[cgmlst-dists](https://github.com/tseemann/cgmlst-dists)

ID = 758, Stars = 5, Language = C, Update = 2020-03-04

üêª
‚áî
üê®
Calculate distance matrix from ChewBBACA cgMLST allele call tables
##### [tseemann](https://github.com/tseemann)/[skrofula](https://github.com/tseemann/skrofula)

ID = 759, Stars = 5, Language = Perl, Update = 2020-03-02

Yet another M.tuberculosis typing and resistance tool, but for the impatient (not in-patient)
##### [statgen](https://github.com/statgen)/[topmed_variant_calling](https://github.com/statgen/topmed_variant_calling) *new+*

ID = 760, Stars = 5, Language = C++, Update = 2020-05-07

None
##### [statgen](https://github.com/statgen)/[LDServer](https://github.com/statgen/LDServer)

ID = 761, Stars = 5, Language = C++, Update = 2020-02-18

Fast API server for calculating linkage disequilibrium
##### [samtools](https://github.com/samtools)/[www.htslib.org](https://github.com/samtools/www.htslib.org)

ID = 762, Stars = 5, Language = HTML, Update = 2020-04-20

The http://www.htslib.org/ web site
##### [vgteam](https://github.com/vgteam)/[vg_wdl](https://github.com/vgteam/vg_wdl)

ID = 763, Stars = 5, Language = wdl, Update = 2020-05-10

Workflow Description Language (WDL) scripts for common vg workflows
##### [nf-core](https://github.com/nf-core)/[epitopeprediction](https://github.com/nf-core/epitopeprediction)

ID = 764, Stars = 5, Language = Python, Update = 2020-05-04

A bioinformatics best-practice analysis pipeline for epitope prediction and annotation
##### [broadinstitute](https://github.com/broadinstitute)/[dig-diabetes-portal](https://github.com/broadinstitute/dig-diabetes-portal)

ID = 765, Stars = 5, Language = JavaScript, Update = 2020-05-08

Diabetes portal for the Medical Population Genetics department at the Broad Institute
##### [broadinstitute](https://github.com/broadinstitute)/[barclay](https://github.com/broadinstitute/barclay)

ID = 766, Stars = 5, Language = Java, Update = 2020-04-28

Command line argument parser and online documentation generation utilities for java command line programs.
##### [broadinstitute](https://github.com/broadinstitute)/[gistic2](https://github.com/broadinstitute/gistic2)

ID = 767, Stars = 5, Language = MATLAB, Update = 2020-04-10

Genomic Identification of Significant Targets in Cancer (GISTIC), version 2
##### [broadinstitute](https://github.com/broadinstitute)/[wdl-runner](https://github.com/broadinstitute/wdl-runner)

ID = 768, Stars = 5, Language = Python, Update = 2020-03-23

Easily run WDL workflows on GCP
##### [umcugenetics](https://github.com/umcugenetics)/[guix-additions](https://github.com/UMCUGenetics/guix-additions)

ID = 769, Stars = 5, Language = Scheme, Update = 2020-05-04

None
##### [papenfusslab](https://github.com/papenfusslab)/[bionix](https://github.com/PapenfussLab/bionix)

ID = 770, Stars = 5, Language = Nix, Update = 2020-04-26

Functional highly reproducible bioinformatics pipelines
##### [qiime2](https://github.com/qiime2)/[q2-types](https://github.com/qiime2/q2-types)

ID = 771, Stars = 5, Language = Python, Update = 2020-03-26

Definitions of common QIIME 2 types.
##### [qiime2](https://github.com/qiime2)/[q2-feature-classifier](https://github.com/qiime2/q2-feature-classifier)

ID = 772, Stars = 5, Language = Python, Update = 2020-03-17

QIIME 2 plugin supporting taxonomic classification
##### [ensembl](https://github.com/ensembl)/[public-plugins](https://github.com/Ensembl/public-plugins)

ID = 773, Stars = 5, Language = HTML, Update = 2020-05-10

Plugins for an Ensembl website
##### [ensembl](https://github.com/ensembl)/[homebrew-ensembl](https://github.com/Ensembl/homebrew-ensembl)

ID = 774, Stars = 5, Language = Ruby, Update = 2020-04-30

Core formula for Ensembl
##### [ensembl](https://github.com/ensembl)/[ensembl-test](https://github.com/Ensembl/ensembl-test)

ID = 775, Stars = 5, Language = Perl, Update = 2020-04-17

Test libraries and harnesses used for running the Ensembl test suite
##### [biowdl](https://github.com/biowdl)/[somatic-variantcalling](https://github.com/biowdl/somatic-variantcalling)

ID = 776, Stars = 5, Language = wdl, Update = 2020-04-21

A pipeline for somatic variantcalling. Category:Single-Sample
##### [ncbi](https://github.com/ncbi)/[HYDROID](https://github.com/ncbi/HYDROID)

ID = 777, Stars = 5, Language = Python, Update = 2020-03-23

Python package for analyzing hydroxyl-radical footprinting experiments of DNA-protein complexes
##### [ncbi](https://github.com/ncbi)/[pm4ngs](https://github.com/ncbi/pm4ngs)

ID = 778, Stars = 5, Language = Jupyter Notebook, Update = 2020-02-27

Project Manager for NGS data analysis
