# Bioinfomatics Repositories

Collection of bioinformatician and their repositories.

- [Top50](https://github.com/xhqsm/bioinfomatics-repositories#Top50)
- [Repositories](https://github.com/xhqsm/bioinfomatics-repositories#repositories)

**Update 2020-09-22**

## Top50

| ID | User | Type |Name | Stars |
| :-: | :-: | :-: | :-: | :-: |
| 1 | [lh3](https://github.com/lh3) | One |Heng Li | 5269 |
| 2 | [broadinstitute](https://github.com/broadinstitute) | Org |Broad Institute | 5238 |
| 3 | [brentp](https://github.com/brentp) | One |Brent Pedersen | 3217 |
| 4 | [biopython](https://github.com/biopython) | Org |Biopython Project | 2567 |
| 5 | [samtools](https://github.com/samtools) | Org |samtools | 2479 |
| 6 | [shenwei356](https://github.com/shenwei356) | One |Wei Shen | 1836 |
| 7 | [arq5x](https://github.com/arq5x) | One |Aaron Quinlan | 1797 |
| 8 | [opengene](https://github.com/opengene) | Org |OpenGene - Open Source Genetics Toolbox | 1626 |
| 9 | [illumina](https://github.com/illumina) | Org |Illumina | 1618 |
| 10 | [ncbi](https://github.com/ncbi) | Org |NCBI - National Center for Biotechnology Information/NLM/NIH | 1564 |
| 11 | [bioconda](https://github.com/bioconda) | Org |Bioconda | 1283 |
| 12 | [ekg](https://github.com/ekg) | One |Erik Garrison | 1217 |
| 13 | [ewels](https://github.com/ewels) | One |Phil Ewels | 1052 |
| 14 | [deeptools](https://github.com/deeptools) | Org |The deepTools ecosystem | 1040 |
| 15 | [benlangmead](https://github.com/benlangmead) | One |Ben | 1022 |
| 16 | [igvteam](https://github.com/igvteam) | Org |igvteam | 977 |
| 17 | [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry) | Org |Edinburgh Genome Foundry | 968 |
| 18 | [ensembl](https://github.com/ensembl) | Org |Ensembl Project | 871 |
| 19 | [snakemake](https://github.com/snakemake) | Org |Snakemake | 763 |
| 20 | [daehwankimlab](https://github.com/daehwankimlab) | Org |Kim Lab at UT Southwestern Medical Center | 521 |
| 21 | [pysam-developers](https://github.com/pysam-developers) | Org |pysam-developers | 455 |
| 22 | [cgatoxford](https://github.com/cgatoxford) | Org |CGAT | 437 |
| 23 | [biod](https://github.com/biod) | Org |BioD | 431 |
| 24 | [wglab](https://github.com/wglab) | Org |Wang Genomics Lab | 426 |
| 25 | [felixkrueger](https://github.com/felixkrueger) | One |Felix Krueger | 424 |
| 26 | [bbuchfink](https://github.com/bbuchfink) | One |Benjamin Buchfink | 418 |
| 27 | [gpertea](https://github.com/gpertea) | One |Geo Pertea | 416 |
| 28 | [mdshw5](https://github.com/mdshw5) | One |Matt Shirley | 406 |
| 29 | [bioinform](https://github.com/bioinform) | Org |The Bioinformatics Repository | 388 |
| 30 | [qiime2](https://github.com/qiime2) | Org |QIIME 2 | 347 |
| 31 | [bwa-mem2](https://github.com/bwa-mem2) | Org |bwa-mem2 | 345 |
| 32 | [dzerbino](https://github.com/dzerbino) | One |Daniel Zerbino | 343 |
| 33 | [ding-lab](https://github.com/ding-lab) | Org |Li Ding's Lab | 297 |
| 34 | [vcftools](https://github.com/vcftools) | Org |vcftools | 296 |
| 35 | [refresh-bio](https://github.com/refresh-bio) | Org |REFRESH Bioinformatics Group | 295 |
| 36 | [astrazeneca-ngs](https://github.com/astrazeneca-ngs) | Org |AstraZeneca - NGS Team | 289 |
| 37 | [etal](https://github.com/etal) | One |Eric Talevich | 258 |
| 38 | [pcingola](https://github.com/pcingola) | One |Pablo Cingolani | 250 |
| 39 | [biothings](https://github.com/biothings) | Org |BioThings | 241 |
| 40 | [dellytools](https://github.com/dellytools) | Org |delly | 227 |
| 41 | [s-andrews](https://github.com/s-andrews) | One |Simon Andrews | 184 |
| 42 | [gregoryfaust](https://github.com/gregoryfaust) | One |Gregory Faust | 172 |
| 43 | [gt1](https://github.com/gt1) | One |German Tischler | 161 |
| 44 | [atks](https://github.com/atks) | One |Adrian Tan | 151 |
| 45 | [mozack](https://github.com/mozack) | One |Lisle Mose | 141 |
| 46 | [abyzovlab](https://github.com/abyzovlab) | Org |Abyzov lab | 138 |
| 47 | [humanlongevity](https://github.com/humanlongevity) | Org |Human Longevity, Inc. | 80 |
| 48 | [mathonco](https://github.com/mathonco) | Org |IMO | 68 |
| 49 | [shubhamchandak94](https://github.com/shubhamchandak94) | One |Shubham Chandak | 51 |
| 50 | [seppinho](https://github.com/seppinho) | One |Sebastian Schoenherr | 47 |
## Repositories

##### [biopython](https://github.com/biopython)/[biopython](https://github.com/biopython/biopython)

ID = 1, Stars = 2432, Language = Python, Update = 2020-09-21

Official git repository for Biopython (originally converted from CVS)
##### [bioconda](https://github.com/bioconda)/[bioconda-recipes](https://github.com/bioconda/bioconda-recipes)

ID = 2, Stars = 1198, Language = Shell, Update = 2020-09-21

Conda recipes for the bioconda channel.
##### [samtools](https://github.com/samtools)/[samtools](https://github.com/samtools/samtools)

ID = 3, Stars = 979, Language = C, Update = 2020-09-21

Tools (written in C using htslib) for manipulating next-generation sequencing data
##### [lh3](https://github.com/lh3)/[bwa](https://github.com/lh3/bwa)

ID = 4, Stars = 893, Language = C, Update = 2020-08-19

Burrow-Wheeler Aligner for short-read alignment (see minimap2 for long-read alignment)
##### [broadinstitute](https://github.com/broadinstitute)/[gatk](https://github.com/broadinstitute/gatk)

ID = 5, Stars = 891, Language = Java, Update = 2020-09-21

Official code repository for GATK versions 4 and up
##### [opengene](https://github.com/opengene)/[fastp](https://github.com/OpenGene/fastp)

ID = 6, Stars = 836, Language = C++, Update = 2020-09-17

An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...)
##### [lh3](https://github.com/lh3)/[minimap2](https://github.com/lh3/minimap2)

ID = 7, Stars = 790, Language = C, Update = 2020-08-28

A versatile pairwise aligner for genomic and spliced nucleotide sequences
##### [lh3](https://github.com/lh3)/[seqtk](https://github.com/lh3/seqtk)

ID = 8, Stars = 726, Language = C, Update = 2020-06-16

Toolkit for processing sequences in FASTA/Q formats
##### [snakemake](https://github.com/snakemake)/[snakemake](https://github.com/snakemake/snakemake)

ID = 9, Stars = 655, Language = Python, Update = 2020-09-20

This is the development home of the workflow management system Snakemake. For general information, see
##### [ewels](https://github.com/ewels)/[MultiQC](https://github.com/ewels/MultiQC)

ID = 10, Stars = 646, Language = Python, Update = 2020-09-14

Aggregate results from bioinformatics analyses across many samples into a single report.
##### [arq5x](https://github.com/arq5x)/[bedtools2](https://github.com/arq5x/bedtools2)

ID = 11, Stars = 627, Language = C, Update = 2020-09-15

bedtools - the swiss army knife for genome arithmetic
##### [broadinstitute](https://github.com/broadinstitute)/[picard](https://github.com/broadinstitute/picard)

ID = 12, Stars = 599, Language = Java, Update = 2020-09-21

A set of command line tools (in Java) for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.
##### [broadinstitute](https://github.com/broadinstitute)/[cromwell](https://github.com/broadinstitute/cromwell)

ID = 13, Stars = 596, Language = Scala, Update = 2020-09-21

Scientific workflow engine designed for simplicity & scalability. Trivially transition between one off use cases to massive scale production environments
##### [shenwei356](https://github.com/shenwei356)/[seqkit](https://github.com/shenwei356/seqkit)

ID = 14, Stars = 531, Language = Go, Update = 2020-09-20

A cross-platform and ultrafast toolkit for FASTA/Q file manipulation in Golang
##### [broadinstitute](https://github.com/broadinstitute)/[keras-rcnn](https://github.com/broadinstitute/keras-rcnn)

ID = 15, Stars = 516, Language = Python, Update = 2020-05-20

Keras package for region-based convolutional neural networks (RCNNs)
##### [shenwei356](https://github.com/shenwei356)/[csvtk](https://github.com/shenwei356/csvtk)

ID = 16, Stars = 511, Language = Go, Update = 2020-09-09

A cross-platform, efficient and practical CSV/TSV toolkit in Golang
##### [samtools](https://github.com/samtools)/[htslib](https://github.com/samtools/htslib)

ID = 17, Stars = 491, Language = C, Update = 2020-09-18

C library for high-throughput sequencing data formats
##### [ekg](https://github.com/ekg)/[freebayes](https://github.com/ekg/freebayes)

ID = 18, Stars = 488, Language = C++, Update = 2020-08-19

Bayesian haplotype-based genetic polymorphism discovery and genotyping.
##### [ncbi](https://github.com/ncbi)/[sra-tools](https://github.com/ncbi/sra-tools)

ID = 19, Stars = 455, Language = C, Update = 2020-09-21

SRA Tools
##### [pysam-developers](https://github.com/pysam-developers)/[pysam](https://github.com/pysam-developers/pysam)

ID = 20, Stars = 455, Language = C, Update = 2020-06-30

Pysam is a Python module for reading and manipulating SAM/BAM/VCF/BCF files. It's a lightweight wrapper of the htslib C-API, the same one that powers samtools, bcftools, and tabix.
##### [lh3](https://github.com/lh3)/[bioawk](https://github.com/lh3/bioawk)

ID = 21, Stars = 436, Language = C, Update = 2019-09-15

BWK awk modified for biological data
##### [bbuchfink](https://github.com/bbuchfink)/[diamond](https://github.com/bbuchfink/diamond)

ID = 22, Stars = 418, Language = C++, Update = 2020-09-15

Accelerated BLAST compatible local sequence aligner.
##### [deeptools](https://github.com/deeptools)/[deepTools](https://github.com/deeptools/deepTools)

ID = 23, Stars = 418, Language = Python, Update = 2020-09-18

Tools to process and analyze deep sequencing data.
##### [samtools](https://github.com/samtools)/[hts-specs](https://github.com/samtools/hts-specs)

ID = 24, Stars = 393, Language = TeX, Update = 2020-09-07

Specifications of SAM/BAM and related high-throughput sequencing file formats
##### [igvteam](https://github.com/igvteam)/[igv](https://github.com/igvteam/igv)

ID = 25, Stars = 393, Language = Java, Update = 2020-09-21

Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
##### [biod](https://github.com/biod)/[sambamba](https://github.com/biod/sambamba)

ID = 26, Stars = 388, Language = D, Update = 2020-08-12

Tools for working with SAM/BAM/CRAM data
##### [deeptools](https://github.com/deeptools)/[pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks)

ID = 27, Stars = 386, Language = Python, Update = 2020-09-01

python module to plot beautiful and highly customizable genome browser tracks
##### [igvteam](https://github.com/igvteam)/[igv.js](https://github.com/igvteam/igv.js)

ID = 28, Stars = 381, Language = JavaScript, Update = 2020-09-21

Embeddable genomic visualization component based on the Integrative Genomics Viewer
##### [shenwei356](https://github.com/shenwei356)/[rush](https://github.com/shenwei356/rush)

ID = 29, Stars = 364, Language = Go, Update = 2020-05-02

A cross-platform command-line tool for executing jobs in parallel
##### [bwa-mem2](https://github.com/bwa-mem2)/[bwa-mem2](https://github.com/bwa-mem2/bwa-mem2)

ID = 30, Stars = 345, Language = C++, Update = 2020-09-21

The next version of bwa-mem
##### [brentp](https://github.com/brentp)/[mosdepth](https://github.com/brentp/mosdepth)

ID = 31, Stars = 341, Language = Nim, Update = 2020-09-16

fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing
##### [samtools](https://github.com/samtools)/[bcftools](https://github.com/samtools/bcftools)

ID = 32, Stars = 328, Language = C, Update = 2020-09-21

This is the official development repository for BCFtools. To compile, the develop branch of htslib is needed: git clone --branch=develop git://github.com/samtools/htslib.git htslib
##### [benlangmead](https://github.com/benlangmead)/[bowtie2](https://github.com/BenLangmead/bowtie2)

ID = 33, Stars = 320, Language = C++, Update = 2020-09-15

A fast and sensitive gapped read aligner
##### [vcftools](https://github.com/vcftools)/[vcftools](https://github.com/vcftools/vcftools)

ID = 34, Stars = 296, Language = C++, Update = 2020-03-10

A set of tools written in Perl and C++ for working with VCF files, such as those generated by the 1000 Genomes Project.
##### [cgatoxford](https://github.com/cgatoxford)/[UMI-tools](https://github.com/CGATOxford/UMI-tools)

ID = 35, Stars = 282, Language = Python, Update = 2020-09-14

Tools for handling Unique Molecular Identifiers in NGS data sets
##### [mdshw5](https://github.com/mdshw5)/[pyfaidx](https://github.com/mdshw5/pyfaidx)

ID = 36, Stars = 280, Language = Python, Update = 2020-08-06

Efficient pythonic random access to fasta subsequences
##### [arq5x](https://github.com/arq5x)/[gemini](https://github.com/arq5x/gemini)

ID = 37, Stars = 274, Language = Python, Update = 2020-04-28

a lightweight db framework for exploring genetic variation.
##### [daehwankimlab](https://github.com/daehwankimlab)/[hisat2](https://github.com/DaehwanKimLab/hisat2)

ID = 38, Stars = 259, Language = C++, Update = 2020-09-21

Graph-based alignment (Hierarchical Graph FM index)
##### [etal](https://github.com/etal)/[cnvkit](https://github.com/etal/cnvkit)

ID = 39, Stars = 258, Language = Python, Update = 2020-09-15

Copy number variant detection from targeted DNA sequencing
##### [brentp](https://github.com/brentp)/[vcfanno](https://github.com/brentp/vcfanno)

ID = 40, Stars = 243, Language = Go, Update = 2020-04-15

annotate a VCF with other VCFs/BEDs/tabixed files
##### [illumina](https://github.com/illumina)/[manta](https://github.com/Illumina/manta)

ID = 41, Stars = 243, Language = C++, Update = 2019-07-17

Structural variant and indel caller for mapped sequencing data
##### [qiime2](https://github.com/qiime2)/[qiime2](https://github.com/qiime2/qiime2)

ID = 42, Stars = 235, Language = Python, Update = 2020-09-09

Official repository for the QIIME 2 framework.
##### [broadinstitute](https://github.com/broadinstitute)/[keras-resnet](https://github.com/broadinstitute/keras-resnet)

ID = 43, Stars = 234, Language = Python, Update = 2020-07-25

Keras package for deep residual networks
##### [dzerbino](https://github.com/dzerbino)/[velvet](https://github.com/dzerbino/velvet)

ID = 44, Stars = 226, Language = C, Update = 2018-07-05

Short read de novo assembler using de Bruijn graphs
##### [ensembl](https://github.com/ensembl)/[ensembl-vep](https://github.com/Ensembl/ensembl-vep)

ID = 45, Stars = 225, Language = Perl, Update = 2020-09-21

The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
##### [illumina](https://github.com/illumina)/[hap.py](https://github.com/Illumina/hap.py)

ID = 46, Stars = 225, Language = C++, Update = 2020-02-27

Haplotype VCF comparison tools
##### [arq5x](https://github.com/arq5x)/[poretools](https://github.com/arq5x/poretools)

ID = 47, Stars = 223, Language = Jupyter Notebook, Update = 2017-12-16

a toolkit for working with Oxford nanopore data
##### [brentp](https://github.com/brentp)/[cyvcf2](https://github.com/brentp/cyvcf2)

ID = 48, Stars = 218, Language = Python, Update = 2020-09-15

cython + htslib == fast VCF and BCF processing
##### [illumina](https://github.com/illumina)/[strelka](https://github.com/Illumina/strelka)

ID = 49, Stars = 218, Language = C++, Update = 2019-09-26

Strelka2 germline and somatic small variant caller
##### [dellytools](https://github.com/dellytools)/[delly](https://github.com/dellytools/delly)

ID = 50, Stars = 216, Language = C++, Update = 2020-08-28

DELLY2: Structural variant discovery by integrated paired-end and split-read analysis
##### [samtools](https://github.com/samtools)/[htsjdk](https://github.com/samtools/htsjdk)

ID = 51, Stars = 213, Language = Java, Update = 2020-09-02

A Java API for high-throughput sequencing data (HTS) formats.
##### [benlangmead](https://github.com/benlangmead)/[comp-genomics-class](https://github.com/BenLangmead/comp-genomics-class)

ID = 52, Stars = 211, Language = Jupyter Notebook, Update = 2019-10-29

Code and examples for JHU Computational Genomics class
##### [felixkrueger](https://github.com/felixkrueger)/[TrimGalore](https://github.com/FelixKrueger/TrimGalore)

ID = 53, Stars = 211, Language = Perl, Update = 2020-09-04

A wrapper around Cutadapt and FastQC to consistently apply adapter and quality trimming to FastQ files, with extra functionality for RRBS data
##### [benlangmead](https://github.com/benlangmead)/[bowtie](https://github.com/BenLangmead/bowtie)

ID = 54, Stars = 210, Language = C++, Update = 2020-08-24

An ultrafast memory-efficient short read aligner
##### [benlangmead](https://github.com/benlangmead)/[c-cpp-notes](https://github.com/BenLangmead/c-cpp-notes)

ID = 55, Stars = 210, Language = Python, Update = 2020-05-09

Lecture notes and example code for teaching C & C++
##### [lh3](https://github.com/lh3)/[miniasm](https://github.com/lh3/miniasm)

ID = 56, Stars = 209, Language = TeX, Update = 2020-04-03

Ultrafast de novo assembly for long noisy reads (though having no consensus step)
##### [arq5x](https://github.com/arq5x)/[lumpy-sv](https://github.com/arq5x/lumpy-sv)

ID = 57, Stars = 205, Language = C, Update = 2020-09-14

lumpy: a general probabilistic framework for structural variant discovery
##### [broadinstitute](https://github.com/broadinstitute)/[pilon](https://github.com/broadinstitute/pilon)

ID = 58, Stars = 205, Language = Scala, Update = 2020-02-20

Pilon is an automated genome assembly improvement and variant detection tool
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[DnaFeaturesViewer](https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer)

ID = 59, Stars = 199, Language = Python, Update = 2020-09-20

üëÅÔ∏è
Python library to plot DNA sequence features (e.g. from Genbank files)
##### [felixkrueger](https://github.com/felixkrueger)/[Bismark](https://github.com/FelixKrueger/Bismark)

ID = 60, Stars = 179, Language = HTML, Update = 2020-04-28

A tool to map bisulfite converted sequence reads and determine cytosine methylation states
##### [ekg](https://github.com/ekg)/[intervaltree](https://github.com/ekg/intervaltree)

ID = 61, Stars = 170, Language = C++, Update = 2020-06-18

a minimal C++ interval tree implementation
##### [gpertea](https://github.com/gpertea)/[stringtie](https://github.com/gpertea/stringtie)

ID = 62, Stars = 170, Language = C++, Update = 2020-09-19

Transcript assembly and quantification for RNA-Seq
##### [broadinstitute](https://github.com/broadinstitute)/[infercnv](https://github.com/broadinstitute/infercnv)

ID = 63, Stars = 169, Language = R, Update = 2020-07-20

Inferring CNV from Single-Cell RNA-Seq
##### [opengene](https://github.com/opengene)/[AfterQC](https://github.com/OpenGene/AfterQC)

ID = 64, Stars = 166, Language = Python, Update = 2020-05-14

Automatic Filtering, Trimming, Error Removing and Quality Control for fastq data
##### [lh3](https://github.com/lh3)/[wgsim](https://github.com/lh3/wgsim)

ID = 65, Stars = 165, Language = C, Update = 2019-10-02

Reads simulator
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-pipeline](https://github.com/broadinstitute/gtex-pipeline)

ID = 66, Stars = 165, Language = Python, Update = 2020-08-21

GTEx & TOPMed data production and analysis pipelines
##### [brentp](https://github.com/brentp)/[goleft](https://github.com/brentp/goleft)

ID = 67, Stars = 164, Language = Go, Update = 2020-02-04

goleft is a collection of bioinformatics tools distributed under MIT license in a single static binary
##### [ncbi](https://github.com/ncbi)/[PubReader](https://github.com/ncbi/PubReader)

ID = 68, Stars = 159, Language = JavaScript, Update = 2018-06-21

A new way to view journal articles
##### [lh3](https://github.com/lh3)/[minigraph](https://github.com/lh3/minigraph)

ID = 69, Stars = 152, Language = C, Update = 2020-09-19

Proof-of-concept seq-to-graph mapper and graph generator
##### [atks](https://github.com/atks)/[vt](https://github.com/atks/vt)

ID = 70, Stars = 151, Language = C, Update = 2020-03-31

A tool set for short variant discovery in genetic sequence data.
##### [gregoryfaust](https://github.com/gregoryfaust)/[samblaster](https://github.com/GregoryFaust/samblaster)

ID = 71, Stars = 151, Language = C++, Update = 2020-06-04

samblaster: a tool to mark duplicates and extract discordant and split reads from sam files.
##### [brentp](https://github.com/brentp)/[bio-playground](https://github.com/brentp/bio-playground)

ID = 72, Stars = 150, Language = C, Update = 2018-11-26

miscellaneous scripts for bioinformatics/genomics that dont merit their own repo.
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[Flametree](https://github.com/Edinburgh-Genome-Foundry/Flametree)

ID = 73, Stars = 147, Language = Python, Update = 2020-09-12

üî•
Python file and zip operations made easy
##### [gpertea](https://github.com/gpertea)/[gffread](https://github.com/gpertea/gffread)

ID = 74, Stars = 145, Language = C++, Update = 2020-09-19

GFF/GTF utility providing format conversions, region filtering, FASTA sequence extraction and more
##### [illumina](https://github.com/illumina)/[SpliceAI](https://github.com/Illumina/SpliceAI)

ID = 75, Stars = 144, Language = Python, Update = 2020-05-15

A deep learning-based tool to identify splice variants
##### [ekg](https://github.com/ekg)/[fraction.js](https://github.com/ekg/fraction.js)

ID = 76, Stars = 141, Language = JavaScript, Update = 2018-07-25

A fraction math library in javascript.
##### [broadinstitute](https://github.com/broadinstitute)/[viral-ngs](https://github.com/broadinstitute/viral-ngs)

ID = 77, Stars = 141, Language = Python, Update = 2020-05-20

Viral genomics analysis pipelines
##### [daehwankimlab](https://github.com/daehwankimlab)/[centrifuge](https://github.com/DaehwanKimLab/centrifuge)

ID = 78, Stars = 139, Language = C++, Update = 2020-08-17

Classifier for metagenomic sequences
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[VarDict](https://github.com/AstraZeneca-NGS/VarDict)

ID = 79, Stars = 132, Language = Perl, Update = 2020-09-16

VarDict
##### [arq5x](https://github.com/arq5x)/[bedtools](https://github.com/arq5x/bedtools)

ID = 80, Stars = 131, Language = C++, Update = 2019-03-10

A powerful toolset for genome arithmetic.
##### [brentp](https://github.com/brentp)/[smoove](https://github.com/brentp/smoove)

ID = 81, Stars = 130, Language = Go, Update = 2019-11-11

structural variant calling and genotyping with existing tools, but, smoothly.
##### [s-andrews](https://github.com/s-andrews)/[FastQC](https://github.com/s-andrews/FastQC)

ID = 82, Stars = 128, Language = Java, Update = 2020-09-21

A quality control analysis tool for high throughput sequencing data
##### [brentp](https://github.com/brentp)/[hts-nim](https://github.com/brentp/hts-nim)

ID = 83, Stars = 127, Language = Nim, Update = 2020-09-21

nim wrapper for htslib for parsing genomics data files
##### [abyzovlab](https://github.com/abyzovlab)/[CNVnator](https://github.com/abyzovlab/CNVnator)

ID = 84, Stars = 126, Language = C++, Update = 2020-07-07

a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[pdf_reports](https://github.com/Edinburgh-Genome-Foundry/pdf_reports)

ID = 85, Stars = 126, Language = Python, Update = 2020-09-12

üìï
Python library and CSS theme to generate PDF reports from HTML/Pug
##### [deeptools](https://github.com/deeptools)/[pyBigWig](https://github.com/deeptools/pyBigWig)

ID = 86, Stars = 118, Language = C, Update = 2020-05-02

A python extension for quick access to bigWig and bigBed files
##### [cgatoxford](https://github.com/cgatoxford)/[cgat](https://github.com/CGATOxford/cgat)

ID = 87, Stars = 118, Language = Python, Update = 2018-11-08

Do not use - please refer to our newest code: https://github.com/cgat-developers/cgat-apps
##### [bioinform](https://github.com/bioinform)/[neusomatic](https://github.com/bioinform/neusomatic)

ID = 88, Stars = 117, Language = Python, Update = 2020-09-19

NeuSomatic: Deep convolutional neural networks for accurate somatic mutation detection
##### [brentp](https://github.com/brentp)/[somalier](https://github.com/brentp/somalier)

ID = 89, Stars = 115, Language = Nim, Update = 2020-09-11

fast sample-swap and relatedness checks on BAMs/CRAMs/VCFs/GVCFs... "like damn that is one smart wine guy"
##### [brentp](https://github.com/brentp)/[cruzdb](https://github.com/brentp/cruzdb)

ID = 90, Stars = 115, Language = Python, Update = 2020-08-27

python access to UCSC genomes database
##### [lh3](https://github.com/lh3)/[readfq](https://github.com/lh3/readfq)

ID = 91, Stars = 115, Language = C, Update = 2019-08-09

Fast multi-line FASTA/Q reader in several programming languages
##### [shenwei356](https://github.com/shenwei356)/[brename](https://github.com/shenwei356/brename)

ID = 92, Stars = 115, Language = Go, Update = 2020-05-02

A practical cross-platform command-line tool for safely batch renaming files/directories via regular expression
##### [lh3](https://github.com/lh3)/[kmer-cnt](https://github.com/lh3/kmer-cnt)

ID = 93, Stars = 110, Language = C++, Update = 2020-03-10

Code examples of fast and simple k-mer counters for tutorial purposes
##### [opengene](https://github.com/opengene)/[MutScan](https://github.com/OpenGene/MutScan)

ID = 94, Stars = 110, Language = C, Update = 2020-01-21

Detect and visualize target mutations by scanning FastQ files directly
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[Taskpacker](https://github.com/Edinburgh-Genome-Foundry/Taskpacker)

ID = 95, Stars = 110, Language = Python, Update = 2018-04-16

üéí
Simple schedule optimization library for Python
##### [refresh-bio](https://github.com/refresh-bio)/[KMC](https://github.com/refresh-bio/KMC)

ID = 96, Stars = 110, Language = C++, Update = 2020-09-06

Fast and frugal disk based k-mer counter
##### [lh3](https://github.com/lh3)/[cgranges](https://github.com/lh3/cgranges)

ID = 97, Stars = 108, Language = C, Update = 2020-07-29

A C/C++ library for fast interval overlap queries (with a "bedtools coverage" example)
##### [biopython](https://github.com/biopython)/[biopython.github.io](https://github.com/biopython/biopython.github.io)

ID = 98, Stars = 108, Language = CSS, Update = 2020-09-20

Source of biopython.org website, migrated from MediaWiki
##### [ncbi](https://github.com/ncbi)/[pgap](https://github.com/ncbi/pgap)

ID = 99, Stars = 107, Language = Common Workflow Language, Update = 2020-09-18

NCBI Prokaryotic Genome Annotation Pipeline
##### [wglab](https://github.com/wglab)/[InterVar](https://github.com/WGLab/InterVar)

ID = 100, Stars = 105, Language = Python, Update = 2020-01-02

A bioinformatics software tool for clinical interpretation of genetic variants by the 2015 ACMG-AMP guideline
##### [bioinform](https://github.com/bioinform)/[somaticseq](https://github.com/bioinform/somaticseq)

ID = 101, Stars = 105, Language = Python, Update = 2020-09-20

An ensemble approach to accurately detect somatic mutations using SomaticSeq
##### [ncbi](https://github.com/ncbi)/[ngs](https://github.com/ncbi/ngs)

ID = 102, Stars = 104, Language = C++, Update = 2020-09-14

NGS Language Bindings
##### [igvteam](https://github.com/igvteam)/[igv-jupyter](https://github.com/igvteam/igv-jupyter)

ID = 103, Stars = 104, Language = Python, Update = 2019-10-11

Extension for Jupyter Notebook which integrates igv.js
##### [illumina](https://github.com/illumina)/[pyflow](https://github.com/Illumina/pyflow)

ID = 104, Stars = 103, Language = Python, Update = 2020-07-01

A lightweight parallel task engine
##### [deeptools](https://github.com/deeptools)/[HiCExplorer](https://github.com/deeptools/HiCExplorer)

ID = 105, Stars = 102, Language = Python, Update = 2020-09-21

HiCExplorer is a powerful and easy to use set of tools to process, normalize and visualize Hi-C data.
##### [brentp](https://github.com/brentp)/[nim-plotly](https://github.com/brentp/nim-plotly)

ID = 106, Stars = 101, Language = Nim, Update = 2020-09-16

plotting library for nim-lang
##### [lh3](https://github.com/lh3)/[minimap](https://github.com/lh3/minimap)

ID = 107, Stars = 100, Language = C, Update = 2017-09-20

This repo is DEPRECATED. Please use minimap2, the successor of minimap.
##### [broadinstitute](https://github.com/broadinstitute)/[xtermcolor](https://github.com/broadinstitute/xtermcolor)

ID = 108, Stars = 99, Language = Python, Update = 2017-10-08

Python library for terminal color support (including 256-color support)
##### [ding-lab](https://github.com/ding-lab)/[msisensor](https://github.com/ding-lab/msisensor)

ID = 109, Stars = 98, Language = C++, Update = 2020-07-07

microsatellite instability detection using tumor only or paired tumor-normal data
##### [ensembl](https://github.com/ensembl)/[WiggleTools](https://github.com/Ensembl/WiggleTools)

ID = 110, Stars = 98, Language = C, Update = 2020-08-20

Basic operations on the space of numerical functions defined on the genome using lazy evaluators for flexibility and efficiency
##### [pcingola](https://github.com/pcingola)/[SnpEff](https://github.com/pcingola/SnpEff)

ID = 111, Stars = 97, Language = Java, Update = 2020-09-21

None
##### [ensembl](https://github.com/ensembl)/[ensembl-rest](https://github.com/Ensembl/ensembl-rest)

ID = 112, Stars = 97, Language = Perl, Update = 2020-09-18

Language agnostic RESTful data access to Ensembl data over HTTP
##### [brentp](https://github.com/brentp)/[peddy](https://github.com/brentp/peddy)

ID = 113, Stars = 96, Language = Jupyter Notebook, Update = 2020-07-03

genotype :: ped correspondence check, ancestry check, sex check. directly, quickly on VCF
##### [lh3](https://github.com/lh3)/[biofast](https://github.com/lh3/biofast)

ID = 114, Stars = 96, Language = C, Update = 2020-07-08

Benchmarking programming languages/implementations for common tasks in Bioinformatics
##### [brentp](https://github.com/brentp)/[slivar](https://github.com/brentp/slivar)

ID = 115, Stars = 92, Language = Nim, Update = 2020-09-17

variant expressions, annotation, and filtering for great good.
##### [illumina](https://github.com/illumina)/[Nirvana](https://github.com/Illumina/Nirvana)

ID = 116, Stars = 92, Language = C#, Update = 2020-09-10

The nimble & robust variant annotator
##### [mdshw5](https://github.com/mdshw5)/[fastqp](https://github.com/mdshw5/fastqp)

ID = 117, Stars = 90, Language = Python, Update = 2018-05-29

Simple FASTQ quality assessment using Python
##### [lh3](https://github.com/lh3)/[psmc](https://github.com/lh3/psmc)

ID = 118, Stars = 89, Language = C, Update = 2016-01-21

Implementation of the Pairwise Sequentially Markovian Coalescent (PSMC) model
##### [illumina](https://github.com/illumina)/[paragraph](https://github.com/Illumina/paragraph)

ID = 119, Stars = 89, Language = C++, Update = 2019-09-24

Graph realignment tools for structural variants
##### [lh3](https://github.com/lh3)/[fermikit](https://github.com/lh3/fermikit)

ID = 120, Stars = 88, Language = TeX, Update = 2017-12-20

De novo assembly based variant calling pipeline for Illumina short reads
##### [lh3](https://github.com/lh3)/[bedtk](https://github.com/lh3/bedtk)

ID = 121, Stars = 87, Language = C, Update = 2020-08-19

A simple toolset for BED files (warning: CLI may change before bedtk becomes stable)
##### [gpertea](https://github.com/gpertea)/[gffcompare](https://github.com/gpertea/gffcompare)

ID = 122, Stars = 87, Language = C++, Update = 2020-09-19

classify, merge, tracking and annotation of GFF files by comparing to a reference annotation GFF
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[VarDictJava](https://github.com/AstraZeneca-NGS/VarDictJava)

ID = 123, Stars = 87, Language = Java, Update = 2020-09-16

VarDict Java port
##### [ewels](https://github.com/ewels)/[ClusterFlow](https://github.com/ewels/ClusterFlow)

ID = 124, Stars = 86, Language = Perl, Update = 2019-06-08

A pipelining tool to automate and standardise bioinformatics analyses on cluster environments.
##### [shenwei356](https://github.com/shenwei356)/[taxonkit](https://github.com/shenwei356/taxonkit)

ID = 125, Stars = 86, Language = Go, Update = 2020-08-21

A Cross-platform and Efficient NCBI Taxonomy Toolkit
##### [daehwankimlab](https://github.com/daehwankimlab)/[tophat](https://github.com/DaehwanKimLab/tophat)

ID = 126, Stars = 86, Language = C++, Update = 2018-05-24

Spliced read mapper for RNA-Seq
##### [broadinstitute](https://github.com/broadinstitute)/[mutect](https://github.com/broadinstitute/mutect)

ID = 127, Stars = 86, Language = Java, Update = 2015-08-26

MuTect -- Accurate and sensitive cancer mutation detection
##### [pcingola](https://github.com/pcingola)/[BigDataScript](https://github.com/pcingola/BigDataScript)

ID = 128, Stars = 85, Language = Java, Update = 2020-09-08

BigDataScript: Scirpting language for big data
##### [broadinstitute](https://github.com/broadinstitute)/[ssGSEA2.0](https://github.com/broadinstitute/ssGSEA2.0)

ID = 129, Stars = 85, Language = R, Update = 2020-09-21

Single sample Gene Set Enrichment analysis (ssGSEA) and PTM Enrichment Analysis (PTM-SEA)
##### [ewels](https://github.com/ewels)/[sra-explorer](https://github.com/ewels/sra-explorer)

ID = 130, Stars = 83, Language = HTML, Update = 2020-09-20

Web application to explore the Sequence Read Archive.
##### [gt1](https://github.com/gt1)/[biobambam2](https://github.com/gt1/biobambam2)

ID = 131, Stars = 82, Language = C++, Update = 2019-03-12

Tools for early stage alignment file processing
##### [brentp](https://github.com/brentp)/[gargs](https://github.com/brentp/gargs)

ID = 132, Stars = 81, Language = Go, Update = 2018-01-26

better(?) xargs in go
##### [brentp](https://github.com/brentp)/[bwa-meth](https://github.com/brentp/bwa-meth)

ID = 133, Stars = 79, Language = PostScript, Update = 2018-08-07

fast and accurate alignment of BS-Seq reads using bwa-mem and a 3-letter genome
##### [lh3](https://github.com/lh3)/[ksw2](https://github.com/lh3/ksw2)

ID = 134, Stars = 79, Language = C, Update = 2019-04-29

Global alignment and alignment extension
##### [lh3](https://github.com/lh3)/[bgt](https://github.com/lh3/bgt)

ID = 135, Stars = 78, Language = C, Update = 2019-09-04

Flexible genotype query among 30,000+ samples whole-genome
##### [arq5x](https://github.com/arq5x)/[grabix](https://github.com/arq5x/grabix)

ID = 136, Stars = 78, Language = C, Update = 2018-05-10

a wee tool for random access into BGZF files.
##### [broadinstitute](https://github.com/broadinstitute)/[wot](https://github.com/broadinstitute/wot)

ID = 137, Stars = 78, Language = Jupyter Notebook, Update = 2020-08-24

A software package for analyzing snapshots of developmental processes
##### [broadinstitute](https://github.com/broadinstitute)/[ichorCNA](https://github.com/broadinstitute/ichorCNA)

ID = 138, Stars = 78, Language = R, Update = 2020-05-06

Estimating tumor fraction in cell-free DNA from ultra-low-pass whole genome sequencing.
##### [brentp](https://github.com/brentp)/[combat.py](https://github.com/brentp/combat.py)

ID = 139, Stars = 77, Language = Python, Update = 2018-12-31

python / numpy / pandas / patsy version of ComBat for removing batch effects.
##### [brentp](https://github.com/brentp)/[pyfasta](https://github.com/brentp/pyfasta)

ID = 140, Stars = 77, Language = Python, Update = 2017-12-11

fast, memory-efficient, pythonic (and command-line) access to fasta sequence files
##### [opengene](https://github.com/opengene)/[repaq](https://github.com/OpenGene/repaq)

ID = 141, Stars = 76, Language = C, Update = 2020-06-14

A fast lossless FASTQ compressor with ultra-high compression ratio
##### [lh3](https://github.com/lh3)/[gfatools](https://github.com/lh3/gfatools)

ID = 142, Stars = 75, Language = C, Update = 2020-07-14

Tools for manipulating sequence graphs in the GFA and rGFA formats
##### [samtools](https://github.com/samtools)/[tabix](https://github.com/samtools/tabix)

ID = 143, Stars = 75, Language = C, Update = 2020-07-27

Note: tabix and bgzip binaries are now part of the HTSlib project.
##### [shenwei356](https://github.com/shenwei356)/[bio_scripts](https://github.com/shenwei356/bio_scripts)

ID = 144, Stars = 74, Language = Perl, Update = 2019-02-12

Practical, reusable scripts for bioinformatics
##### [biothings](https://github.com/biothings)/[mygene.info](https://github.com/biothings/mygene.info)

ID = 145, Stars = 74, Language = Python, Update = 2020-09-02

MyGene.info: A BioThings API for gene annotations
##### [ncbi](https://github.com/ncbi)/[robotframework-pageobjects](https://github.com/ncbi/robotframework-pageobjects)

ID = 146, Stars = 74, Language = Python, Update = 2018-11-30

Implementation of the Page Object pattern with Robot Framework and selenium. Also facilitates page object pattern independent of Robot Framework
##### [ensembl](https://github.com/ensembl)/[VEP_plugins](https://github.com/Ensembl/VEP_plugins)

ID = 147, Stars = 73, Language = Perl, Update = 2020-09-18

Plugins for the Ensembl Variant Effect Predictor (VEP)
##### [ekg](https://github.com/ekg)/[seqwish](https://github.com/ekg/seqwish)

ID = 148, Stars = 72, Language = C++, Update = 2020-09-17

alignment to variation graph inducer
##### [opengene](https://github.com/opengene)/[gencore](https://github.com/OpenGene/gencore)

ID = 149, Stars = 71, Language = C++, Update = 2020-08-31

Generate consensus reads to reduce sequencing noises and remove duplications
##### [lh3](https://github.com/lh3)/[fermi](https://github.com/lh3/fermi)

ID = 150, Stars = 69, Language = C, Update = 2013-12-06

A WGS de novo assembler based on the FMD-index for large genomes
##### [brentp](https://github.com/brentp)/[duphold](https://github.com/brentp/duphold)

ID = 151, Stars = 68, Language = Nim, Update = 2020-05-27

don't get DUP'ed or DEL'ed by your putative SVs.
##### [biothings](https://github.com/biothings)/[myvariant.info](https://github.com/biothings/myvariant.info)

ID = 152, Stars = 68, Language = Python, Update = 2020-08-17

MyVariant.info: A BioThings API for human variant annotations
##### [illumina](https://github.com/illumina)/[ExpansionHunter](https://github.com/Illumina/ExpansionHunter)

ID = 153, Stars = 68, Language = C++, Update = 2020-07-13

A tool for estimating repeat sizes
##### [broadinstitute](https://github.com/broadinstitute)/[Drop-seq](https://github.com/broadinstitute/Drop-seq)

ID = 154, Stars = 68, Language = Java, Update = 2020-09-15

Java tools for analyzing Drop-seq data
##### [brentp](https://github.com/brentp)/[slurmpy](https://github.com/brentp/slurmpy)

ID = 155, Stars = 67, Language = Python, Update = 2020-09-03

submit jobs to slurm with quick-and-dirty python
##### [arq5x](https://github.com/arq5x)/[bedtools-protocols](https://github.com/arq5x/bedtools-protocols)

ID = 156, Stars = 67, Language = CSS, Update = 2014-03-06

None
##### [ncbi](https://github.com/ncbi)/[TPMCalculator](https://github.com/ncbi/TPMCalculator)

ID = 157, Stars = 66, Language = C++, Update = 2020-08-13

TPMCalculator quantifies mRNA abundance directly from the alignments by parsing BAM files
##### [ding-lab](https://github.com/ding-lab)/[CharGer](https://github.com/ding-lab/CharGer)

ID = 158, Stars = 65, Language = Python, Update = 2020-04-10

Characterization of Germline variants
##### [ncbi](https://github.com/ncbi)/[ncbi-vdb](https://github.com/ncbi/ncbi-vdb)

ID = 159, Stars = 65, Language = C, Update = 2020-09-21

ncbi-vdb
##### [lh3](https://github.com/lh3)/[hickit](https://github.com/lh3/hickit)

ID = 160, Stars = 63, Language = C, Update = 2019-05-13

TAD calling, phase imputation, 3D modeling and more for diploid single-cell Hi-C (Dip-C) and general Hi-C
##### [illumina](https://github.com/illumina)/[Pisces](https://github.com/Illumina/Pisces)

ID = 161, Stars = 63, Language = C#, Update = 2019-08-09

Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows.
##### [snakemake](https://github.com/snakemake)/[snakemake-wrappers](https://github.com/snakemake/snakemake-wrappers)

ID = 162, Stars = 63, Language = Perl, Update = 2020-09-21

This is the development home of the Snakemake wrapper repository, see
##### [bioinform](https://github.com/bioinform)/[rnacocktail](https://github.com/bioinform/rnacocktail)

ID = 163, Stars = 63, Language = Jupyter Notebook, Update = 2019-11-23

None
##### [humanlongevity](https://github.com/humanlongevity)/[HLA](https://github.com/humanlongevity/HLA)

ID = 164, Stars = 63, Language = Jupyter Notebook, Update = 2018-10-11

xHLA: Fast and accurate HLA typing from short read sequence data
##### [dzerbino](https://github.com/dzerbino)/[ascii_plots](https://github.com/dzerbino/ascii_plots)

ID = 165, Stars = 62, Language = Python, Update = 2017-10-30

Convenience function for quick and dirty data analysis
##### [lh3](https://github.com/lh3)/[fermi-lite](https://github.com/lh3/fermi-lite)

ID = 166, Stars = 62, Language = C, Update = 2020-06-26

Standalone C library for assembling Illumina short reads in small regions
##### [opengene](https://github.com/opengene)/[GeneFuse](https://github.com/OpenGene/GeneFuse)

ID = 167, Stars = 62, Language = C, Update = 2020-09-17

Gene fusion detection and visualization
##### [bioinform](https://github.com/bioinform)/[varsim](https://github.com/bioinform/varsim)

ID = 168, Stars = 62, Language = Java, Update = 2020-05-15

VarSim: A high-fidelity simulation validation framework for high-throughput genome sequencing with cancer applications
##### [bioconda](https://github.com/bioconda)/[bioconda-utils](https://github.com/bioconda/bioconda-utils)

ID = 169, Stars = 62, Language = Python, Update = 2020-09-13

Utilities for building and managing bioconda recipes
##### [mozack](https://github.com/mozack)/[abra](https://github.com/mozack/abra)

ID = 170, Stars = 60, Language = Java, Update = 2018-05-24

Assembly Based ReAligner
##### [broadinstitute](https://github.com/broadinstitute)/[oncotator](https://github.com/broadinstitute/oncotator)

ID = 171, Stars = 60, Language = Python, Update = 2020-02-26

None
##### [opengene](https://github.com/opengene)/[OpenGene.jl](https://github.com/OpenGene/OpenGene.jl)

ID = 172, Stars = 59, Language = Julia, Update = 2017-10-08

(No maintenance) OpenGene, core libraries for NGS data analysis and bioinformatics in Julia
##### [ewels](https://github.com/ewels)/[AWS-iGenomes](https://github.com/ewels/AWS-iGenomes)

ID = 173, Stars = 58, Language = HTML, Update = 2019-11-06

Documentation and description of AWS iGenomes S3 resource.
##### [lh3](https://github.com/lh3)/[bfc](https://github.com/lh3/bfc)

ID = 174, Stars = 58, Language = TeX, Update = 2016-05-31

High-performance error correction for Illumina resequencing data
##### [ncbi](https://github.com/ncbi)/[dbsnp](https://github.com/ncbi/dbsnp)

ID = 175, Stars = 58, Language = Jupyter Notebook, Update = 2020-04-22

dbSNP
##### [illumina](https://github.com/illumina)/[PrimateAI](https://github.com/Illumina/PrimateAI)

ID = 176, Stars = 57, Language = Python, Update = 2019-02-26

deep residual neural network for classifying the pathogenicity of missense mutations.
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[lala](https://github.com/Edinburgh-Genome-Foundry/lala)

ID = 177, Stars = 57, Language = Python, Update = 2020-09-02

üåé
Analyze and generate reports of web logs (NGINX)
##### [shenwei356](https://github.com/shenwei356)/[bio](https://github.com/shenwei356/bio)

ID = 178, Stars = 56, Language = Go, Update = 2020-07-07

A lightweight and high-performance bioinformatics package in Golang
##### [mozack](https://github.com/mozack)/[abra2](https://github.com/mozack/abra2)

ID = 179, Stars = 56, Language = Java, Update = 2020-09-14

ABRA2
##### [opengene](https://github.com/opengene)/[fastv](https://github.com/OpenGene/fastv)

ID = 180, Stars = 56, Language = C++, Update = 2020-08-06

An ultra-fast tool for identification of SARS-CoV-2 and other microbes from sequencing data. This tool can be used to detect viral infectious diseases, like COVID-19.
##### [ensembl](https://github.com/ensembl)/[ensembl](https://github.com/Ensembl/ensembl)

ID = 181, Stars = 56, Language = Perl, Update = 2020-09-21

The Ensembl Core Perl API and SQL schema
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[DnaChisel](https://github.com/Edinburgh-Genome-Foundry/DnaChisel)

ID = 182, Stars = 56, Language = Python, Update = 2020-09-20

‚úèÔ∏è
A versatile DNA sequence optimizer
##### [dzerbino](https://github.com/dzerbino)/[oases](https://github.com/dzerbino/oases)

ID = 183, Stars = 55, Language = C, Update = 2018-03-12

De novo transcriptome assembler for short reads
##### [lh3](https://github.com/lh3)/[dna-nn](https://github.com/lh3/dna-nn)

ID = 184, Stars = 55, Language = C, Update = 2019-06-16

Model and predict short DNA sequence features with neural networks
##### [ekg](https://github.com/ekg)/[fastahack](https://github.com/ekg/fastahack)

ID = 185, Stars = 54, Language = C++, Update = 2020-01-06

utilities for indexing and sequence extraction from FASTA files
##### [ncbi](https://github.com/ncbi)/[SKESA](https://github.com/ncbi/SKESA)

ID = 186, Stars = 54, Language = C++, Update = 2020-08-10

SKESA assembler
##### [wglab](https://github.com/wglab)/[DeepMod](https://github.com/WGLab/DeepMod)

ID = 187, Stars = 53, Language = Python, Update = 2020-06-24

DeepMod: a deep-learning tool for genomic-scale, strand-sensitive and single-nucleotide based detection of DNA modifications
##### [ncbi](https://github.com/ncbi)/[ngs-tools](https://github.com/ncbi/ngs-tools)

ID = 188, Stars = 53, Language = C++, Update = 2020-08-19

None
##### [broadinstitute](https://github.com/broadinstitute)/[tensorqtl](https://github.com/broadinstitute/tensorqtl)

ID = 189, Stars = 53, Language = Python, Update = 2020-09-18

Ultrafast GPU-based QTL mapper
##### [illumina](https://github.com/illumina)/[interop](https://github.com/Illumina/interop)

ID = 190, Stars = 52, Language = C++, Update = 2020-08-25

C++ Library to parse Illumina InterOp files
##### [brentp](https://github.com/brentp)/[xopen](https://github.com/brentp/xopen)

ID = 191, Stars = 51, Language = Go, Update = 2018-11-16

open files for buffered reading and writing in #golang
##### [ewels](https://github.com/ewels)/[MegaQC](https://github.com/ewels/MegaQC)

ID = 192, Stars = 51, Language = Python, Update = 2020-09-18

Web application to collect and visualise data across multiple MultiQC runs.
##### [lh3](https://github.com/lh3)/[tabtk](https://github.com/lh3/tabtk)

ID = 193, Stars = 51, Language = C, Update = 2016-09-08

Toolkit for processing TAB-delimited format
##### [arq5x](https://github.com/arq5x)/[cyvcf](https://github.com/arq5x/cyvcf)

ID = 194, Stars = 51, Language = Python, Update = 2018-03-27

A fast Python library for VCF files leveraging Cython for speed.
##### [ncbi](https://github.com/ncbi)/[icn3d](https://github.com/ncbi/icn3d)

ID = 195, Stars = 51, Language = JavaScript, Update = 2020-09-15

WebGL-based protein structure viewer
##### [broadinstitute](https://github.com/broadinstitute)/[ABC-Enhancer-Gene-Prediction](https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction)

ID = 196, Stars = 51, Language = Python, Update = 2020-09-17

Cell type specific enhancer-gene predictions using ABC model (Fulco, Nasser et al, Nature Genetics 2019)
##### [ewels](https://github.com/ewels)/[CPT-Bootstrap-Carousel](https://github.com/ewels/CPT-Bootstrap-Carousel)

ID = 197, Stars = 50, Language = PHP, Update = 2020-04-17

WordPress plugin which generates a custom post type for choosing images and content. Outputs Bootstrap Image Carousel (slider) HTML from a shortcode.
##### [lh3](https://github.com/lh3)/[ropebwt2](https://github.com/lh3/ropebwt2)

ID = 198, Stars = 49, Language = TeX, Update = 2020-04-17

Incremental construction of FM-index for DNA sequences
##### [ensembl](https://github.com/ensembl)/[ensembl-pipeline](https://github.com/Ensembl/ensembl-pipeline)

ID = 199, Stars = 49, Language = Perl, Update = 2020-01-30

*DEPRECATED* Job management for the Ensembl Genebuild pipeline
##### [wglab](https://github.com/wglab)/[PennCNV](https://github.com/WGLab/PennCNV)

ID = 200, Stars = 49, Language = C, Update = 2020-08-15

Copy number vaiation detection from SNP arrays
##### [ncbi](https://github.com/ncbi)/[amr](https://github.com/ncbi/amr)

ID = 201, Stars = 49, Language = C++, Update = 2020-09-21

amr_finder
##### [gt1](https://github.com/gt1)/[biobambam](https://github.com/gt1/biobambam)

ID = 202, Stars = 48, Language = C++, Update = 2015-04-20

Tools for bam file processing
##### [pcingola](https://github.com/pcingola)/[jFuzzyLogic](https://github.com/pcingola/jFuzzyLogic)

ID = 203, Stars = 47, Language = Java, Update = 2018-07-17

The most complete fuzzy logic library in Java. The de-facto standard for research and industrial applications.
##### [illumina](https://github.com/illumina)/[akt](https://github.com/Illumina/akt)

ID = 204, Stars = 47, Language = C++, Update = 2020-05-22

Ancestry and Kinship Tools
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[Proglog](https://github.com/Edinburgh-Genome-Foundry/Proglog)

ID = 205, Stars = 47, Language = Python, Update = 2020-05-03

üìù
Logs and progress bars manager for Python
##### [mathonco](https://github.com/mathonco)/[NeoPredPipe](https://github.com/MathOnco/NeoPredPipe)

ID = 206, Stars = 47, Language = Python, Update = 2020-09-21

Neoantigens prediction pipeline for multi- or single-region vcf files using ANNOVAR and netMHCpan.
##### [brentp](https://github.com/brentp)/[hts-python](https://github.com/brentp/hts-python)

ID = 207, Stars = 46, Language = Python, Update = 2017-03-03

pythonic wrapper for libhts (moved to: https://github.com/quinlan-lab/hts-python)
##### [ding-lab](https://github.com/ding-lab)/[MuSiC2](https://github.com/ding-lab/MuSiC2)

ID = 208, Stars = 46, Language = Perl, Update = 2019-05-19

identifying mutational significance in cancer genomes
##### [opengene](https://github.com/opengene)/[CfdnaPattern](https://github.com/OpenGene/CfdnaPattern)

ID = 209, Stars = 46, Language = Python, Update = 2018-08-03

Pattern Recognition for Cell-free DNA
##### [refresh-bio](https://github.com/refresh-bio)/[kmer-db](https://github.com/refresh-bio/kmer-db)

ID = 210, Stars = 46, Language = C++, Update = 2020-08-19

None
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad-browser](https://github.com/broadinstitute/gnomad-browser)

ID = 211, Stars = 46, Language = JavaScript, Update = 2020-09-18

JavaScript tools for exploring genomic data
##### [arq5x](https://github.com/arq5x)/[ggd](https://github.com/arq5x/ggd)

ID = 212, Stars = 45, Language = Python, Update = 2016-04-20

None
##### [broadinstitute](https://github.com/broadinstitute)/[docker-terraform](https://github.com/broadinstitute/docker-terraform)

ID = 213, Stars = 45, Language = Dockerfile, Update = 2020-09-09

Docker container for running the Terraform application
##### [brentp](https://github.com/brentp)/[irelate](https://github.com/brentp/irelate)

ID = 214, Stars = 44, Language = Go, Update = 2020-07-12

Streaming relation (overlap, distance, KNN) of (any number of) sorted genomic interval sets. #golang
##### [brentp](https://github.com/brentp)/[vcfgo](https://github.com/brentp/vcfgo)

ID = 215, Stars = 44, Language = Go, Update = 2020-03-14

a golang library to read, write and manipulate files in the variant call format.
##### [refresh-bio](https://github.com/refresh-bio)/[DSRC](https://github.com/refresh-bio/DSRC)

ID = 216, Stars = 44, Language = C++, Update = 2020-06-24

DSRC - DNA Sequence Reads Compressor
##### [brentp](https://github.com/brentp)/[fishers_exact_test](https://github.com/brentp/fishers_exact_test)

ID = 217, Stars = 43, Language = Python, Update = 2019-10-02

Fishers Exact Test for Python (Cython)
##### [brentp](https://github.com/brentp)/[intintmap](https://github.com/brentp/intintmap)

ID = 218, Stars = 43, Language = Go, Update = 2019-02-11

fast int64-int64 map for go
##### [brentp](https://github.com/brentp)/[interlap](https://github.com/brentp/interlap)

ID = 219, Stars = 43, Language = Python, Update = 2018-08-31

fast, pure-python interval overlap testing
##### [lh3](https://github.com/lh3)/[samtools](https://github.com/lh3/samtools)

ID = 220, Stars = 43, Language = C, Update = 2017-06-09

This is *NOT* the official repository of samtools.
##### [arq5x](https://github.com/arq5x)/[filo](https://github.com/arq5x/filo)

ID = 221, Stars = 43, Language = C++, Update = 2015-12-21

Useful FILe and stream Operations
##### [biod](https://github.com/biod)/[BioD](https://github.com/biod/BioD)

ID = 222, Stars = 43, Language = D, Update = 2019-12-17

A D library for computational biology and bioinformatics
##### [niu-lab](https://github.com/niu-lab)/[msisensor2](https://github.com/niu-lab/msisensor2)

ID = 223, Stars = 43, Language = R, Update = 2019-11-21

Microsatellite instability (MSI) detection for tumor only data.
##### [brentp](https://github.com/brentp)/[go-chartjs](https://github.com/brentp/go-chartjs)

ID = 224, Stars = 42, Language = Go, Update = 2017-09-01

golang library to make https://chartjs.org/ plots (this is vanilla #golang, not gopherjs)
##### [biothings](https://github.com/biothings)/[mygene.py](https://github.com/biothings/mygene.py)

ID = 225, Stars = 42, Language = Python, Update = 2020-03-05

mygene is an easy-to-use Python wrapper to access MyGene.Info services.
##### [ensembl](https://github.com/ensembl)/[ensembl-hive](https://github.com/Ensembl/ensembl-hive)

ID = 226, Stars = 42, Language = Perl, Update = 2020-09-18

EnsEMBL Hive - a system for creating and running pipelines on a distributed compute resource
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[reference_data](https://github.com/AstraZeneca-NGS/reference_data)

ID = 227, Stars = 42, Language = Python, Update = 2017-11-28

Reference data: BED files, genes, transcripts, variations.
##### [broadinstitute](https://github.com/broadinstitute)/[catch](https://github.com/broadinstitute/catch)

ID = 228, Stars = 42, Language = Python, Update = 2020-06-12

A package for designing compact and comprehensive capture probe sets.
##### [brentp](https://github.com/brentp)/[hts-nim-tools](https://github.com/brentp/hts-nim-tools)

ID = 229, Stars = 41, Language = Nim, Update = 2020-08-07

useful command-line tools written to show-case hts-nim
##### [wglab](https://github.com/wglab)/[phenolyzer](https://github.com/WGLab/phenolyzer)

ID = 230, Stars = 41, Language = Perl, Update = 2019-12-04

phenotype-based prioritization of candidate genes for human diseases
##### [bioinform](https://github.com/bioinform)/[metasv](https://github.com/bioinform/metasv)

ID = 231, Stars = 41, Language = Python, Update = 2017-06-30

MetaSV: An accurate and integrative structural-variant caller for next generation sequencing
##### [ncbi](https://github.com/ncbi)/[BAMscale](https://github.com/ncbi/BAMscale)

ID = 232, Stars = 40, Language = C, Update = 2020-09-18

BAMscale is a one-step tool for either 1) quantifying and normalizing the coverage of peaks or 2) generated scaled BigWig files for easy visualization of commonly used DNA-seq capture based methods.
##### [brentp](https://github.com/brentp)/[jigv](https://github.com/brentp/jigv)

ID = 233, Stars = 39, Language = Nim, Update = 2020-04-07

igv.js server and automatic configuration to view bam/cram/vcf/bed. "working in under 1 minute"
##### [brentp](https://github.com/brentp)/[indelope](https://github.com/brentp/indelope)

ID = 234, Stars = 39, Language = Nim, Update = 2017-12-03

find large indels (in the blind spot between GATK/freebayes and SV callers)
##### [shubhamchandak94](https://github.com/shubhamchandak94)/[Spring](https://github.com/shubhamchandak94/Spring)

ID = 235, Stars = 39, Language = C++, Update = 2020-01-28

FASTQ compression
##### [ncbi](https://github.com/ncbi)/[JATSPreviewStylesheets](https://github.com/ncbi/JATSPreviewStylesheets)

ID = 236, Stars = 39, Language = XSLT, Update = 2014-05-13

JATS Preview Stylesheets
##### [igvteam](https://github.com/igvteam)/[igv-webapp](https://github.com/igvteam/igv-webapp)

ID = 237, Stars = 39, Language = JavaScript, Update = 2020-09-16

IGV Web App
##### [broadinstitute](https://github.com/broadinstitute)/[gamgee](https://github.com/broadinstitute/gamgee)

ID = 238, Stars = 39, Language = C++, Update = 2019-08-05

A C++14 library for NGS data formats
##### [ewels](https://github.com/ewels)/[nf-core-rnaseq](https://github.com/ewels/nf-core-rnaseq)

ID = 239, Stars = 38, Language = Nextflow, Update = 2020-06-17

See the main fork of this repository here >>>
##### [lh3](https://github.com/lh3)/[CHM-eval](https://github.com/lh3/CHM-eval)

ID = 240, Stars = 38, Language = TeX, Update = 2020-06-24

None
##### [benlangmead](https://github.com/benlangmead)/[crossbow](https://github.com/BenLangmead/crossbow)

ID = 241, Stars = 37, Language = Perl, Update = 2014-01-14

Variant calling from sequence reads using cloud computing
##### [lh3](https://github.com/lh3)/[htsbox](https://github.com/lh3/htsbox)

ID = 242, Stars = 37, Language = C, Update = 2018-03-29

My experimental tools on top of htslib. NOT OFFICIAL!!!
##### [daehwankimlab](https://github.com/daehwankimlab)/[hisat](https://github.com/DaehwanKimLab/hisat)

ID = 243, Stars = 37, Language = C++, Update = 2015-09-08

Fast spliced aligner with low memory requirements
##### [ensembl](https://github.com/ensembl)/[ensembl-compara](https://github.com/Ensembl/ensembl-compara)

ID = 244, Stars = 37, Language = Perl, Update = 2020-09-21

The Ensembl Compara Perl API and SQL schema
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[blabel](https://github.com/Edinburgh-Genome-Foundry/blabel)

ID = 245, Stars = 37, Language = Python, Update = 2020-09-04

üè∑Ô∏è
Python label/sticker PDF generation. HTML templates, built-in barcodes, qr codes, and other goodies
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-viz](https://github.com/broadinstitute/gtex-viz)

ID = 246, Stars = 37, Language = JavaScript, Update = 2020-01-28

GTEx Visualizations
##### [broadinstitute](https://github.com/broadinstitute)/[dsde-deep-learning](https://github.com/broadinstitute/dsde-deep-learning)

ID = 247, Stars = 37, Language = Python, Update = 2019-03-28

DSDE Deep Learning Club
##### [cgatoxford](https://github.com/cgatoxford)/[CGATPipelines](https://github.com/CGATOxford/CGATPipelines)

ID = 248, Stars = 37, Language = Python, Update = 2018-11-08

Collection of CGAT NGS Pipelines
##### [mdshw5](https://github.com/mdshw5)/[simplesam](https://github.com/mdshw5/simplesam)

ID = 249, Stars = 36, Language = Python, Update = 2020-06-02

Simple pure Python SAM parser and objects for working with SAM records
##### [lh3](https://github.com/lh3)/[yak](https://github.com/lh3/yak)

ID = 250, Stars = 36, Language = C, Update = 2020-08-03

Yet another k-mer analyzer
##### [ekg](https://github.com/ekg)/[bamaddrg](https://github.com/ekg/bamaddrg)

ID = 251, Stars = 36, Language = C++, Update = 2017-09-21

adds sample names and read-group (RG) tags to BAM alignments
##### [ekg](https://github.com/ekg)/[glia](https://github.com/ekg/glia)

ID = 252, Stars = 35, Language = C++, Update = 2016-07-05

a string to graph aligner
##### [ding-lab](https://github.com/ding-lab)/[hotspot3d](https://github.com/ding-lab/hotspot3d)

ID = 253, Stars = 35, Language = Perl, Update = 2019-08-22

3D hotspot mutation proximity analysis tool
##### [pharmgkb](https://github.com/pharmgkb)/[PharmCAT](https://github.com/PharmGKB/PharmCAT)

ID = 254, Stars = 35, Language = Java, Update = 2020-09-12

The Pharmacogenomic Clinical Annotation Tool
##### [broadinstitute](https://github.com/broadinstitute)/[CellBender](https://github.com/broadinstitute/CellBender)

ID = 255, Stars = 35, Language = Python, Update = 2020-06-19

CellBender is a software package for eliminating technical artifacts from high-throughput single-cell RNA sequencing (scRNA-seq) data.
##### [brentp](https://github.com/brentp)/[bigly](https://github.com/brentp/bigly)

ID = 256, Stars = 34, Language = Go, Update = 2018-01-17

a pileup library that embraces the huge
##### [lh3](https://github.com/lh3)/[etrf](https://github.com/lh3/etrf)

ID = 257, Stars = 34, Language = C, Update = 2019-10-22

Exact Tandem Repeat Finder (not a TRF replacement)
##### [igvteam](https://github.com/igvteam)/[igv-reports](https://github.com/igvteam/igv-reports)

ID = 258, Stars = 34, Language = Python, Update = 2020-07-21

Python application to generate self-contained igv.js pages that can be opened within a browser with "file" protocol.
##### [snakemake](https://github.com/snakemake)/[snakefmt](https://github.com/snakemake/snakefmt)

ID = 259, Stars = 33, Language = Python, Update = 2020-09-22

The uncompromising Snakemake code formatter
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_portal_core](https://github.com/broadinstitute/single_cell_portal_core)

ID = 260, Stars = 33, Language = Ruby, Update = 2020-09-21

Rails/Docker application for the Broad Institute's single cell RNA-seq data portal
##### [broadinstitute](https://github.com/broadinstitute)/[long-read-pipelines](https://github.com/broadinstitute/long-read-pipelines)

ID = 261, Stars = 33, Language = Jupyter Notebook, Update = 2020-09-21

Long read production pipelines
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_portal](https://github.com/broadinstitute/single_cell_portal)

ID = 262, Stars = 33, Language = Python, Update = 2020-09-16

Tutorials, workflows, and convenience scripts for Single Cell Portal
##### [lh3](https://github.com/lh3)/[klib.nim](https://github.com/lh3/klib.nim)

ID = 263, Stars = 32, Language = Nim, Update = 2020-04-20

Experimental getopt, gzip reader, FASTA/Q parser and interval queries in nim-lang
##### [seppinho](https://github.com/seppinho)/[haplogrep-cmd](https://github.com/seppinho/haplogrep-cmd)

ID = 264, Stars = 32, Language = Java, Update = 2020-08-20

HaploGrep - mtDNA haplogroup classification. Supporting rCRS and RSRS.
##### [steven-n-hart](https://github.com/steven-n-hart)/[vcf-miner](https://github.com/Steven-N-Hart/vcf-miner)

ID = 265, Stars = 32, Language = HTML, Update = 2019-11-19

VCF-Miner: A graphical user interface for sorting, filtering and querying annotated VCF Files
##### [ensembl](https://github.com/ensembl)/[ensembl-tools](https://github.com/Ensembl/ensembl-tools)

ID = 266, Stars = 32, Language = Perl, Update = 2020-08-19

Ensembl tools
##### [brentp](https://github.com/brentp)/[450k-analysis-guide](https://github.com/brentp/450k-analysis-guide)

ID = 267, Stars = 31, Language = TeX, Update = 2014-10-02

A Practical (And Opinionated) Guide To Analyzing 450K Data
##### [lh3](https://github.com/lh3)/[bioseq-js](https://github.com/lh3/bioseq-js)

ID = 268, Stars = 31, Language = HTML, Update = 2019-08-08

For live demo, see http://lh3lh3.users.sourceforge.net/bioseq.shtml
##### [arq5x](https://github.com/arq5x)/[nanopore-scripts](https://github.com/arq5x/nanopore-scripts)

ID = 269, Stars = 31, Language = Python, Update = 2016-04-17

Various scripts and recipes for working with nanopore data
##### [shenwei356](https://github.com/shenwei356)/[ClipboardTextJoiner](https://github.com/shenwei356/ClipboardTextJoiner)

ID = 270, Stars = 31, Language = Perl, Update = 2016-07-22

Monitoring system clipboard change and joining multi-line text. It's very useful when copying multi-line text from PDF files.
##### [brentp](https://github.com/brentp)/[quicksect](https://github.com/brentp/quicksect)

ID = 271, Stars = 30, Language = Python, Update = 2019-06-04

a cythonized, extended version of the interval search tree in bx
##### [ekg](https://github.com/ekg)/[thesis](https://github.com/ekg/thesis)

ID = 272, Stars = 30, Language = TeX, Update = 2019-07-10

my PhD thesis
##### [brentp](https://github.com/brentp)/[align](https://github.com/brentp/align)

ID = 273, Stars = 29, Language = Python, Update = 2016-12-21

sequence alignment. global, local, glocal.
##### [ewels](https://github.com/ewels)/[Labrador](https://github.com/ewels/Labrador)

ID = 274, Stars = 29, Language = PHP, Update = 2018-11-16

A web based tool to manage and automate the processing of publicly available datasets.
##### [s-andrews](https://github.com/s-andrews)/[BamQC](https://github.com/s-andrews/BamQC)

ID = 275, Stars = 29, Language = Java, Update = 2018-05-16

Mapped QC analysis program
##### [ding-lab](https://github.com/ding-lab)/[PanCanAtlasGermline](https://github.com/ding-lab/PanCanAtlasGermline)

ID = 276, Stars = 29, Language = R, Update = 2018-10-27

None
##### [brentp](https://github.com/brentp)/[combined-pvalues](https://github.com/brentp/combined-pvalues)

ID = 277, Stars = 28, Language = Python, Update = 2020-08-24

combining p-values using modified stouffer-liptak for spatially correlated results (probes)
##### [brentp](https://github.com/brentp)/[bsub](https://github.com/brentp/bsub)

ID = 278, Stars = 28, Language = Python, Update = 2020-02-27

python wrapper to submit jobs to bsub (and later qsub)
##### [felixkrueger](https://github.com/felixkrueger)/[SNPsplit](https://github.com/FelixKrueger/SNPsplit)

ID = 279, Stars = 28, Language = Perl, Update = 2020-07-10

Allele-specific alignment sorting
##### [broadinstitute](https://github.com/broadinstitute)/[cromshell](https://github.com/broadinstitute/cromshell)

ID = 280, Stars = 28, Language = Shell, Update = 2020-09-18

CLI for interacting with Cromwell servers
##### [broadinstitute](https://github.com/broadinstitute)/[genomics-in-the-cloud](https://github.com/broadinstitute/genomics-in-the-cloud)

ID = 281, Stars = 28, Language = Jupyter Notebook, Update = 2020-05-29

Source code and related materials for the O'Reilly book
##### [broadinstitute](https://github.com/broadinstitute)/[SignatureAnalyzer-GPU](https://github.com/broadinstitute/SignatureAnalyzer-GPU)

ID = 282, Stars = 28, Language = Python, Update = 2019-12-09

GPU implementation of ARD NMF
##### [brentp](https://github.com/brentp)/[gsort](https://github.com/brentp/gsort)

ID = 283, Stars = 27, Language = Go, Update = 2020-05-26

sort genomic data
##### [lh3](https://github.com/lh3)/[hapdip](https://github.com/lh3/hapdip)

ID = 284, Stars = 27, Language = JavaScript, Update = 2018-02-20

The CHM1-NA12878 benchmark for single-sample SNP/INDEL calling from WGS Illumina data
##### [s-andrews](https://github.com/s-andrews)/[SeqMonk](https://github.com/s-andrews/SeqMonk)

ID = 285, Stars = 27, Language = Java, Update = 2020-07-29

SeqMonk NGS visualisation and analysis tool
##### [illumina](https://github.com/illumina)/[BeadArrayFiles](https://github.com/Illumina/BeadArrayFiles)

ID = 286, Stars = 27, Language = Python, Update = 2020-02-17

Python library to parse file formats related to Illumina bead arrays
##### [brentp](https://github.com/brentp)/[poverlap](https://github.com/brentp/poverlap)

ID = 287, Stars = 26, Language = Python, Update = 2020-07-11

significance testing over interval overlaps
##### [brentp](https://github.com/brentp)/[geneimpacts](https://github.com/brentp/geneimpacts)

ID = 288, Stars = 26, Language = Python, Update = 2019-10-29

prioritize effects of variant annotations from VEP, SnpEff, et al.
##### [lh3](https://github.com/lh3)/[pre-pe](https://github.com/lh3/pre-pe)

ID = 289, Stars = 26, Language = C, Update = 2018-06-28

Preprocessing paired-end reads produced with experiment-specific protocols
##### [wglab](https://github.com/wglab)/[VirTect](https://github.com/WGLab/VirTect)

ID = 290, Stars = 26, Language = Python, Update = 2020-06-02

Detection of viruses from RNA-Seq on human samples
##### [ncbi](https://github.com/ncbi)/[docker](https://github.com/ncbi/docker)

ID = 291, Stars = 26, Language = C++, Update = 2020-07-09

None
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad_methods](https://github.com/broadinstitute/gnomad_methods)

ID = 292, Stars = 26, Language = Python, Update = 2020-09-18

Hail helper functions for gnomAD/MacArthur lab
##### [broadinstitute](https://github.com/broadinstitute)/[PhylogicNDT](https://github.com/broadinstitute/PhylogicNDT)

ID = 293, Stars = 26, Language = HTML, Update = 2020-09-02

None
##### [brentp](https://github.com/brentp)/[fastbit-python](https://github.com/brentp/fastbit-python)

ID = 294, Stars = 25, Language = Python, Update = 2018-08-10

pythonic access to fastbit
##### [shenwei356](https://github.com/shenwei356)/[unikmer](https://github.com/shenwei356/unikmer)

ID = 295, Stars = 25, Language = Go, Update = 2020-09-07

unikmer: toolkit for nucleic acid k-mer analysis, including set operation, indexing, and searching on k-mers optional with TaxIDs but without count information.
##### [mozack](https://github.com/mozack)/[vdjer](https://github.com/mozack/vdjer)

ID = 296, Stars = 25, Language = C, Update = 2017-04-11

V'DJer - B Cell Receptor Repertoire Reconstruction from short read mRNA-Seq data
##### [opengene](https://github.com/opengene)/[ctdna-pipeline](https://github.com/OpenGene/ctdna-pipeline)

ID = 297, Stars = 25, Language = Shell, Update = 2017-09-23

A simplified pipeline for ctDNA sequencing data analysis
##### [opengene](https://github.com/opengene)/[FusionDirect.jl](https://github.com/OpenGene/FusionDirect.jl)

ID = 298, Stars = 25, Language = Julia, Update = 2017-08-13

(No maintenance) Detect gene fusion directly from raw fastq files
##### [ensembl](https://github.com/ensembl)/[postgap](https://github.com/Ensembl/postgap)

ID = 299, Stars = 25, Language = Jupyter Notebook, Update = 2020-09-14

Linking GWAS studies to genes through cis-regulatory datasets
##### [illumina](https://github.com/illumina)/[GTCtoVCF](https://github.com/Illumina/GTCtoVCF)

ID = 300, Stars = 25, Language = Python, Update = 2020-07-09

Script to convert GTC/BPM files to VCF
##### [illumina](https://github.com/illumina)/[gvcfgenotyper](https://github.com/Illumina/gvcfgenotyper)

ID = 301, Stars = 25, Language = C++, Update = 2019-02-26

A utility for merging and genotyping Illumina-style GVCFs.
##### [broadinstitute](https://github.com/broadinstitute)/[single_cell_analysis](https://github.com/broadinstitute/single_cell_analysis)

ID = 302, Stars = 25, Language = HTML, Update = 2017-02-05

Documents used for workshops on single cell analysis
##### [benlangmead](https://github.com/benlangmead)/[qtip](https://github.com/BenLangmead/qtip)

ID = 303, Stars = 24, Language = C++, Update = 2019-10-31

Qtip: a tandem simulation approach for accurately predicting read alignment mapping qualities
##### [lh3](https://github.com/lh3)/[asub](https://github.com/lh3/asub)

ID = 304, Stars = 24, Language = Perl, Update = 2019-11-29

A unified array job submitter for LSF, SGE/UGE and Slurm
##### [wglab](https://github.com/wglab)/[NanoMod](https://github.com/WGLab/NanoMod)

ID = 305, Stars = 24, Language = Python, Update = 2020-07-24

NanoMod: a computational tool to detect DNA modifications using Nanopore long-read sequencing data
##### [wglab](https://github.com/wglab)/[EHR-Phenolyzer](https://github.com/WGLab/EHR-Phenolyzer)

ID = 306, Stars = 24, Language = Python, Update = 2019-10-30

A pipeline that takes clinical notes from EHRs and generate phenotype terms, then generate ranked gene list
##### [broadinstitute](https://github.com/broadinstitute)/[gdctools](https://github.com/broadinstitute/gdctools)

ID = 307, Stars = 24, Language = Python, Update = 2018-10-11

Python and UNIX CLI utilities to simplify interaction with the NIH/NCI Genomics Data Commons
##### [lh3](https://github.com/lh3)/[fermi2](https://github.com/lh3/fermi2)

ID = 308, Stars = 23, Language = C, Update = 2018-02-07

None
##### [wglab](https://github.com/wglab)/[Phen2Gene](https://github.com/WGLab/Phen2Gene)

ID = 309, Stars = 23, Language = Python, Update = 2020-06-12

Phenotype driven gene prioritization for HPO
##### [bioconda](https://github.com/bioconda)/[bioconda.github.io](https://github.com/bioconda/bioconda.github.io)

ID = 310, Stars = 23, Language = HTML, Update = 2020-09-21

Bioconda is a distribution of bioinformatics software realized as a channel for the versatile package manager Conda.
##### [brentp](https://github.com/brentp)/[nim-lapper](https://github.com/brentp/nim-lapper)

ID = 311, Stars = 22, Language = Nim, Update = 2020-06-09

fast easy interval overlapping for nim-lang
##### [brentp](https://github.com/brentp)/[cigar](https://github.com/brentp/cigar)

ID = 312, Stars = 22, Language = Python, Update = 2019-09-26

simple library for dealing with SAM cigar strings
##### [lh3](https://github.com/lh3)/[dipcall](https://github.com/lh3/dipcall)

ID = 313, Stars = 22, Language = JavaScript, Update = 2020-09-14

Reference-based variant calling pipeline for a pair of phased haplotype assemblies
##### [arq5x](https://github.com/arq5x)/[bits](https://github.com/arq5x/bits)

ID = 314, Stars = 22, Language = C++, Update = 2017-04-26

BITS: Binary Interval Search
##### [opengene](https://github.com/opengene)/[VisualMSI](https://github.com/OpenGene/VisualMSI)

ID = 315, Stars = 22, Language = C++, Update = 2019-06-04

Detect and visualize microsatellite instability(MSI) from NGS data
##### [opengene](https://github.com/opengene)/[defq](https://github.com/OpenGene/defq)

ID = 316, Stars = 22, Language = C, Update = 2018-07-28

Ultra-fast Multi-threaded FASTQ Demultiplexing
##### [ensembl](https://github.com/ensembl)/[ensembl-variation](https://github.com/Ensembl/ensembl-variation)

ID = 317, Stars = 22, Language = Perl, Update = 2020-09-21

The Ensembl Variation Perl API and SQL schema
##### [illumina](https://github.com/illumina)/[ExpansionHunterDenovo](https://github.com/Illumina/ExpansionHunterDenovo)

ID = 318, Stars = 22, Language = C++, Update = 2020-08-26

A suite of tools for detecting expansions of short tandem repeats
##### [refresh-bio](https://github.com/refresh-bio)/[FAMSA](https://github.com/refresh-bio/FAMSA)

ID = 319, Stars = 22, Language = C++, Update = 2020-06-28

Algorithm for large-scale multiple sequence alignments (400k proteins in 2 hours and 8BG of RAM)
##### [broadinstitute](https://github.com/broadinstitute)/[gatk-sv](https://github.com/broadinstitute/gatk-sv)

ID = 320, Stars = 22, Language = wdl, Update = 2020-09-20

None
##### [pcingola](https://github.com/pcingola)/[SnpSift](https://github.com/pcingola/SnpSift)

ID = 321, Stars = 21, Language = Java, Update = 2020-08-09

None
##### [gregoryfaust](https://github.com/gregoryfaust)/[yaha](https://github.com/GregoryFaust/yaha)

ID = 322, Stars = 21, Language = C, Update = 2017-12-06

yaha: a flexible, sensitive and accurate DNA alignment tool designed to find optimal split-read mappings on single-end queries from 100bp to 32kb.
##### [ekg](https://github.com/ekg)/[edyeet](https://github.com/ekg/edyeet)

ID = 323, Stars = 21, Language = C++, Update = 2020-09-02

base-accurate DNA sequence alignments using edlib and mashmap2
##### [ncbi](https://github.com/ncbi)/[magicblast](https://github.com/ncbi/magicblast)

ID = 324, Stars = 21, Language = Python, Update = 2020-07-18

None
##### [ncbi](https://github.com/ncbi)/[consul-announcer](https://github.com/ncbi/consul-announcer)

ID = 325, Stars = 21, Language = Python, Update = 2016-12-12

Service announcer for Consul (https://www.consul.io/).
##### [biopython](https://github.com/biopython)/[biopython_docker](https://github.com/biopython/biopython_docker)

ID = 326, Stars = 21, Language = Shell, Update = 2016-08-30

Dockerfiles to build Biopython instances
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-orchestration](https://github.com/broadinstitute/firecloud-orchestration)

ID = 327, Stars = 21, Language = Scala, Update = 2020-09-21

None
##### [broadinstitute](https://github.com/broadinstitute)/[2019_scWorkshop](https://github.com/broadinstitute/2019_scWorkshop)

ID = 328, Stars = 21, Language = TeX, Update = 2019-11-09

Repo for Physalia course Analysis of Single Cell RNA-Seq data
##### [lh3](https://github.com/lh3)/[sdust](https://github.com/lh3/sdust)

ID = 329, Stars = 20, Language = C, Update = 2019-10-03

Symmetric DUST for finding low-complexity regions in DNA sequences
##### [ekg](https://github.com/ekg)/[1000G-integration](https://github.com/ekg/1000G-integration)

ID = 330, Stars = 20, Language = Shell, Update = 2018-01-16

variant integration methods for the 1000 Genomes Project
##### [ensembl](https://github.com/ensembl)/[Bio-DB-HTS](https://github.com/Ensembl/Bio-DB-HTS)

ID = 331, Stars = 20, Language = Perl, Update = 2020-07-31

Git repo for Bio::DB::HTS module on CPAN, providing Perl links into HTSlib
##### [wglab](https://github.com/wglab)/[SeqMule](https://github.com/WGLab/SeqMule)

ID = 332, Stars = 20, Language = C++, Update = 2020-01-05

Automated human exome/genome variants detection from FASTQ files
##### [ncbi](https://github.com/ncbi)/[JUDI](https://github.com/ncbi/JUDI)

ID = 333, Stars = 20, Language = Python, Update = 2020-09-20

This repository contains the source code of JUDI, a workflow management system for developing complex bioinformatics software with many parameter settings. Bioinformatics pipeline: Just Do It!
##### [refresh-bio](https://github.com/refresh-bio)/[Whisper](https://github.com/refresh-bio/Whisper)

ID = 334, Stars = 20, Language = C++, Update = 2020-07-02

None
##### [broadinstitute](https://github.com/broadinstitute)/[gatk-docs](https://github.com/broadinstitute/gatk-docs)

ID = 335, Stars = 20, Language = HTML, Update = 2020-01-06

Documentation archive for GATK tools and workflows
##### [brentp](https://github.com/brentp)/[toolshed](https://github.com/brentp/toolshed)

ID = 336, Stars = 19, Language = Python, Update = 2019-12-18

python stuff I use
##### [ekg](https://github.com/ekg)/[hhga](https://github.com/ekg/hhga)

ID = 337, Stars = 19, Language = C++, Update = 2019-06-13

haplotypes genotypes and alleles example decision synthesizer
##### [shenwei356](https://github.com/shenwei356)/[easy_qsub](https://github.com/shenwei356/easy_qsub)

ID = 338, Stars = 19, Language = Python, Update = 2019-06-18

Easily submitting multiple PBS jobs or running local jobs in parallel. Multiple input files supported.
##### [opengene](https://github.com/opengene)/[MrBam](https://github.com/OpenGene/MrBam)

ID = 339, Stars = 19, Language = Python, Update = 2018-11-26

Query Mutated Reads from a Bam
##### [opengene](https://github.com/opengene)/[SeqMaker.jl](https://github.com/OpenGene/SeqMaker.jl)

ID = 340, Stars = 19, Language = Julia, Update = 2017-08-10

(No maintenance) Next Generation Sequencing Simulation with SNP, Variation and Sequencing Error Integrated
##### [ensembl](https://github.com/ensembl)/[ensembl-webcode](https://github.com/Ensembl/ensembl-webcode)

ID = 341, Stars = 19, Language = Perl, Update = 2020-09-21

The code to run an Ensembl website
##### [illumina](https://github.com/illumina)/[Isaac3](https://github.com/Illumina/Isaac3)

ID = 342, Stars = 19, Language = C++, Update = 2018-02-13

Aligner for sequencing data
##### [ncbi](https://github.com/ncbi)/[gprobe](https://github.com/ncbi/gprobe)

ID = 343, Stars = 19, Language = Go, Update = 2020-01-31

client app for the gRPC health-checking protocol
##### [broadinstitute](https://github.com/broadinstitute)/[getzlab-SignatureAnalyzer](https://github.com/broadinstitute/getzlab-SignatureAnalyzer)

ID = 344, Stars = 19, Language = Python, Update = 2020-09-21

Updated SignatureAnalyzer-GPU with mutational spectra & RNA expression compatibility.
##### [broadinstitute](https://github.com/broadinstitute)/[BipolarCell2016](https://github.com/broadinstitute/BipolarCell2016)

ID = 345, Stars = 19, Language = R, Update = 2020-02-07

None
##### [broadinstitute](https://github.com/broadinstitute)/[BARD](https://github.com/broadinstitute/BARD)

ID = 346, Stars = 19, Language = Groovy, Update = 2019-01-29

BioAssay Research Database
##### [genedx](https://github.com/genedx)/[scramble](https://github.com/GeneDx/scramble)

ID = 347, Stars = 19, Language = R, Update = 2020-09-16

None
##### [gt1](https://github.com/gt1)/[daccord](https://github.com/gt1/daccord)

ID = 348, Stars = 18, Language = C++, Update = 2018-09-12

d'accord is a non hybrid long read consensus program based on local de Bruijn graph assembly
##### [ding-lab](https://github.com/ding-lab)/[BreakPointSurveyor](https://github.com/ding-lab/BreakPointSurveyor)

ID = 349, Stars = 18, Language = Shell, Update = 2020-01-28

A comprehensive pipeline to analyze and visualize structural variants
##### [biothings](https://github.com/biothings)/[biothings.api](https://github.com/biothings/biothings.api)

ID = 350, Stars = 18, Language = CSS, Update = 2020-08-14

BioThings API framework - Making high-performance API for biological annotation data
##### [wglab](https://github.com/wglab)/[GenGen](https://github.com/WGLab/GenGen)

ID = 351, Stars = 18, Language = C, Update = 2019-01-16

A set of software tools to facilitate GWAS analysis
##### [illumina](https://github.com/illumina)/[SMNCopyNumberCaller](https://github.com/Illumina/SMNCopyNumberCaller)

ID = 352, Stars = 18, Language = Python, Update = 2020-08-27

A copy number caller for SMN1 and SMN2 to enable SMA diagnosis and carrier screening with WGS
##### [xjtu-omics](https://github.com/xjtu-omics)/[msisensor-pro](https://github.com/xjtu-omics/msisensor-pro)

ID = 353, Stars = 18, Language = C++, Update = 2020-09-17

Microsatellite Instability (MSI) detection using high-throughput sequencing data.
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[Primavera](https://github.com/Edinburgh-Genome-Foundry/Primavera)

ID = 354, Stars = 18, Language = Python, Update = 2020-09-17

üå∏
Python library for primer-based verification of DNA assemblies: primer selection, data analyis, etc.
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[DnaCauldron](https://github.com/Edinburgh-Genome-Foundry/DnaCauldron)

ID = 355, Stars = 18, Language = Python, Update = 2020-09-11

‚öóÔ∏è
Simple cloning simulator (Golden Gate etc.) for single and combinatorial assemblies
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[sequenticon](https://github.com/Edinburgh-Genome-Foundry/sequenticon)

ID = 356, Stars = 18, Language = Python, Update = 2020-09-04

üëæ
Generate identicons for DNA sequences with Python
##### [broadinstitute](https://github.com/broadinstitute)/[gnomad_qc](https://github.com/broadinstitute/gnomad_qc)

ID = 357, Stars = 18, Language = Python, Update = 2020-09-21

None
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-ui](https://github.com/broadinstitute/firecloud-ui)

ID = 358, Stars = 18, Language = Clojure, Update = 2020-08-26

FireCloud user interface for web browsers.
##### [broadinstitute](https://github.com/broadinstitute)/[wdl-ide](https://github.com/broadinstitute/wdl-ide)

ID = 359, Stars = 18, Language = Python, Update = 2019-11-14

Rich IDE support for Workflow Description Language
##### [ekg](https://github.com/ekg)/[viral-assembly](https://github.com/ekg/viral-assembly)

ID = 360, Stars = 17, Language = Shell, Update = 2020-05-18

exploring viral genome assembly with variation graph tools
##### [ekg](https://github.com/ekg)/[multichoose](https://github.com/ekg/multichoose)

ID = 361, Stars = 17, Language = C++, Update = 2019-12-09

generate multiset combinations (n multichoose k)
##### [wglab](https://github.com/wglab)/[RepeatHMM](https://github.com/WGLab/RepeatHMM)

ID = 362, Stars = 17, Language = Python, Update = 2020-08-04

a hidden Markov model to infer simple repeats from genome sequences
##### [illumina](https://github.com/illumina)/[GraphAlignmentViewer](https://github.com/Illumina/GraphAlignmentViewer)

ID = 363, Stars = 17, Language = Python, Update = 2020-06-05

None
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[disambiguate](https://github.com/AstraZeneca-NGS/disambiguate)

ID = 364, Stars = 17, Language = C++, Update = 2018-05-08

Disambiguation algorithm for reads aligned to human and mouse genomes using Tophat or BWA mem
##### [broadinstitute](https://github.com/broadinstitute)/[sam](https://github.com/broadinstitute/sam)

ID = 365, Stars = 17, Language = Scala, Update = 2020-09-21

workbench identity and access management
##### [broadinstitute](https://github.com/broadinstitute)/[deepometry](https://github.com/broadinstitute/deepometry)

ID = 366, Stars = 17, Language = Python, Update = 2020-09-12

Image classification for imaging flow cytometry.
##### [broadinstitute](https://github.com/broadinstitute)/[cromwell-tools](https://github.com/broadinstitute/cromwell-tools)

ID = 367, Stars = 17, Language = Python, Update = 2020-02-21

A collection of Python clients and accessory scripts for interacting with the Cromwell
##### [humanlongevity](https://github.com/humanlongevity)/[tredparse](https://github.com/humanlongevity/tredparse)

ID = 368, Stars = 17, Language = Python, Update = 2020-08-20

TREDPARSE: HLI Short Tandem Repeat (STR) caller
##### [brentp](https://github.com/brentp)/[genoiser](https://github.com/brentp/genoiser)

ID = 369, Stars = 16, Language = Nim, Update = 2020-04-15

use the noise
##### [lh3](https://github.com/lh3)/[calN50](https://github.com/lh3/calN50)

ID = 370, Stars = 16, Language = JavaScript, Update = 2020-07-14

Compute N50/NG50 and auN/auNG
##### [lh3](https://github.com/lh3)/[lianti](https://github.com/lh3/lianti)

ID = 371, Stars = 16, Language = C, Update = 2019-02-20

Tools to process LIANTI sequence data
##### [ekg](https://github.com/ekg)/[drawbwt](https://github.com/ekg/drawbwt)

ID = 372, Stars = 16, Language = C++, Update = 2019-04-29

vector illustrations of BWT based searching on small strings
##### [igvteam](https://github.com/igvteam)/[igv.js-flask](https://github.com/igvteam/igv.js-flask)

ID = 373, Stars = 16, Language = Python, Update = 2020-04-06

Example project for integrating igv.js and flask
##### [broadinstitute](https://github.com/broadinstitute)/[wordpress-crowd-plugin](https://github.com/broadinstitute/wordpress-crowd-plugin)

ID = 374, Stars = 16, Language = PHP, Update = 2018-01-11

Crowd Authentication Plugin for Wordpress
##### [genedx](https://github.com/genedx)/[phenopy](https://github.com/GeneDx/phenopy)

ID = 375, Stars = 16, Language = Python, Update = 2020-08-31

Phenotype comparison tools using semantic similarity.
##### [ekg](https://github.com/ekg)/[smoothxg](https://github.com/ekg/smoothxg)

ID = 376, Stars = 15, Language = C++, Update = 2020-09-14

linearize and simplify variation graphs using blocked partial order alignment
##### [seppinho](https://github.com/seppinho)/[mutserve](https://github.com/seppinho/mutserve)

ID = 377, Stars = 15, Language = Java, Update = 2020-09-16

mtDNA Variant Caller
##### [qiime2](https://github.com/qiime2)/[q2cli](https://github.com/qiime2/q2cli)

ID = 378, Stars = 15, Language = Python, Update = 2020-08-22

Command line interface for QIIME 2
##### [broadinstitute](https://github.com/broadinstitute)/[pyfrost](https://github.com/broadinstitute/pyfrost)

ID = 379, Stars = 15, Language = C++, Update = 2020-09-18

Python bindings for Bifrost with a NetworkX compatible API
##### [brentp](https://github.com/brentp)/[lua-stringy](https://github.com/brentp/lua-stringy)

ID = 380, Stars = 14, Language = C, Update = 2020-03-21

fast lua string operations
##### [gpertea](https://github.com/gpertea)/[gclib](https://github.com/gpertea/gclib)

ID = 381, Stars = 14, Language = C++, Update = 2020-09-19

GCLib - Genomic C++ library of reusable code for bioinformatics projects
##### [qiime2](https://github.com/qiime2)/[docs](https://github.com/qiime2/docs)

ID = 382, Stars = 14, Language = Python, Update = 2020-09-22

Source repository for https://docs.qiime2.org
##### [illumina](https://github.com/illumina)/[witty.er](https://github.com/Illumina/witty.er)

ID = 383, Stars = 14, Language = C#, Update = 2020-08-03

What is true, thank you, ernestly. A large variant benchmarking tool analogous to hap.py for small variants.
##### [illumina](https://github.com/illumina)/[BaseSpace_Clarity_LIMS](https://github.com/Illumina/BaseSpace_Clarity_LIMS)

ID = 384, Stars = 14, Language = Python, Update = 2020-07-03

API libraries, application examples, and custom tools for BaseSpace Clarity LIMS
##### [ncbi](https://github.com/ncbi)/[pipelines](https://github.com/ncbi/pipelines)

ID = 385, Stars = 14, Language = Common Workflow Language, Update = 2018-11-28

None
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[crazydoc](https://github.com/Edinburgh-Genome-Foundry/crazydoc)

ID = 386, Stars = 14, Language = Python, Update = 2020-09-16

Read DNA sequences from colourful Microsoft Word documents
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[Plateo](https://github.com/Edinburgh-Genome-Foundry/Plateo)

ID = 387, Stars = 14, Language = Python, Update = 2020-02-06

ü§ñ
Python biolab automation library: parsers, reports generators, picklists simulators, and more
##### [brentp](https://github.com/brentp)/[bigwig-nim](https://github.com/brentp/bigwig-nim)

ID = 388, Stars = 13, Language = Nim, Update = 2020-05-02

command-line querying+conversion of bigwigs and a nim wrapper for dpryan's libbigwig
##### [andreas-wilm](https://github.com/andreas-wilm)/[lofreq3](https://github.com/andreas-wilm/lofreq3)

ID = 389, Stars = 13, Language = Nim, Update = 2020-09-10

LoFreq Version 3
##### [lh3](https://github.com/lh3)/[gfa1](https://github.com/lh3/gfa1)

ID = 390, Stars = 13, Language = C, Update = 2018-08-17

This repo is deprecated. Please use gfatools instead.
##### [gt1](https://github.com/gt1)/[libmaus2](https://github.com/gt1/libmaus2)

ID = 391, Stars = 13, Language = C++, Update = 2019-03-19

collection of data structures and algorithms
##### [ekg](https://github.com/ekg)/[tabixpp](https://github.com/ekg/tabixpp)

ID = 392, Stars = 13, Language = C++, Update = 2020-05-31

C++ wrapper to tabix indexer
##### [shenwei356](https://github.com/shenwei356)/[bwt](https://github.com/shenwei356/bwt)

ID = 393, Stars = 13, Language = Go, Update = 2020-04-18

Burrows-Wheeler Transform and FM-index in golang
##### [biothings](https://github.com/biothings)/[myvariant.py](https://github.com/biothings/myvariant.py)

ID = 394, Stars = 13, Language = Python, Update = 2020-02-21

Python client for MyVariant.info web services.
##### [wglab](https://github.com/wglab)/[LinkedSV](https://github.com/WGLab/LinkedSV)

ID = 395, Stars = 13, Language = C, Update = 2020-04-25

None
##### [wglab](https://github.com/wglab)/[icages](https://github.com/WGLab/icages)

ID = 396, Stars = 13, Language = Perl, Update = 2019-03-21

iCAGES (integrated CAncer GEnome Score) is an effective tool for prioritizing cancer driver genes for a patient
##### [illumina](https://github.com/illumina)/[Cyrius](https://github.com/Illumina/Cyrius)

ID = 397, Stars = 13, Language = Python, Update = 2020-07-24

A tool to genotype CYP2D6 with WGS data
##### [ncbi](https://github.com/ncbi)/[cwl-ngs-workflows-cbb](https://github.com/ncbi/cwl-ngs-workflows-cbb)

ID = 398, Stars = 13, Language = Common Workflow Language, Update = 2020-09-05

A set of CWL tools and workflows used by NCBI Computational Biology Branch for NGS data analysis
##### [refresh-bio](https://github.com/refresh-bio)/[ORCOM](https://github.com/refresh-bio/ORCOM)

ID = 399, Stars = 13, Language = C++, Update = 2020-06-24

Overlapping Reads COmpression with Minimizers
##### [refresh-bio](https://github.com/refresh-bio)/[GTC](https://github.com/refresh-bio/GTC)

ID = 400, Stars = 13, Language = C++, Update = 2020-06-24

GenoTypes Compressor
##### [broadinstitute](https://github.com/broadinstitute)/[fiss](https://github.com/broadinstitute/fiss)

ID = 401, Stars = 13, Language = Python, Update = 2020-09-17

FireCloud Service Selector (FISS) -- Python bindings and CLI for FireCloud execution engine
##### [broadinstitute](https://github.com/broadinstitute)/[rawls](https://github.com/broadinstitute/rawls)

ID = 402, Stars = 13, Language = Scala, Update = 2020-09-16

Rawls service for DSDE
##### [broadinstitute](https://github.com/broadinstitute)/[python-cert_manager](https://github.com/broadinstitute/python-cert_manager)

ID = 403, Stars = 13, Language = Python, Update = 2020-09-15

Python interface to the Sectigo Certificate Manager REST API
##### [broadinstitute](https://github.com/broadinstitute)/[2020_scWorkshop](https://github.com/broadinstitute/2020_scWorkshop)

ID = 404, Stars = 13, Language = Shell, Update = 2020-04-15

Code and data repository for the 2020 physalia course on single cell RNA sequencing.
##### [shubhamchandak94](https://github.com/shubhamchandak94)/[HARC](https://github.com/shubhamchandak94/HARC)

ID = 405, Stars = 12, Language = C++, Update = 2019-10-03

Fastq compression
##### [lh3](https://github.com/lh3)/[lh3-snippets](https://github.com/lh3/lh3-snippets)

ID = 406, Stars = 12, Language = C, Update = 2020-01-23

None
##### [ekg](https://github.com/ekg)/[mmmulti](https://github.com/ekg/mmmulti)

ID = 407, Stars = 12, Language = C++, Update = 2020-08-29

memory mapped multimap, multiset, and implicit interval tree based on an in-place parallel sort
##### [abyzovlab](https://github.com/abyzovlab)/[CNVpytor](https://github.com/abyzovlab/CNVpytor)

ID = 408, Stars = 12, Language = Python, Update = 2020-09-16

a python extension of CNVnator -- a tool for CNV analysis from depth-of-coverage by mapped reads
##### [opengene](https://github.com/opengene)/[UniqueKMER](https://github.com/OpenGene/UniqueKMER)

ID = 409, Stars = 12, Language = C, Update = 2020-07-28

Generate unique KMERs for every contig in a FASTA file
##### [ensembl](https://github.com/ensembl)/[ensembl-funcgen](https://github.com/Ensembl/ensembl-funcgen)

ID = 410, Stars = 12, Language = Perl, Update = 2020-09-03

Ensembl Funcgen Perl API and SQL schema
##### [illumina](https://github.com/illumina)/[Isaac4](https://github.com/Illumina/Isaac4)

ID = 411, Stars = 12, Language = C++, Update = 2020-06-19

Isaac aligner version 4
##### [ncbi](https://github.com/ncbi)/[cxx-toolkit](https://github.com/ncbi/cxx-toolkit)

ID = 412, Stars = 12, Language = CSS, Update = 2020-09-21

None
##### [ncbi](https://github.com/ncbi)/[osiris](https://github.com/ncbi/osiris)

ID = 413, Stars = 12, Language = C++, Update = 2020-06-04

OSIRIS is a public domain quality assurance software package that facilitates the assessment of multiplex short tandem repeat (STR) DNA profiles based on laboratory-specific protocols. OSIRIS evaluates the raw electrophoresis data contained in .fsa or .hid files using an independently derived mathematically-based sizing algorithm. OSIRIS current‚Ä¶
##### [snakemake](https://github.com/snakemake)/[snakemake-github-action](https://github.com/snakemake/snakemake-github-action)

ID = 414, Stars = 12, Language = Shell, Update = 2020-09-15

A Github action for running a Snakemake workflow
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[CAB](https://github.com/Edinburgh-Genome-Foundry/CAB)

ID = 415, Stars = 12, Language = Vue, Update = 2020-06-05

üöñ
The friendly Computational App Boilerplate. Django + Vue.JS + Redis queues + NginX
##### [refresh-bio](https://github.com/refresh-bio)/[FaStore](https://github.com/refresh-bio/FaStore)

ID = 416, Stars = 12, Language = C++, Update = 2020-06-24

FaStore - high-performance FASTQ files compressor
##### [broadinstitute](https://github.com/broadinstitute)/[covid19-testing](https://github.com/broadinstitute/covid19-testing)

ID = 417, Stars = 12, Language = HTML, Update = 2020-09-21

COVID-19 Diagnostic Processing Dashboard
##### [broadinstitute](https://github.com/broadinstitute)/[cms](https://github.com/broadinstitute/cms)

ID = 418, Stars = 12, Language = Python, Update = 2020-07-04

Composite of Multiple Signals: tests for selection in meiotically recombinant populations
##### [brentp](https://github.com/brentp)/[nim-kmer](https://github.com/brentp/nim-kmer)

ID = 419, Stars = 11, Language = Nim, Update = 2020-04-11

DNA kmer operations for nim
##### [andreas-wilm](https://github.com/andreas-wilm)/[nimreadfq](https://github.com/andreas-wilm/nimreadfq)

ID = 420, Stars = 11, Language = C, Update = 2019-11-22

A wrapper to Heng Li's kseq/readfq, an efficient FastQ/Fasta parser
##### [ewels](https://github.com/ewels)/[MultiQC_TestData](https://github.com/ewels/MultiQC_TestData)

ID = 421, Stars = 11, Language = HTML, Update = 2020-09-14

Test data for MultiQC. See https://github.com/ewels/MultiQC
##### [ekg](https://github.com/ekg)/[gimbricate](https://github.com/ekg/gimbricate)

ID = 422, Stars = 11, Language = C++, Update = 2020-08-30

recompute GFA link overlaps
##### [shenwei356](https://github.com/shenwei356)/[taxid-changelog](https://github.com/shenwei356/taxid-changelog)

ID = 423, Stars = 11, Language = R, Update = 2020-09-03

NCBI taxonomic identifier (taxid) changelog, including taxids deletion, new adding, merge, reuse, and rank/name changes.
##### [dellytools](https://github.com/dellytools)/[svprops](https://github.com/dellytools/svprops)

ID = 424, Stars = 11, Language = C++, Update = 2020-04-14

Computes various SV statistics
##### [qiime2](https://github.com/qiime2)/[q2studio](https://github.com/qiime2/q2studio)

ID = 425, Stars = 11, Language = JavaScript, Update = 2020-08-27

Prototype graphical user interface for QIIME 2
##### [illumina](https://github.com/illumina)/[happyR](https://github.com/Illumina/happyR)

ID = 426, Stars = 11, Language = R, Update = 2019-07-12

R tools to interact with hap.py output
##### [astrazeneca-ngs](https://github.com/astrazeneca-ngs)/[simple_sv_annotation](https://github.com/AstraZeneca-NGS/simple_sv_annotation)

ID = 427, Stars = 11, Language = Python, Update = 2019-02-18

Simplify snpEff annotations for interesting cases
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[Geneblocks](https://github.com/Edinburgh-Genome-Foundry/Geneblocks)

ID = 428, Stars = 11, Language = Python, Update = 2020-09-09

üí†
Find common blocks and differences between DNA sequences
##### [mathonco](https://github.com/mathonco)/[EvoFreq](https://github.com/MathOnco/EvoFreq)

ID = 429, Stars = 11, Language = R, Update = 2020-04-08

Evolutionary frequency visualization tool of temporal data
##### [broadinstitute](https://github.com/broadinstitute)/[viral-pipelines](https://github.com/broadinstitute/viral-pipelines)

ID = 430, Stars = 11, Language = wdl, Update = 2020-09-10

viral-ngs: complete pipelines
##### [benlangmead](https://github.com/benlangmead)/[aws-indexes](https://github.com/BenLangmead/aws-indexes)

ID = 431, Stars = 10, Language = Python, Update = 2020-09-21

Catalog of genomic indexes freely available from public clouds
##### [biothings](https://github.com/biothings)/[biothings_explorer](https://github.com/biothings/biothings_explorer)

ID = 432, Stars = 10, Language = Jupyter Notebook, Update = 2020-09-21

BioThings Explorer: a schema-based client for API interoperability
##### [ensembl](https://github.com/ensembl)/[ensembl-analysis](https://github.com/Ensembl/ensembl-analysis)

ID = 433, Stars = 10, Language = Perl, Update = 2020-09-17

Modules to interface with tools used in Ensembl Gene Annotation Process and scripts to run pipelines
##### [qiime2](https://github.com/qiime2)/[q2-fragment-insertion](https://github.com/qiime2/q2-fragment-insertion)

ID = 434, Stars = 10, Language = Python, Update = 2020-08-22

QIIME 2 plugin for fragment insertion
##### [qiime2](https://github.com/qiime2)/[q2-dada2](https://github.com/qiime2/q2-dada2)

ID = 435, Stars = 10, Language = Python, Update = 2020-08-22

QIIME 2 plugin wrapping DADA2
##### [ncbi](https://github.com/ncbi)/[scPopCorn](https://github.com/ncbi/scPopCorn)

ID = 436, Stars = 10, Language = Jupyter Notebook, Update = 2019-10-25

A python tool to do comparative analysis of mulitple single cell datasets.
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[genedom](https://github.com/Edinburgh-Genome-Foundry/genedom)

ID = 437, Stars = 10, Language = Python, Update = 2020-09-20

Batch domestication of genetic parts with Python
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[DnaWeaver](https://github.com/Edinburgh-Genome-Foundry/DnaWeaver)

ID = 438, Stars = 10, Language = Python, Update = 2020-05-28

A route planner for DNA assembly
##### [igvteam](https://github.com/igvteam)/[juicebox.js](https://github.com/igvteam/juicebox.js)

ID = 439, Stars = 10, Language = JavaScript, Update = 2020-09-21

HiC contact matrix visualization
##### [mathonco](https://github.com/mathonco)/[HAL](https://github.com/MathOnco/HAL)

ID = 440, Stars = 10, Language = Java, Update = 2020-07-09

HAL (Hybrid Automata Library) is the one-stop shop for all of your hybrid modeling needs
##### [refresh-bio](https://github.com/refresh-bio)/[fqsqueezer](https://github.com/refresh-bio/fqsqueezer)

ID = 441, Stars = 10, Language = C++, Update = 2020-06-16

FQSqueezer - FASTQ compressor based on k-mer statistics
##### [broadinstitute](https://github.com/broadinstitute)/[sparklespray](https://github.com/broadinstitute/sparklespray)

ID = 442, Stars = 10, Language = HTML, Update = 2020-08-22

Easy batch submission of adhoc jobs onto GCP
##### [broadinstitute](https://github.com/broadinstitute)/[cell-health](https://github.com/broadinstitute/cell-health)

ID = 443, Stars = 10, Language = HTML, Update = 2020-08-14

Predicting Cell Health with Morphological Profiles
##### [broadinstitute](https://github.com/broadinstitute)/[vim-wdl](https://github.com/broadinstitute/vim-wdl)

ID = 444, Stars = 10, Language = Vim script, Update = 2020-01-29

Vim syntax highlighting for WDL
##### [lh3](https://github.com/lh3)/[unicall](https://github.com/lh3/unicall)

ID = 445, Stars = 9, Language = Perl, Update = 2019-07-21

A wrapper for calling small variants from human germline high-coverage single-sample Illumina data
##### [ekg](https://github.com/ekg)/[guix-genomics](https://github.com/ekg/guix-genomics)

ID = 446, Stars = 9, Language = Scheme, Update = 2020-08-31

guix packages for bioinformatics software
##### [biothings](https://github.com/biothings)/[mychem.info](https://github.com/biothings/mychem.info)

ID = 447, Stars = 9, Language = Python, Update = 2020-07-31

MyChem.info: A BioThings API for chemical/drug annotations
##### [ensembl](https://github.com/ensembl)/[ensembl-production](https://github.com/Ensembl/ensembl-production)

ID = 448, Stars = 9, Language = Perl, Update = 2020-09-18

Ensembl Production code
##### [ensembl](https://github.com/ensembl)/[ensj-healthcheck](https://github.com/Ensembl/ensj-healthcheck)

ID = 449, Stars = 9, Language = Java, Update = 2020-09-04

Ensembl's Automated QC Framework
##### [qiime2](https://github.com/qiime2)/[q2-longitudinal](https://github.com/qiime2/q2-longitudinal)

ID = 450, Stars = 9, Language = Python, Update = 2020-09-04

QIIME 2 plugin for paired sample comparisons
##### [qiime2](https://github.com/qiime2)/[q2cwl](https://github.com/qiime2/q2cwl)

ID = 451, Stars = 9, Language = Python, Update = 2020-02-13

Prototype interface for automatically generating CWL tools from QIIME 2 actions
##### [deeptools](https://github.com/deeptools)/[HiCBrowser](https://github.com/deeptools/HiCBrowser)

ID = 452, Stars = 9, Language = JavaScript, Update = 2019-12-12

Simple web browser to visualize HiC tracks
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[codon-usage-tables](https://github.com/Edinburgh-Genome-Foundry/codon-usage-tables)

ID = 453, Stars = 9, Language = Python, Update = 2020-09-06

üìä
Codon usage tables in code-friendly format + Python bindings
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[BandWagon](https://github.com/Edinburgh-Genome-Foundry/BandWagon)

ID = 454, Stars = 9, Language = Python, Update = 2020-09-05

üé∫
Plot DNA digestion band patterns with Python
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[genome_collector](https://github.com/Edinburgh-Genome-Foundry/genome_collector)

ID = 455, Stars = 9, Language = Python, Update = 2019-11-12

üå†
Easily download genomes and build BLAST/Bowtie indexes in Python
##### [broadinstitute](https://github.com/broadinstitute)/[Tangram](https://github.com/broadinstitute/Tangram) *new+*

ID = 456, Stars = 9, Language = Jupyter Notebook, Update = 2020-09-21

None
##### [broadinstitute](https://github.com/broadinstitute)/[agora](https://github.com/broadinstitute/agora)

ID = 457, Stars = 9, Language = Scala, Update = 2020-07-31

It's a methods repository!
##### [broadinstitute](https://github.com/broadinstitute)/[scRNA-Seq](https://github.com/broadinstitute/scRNA-Seq)

ID = 458, Stars = 9, Language = Python, Update = 2019-07-26

None
##### [genedx](https://github.com/genedx)/[txt2hpo](https://github.com/GeneDx/txt2hpo)

ID = 459, Stars = 9, Language = Python, Update = 2020-05-09

Python library for extracting HPO encoded phenotypes from text
##### [ekg](https://github.com/ekg)/[yeast-pangenome](https://github.com/ekg/yeast-pangenome)

ID = 460, Stars = 8, Language = Shell, Update = 2020-07-29

yeast pangenome
##### [ncbi](https://github.com/ncbi)/[blast-cloud](https://github.com/ncbi/blast-cloud)

ID = 461, Stars = 8, Language = CSS, Update = 2020-08-27

Documentation for NCBI BLAST AMI
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[CUBA](https://github.com/Edinburgh-Genome-Foundry/CUBA)

ID = 462, Stars = 8, Language = Vue, Update = 2020-09-09

üèñÔ∏è
The EGF Collection of Useful Bio Apps - Web demos of EGF software
##### [broadinstitute](https://github.com/broadinstitute)/[wdl-runner](https://github.com/broadinstitute/wdl-runner)

ID = 463, Stars = 8, Language = Python, Update = 2020-08-31

Easily run WDL workflows on GCP
##### [broadinstitute](https://github.com/broadinstitute)/[gtex-v8](https://github.com/broadinstitute/gtex-v8)

ID = 464, Stars = 8, Language = Jupyter Notebook, Update = 2020-07-05

Notebooks and scripts for reproducing analyses and figures from the V8 GTEx Consortium paper
##### [lh3](https://github.com/lh3)/[lh3.github.com](https://github.com/lh3/lh3.github.com)

ID = 465, Stars = 7, Language = HTML, Update = 2020-05-27

None
##### [ekg](https://github.com/ekg)/[gyeet](https://github.com/ekg/gyeet)

ID = 466, Stars = 7, Language = C++, Update = 2020-05-25

sequence to graph mapper
##### [biothings](https://github.com/biothings)/[biothings_client.py](https://github.com/biothings/biothings_client.py)

ID = 467, Stars = 7, Language = Python, Update = 2020-09-08

This is a generic Python client for BioThings APIs
##### [ensembl](https://github.com/ensembl)/[public-plugins](https://github.com/Ensembl/public-plugins)

ID = 468, Stars = 7, Language = HTML, Update = 2020-09-21

Plugins for an Ensembl website
##### [ensembl](https://github.com/ensembl)/[ensembl-io](https://github.com/Ensembl/ensembl-io)

ID = 469, Stars = 7, Language = Perl, Update = 2020-09-18

File parsing and writing code for Ensembl
##### [ensembl](https://github.com/ensembl)/[trackhub-registry](https://github.com/Ensembl/trackhub-registry)

ID = 470, Stars = 7, Language = Perl, Update = 2020-07-27

Specifications and implementation of the TrackHub registry
##### [qiime2](https://github.com/qiime2)/[q2view](https://github.com/qiime2/q2view)

ID = 471, Stars = 7, Language = JavaScript, Update = 2020-06-24

A web-based "serverless" viewer of QIIME 2 Artifacts and Visualizations
##### [deeptools](https://github.com/deeptools)/[HiCMatrix](https://github.com/deeptools/HiCMatrix)

ID = 472, Stars = 7, Language = Python, Update = 2020-09-01

None
##### [ncbi](https://github.com/ncbi)/[pm4ngs](https://github.com/ncbi/pm4ngs)

ID = 473, Stars = 7, Language = Python, Update = 2020-09-19

Project Manager for NGS data analysis
##### [ncbi](https://github.com/ncbi)/[HYDROID](https://github.com/ncbi/HYDROID)

ID = 474, Stars = 7, Language = Python, Update = 2020-09-16

Python package for analyzing hydroxyl-radical footprinting experiments of DNA-protein complexes
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[BandWitch](https://github.com/Edinburgh-Genome-Foundry/BandWitch)

ID = 475, Stars = 7, Language = Python, Update = 2020-09-20

üí´
Computer-aided DNA assembly validation and identification from restriction digests.
##### [broadinstitute](https://github.com/broadinstitute)/[ccle_processing](https://github.com/broadinstitute/ccle_processing)

ID = 476, Stars = 7, Language = Jupyter Notebook, Update = 2020-08-31

What you need to process the Quarterly DepMap-Omics releases from Terra
##### [broadinstitute](https://github.com/broadinstitute)/[firecloud-tools](https://github.com/broadinstitute/firecloud-tools)

ID = 477, Stars = 7, Language = Python, Update = 2020-05-29

Tools for use with FireCloud
##### [ekg](https://github.com/ekg)/[wfmash](https://github.com/ekg/wfmash) *new+*

ID = 478, Stars = 6, Language = C++, Update = 2020-09-12

base-accurate DNA sequence alignments using WFA and mashmap2
##### [felixkrueger](https://github.com/felixkrueger)/[nextflow_pipelines](https://github.com/FelixKrueger/nextflow_pipelines)

ID = 479, Stars = 6, Language = Nextflow, Update = 2020-07-23

The set of NGS processing pipelines used at Babraham
##### [ding-lab](https://github.com/ding-lab)/[somaticwrapper](https://github.com/ding-lab/somaticwrapper)

ID = 480, Stars = 6, Language = Perl, Update = 2020-07-02

Detect somatic variants from tumor and normal exome data
##### [qiime2](https://github.com/qiime2)/[vm-playbooks](https://github.com/qiime2/vm-playbooks)

ID = 481, Stars = 6, Language = Shell, Update = 2020-08-28

Ansible playbooks for QIIME 2 environments
##### [qiime2](https://github.com/qiime2)/[q2-types](https://github.com/qiime2/q2-types)

ID = 482, Stars = 6, Language = Python, Update = 2020-08-22

Definitions of common QIIME 2 types.
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[zymp](https://github.com/Edinburgh-Genome-Foundry/zymp)

ID = 483, Stars = 6, Language = Python, Update = 2020-09-11

‚úÇÔ∏è
Design compact restriction sites arrays (python utility)
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[kappagate](https://github.com/Edinburgh-Genome-Foundry/kappagate)

ID = 484, Stars = 6, Language = Python, Update = 2020-07-24

üîÆ
Predict DNA assembly clone validity rates - powered by Kappa
##### [biopython](https://github.com/biopython)/[biopython-wheels](https://github.com/biopython/biopython-wheels)

ID = 485, Stars = 6, Language = Shell, Update = 2020-09-03

Automated compilation of Python wheel binary packages for Biopython
##### [broadinstitute](https://github.com/broadinstitute)/[barclay](https://github.com/broadinstitute/barclay)

ID = 486, Stars = 6, Language = Java, Update = 2020-09-21

Command line argument parser and online documentation generation utilities for java command line programs.
##### [broadinstitute](https://github.com/broadinstitute)/[Celligner_ms](https://github.com/broadinstitute/Celligner_ms)

ID = 487, Stars = 6, Language = R, Update = 2020-06-29

Code related to the Celligner manuscript
##### [broadinstitute](https://github.com/broadinstitute)/[DLBCL_Nat_Med_April_2018](https://github.com/broadinstitute/DLBCL_Nat_Med_April_2018)

ID = 488, Stars = 6, Language = R, Update = 2020-06-11

None
##### [ensembl](https://github.com/ensembl)/[ensembl-client](https://github.com/Ensembl/ensembl-client)

ID = 489, Stars = 5, Language = TypeScript, Update = 2020-09-21

Ensembl 2020 client
##### [ensembl](https://github.com/ensembl)/[ensembl-test](https://github.com/Ensembl/ensembl-test)

ID = 490, Stars = 5, Language = Perl, Update = 2020-09-18

Test libraries and harnesses used for running the Ensembl test suite
##### [ensembl](https://github.com/ensembl)/[homebrew-ensembl](https://github.com/Ensembl/homebrew-ensembl)

ID = 491, Stars = 5, Language = Ruby, Update = 2020-08-20

Core formula for Ensembl
##### [qiime2](https://github.com/qiime2)/[Keemei](https://github.com/qiime2/Keemei)

ID = 492, Stars = 5, Language = JavaScript, Update = 2020-08-24

Validate tabular bioinformatics file formats in Google Sheets
##### [qiime2](https://github.com/qiime2)/[q2-feature-classifier](https://github.com/qiime2/q2-feature-classifier)

ID = 493, Stars = 5, Language = Python, Update = 2020-08-22

QIIME 2 plugin supporting taxonomic classification
##### [qiime2](https://github.com/qiime2)/[q2-sample-classifier](https://github.com/qiime2/q2-sample-classifier)

ID = 494, Stars = 5, Language = Python, Update = 2020-08-22

QIIME 2 plugin for machine learning prediction of sample data.
##### [pharmgkb](https://github.com/pharmgkb)/[PGxPOP](https://github.com/PharmGKB/PGxPOP)

ID = 495, Stars = 5, Language = Python, Update = 2020-09-12

PGxPOP
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[GoldenHinges](https://github.com/Edinburgh-Genome-Foundry/GoldenHinges)

ID = 496, Stars = 5, Language = Python, Update = 2020-09-11

üîó
Short overhangs design for DNA assembly
##### [edinburgh-genome-foundry](https://github.com/edinburgh-genome-foundry)/[tatapov](https://github.com/Edinburgh-Genome-Foundry/tatapov)

ID = 497, Stars = 5, Language = Python, Update = 2020-09-08

üêæ
DNA overhang misannealing data for Python
##### [refresh-bio](https://github.com/refresh-bio)/[RADULS](https://github.com/refresh-bio/RADULS)

ID = 498, Stars = 5, Language = C++, Update = 2020-06-23

Ultra fast MSD radix sorter
##### [broadinstitute](https://github.com/broadinstitute)/[wfl](https://github.com/broadinstitute/wfl)

ID = 499, Stars = 5, Language = Clojure, Update = 2020-09-21

WorkFlow Launcher (n√©e Zero)
##### [broadinstitute](https://github.com/broadinstitute)/[warp](https://github.com/broadinstitute/warp)

ID = 500, Stars = 5, Language = wdl, Update = 2020-09-21

WDL Analysis Research Pipelines
##### [broadinstitute](https://github.com/broadinstitute)/[viral-assemble](https://github.com/broadinstitute/viral-assemble) *new+*

ID = 501, Stars = 5, Language = Python, Update = 2020-08-21

viral-ngs: genome assembly and scaffolding
##### [broadinstitute](https://github.com/broadinstitute)/[taiga](https://github.com/broadinstitute/taiga)

ID = 502, Stars = 5, Language = Python, Update = 2020-08-18

Taiga for Dataset management
##### [broadinstitute](https://github.com/broadinstitute)/[dig-diabetes-portal](https://github.com/broadinstitute/dig-diabetes-portal)

ID = 503, Stars = 5, Language = JavaScript, Update = 2020-07-23

Diabetes portal for the Medical Population Genetics department at the Broad Institute
##### [broadinstitute](https://github.com/broadinstitute)/[lincs-cell-painting](https://github.com/broadinstitute/lincs-cell-painting)

ID = 504, Stars = 5, Language = Jupyter Notebook, Update = 2020-07-02

Processed Cell Painting Data for the LINCS Drug Repurposing Project
